ids	gene_num	othergene_num	p-value	q-value	sig	name	cluster	go_class
GO:0005525	30	13	0.008046107522412445	0.9996157290847086	-	GTP binding	0	F
GO:0003924	30	15	0.02096006801787435	0.9996157290847086	-	GTPase activity	0	F
GO:0003674	25	13	0.042425887385648835	0.9996157290847086	-	molecular_function	0	F
GO:0003713	8	2	0.059850004314433755	0.9996157290847086	-	transcription coactivator activity	0	F
GO:0005161	6	1	0.06744776696746518	0.9996157290847086	-	platelet-derived growth factor receptor binding	0	F
GO:0046982	33	21	0.07842542955752284	0.9996157290847086	-	protein heterodimerization activity	0	F
GO:0005198	9	3	0.08010261613434412	0.9996157290847086	-	structural molecule activity	0	F
GO:0051015	24	15	0.11464548347333983	0.9996157290847086	-	actin filament binding	0	F
GO:0046872	57	43	0.11608407606779778	0.9996157290847086	-	metal ion binding	0	F
GO:0008565	5	1	0.11662770242832539	0.9996157290847086	-	protein transporter activity	0	F
GO:0005178	15	8	0.11758989468400823	0.9996157290847086	-	integrin binding	0	F
GO:0000166	8	3	0.1230170146261113	0.9996157290847086	-	nucleotide binding	0	F
GO:0044325	17	10	0.13924051364526324	0.9996157290847086	-	ion channel binding	0	F
GO:0003779	25	17	0.16025544496658456	0.9996157290847086	-	actin binding	0	F
GO:0005080	7	3	0.18458951788137756	0.9996157290847086	-	protein kinase C binding	0	F
GO:0097110	4	1	0.19740748702539465	0.9996157290847086	-	scaffold protein binding	0	F
GO:0004601	4	1	0.19740748702539465	0.9996157290847086	-	peroxidase activity	0	F
GO:0017112	4	1	0.19740748702539465	0.9996157290847086	-	Rab guanyl-nucleotide exchange factor activity	0	F
GO:0001786	4	1	0.19740748702539465	0.9996157290847086	-	phosphatidylserine binding	0	F
GO:0043130	4	1	0.19740748702539465	0.9996157290847086	-	ubiquitin binding	0	F
GO:0008083	8	4	0.20884581160964563	0.9996157290847086	-	growth factor activity	0	F
GO:0019003	9	5	0.22905054137018013	0.9996157290847086	-	GDP binding	0	F
GO:0035615	5	2	0.23957754108047588	0.9996157290847086	-	clathrin adaptor activity	0	F
GO:0001618	5	2	0.23957754108047588	0.9996157290847086	-	virus receptor activity	0	F
GO:0005547	6	3	0.2695380935934397	0.9996157290847086	-	phosphatidylinositol-3,4,5-trisphosphate binding	0	F
GO:0005543	6	3	0.2695380935934397	0.9996157290847086	-	phospholipid binding	0	F
GO:0004672	12	8	0.2741656137988785	0.9996157290847086	-	protein kinase activity	0	F
GO:0005200	12	8	0.2741656137988785	0.9996157290847086	-	structural constituent of cytoskeleton	0	F
GO:0000980	7	4	0.2923413985780936	0.9996157290847086	-	RNA polymerase II distal enhancer sequence-specific DNA binding	0	F
GO:0008201	7	4	0.2923413985780936	0.9996157290847086	-	heparin binding	0	F
GO:0043621	3	1	0.3246628856507586	0.9996157290847086	-	protein self-association	0	F
GO:0005160	3	1	0.3246628856507586	0.9996157290847086	-	transforming growth factor beta receptor binding	0	F
GO:0008093	3	1	0.3246628856507586	0.9996157290847086	-	cytoskeletal adaptor activity	0	F
GO:0050840	3	1	0.3246628856507586	0.9996157290847086	-	extracellular matrix binding	0	F
GO:0032266	3	1	0.3246628856507586	0.9996157290847086	-	phosphatidylinositol-3-phosphate binding	0	F
GO:0005484	3	1	0.3246628856507586	0.9996157290847086	-	SNAP receptor activity	0	F
GO:0019901	30	25	0.3346070731933105	0.9996157290847086	-	protein kinase binding	0	F
GO:0050681	4	2	0.3589670562195333	0.9996157290847086	-	androgen receptor binding	0	F
GO:0005201	4	2	0.3589670562195333	0.9996157290847086	-	extracellular matrix structural constituent	0	F
GO:0048156	4	2	0.3589670562195333	0.9996157290847086	-	tau protein binding	0	F
GO:0031492	4	2	0.3589670562195333	0.9996157290847086	-	nucleosomal DNA binding	0	F
GO:0001968	4	2	0.3589670562195333	0.9996157290847086	-	fibronectin binding	0	F
GO:0004867	4	2	0.3589670562195333	0.9996157290847086	-	serine-type endopeptidase inhibitor activity	0	F
GO:0008009	5	3	0.3810502149021172	0.9996157290847086	-	chemokine activity	0	F
GO:0005070	5	3	0.3810502149021172	0.9996157290847086	-	SH3/SH2 adaptor activity	0	F
GO:0005085	6	4	0.39696095716153285	0.9996157290847086	-	guanyl-nucleotide exchange factor activity	0	F
GO:0030145	6	4	0.39696095716153285	0.9996157290847086	-	manganese ion binding	0	F
GO:0004842	6	4	0.39696095716153285	0.9996157290847086	-	ubiquitin-protein transferase activity	0	F
GO:0061630	6	4	0.39696095716153285	0.9996157290847086	-	ubiquitin protein ligase activity	0	F
GO:0017048	7	5	0.4092352203339207	0.9996157290847086	-	Rho GTPase binding	0	F
GO:0045296	21	18	0.41296782714245467	0.9996157290847086	-	cadherin binding	0	F
GO:0005096	11	9	0.44067979300319743	0.9996157290847086	-	GTPase activator activity	0	F
GO:0031852	2	1	0.5124257791032786	0.9996157290847086	-	mu-type opioid receptor binding	0	F
GO:0019960	2	1	0.5124257791032786	0.9996157290847086	-	C-X3-C chemokine binding	0	F
GO:0017080	2	1	0.5124257791032786	0.9996157290847086	-	sodium channel regulator activity	0	F
GO:0034236	2	1	0.5124257791032786	0.9996157290847086	-	protein kinase A catalytic subunit binding	0	F
GO:0048487	2	1	0.5124257791032786	0.9996157290847086	-	beta-tubulin binding	0	F
GO:0004721	2	1	0.5124257791032786	0.9996157290847086	-	phosphoprotein phosphatase activity	0	F
GO:0004602	2	1	0.5124257791032786	0.9996157290847086	-	glutathione peroxidase activity	0	F
GO:0050431	2	1	0.5124257791032786	0.9996157290847086	-	transforming growth factor beta binding	0	F
GO:0030159	2	1	0.5124257791032786	0.9996157290847086	-	receptor signaling complex scaffold activity	0	F
GO:0004622	2	1	0.5124257791032786	0.9996157290847086	-	lysophospholipase activity	0	F
GO:0005244	2	1	0.5124257791032786	0.9996157290847086	-	voltage-gated ion channel activity	0	F
GO:0005254	2	1	0.5124257791032786	0.9996157290847086	-	chloride channel activity	0	F
GO:0042626	2	1	0.5124257791032786	0.9996157290847086	-	ATPase activity, coupled to transmembrane movement of substances	0	F
GO:0043208	2	1	0.5124257791032786	0.9996157290847086	-	glycosphingolipid binding	0	F
GO:0008179	2	1	0.5124257791032786	0.9996157290847086	-	adenylate cyclase binding	0	F
GO:0008047	2	1	0.5124257791032786	0.9996157290847086	-	enzyme activator activity	0	F
GO:0005123	2	1	0.5124257791032786	0.9996157290847086	-	death receptor binding	0	F
GO:0043225	2	1	0.5124257791032786	0.9996157290847086	-	ATPase-coupled anion transmembrane transporter activity	0	F
GO:0098639	2	1	0.5124257791032786	0.9996157290847086	-	collagen binding involved in cell-matrix adhesion	0	F
GO:0003712	2	1	0.5124257791032786	0.9996157290847086	-	transcription coregulator activity	0	F
GO:0005536	2	1	0.5124257791032786	0.9996157290847086	-	glucose binding	0	F
GO:0080025	2	1	0.5124257791032786	0.9996157290847086	-	phosphatidylinositol-3,5-bisphosphate binding	0	F
GO:0003785	2	1	0.5124257791032786	0.9996157290847086	-	actin monomer binding	0	F
GO:0003824	2	1	0.5124257791032786	0.9996157290847086	-	catalytic activity	0	F
GO:0017046	2	1	0.5124257791032786	0.9996157290847086	-	peptide hormone binding	0	F
GO:0017075	2	1	0.5124257791032786	0.9996157290847086	-	syntaxin-1 binding	0	F
GO:0017022	2	1	0.5124257791032786	0.9996157290847086	-	myosin binding	0	F
GO:0034713	2	1	0.5124257791032786	0.9996157290847086	-	type I transforming growth factor beta receptor binding	0	F
GO:0015297	2	1	0.5124257791032786	0.9996157290847086	-	antiporter activity	0	F
GO:0015026	3	2	0.5155459835888045	0.9996157290847086	-	coreceptor activity	0	F
GO:0043531	3	2	0.5155459835888045	0.9996157290847086	-	ADP binding	0	F
GO:0008013	3	2	0.5155459835888045	0.9996157290847086	-	beta-catenin binding	0	F
GO:0050750	3	2	0.5155459835888045	0.9996157290847086	-	low-density lipoprotein particle receptor binding	0	F
GO:0003774	3	2	0.5155459835888045	0.9996157290847086	-	motor activity	0	F
GO:0070888	3	2	0.5155459835888045	0.9996157290847086	-	E-box binding	0	F
GO:0050998	3	2	0.5155459835888045	0.9996157290847086	-	nitric-oxide synthase binding	0	F
GO:0042169	4	3	0.5181530764049431	0.9996157290847086	-	SH2 domain binding	0	F
GO:0005088	4	3	0.5181530764049431	0.9996157290847086	-	Ras guanyl-nucleotide exchange factor activity	0	F
GO:0051082	4	3	0.5181530764049431	0.9996157290847086	-	unfolded protein binding	0	F
GO:0001085	4	3	0.5181530764049431	0.9996157290847086	-	RNA polymerase II transcription factor binding	0	F
GO:0005518	4	3	0.5181530764049431	0.9996157290847086	-	collagen binding	0	F
GO:0005506	6	5	0.5225044274616604	0.9996157290847086	-	iron ion binding	0	F
GO:0047485	7	6	0.5244015471869525	0.9996157290847086	-	protein N-terminus binding	0	F
GO:0003682	8	7	0.5261678814544681	0.9996157290847086	-	chromatin binding	0	F
GO:0005509	27	26	0.5494517744098916	0.9996157290847086	-	calcium ion binding	0	F
GO:0031625	15	15	0.6093230335513092	0.9996157290847086	-	ubiquitin protein ligase binding	0	F
GO:0042802	50	50	0.6101766323853448	0.9996157290847086	-	identical protein binding	0	F
GO:0004674	14	14	0.6105023908004474	0.9996157290847086	-	protein serine/threonine kinase activity	0	F
GO:0000287	7	7	0.629654289127727	0.9996157290847086	-	magnesium ion binding	0	F
GO:0005524	42	43	0.6502802726940164	0.9996157290847086	-	ATP binding	0	F
GO:0003690	4	4	0.6552559379077691	0.9996157290847086	-	double-stranded DNA binding	0	F
GO:0035091	4	4	0.6552559379077691	0.9996157290847086	-	phosphatidylinositol binding	0	F
GO:0031490	4	4	0.6552559379077691	0.9996157290847086	-	chromatin DNA binding	0	F
GO:0004715	4	4	0.6552559379077691	0.9996157290847086	-	non-membrane spanning protein tyrosine kinase activity	0	F
GO:0005515	350	344	0.6672766234138652	0.9996157290847086	-	protein binding	0	F
GO:0016887	3	3	0.6721249109580756	0.9996157290847086	-	ATPase activity	0	F
GO:0004930	3	3	0.6721249109580756	0.9996157290847086	-	G protein-coupled receptor activity	0	F
GO:0001671	3	3	0.6721249109580756	0.9996157290847086	-	ATPase activator activity	0	F
GO:0005544	3	3	0.6721249109580756	0.9996157290847086	-	calcium-dependent phospholipid binding	0	F
GO:0008198	3	3	0.6721249109580756	0.9996157290847086	-	ferrous iron binding	0	F
GO:0003697	3	3	0.6721249109580756	0.9996157290847086	-	single-stranded DNA binding	0	F
GO:0030165	3	3	0.6721249109580756	0.9996157290847086	-	PDZ domain binding	0	F
GO:0019904	15	16	0.6765947219401011	0.9996157290847086	-	protein domain specific binding	0	F
GO:0070412	2	2	0.7001886725557986	0.9996157290847086	-	R-SMAD binding	0	F
GO:0003755	2	2	0.7001886725557986	0.9996157290847086	-	peptidyl-prolyl cis-trans isomerase activity	0	F
GO:0003823	2	2	0.7001886725557986	0.9996157290847086	-	antigen binding	0	F
GO:0005246	2	2	0.7001886725557986	0.9996157290847086	-	calcium channel regulator activity	0	F
GO:0005523	2	2	0.7001886725557986	0.9996157290847086	-	tropomyosin binding	0	F
GO:0042056	2	2	0.7001886725557986	0.9996157290847086	-	chemoattractant activity	0	F
GO:0005522	2	2	0.7001886725557986	0.9996157290847086	-	profilin binding	0	F
GO:0004322	2	2	0.7001886725557986	0.9996157290847086	-	ferroxidase activity	0	F
GO:0097718	2	2	0.7001886725557986	0.9996157290847086	-	disordered domain specific binding	0	F
GO:0008137	2	2	0.7001886725557986	0.9996157290847086	-	NADH dehydrogenase (ubiquinone) activity	0	F
GO:0043539	2	2	0.7001886725557986	0.9996157290847086	-	protein serine/threonine kinase activator activity	0	F
GO:0000976	2	2	0.7001886725557986	0.9996157290847086	-	transcription regulatory region sequence-specific DNA binding	0	F
GO:0004722	2	2	0.7001886725557986	0.9996157290847086	-	protein serine/threonine phosphatase activity	0	F
GO:0004407	2	2	0.7001886725557986	0.9996157290847086	-	histone deacetylase activity	0	F
GO:0050660	2	2	0.7001886725557986	0.9996157290847086	-	flavin adenine dinucleotide binding	0	F
GO:0008199	2	2	0.7001886725557986	0.9996157290847086	-	ferric iron binding	0	F
GO:0051959	2	2	0.7001886725557986	0.9996157290847086	-	dynein light intermediate chain binding	0	F
GO:0008017	8	9	0.7112086780959164	0.9996157290847086	-	microtubule binding	0	F
GO:0017137	6	7	0.7312354237914973	0.9996157290847086	-	Rab GTPase binding	0	F
GO:0048306	5	6	0.7450992329583398	0.9996157290847086	-	calcium-dependent protein binding	0	F
GO:0008474	1	1	0.758451630536907	0.9996157290847086	-	palmitoyl-(protein) hydrolase activity	0	F
GO:0005095	1	1	0.758451630536907	0.9996157290847086	-	GTPase inhibitor activity	0	F
GO:0005086	1	1	0.758451630536907	0.9996157290847086	-	ARF guanyl-nucleotide exchange factor activity	0	F
GO:0005068	1	1	0.758451630536907	0.9996157290847086	-	transmembrane receptor protein tyrosine kinase adaptor activity	0	F
GO:0005355	1	1	0.758451630536907	0.9996157290847086	-	glucose transmembrane transporter activity	0	F
GO:0005521	1	1	0.758451630536907	0.9996157290847086	-	lamin binding	0	F
GO:0031404	1	1	0.758451630536907	0.9996157290847086	-	chloride ion binding	0	F
GO:0031802	1	1	0.758451630536907	0.9996157290847086	-	type 5 metabotropic glutamate receptor binding	0	F
GO:0031762	1	1	0.758451630536907	0.9996157290847086	-	follicle-stimulating hormone receptor binding	0	F
GO:0031748	1	1	0.758451630536907	0.9996157290847086	-	D1 dopamine receptor binding	0	F
GO:0031730	1	1	0.758451630536907	0.9996157290847086	-	CCR5 chemokine receptor binding	0	F
GO:0031726	1	1	0.758451630536907	0.9996157290847086	-	CCR1 chemokine receptor binding	0	F
GO:0031701	1	1	0.758451630536907	0.9996157290847086	-	angiotensin receptor binding	0	F
GO:0031692	1	1	0.758451630536907	0.9996157290847086	-	alpha-1B adrenergic receptor binding	0	F
GO:0030169	1	1	0.758451630536907	0.9996157290847086	-	low-density lipoprotein particle binding	0	F
GO:0008432	1	1	0.758451630536907	0.9996157290847086	-	JUN kinase binding	0	F
GO:0008301	1	1	0.758451630536907	0.9996157290847086	-	DNA binding, bending	0	F
GO:0015293	1	1	0.758451630536907	0.9996157290847086	-	symporter activity	0	F
GO:0010314	1	1	0.758451630536907	0.9996157290847086	-	phosphatidylinositol-5-phosphate binding	0	F
GO:0016787	1	1	0.758451630536907	0.9996157290847086	-	hydrolase activity	0	F
GO:0016705	1	1	0.758451630536907	0.9996157290847086	-	oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen	0	F
GO:0030955	1	1	0.758451630536907	0.9996157290847086	-	potassium ion binding	0	F
GO:0031691	1	1	0.758451630536907	0.9996157290847086	-	alpha-1A adrenergic receptor binding	0	F
GO:0031624	1	1	0.758451630536907	0.9996157290847086	-	ubiquitin conjugating enzyme binding	0	F
GO:0004222	1	1	0.758451630536907	0.9996157290847086	-	metalloendopeptidase activity	0	F
GO:0030957	1	1	0.758451630536907	0.9996157290847086	-	Tat protein binding	0	F
GO:0016595	1	1	0.758451630536907	0.9996157290847086	-	glutamate binding	0	F
GO:0019855	1	1	0.758451630536907	0.9996157290847086	-	calcium channel inhibitor activity	0	F
GO:0097199	1	1	0.758451630536907	0.9996157290847086	-	cysteine-type endopeptidase activity involved in apoptotic signaling pathway	0	F
GO:0072320	1	1	0.758451630536907	0.9996157290847086	-	volume-sensitive chloride channel activity	0	F
GO:1905538	1	1	0.758451630536907	0.9996157290847086	-	polysome binding	0	F
GO:0004198	1	1	0.758451630536907	0.9996157290847086	-	calcium-dependent cysteine-type endopeptidase activity	0	F
GO:0004180	1	1	0.758451630536907	0.9996157290847086	-	carboxypeptidase activity	0	F
GO:0070053	1	1	0.758451630536907	0.9996157290847086	-	thrombospondin receptor activity	0	F
GO:0070006	1	1	0.758451630536907	0.9996157290847086	-	metalloaminopeptidase activity	0	F
GO:0051537	1	1	0.758451630536907	0.9996157290847086	-	2 iron, 2 sulfur cluster binding	0	F
GO:0051213	1	1	0.758451630536907	0.9996157290847086	-	dioxygenase activity	0	F
GO:0072542	1	1	0.758451630536907	0.9996157290847086	-	protein phosphatase activator activity	0	F
GO:0098973	1	1	0.758451630536907	0.9996157290847086	-	structural constituent of postsynaptic actin cytoskeleton	0	F
GO:0019894	1	1	0.758451630536907	0.9996157290847086	-	kinesin binding	0	F
GO:0019869	1	1	0.758451630536907	0.9996157290847086	-	chloride channel inhibitor activity	0	F
GO:0031434	1	1	0.758451630536907	0.9996157290847086	-	mitogen-activated protein kinase kinase binding	0	F
GO:0019828	1	1	0.758451630536907	0.9996157290847086	-	aspartic-type endopeptidase inhibitor activity	0	F
GO:0019809	1	1	0.758451630536907	0.9996157290847086	-	spermidine binding	0	F
GO:0030395	1	1	0.758451630536907	0.9996157290847086	-	lactose binding	0	F
GO:0030332	1	1	0.758451630536907	0.9996157290847086	-	cyclin binding	0	F
GO:0030296	1	1	0.758451630536907	0.9996157290847086	-	protein tyrosine kinase activator activity	0	F
GO:0033691	1	1	0.758451630536907	0.9996157290847086	-	sialic acid binding	0	F
GO:0033558	1	1	0.758451630536907	0.9996157290847086	-	protein deacetylase activity	0	F
GO:0032041	1	1	0.758451630536907	0.9996157290847086	-	NAD-dependent histone deacetylase activity (H3-K14 specific)	0	F
GO:0031994	1	1	0.758451630536907	0.9996157290847086	-	insulin-like growth factor I binding	0	F
GO:0019706	1	1	0.758451630536907	0.9996157290847086	-	protein-cysteine S-palmitoyltransferase activity	0	F
GO:0019208	1	1	0.758451630536907	0.9996157290847086	-	phosphatase regulator activity	0	F
GO:0030170	1	1	0.758451630536907	0.9996157290847086	-	pyridoxal phosphate binding	0	F
GO:0048185	1	1	0.758451630536907	0.9996157290847086	-	activin binding	0	F
GO:0000339	1	1	0.758451630536907	0.9996157290847086	-	RNA cap binding	0	F
GO:0001222	1	1	0.758451630536907	0.9996157290847086	-	transcription corepressor binding	0	F
GO:0016004	1	1	0.758451630536907	0.9996157290847086	-	phospholipase activator activity	0	F
GO:0000987	1	1	0.758451630536907	0.9996157290847086	-	proximal promoter sequence-specific DNA binding	0	F
GO:0042277	1	1	0.758451630536907	0.9996157290847086	-	peptide binding	0	F
GO:0043015	1	1	0.758451630536907	0.9996157290847086	-	gamma-tubulin binding	0	F
GO:0050811	1	1	0.758451630536907	0.9996157290847086	-	GABA receptor binding	0	F
GO:0050699	1	1	0.758451630536907	0.9996157290847086	-	WW domain binding	0	F
GO:0036134	1	1	0.758451630536907	0.9996157290847086	-	12-hydroxyheptadecatrienoic acid synthase activity	0	F
GO:0035605	1	1	0.758451630536907	0.9996157290847086	-	peptidyl-cysteine S-nitrosylase activity	0	F
GO:0001882	1	1	0.758451630536907	0.9996157290847086	-	nucleoside binding	0	F
GO:0048029	1	1	0.758451630536907	0.9996157290847086	-	monosaccharide binding	0	F
GO:0048020	1	1	0.758451630536907	0.9996157290847086	-	CCR chemokine receptor binding	0	F
GO:0000993	1	1	0.758451630536907	0.9996157290847086	-	RNA polymerase II complex binding	0	F
GO:0098772	1	1	0.758451630536907	0.9996157290847086	-	molecular function regulator	0	F
GO:0070700	1	1	0.758451630536907	0.9996157290847086	-	BMP receptor binding	0	F
GO:0070324	1	1	0.758451630536907	0.9996157290847086	-	thyroid hormone binding	0	F
GO:0070273	1	1	0.758451630536907	0.9996157290847086	-	phosphatidylinositol-4-phosphate binding	0	F
GO:0070063	1	1	0.758451630536907	0.9996157290847086	-	RNA polymerase binding	0	F
GO:0102121	1	1	0.758451630536907	0.9996157290847086	-	ceramidase activity	0	F
GO:0045503	1	1	0.758451630536907	0.9996157290847086	-	dynein light chain binding	0	F
GO:0048037	1	1	0.758451630536907	0.9996157290847086	-	cofactor binding	0	F
GO:0051059	1	1	0.758451630536907	0.9996157290847086	-	NF-kappaB binding	0	F
GO:0004145	1	1	0.758451630536907	0.9996157290847086	-	diamine N-acetyltransferase activity	0	F
GO:0004012	1	1	0.758451630536907	0.9996157290847086	-	phospholipid-translocating ATPase activity	0	F
GO:0004356	1	1	0.758451630536907	0.9996157290847086	-	glutamate-ammonia ligase activity	0	F
GO:0000340	1	1	0.758451630536907	0.9996157290847086	-	RNA 7-methylguanosine cap binding	0	F
GO:0017040	1	1	0.758451630536907	0.9996157290847086	-	N-acylsphingosine amidohydrolase activity	0	F
GO:0016936	1	1	0.758451630536907	0.9996157290847086	-	galactoside binding	0	F
GO:0004698	1	1	0.758451630536907	0.9996157290847086	-	calcium-dependent protein kinase C activity	0	F
GO:0004623	1	1	0.758451630536907	0.9996157290847086	-	phospholipase A2 activity	0	F
GO:0004801	1	1	0.758451630536907	0.9996157290847086	-	sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase activity	0	F
GO:0004796	1	1	0.758451630536907	0.9996157290847086	-	thromboxane-A synthase activity	0	F
GO:0004727	1	1	0.758451630536907	0.9996157290847086	-	prenylated protein tyrosine phosphatase activity	0	F
GO:0004726	1	1	0.758451630536907	0.9996157290847086	-	non-membrane spanning protein tyrosine phosphatase activity	0	F
GO:0005527	1	1	0.758451630536907	0.9996157290847086	-	macrolide binding	0	F
GO:0008073	1	1	0.758451630536907	0.9996157290847086	-	ornithine decarboxylase inhibitor activity	0	F
GO:0008121	1	1	0.758451630536907	0.9996157290847086	-	ubiquinol-cytochrome-c reductase activity	0	F
GO:0004497	1	1	0.758451630536907	0.9996157290847086	-	monooxygenase activity	0	F
GO:0004364	1	1	0.758451630536907	0.9996157290847086	-	glutathione transferase activity	0	F
GO:0043395	1	1	0.758451630536907	0.9996157290847086	-	heparan sulfate proteoglycan binding	0	F
GO:0043548	1	1	0.758451630536907	0.9996157290847086	-	phosphatidylinositol 3-kinase binding	0	F
GO:0008177	1	1	0.758451630536907	0.9996157290847086	-	succinate dehydrogenase (ubiquinone) activity	0	F
GO:0008190	1	1	0.758451630536907	0.9996157290847086	-	eukaryotic initiation factor 4E binding	0	F
GO:0004365	1	1	0.758451630536907	0.9996157290847086	-	glyceraldehyde-3-phosphate dehydrogenase (NAD+) (phosphorylating) activity	0	F
GO:0008080	1	1	0.758451630536907	0.9996157290847086	-	N-acetyltransferase activity	0	F
GO:0004725	4	5	0.7631083950036134	0.9996157290847086	-	protein tyrosine phosphatase activity	0	F
GO:0008289	4	5	0.7631083950036134	0.9996157290847086	-	lipid binding	0	F
GO:0020037	4	5	0.7631083950036134	0.9996157290847086	-	heme binding	0	F
GO:0008270	21	24	0.7652210141454008	0.9996157290847086	-	zinc ion binding	0	F
GO:0005102	10	12	0.7655557192219142	0.9996157290847086	-	signaling receptor binding	0	F
GO:0070491	3	4	0.7876037868729249	0.9996157290847086	-	repressing transcription factor binding	0	F
GO:0000149	3	4	0.7876037868729249	0.9996157290847086	-	SNARE binding	0	F
GO:0098641	3	4	0.7876037868729249	0.9996157290847086	-	cadherin binding involved in cell-cell adhesion	0	F
GO:0043236	3	4	0.7876037868729249	0.9996157290847086	-	laminin binding	0	F
GO:0046332	3	4	0.7876037868729249	0.9996157290847086	-	SMAD binding	0	F
GO:0030674	3	4	0.7876037868729249	0.9996157290847086	-	protein binding, bridging	0	F
GO:0000979	3	4	0.7876037868729249	0.9996157290847086	-	RNA polymerase II core promoter sequence-specific DNA binding	0	F
GO:0032947	3	4	0.7876037868729249	0.9996157290847086	-	protein-containing complex scaffold activity	0	F
GO:0042803	39	44	0.8007421484343773	0.9996157290847086	-	protein homodimerization activity	0	F
GO:0044877	12	15	0.8070042008669959	0.9996157290847086	-	protein-containing complex binding	0	F
GO:0003714	6	8	0.8074212747893253	0.9996157290847086	-	transcription corepressor activity	0	F
GO:0042393	2	3	0.823283798533795	0.9996157290847086	-	histone binding	0	F
GO:0003676	2	3	0.823283798533795	0.9996157290847086	-	nucleic acid binding	0	F
GO:0030971	2	3	0.823283798533795	0.9996157290847086	-	receptor tyrosine kinase binding	0	F
GO:0004866	2	3	0.823283798533795	0.9996157290847086	-	endopeptidase inhibitor activity	0	F
GO:0008144	2	3	0.823283798533795	0.9996157290847086	-	drug binding	0	F
GO:0016301	2	3	0.823283798533795	0.9996157290847086	-	kinase activity	0	F
GO:0015459	2	3	0.823283798533795	0.9996157290847086	-	potassium channel regulator activity	0	F
GO:0008656	2	3	0.823283798533795	0.9996157290847086	-	cysteine-type endopeptidase activator activity involved in apoptotic process	0	F
GO:0001965	2	3	0.823283798533795	0.9996157290847086	-	G-protein alpha-subunit binding	0	F
GO:0008191	2	3	0.823283798533795	0.9996157290847086	-	metalloendopeptidase inhibitor activity	0	F
GO:0043565	9	12	0.8310397246610084	0.9996157290847086	-	sequence-specific DNA binding	0	F
GO:0051117	4	6	0.8425674743964261	0.9996157290847086	-	ATPase binding	0	F
GO:0005546	4	6	0.8425674743964261	0.9996157290847086	-	phosphatidylinositol-4,5-bisphosphate binding	0	F
GO:0019900	4	6	0.8425674743964261	0.9996157290847086	-	kinase binding	0	F
GO:0003730	4	6	0.8425674743964261	0.9996157290847086	-	mRNA 3'-UTR binding	0	F
GO:0003725	3	5	0.8670124962520183	0.9996157290847086	-	double-stranded RNA binding	0	F
GO:0008092	3	5	0.8670124962520183	0.9996157290847086	-	cytoskeletal protein binding	0	F
GO:0051087	3	5	0.8670124962520183	0.9996157290847086	-	chaperone binding	0	F
GO:0019903	3	5	0.8670124962520183	0.9996157290847086	-	protein phosphatase binding	0	F
GO:0003677	22	28	0.8718896984690407	0.9996157290847086	-	DNA binding	0	F
GO:0008022	5	8	0.8806598975087073	0.9996157290847086	-	protein C-terminus binding	0	F
GO:0019834	1	2	0.8814645562537212	0.9996157290847086	-	phospholipase A2 inhibitor activity	0	F
GO:0051219	1	2	0.8814645562537212	0.9996157290847086	-	phosphoprotein binding	0	F
GO:0089720	1	2	0.8814645562537212	0.9996157290847086	-	caspase binding	0	F
GO:0031996	1	2	0.8814645562537212	0.9996157290847086	-	thioesterase binding	0	F
GO:0003743	1	2	0.8814645562537212	0.9996157290847086	-	translation initiation factor activity	0	F
GO:0017056	1	2	0.8814645562537212	0.9996157290847086	-	structural constituent of nuclear pore	0	F
GO:0017025	1	2	0.8814645562537212	0.9996157290847086	-	TBP-class protein binding	0	F
GO:0015631	1	2	0.8814645562537212	0.9996157290847086	-	tubulin binding	0	F
GO:0097153	1	2	0.8814645562537212	0.9996157290847086	-	cysteine-type endopeptidase activity involved in apoptotic process	0	F
GO:0051022	1	2	0.8814645562537212	0.9996157290847086	-	Rho GDP-dissociation inhibitor binding	0	F
GO:0004190	1	2	0.8814645562537212	0.9996157290847086	-	aspartic-type endopeptidase activity	0	F
GO:0004386	1	2	0.8814645562537212	0.9996157290847086	-	helicase activity	0	F
GO:0005528	1	2	0.8814645562537212	0.9996157290847086	-	FK506 binding	0	F
GO:0004860	1	2	0.8814645562537212	0.9996157290847086	-	protein kinase inhibitor activity	0	F
GO:0030414	1	2	0.8814645562537212	0.9996157290847086	-	peptidase inhibitor activity	0	F
GO:0031683	1	2	0.8814645562537212	0.9996157290847086	-	G-protein beta/gamma-subunit complex binding	0	F
GO:0035259	1	2	0.8814645562537212	0.9996157290847086	-	glucocorticoid receptor binding	0	F
GO:0050700	1	2	0.8814645562537212	0.9996157290847086	-	CARD domain binding	0	F
GO:0034237	1	2	0.8814645562537212	0.9996157290847086	-	protein kinase A regulatory subunit binding	0	F
GO:0034988	1	2	0.8814645562537212	0.9996157290847086	-	Fc-gamma receptor I complex binding	0	F
GO:0001102	1	2	0.8814645562537212	0.9996157290847086	-	RNA polymerase II activating transcription factor binding	0	F
GO:0004714	1	2	0.8814645562537212	0.9996157290847086	-	transmembrane receptor protein tyrosine kinase activity	0	F
GO:0043560	1	2	0.8814645562537212	0.9996157290847086	-	insulin receptor substrate binding	0	F
GO:0043394	1	2	0.8814645562537212	0.9996157290847086	-	proteoglycan binding	0	F
GO:0140031	1	2	0.8814645562537212	0.9996157290847086	-	phosphorylation-dependent protein binding	0	F
GO:0015035	1	2	0.8814645562537212	0.9996157290847086	-	protein disulfide oxidoreductase activity	0	F
GO:0016810	1	2	0.8814645562537212	0.9996157290847086	-	hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds	0	F
GO:0016791	1	2	0.8814645562537212	0.9996157290847086	-	phosphatase activity	0	F
GO:0042834	1	2	0.8814645562537212	0.9996157290847086	-	peptidoglycan binding	0	F
GO:0004185	1	2	0.8814645562537212	0.9996157290847086	-	serine-type carboxypeptidase activity	0	F
GO:0046875	1	2	0.8814645562537212	0.9996157290847086	-	ephrin receptor binding	0	F
GO:0050661	1	2	0.8814645562537212	0.9996157290847086	-	NADP binding	0	F
GO:1990763	1	2	0.8814645562537212	0.9996157290847086	-	arrestin family protein binding	0	F
GO:0017160	1	2	0.8814645562537212	0.9996157290847086	-	Ral GTPase binding	0	F
GO:0017124	8	12	0.88596766474305	0.9996157290847086	-	SH3 domain binding	0	F
GO:0005516	7	11	0.8965669107070411	0.9996157290847086	-	calmodulin binding	0	F
GO:0050839	4	7	0.8982636123610467	0.9996157290847086	-	cell adhesion molecule binding	0	F
GO:0002039	2	4	0.8988632423216547	0.9996157290847086	-	p53 binding	0	F
GO:0051287	2	4	0.8988632423216547	0.9996157290847086	-	NAD binding	0	F
GO:0051020	2	4	0.8988632423216547	0.9996157290847086	-	GTPase binding	0	F
GO:0031267	2	4	0.8988632423216547	0.9996157290847086	-	small GTPase binding	0	F
GO:0043325	2	4	0.8988632423216547	0.9996157290847086	-	phosphatidylinositol-3,4-bisphosphate binding	0	F
GO:0008134	10	15	0.9033330438972506	0.9996157290847086	-	transcription factor binding	0	F
GO:0019899	14	20	0.9066978327449536	0.9996157290847086	-	enzyme binding	0	F
GO:0046934	3	6	0.9189645468762208	0.9996157290847086	-	phosphatidylinositol-4,5-bisphosphate 3-kinase activity	0	F
GO:0004175	3	6	0.9189645468762208	0.9996157290847086	-	endopeptidase activity	0	F
GO:0002020	8	13	0.9197694740243065	0.9996157290847086	-	protease binding	0	F
GO:0038023	6	11	0.9387544119160759	0.9996157290847086	-	signaling receptor activity	0	F
GO:0001540	6	11	0.9387544119160759	0.9996157290847086	-	amyloid-beta binding	0	F
GO:0030276	1	3	0.9418898508196953	0.9996157290847086	-	clathrin binding	0	F
GO:0004708	1	3	0.9418898508196953	0.9996157290847086	-	MAP kinase kinase activity	0	F
GO:0003727	1	3	0.9418898508196953	0.9996157290847086	-	single-stranded RNA binding	0	F
GO:0004177	1	3	0.9418898508196953	0.9996157290847086	-	aminopeptidase activity	0	F
GO:0042288	1	3	0.9418898508196953	0.9996157290847086	-	MHC class I protein binding	0	F
GO:0008138	1	3	0.9418898508196953	0.9996157290847086	-	protein tyrosine/serine/threonine phosphatase activity	0	F
GO:0005041	1	3	0.9418898508196953	0.9996157290847086	-	low-density lipoprotein particle receptor activity	0	F
GO:0030544	1	3	0.9418898508196953	0.9996157290847086	-	Hsp70 protein binding	0	F
GO:0071889	1	3	0.9418898508196953	0.9996157290847086	-	14-3-3 protein binding	0	F
GO:0005154	1	3	0.9418898508196953	0.9996157290847086	-	epidermal growth factor receptor binding	0	F
GO:0001530	1	3	0.9418898508196953	0.9996157290847086	-	lipopolysaccharide binding	0	F
GO:0005044	1	3	0.9418898508196953	0.9996157290847086	-	scavenger receptor activity	0	F
GO:0003729	2	5	0.9433670245011466	0.9996157290847086	-	mRNA binding	0	F
GO:0000977	8	14	0.9445664275527034	0.9996157290847086	-	RNA polymerase II regulatory region sequence-specific DNA binding	0	F
GO:0048365	3	7	0.9517061493675614	0.9996157290847086	-	Rac GTPase binding	0	F
GO:0042826	3	7	0.9517061493675614	0.9996157290847086	-	histone deacetylase binding	0	F
GO:0046983	3	7	0.9517061493675614	0.9996157290847086	-	protein dimerization activity	0	F
GO:0005215	3	7	0.9517061493675614	0.9996157290847086	-	transporter activity	0	F
GO:0005125	3	7	0.9517061493675614	0.9996157290847086	-	cytokine activity	0	F
GO:0044212	4	9	0.9603080779435565	0.9996157290847086	-	transcription regulatory region DNA binding	0	F
GO:0004713	4	9	0.9603080779435565	0.9996157290847086	-	protein tyrosine kinase activity	0	F
GO:0000978	13	22	0.9661608883944599	0.9996157290847086	-	RNA polymerase II proximal promoter sequence-specific DNA binding	0	F
GO:0004888	2	6	0.9688185339175227	0.9996157290847086	-	transmembrane signaling receptor activity	0	F
GO:0008233	2	6	0.9688185339175227	0.9996157290847086	-	peptidase activity	0	F
GO:0001784	2	6	0.9688185339175227	0.9996157290847086	-	phosphotyrosine residue binding	0	F
GO:0051019	1	4	0.9715413638911704	0.9996157290847086	-	mitogen-activated protein kinase binding	0	F
GO:0004435	1	4	0.9715413638911704	0.9996157290847086	-	phosphatidylinositol phospholipase C activity	0	F
GO:0004869	1	4	0.9715413638911704	0.9996157290847086	-	cysteine-type endopeptidase inhibitor activity	0	F
GO:0000981	27	40	0.972149427090158	0.9996157290847086	-	DNA-binding transcription factor activity, RNA polymerase II-specific	0	F
GO:0009055	4	10	0.9758921006243002	0.9996157290847086	-	electron transfer activity	0	F
GO:0001664	2	7	0.9830625302150375	0.9996157290847086	-	G protein-coupled receptor binding	0	F
GO:0001227	6	14	0.9835716292309843	0.9996157290847086	-	DNA-binding transcription repressor activity, RNA polymerase II-specific	0	F
GO:0031072	1	5	0.9860769882050736	0.9996157290847086	-	heat shock protein binding	0	F
GO:0023026	1	5	0.9860769882050736	0.9996157290847086	-	MHC class II protein complex binding	0	F
GO:0001228	8	18	0.9891840340865644	0.9996157290847086	-	DNA-binding transcription activator activity, RNA polymerase II-specific	0	F
GO:0003700	15	28	0.9894670612113068	0.9996157290847086	-	DNA-binding transcription factor activity	0	F
GO:0004252	2	9	0.9951581864673293	0.9996157290847086	-	serine-type endopeptidase activity	0	F
GO:0003723	24	44	0.9973923988332474	0.9996157290847086	-	RNA binding	0	F
GO:0004197	1	9	0.9992104979376742	0.9996157290847086	-	cysteine-type endopeptidase activity	0	F
GO:0030246	1	10	0.9996157290847086	0.9996157290847086	-	carbohydrate binding	0	F
GO:0031093	18	5	0.0061426277530148175	0.9998202045939182	-	platelet alpha granule lumen	0	C
GO:0005575	12	2	0.00733135739457189	0.9998202045939182	-	cellular_component	0	C
GO:0002102	8	1	0.02153538235401066	0.9998202045939182	-	podosome	0	C
GO:0005815	11	3	0.03206927847879207	0.9998202045939182	-	microtubule organizing center	0	C
GO:0005923	11	3	0.03206927847879207	0.9998202045939182	-	bicellular tight junction	0	C
GO:0001725	12	4	0.04302309536176781	0.9998202045939182	-	stress fiber	0	C
GO:0005856	30	17	0.046260329408703095	0.9998202045939182	-	cytoskeleton	0	C
GO:0005874	13	5	0.05401749770879116	0.9998202045939182	-	microtubule	0	C
GO:0030018	8	2	0.059850004314433755	0.9998202045939182	-	Z disc	0	C
GO:0072686	6	1	0.06744776696746518	0.9998202045939182	-	mitotic spindle	0	C
GO:0030017	6	1	0.06744776696746518	0.9998202045939182	-	sarcomere	0	C
GO:0045335	9	3	0.08010261613434412	0.9998202045939182	-	phagocytic vesicle	0	C
GO:0072562	9	3	0.08010261613434412	0.9998202045939182	-	blood microparticle	0	C
GO:0005912	5	1	0.11662770242832539	0.9998202045939182	-	adherens junction	0	C
GO:0005796	5	1	0.11662770242832539	0.9998202045939182	-	Golgi lumen	0	C
GO:0000786	5	1	0.11662770242832539	0.9998202045939182	-	nucleosome	0	C
GO:0045202	8	3	0.1230170146261113	0.9998202045939182	-	synapse	0	C
GO:0031092	8	3	0.1230170146261113	0.9998202045939182	-	platelet alpha granule membrane	0	C
GO:0005739	39	28	0.1304903067693727	0.9998202045939182	-	mitochondrion	0	C
GO:0005802	9	4	0.14529867768538351	0.9998202045939182	-	trans-Golgi network	0	C
GO:0031901	9	4	0.14529867768538351	0.9998202045939182	-	early endosome membrane	0	C
GO:0005911	13	7	0.14576207420118203	0.9998202045939182	-	cell-cell junction	0	C
GO:0012505	6	2	0.15469393678749122	0.9998202045939182	-	endomembrane system	0	C
GO:0098978	25	17	0.16025544496658456	0.9998202045939182	-	glutamatergic synapse	0	C
GO:0055038	10	5	0.16474615990768413	0.9998202045939182	-	recycling endosome membrane	0	C
GO:0005913	11	6	0.1818829521274304	0.9998202045939182	-	cell-cell adherens junction	0	C
GO:0005635	7	3	0.18458951788137756	0.9998202045939182	-	nuclear envelope	0	C
GO:0043005	12	7	0.1971234900402606	0.9998202045939182	-	neuron projection	0	C
GO:0031088	4	1	0.19740748702539465	0.9998202045939182	-	platelet dense granule membrane	0	C
GO:0032580	4	1	0.19740748702539465	0.9998202045939182	-	Golgi cisterna membrane	0	C
GO:0000139	34	26	0.21224313835648284	0.9998202045939182	-	Golgi membrane	0	C
GO:0055037	9	5	0.22905054137018013	0.9998202045939182	-	recycling endosome	0	C
GO:0032993	5	2	0.23957754108047588	0.9998202045939182	-	protein-DNA complex	0	C
GO:0016605	5	2	0.23957754108047588	0.9998202045939182	-	PML body	0	C
GO:0005903	5	2	0.23957754108047588	0.9998202045939182	-	brush border	0	C
GO:0005770	5	2	0.23957754108047588	0.9998202045939182	-	late endosome	0	C
GO:0043197	10	6	0.24623861961682567	0.9998202045939182	-	dendritic spine	0	C
GO:0014069	11	7	0.2611123339983916	0.9998202045939182	-	postsynaptic density	0	C
GO:0016604	11	7	0.2611123339983916	0.9998202045939182	-	nuclear body	0	C
GO:0005829	223	205	0.265140944010753	0.9998202045939182	-	cytosol	0	C
GO:0030424	12	8	0.2741656137988785	0.9998202045939182	-	axon	0	C
GO:0005929	7	4	0.2923413985780936	0.9998202045939182	-	cilium	0	C
GO:0005925	48	41	0.3080193913901595	0.9998202045939182	-	focal adhesion	0	C
GO:0005902	8	5	0.3105212258884649	0.9998202045939182	-	microvillus	0	C
GO:0015629	26	21	0.3156223622448358	0.9998202045939182	-	actin cytoskeleton	0	C
GO:0043034	3	1	0.3246628856507586	0.9998202045939182	-	costamere	0	C
GO:0099059	3	1	0.3246628856507586	0.9998202045939182	-	integral component of presynaptic active zone membrane	0	C
GO:0014704	3	1	0.3246628856507586	0.9998202045939182	-	intercalated disc	0	C
GO:0030140	3	1	0.3246628856507586	0.9998202045939182	-	trans-Golgi network transport vesicle	0	C
GO:0005694	3	1	0.3246628856507586	0.9998202045939182	-	chromosome	0	C
GO:0005862	3	1	0.3246628856507586	0.9998202045939182	-	muscle thin filament tropomyosin	0	C
GO:1904115	3	1	0.3246628856507586	0.9998202045939182	-	axon cytoplasm	0	C
GO:0043025	19	15	0.33595124302273377	0.9998202045939182	-	neuronal cell body	0	C
GO:0048471	34	29	0.35061297398156893	0.9998202045939182	-	perinuclear region of cytoplasm	0	C
GO:0031410	12	9	0.358662878519243	0.9998202045939182	-	cytoplasmic vesicle	0	C
GO:0043198	4	2	0.3589670562195333	0.9998202045939182	-	dendritic shaft	0	C
GO:0001650	4	2	0.3589670562195333	0.9998202045939182	-	fibrillar center	0	C
GO:0030864	4	2	0.3589670562195333	0.9998202045939182	-	cortical actin cytoskeleton	0	C
GO:0005901	4	2	0.3589670562195333	0.9998202045939182	-	caveola	0	C
GO:0030863	4	2	0.3589670562195333	0.9998202045939182	-	cortical cytoskeleton	0	C
GO:0031226	4	2	0.3589670562195333	0.9998202045939182	-	intrinsic component of plasma membrane	0	C
GO:0042629	5	3	0.3810502149021172	0.9998202045939182	-	mast cell granule	0	C
GO:0005811	5	3	0.3810502149021172	0.9998202045939182	-	lipid droplet	0	C
GO:0008305	5	3	0.3810502149021172	0.9998202045939182	-	integrin complex	0	C
GO:0017053	5	3	0.3810502149021172	0.9998202045939182	-	transcriptional repressor complex	0	C
GO:0031966	6	4	0.39696095716153285	0.9998202045939182	-	mitochondrial membrane	0	C
GO:0031982	18	15	0.3990507992618792	0.9998202045939182	-	vesicle	0	C
GO:0030659	7	5	0.4092352203339207	0.9998202045939182	-	cytoplasmic vesicle membrane	0	C
GO:0008021	7	5	0.4092352203339207	0.9998202045939182	-	synaptic vesicle	0	C
GO:0005768	15	13	0.4595728605878306	0.9998202045939182	-	endosome	0	C
GO:0030054	17	15	0.4667962094167418	0.9998202045939182	-	cell junction	0	C
GO:0005747	2	1	0.5124257791032786	0.9998202045939182	-	mitochondrial respiratory chain complex I	0	C
GO:0000118	2	1	0.5124257791032786	0.9998202045939182	-	histone deacetylase complex	0	C
GO:0099056	2	1	0.5124257791032786	0.9998202045939182	-	integral component of presynaptic membrane	0	C
GO:0005881	2	1	0.5124257791032786	0.9998202045939182	-	cytoplasmic microtubule	0	C
GO:0030478	2	1	0.5124257791032786	0.9998202045939182	-	actin cap	0	C
GO:0000407	2	1	0.5124257791032786	0.9998202045939182	-	phagophore assembly site	0	C
GO:0000228	2	1	0.5124257791032786	0.9998202045939182	-	nuclear chromosome	0	C
GO:0000151	2	1	0.5124257791032786	0.9998202045939182	-	ubiquitin ligase complex	0	C
GO:0008290	2	1	0.5124257791032786	0.9998202045939182	-	F-actin capping protein complex	0	C
GO:0012506	2	1	0.5124257791032786	0.9998202045939182	-	vesicle membrane	0	C
GO:0016529	2	1	0.5124257791032786	0.9998202045939182	-	sarcoplasmic reticulum	0	C
GO:0014802	2	1	0.5124257791032786	0.9998202045939182	-	terminal cisterna	0	C
GO:0031224	2	1	0.5124257791032786	0.9998202045939182	-	intrinsic component of membrane	0	C
GO:0031252	2	1	0.5124257791032786	0.9998202045939182	-	cell leading edge	0	C
GO:0000421	2	1	0.5124257791032786	0.9998202045939182	-	autophagosome membrane	0	C
GO:0005793	2	1	0.5124257791032786	0.9998202045939182	-	endoplasmic reticulum-Golgi intermediate compartment	0	C
GO:0032585	2	1	0.5124257791032786	0.9998202045939182	-	multivesicular body membrane	0	C
GO:0097440	2	1	0.5124257791032786	0.9998202045939182	-	apical dendrite	0	C
GO:0034666	3	2	0.5155459835888045	0.9998202045939182	-	integrin alpha2-beta1 complex	0	C
GO:0098685	3	2	0.5155459835888045	0.9998202045939182	-	Schaffer collateral - CA1 synapse	0	C
GO:0000776	3	2	0.5155459835888045	0.9998202045939182	-	kinetochore	0	C
GO:0031201	3	2	0.5155459835888045	0.9998202045939182	-	SNARE complex	0	C
GO:0005814	3	2	0.5155459835888045	0.9998202045939182	-	centriole	0	C
GO:0097386	3	2	0.5155459835888045	0.9998202045939182	-	glial cell projection	0	C
GO:0097546	3	2	0.5155459835888045	0.9998202045939182	-	ciliary base	0	C
GO:0043679	3	2	0.5155459835888045	0.9998202045939182	-	axon terminus	0	C
GO:0030133	3	2	0.5155459835888045	0.9998202045939182	-	transport vesicle	0	C
GO:0044754	4	3	0.5181530764049431	0.9998202045939182	-	autolysosome	0	C
GO:0036064	4	3	0.5181530764049431	0.9998202045939182	-	ciliary basal body	0	C
GO:0032154	5	4	0.5204403665379638	0.9998202045939182	-	cleavage furrow	0	C
GO:0016607	10	9	0.5294006291104987	0.9998202045939182	-	nuclear speck	0	C
GO:0005783	29	28	0.5513621134064415	0.9998202045939182	-	endoplasmic reticulum	0	C
GO:0005654	87	85	0.5614434426133867	0.9998202045939182	-	nucleoplasm	0	C
GO:0005737	173	170	0.5961694799712725	0.9998202045939182	-	cytoplasm	0	C
GO:0005794	42	42	0.6073814760808275	0.9998202045939182	-	Golgi apparatus	0	C
GO:0005813	13	13	0.6119136143056737	0.9998202045939182	-	centrosome	0	C
GO:0032587	12	12	0.6136065037229891	0.9998202045939182	-	ruffle membrane	0	C
GO:0042995	10	10	0.6181214652177996	0.9998202045939182	-	cell projection	0	C
GO:0005741	8	8	0.6249120142701703	0.9998202045939182	-	mitochondrial outer membrane	0	C
GO:0005759	8	8	0.6249120142701703	0.9998202045939182	-	mitochondrial matrix	0	C
GO:0031012	6	6	0.6357736345893998	0.9998202045939182	-	extracellular matrix	0	C
GO:0042383	5	5	0.6439197759143948	0.9998202045939182	-	sarcolemma	0	C
GO:1903561	4	4	0.6552559379077691	0.9998202045939182	-	extracellular vesicle	0	C
GO:0030136	4	4	0.6552559379077691	0.9998202045939182	-	clathrin-coated vesicle	0	C
GO:0032991	26	27	0.6576245348671922	0.9998202045939182	-	protein-containing complex	0	C
GO:0000784	3	3	0.6721249109580756	0.9998202045939182	-	nuclear chromosome, telomeric region	0	C
GO:0031594	3	3	0.6721249109580756	0.9998202045939182	-	neuromuscular junction	0	C
GO:0005604	3	3	0.6721249109580756	0.9998202045939182	-	basement membrane	0	C
GO:0044291	3	3	0.6721249109580756	0.9998202045939182	-	cell-cell contact zone	0	C
GO:0090575	2	2	0.7001886725557986	0.9998202045939182	-	RNA polymerase II transcription factor complex	0	C
GO:0036020	2	2	0.7001886725557986	0.9998202045939182	-	endolysosome membrane	0	C
GO:0031941	2	2	0.7001886725557986	0.9998202045939182	-	filamentous actin	0	C
GO:0044853	2	2	0.7001886725557986	0.9998202045939182	-	plasma membrane raft	0	C
GO:0030665	2	2	0.7001886725557986	0.9998202045939182	-	clathrin-coated vesicle membrane	0	C
GO:0008043	2	2	0.7001886725557986	0.9998202045939182	-	intracellular ferritin complex	0	C
GO:0032153	2	2	0.7001886725557986	0.9998202045939182	-	cell division site	0	C
GO:0005719	2	2	0.7001886725557986	0.9998202045939182	-	nuclear euchromatin	0	C
GO:0008180	2	2	0.7001886725557986	0.9998202045939182	-	COP9 signalosome	0	C
GO:0030672	2	2	0.7001886725557986	0.9998202045939182	-	synaptic vesicle membrane	0	C
GO:0030315	2	2	0.7001886725557986	0.9998202045939182	-	T-tubule	0	C
GO:0030139	2	2	0.7001886725557986	0.9998202045939182	-	endocytic vesicle	0	C
GO:0016459	2	2	0.7001886725557986	0.9998202045939182	-	myosin complex	0	C
GO:0030016	2	2	0.7001886725557986	0.9998202045939182	-	myofibril	0	C
GO:0031527	2	2	0.7001886725557986	0.9998202045939182	-	filopodium membrane	0	C
GO:0031258	2	2	0.7001886725557986	0.9998202045939182	-	lamellipodium membrane	0	C
GO:0043195	2	2	0.7001886725557986	0.9998202045939182	-	terminal bouton	0	C
GO:0042470	9	10	0.7037292895318623	0.9998202045939182	-	melanosome	0	C
GO:0001726	9	10	0.7037292895318623	0.9998202045939182	-	ruffle	0	C
GO:0016324	8	9	0.7112086780959164	0.9998202045939182	-	apical plasma membrane	0	C
GO:0031965	8	9	0.7112086780959164	0.9998202045939182	-	nuclear membrane	0	C
GO:0005886	171	173	0.7173298278320976	0.9998202045939182	-	plasma membrane	0	C
GO:0045211	6	7	0.7312354237914973	0.9998202045939182	-	postsynaptic membrane	0	C
GO:0015630	6	7	0.7312354237914973	0.9998202045939182	-	microtubule cytoskeleton	0	C
GO:0043204	5	6	0.7450992329583398	0.9998202045939182	-	perikaryon	0	C
GO:0010008	13	15	0.7463883882701964	0.9998202045939182	-	endosome membrane	0	C
GO:0030027	12	14	0.7520238719810098	0.9998202045939182	-	lamellipodium	0	C
GO:0005750	1	1	0.758451630536907	0.9998202045939182	-	mitochondrial respiratory chain complex III	0	C
GO:0008385	1	1	0.758451630536907	0.9998202045939182	-	IkappaB kinase complex	0	C
GO:0000777	1	1	0.758451630536907	0.9998202045939182	-	condensed chromosome kinetochore	0	C
GO:0034677	1	1	0.758451630536907	0.9998202045939182	-	integrin alpha7-beta1 complex	0	C
GO:0034667	1	1	0.758451630536907	0.9998202045939182	-	integrin alpha3-beta1 complex	0	C
GO:0034665	1	1	0.758451630536907	0.9998202045939182	-	integrin alpha1-beta1 complex	0	C
GO:0032809	1	1	0.758451630536907	0.9998202045939182	-	neuronal cell body membrane	0	C
GO:0035327	1	1	0.758451630536907	0.9998202045939182	-	transcriptionally active chromatin	0	C
GO:0035267	1	1	0.758451630536907	0.9998202045939182	-	NuA4 histone acetyltransferase complex	0	C
GO:0010494	1	1	0.758451630536907	0.9998202045939182	-	cytoplasmic stress granule	0	C
GO:0032433	1	1	0.758451630536907	0.9998202045939182	-	filopodium tip	0	C
GO:0034678	1	1	0.758451630536907	0.9998202045939182	-	integrin alpha8-beta1 complex	0	C
GO:0001740	1	1	0.758451630536907	0.9998202045939182	-	Barr body	0	C
GO:0001669	1	1	0.758451630536907	0.9998202045939182	-	acrosomal vesicle	0	C
GO:0030660	1	1	0.758451630536907	0.9998202045939182	-	Golgi-associated vesicle membrane	0	C
GO:0030485	1	1	0.758451630536907	0.9998202045939182	-	smooth muscle contractile fiber	0	C
GO:0072546	1	1	0.758451630536907	0.9998202045939182	-	ER membrane protein complex	0	C
GO:0099091	1	1	0.758451630536907	0.9998202045939182	-	postsynaptic specialization, intracellular component	0	C
GO:0098871	1	1	0.758451630536907	0.9998202045939182	-	postsynaptic actin cytoskeleton	0	C
GO:0098686	1	1	0.758451630536907	0.9998202045939182	-	hippocampal mossy fiber to CA3 synapse	0	C
GO:0036284	1	1	0.758451630536907	0.9998202045939182	-	tubulobulbar complex	0	C
GO:0097433	1	1	0.758451630536907	0.9998202045939182	-	dense body	0	C
GO:0043296	1	1	0.758451630536907	0.9998202045939182	-	apical junction complex	0	C
GO:0120103	1	1	0.758451630536907	0.9998202045939182	-	centriolar subdistal appendage	0	C
GO:0099699	1	1	0.758451630536907	0.9998202045939182	-	integral component of synaptic membrane	0	C
GO:0099523	1	1	0.758451630536907	0.9998202045939182	-	presynaptic cytosol	0	C
GO:0030904	1	1	0.758451630536907	0.9998202045939182	-	retromer complex	0	C
GO:1990425	1	1	0.758451630536907	0.9998202045939182	-	ryanodine receptor complex	0	C
GO:0097418	1	1	0.758451630536907	0.9998202045939182	-	neurofibrillary tangle	0	C
GO:0034707	1	1	0.758451630536907	0.9998202045939182	-	chloride channel complex	0	C
GO:0034704	1	1	0.758451630536907	0.9998202045939182	-	calcium channel complex	0	C
GO:0034681	1	1	0.758451630536907	0.9998202045939182	-	integrin alpha11-beta1 complex	0	C
GO:0034680	1	1	0.758451630536907	0.9998202045939182	-	integrin alpha10-beta1 complex	0	C
GO:0048787	1	1	0.758451630536907	0.9998202045939182	-	presynaptic active zone membrane	0	C
GO:0031528	1	1	0.758451630536907	0.9998202045939182	-	microvillus membrane	0	C
GO:0005845	1	1	0.758451630536907	0.9998202045939182	-	mRNA cap binding complex	0	C
GO:0005882	1	1	0.758451630536907	0.9998202045939182	-	intermediate filament	0	C
GO:0097452	1	1	0.758451630536907	0.9998202045939182	-	GAIT complex	0	C
GO:0098562	1	1	0.758451630536907	0.9998202045939182	-	cytoplasmic side of membrane	0	C
GO:0000785	1	1	0.758451630536907	0.9998202045939182	-	chromatin	0	C
GO:0030132	1	1	0.758451630536907	0.9998202045939182	-	clathrin coat of coated pit	0	C
GO:0036057	1	1	0.758451630536907	0.9998202045939182	-	slit diaphragm	0	C
GO:0005749	1	1	0.758451630536907	0.9998202045939182	-	mitochondrial respiratory chain complex II, succinate dehydrogenase complex (ubiquinone)	0	C
GO:0044753	1	1	0.758451630536907	0.9998202045939182	-	amphisome	0	C
GO:0043220	1	1	0.758451630536907	0.9998202045939182	-	Schmidt-Lanterman incisure	0	C
GO:0030119	1	1	0.758451630536907	0.9998202045939182	-	AP-type membrane coat adaptor complex	0	C
GO:0016600	1	1	0.758451630536907	0.9998202045939182	-	flotillin complex	0	C
GO:0019005	1	1	0.758451630536907	0.9998202045939182	-	SCF ubiquitin ligase complex	0	C
GO:0005577	1	1	0.758451630536907	0.9998202045939182	-	fibrinogen complex	0	C
GO:0005640	1	1	0.758451630536907	0.9998202045939182	-	nuclear outer membrane	0	C
GO:0005778	1	1	0.758451630536907	0.9998202045939182	-	peroxisomal membrane	0	C
GO:0030688	1	1	0.758451630536907	0.9998202045939182	-	preribosome, small subunit precursor	0	C
GO:0031523	1	1	0.758451630536907	0.9998202045939182	-	Myb complex	0	C
GO:0031519	1	1	0.758451630536907	0.9998202045939182	-	PcG protein complex	0	C
GO:0005662	1	1	0.758451630536907	0.9998202045939182	-	DNA replication factor A complex	0	C
GO:0031904	1	1	0.758451630536907	0.9998202045939182	-	endosome lumen	0	C
GO:0032592	1	1	0.758451630536907	0.9998202045939182	-	integral component of mitochondrial membrane	0	C
GO:0005790	1	1	0.758451630536907	0.9998202045939182	-	smooth endoplasmic reticulum	0	C
GO:0035748	1	1	0.758451630536907	0.9998202045939182	-	myelin sheath abaxonal region	0	C
GO:0031430	1	1	0.758451630536907	0.9998202045939182	-	M band	0	C
GO:0031312	1	1	0.758451630536907	0.9998202045939182	-	extrinsic component of organelle membrane	0	C
GO:0016461	1	1	0.758451630536907	0.9998202045939182	-	unconventional myosin complex	0	C
GO:0016442	1	1	0.758451630536907	0.9998202045939182	-	RISC complex	0	C
GO:0016281	1	1	0.758451630536907	0.9998202045939182	-	eukaryotic translation initiation factor 4F complex	0	C
GO:0035867	1	1	0.758451630536907	0.9998202045939182	-	alphav-beta3 integrin-IGF-1-IGF1R complex	0	C
GO:0005789	22	25	0.7621786459351089	0.9998202045939182	-	endoplasmic reticulum membrane	0	C
GO:0005885	4	5	0.7631083950036134	0.9998202045939182	-	Arp2/3 protein complex	0	C
GO:0036464	4	5	0.7631083950036134	0.9998202045939182	-	cytoplasmic ribonucleoprotein granule	0	C
GO:0035578	10	12	0.7655557192219142	0.9998202045939182	-	azurophil granule lumen	0	C
GO:0043209	9	11	0.7737720548797316	0.9998202045939182	-	myelin sheath	0	C
GO:0000790	8	10	0.7832535760542025	0.9998202045939182	-	nuclear chromatin	0	C
GO:0098793	3	4	0.7876037868729249	0.9998202045939182	-	presynapse	0	C
GO:0016363	3	4	0.7876037868729249	0.9998202045939182	-	nuclear matrix	0	C
GO:0030141	3	4	0.7876037868729249	0.9998202045939182	-	secretory granule	0	C
GO:0001772	3	4	0.7876037868729249	0.9998202045939182	-	immunological synapse	0	C
GO:0009986	33	38	0.8112841352737926	0.9998202045939182	-	cell surface	0	C
GO:0035579	10	13	0.822123073633366	0.9998202045939182	-	specific granule membrane	0	C
GO:0031225	2	3	0.823283798533795	0.9998202045939182	-	anchored component of membrane	0	C
GO:0030122	2	3	0.823283798533795	0.9998202045939182	-	AP-2 adaptor complex	0	C
GO:0045334	2	3	0.823283798533795	0.9998202045939182	-	clathrin-coated endocytic vesicle	0	C
GO:0016328	2	3	0.823283798533795	0.9998202045939182	-	lateral plasma membrane	0	C
GO:0000502	2	3	0.823283798533795	0.9998202045939182	-	proteasome complex	0	C
GO:0005834	2	3	0.823283798533795	0.9998202045939182	-	heterotrimeric G-protein complex	0	C
GO:0005884	8	11	0.8410894207977226	0.9998202045939182	-	actin filament	0	C
GO:0043231	30	36	0.8484731983093005	0.9998202045939182	-	intracellular membrane-bounded organelle	0	C
GO:0005667	7	10	0.8525006138976039	0.9998202045939182	-	transcription factor complex	0	C
GO:0005769	11	15	0.859725064182325	0.9998202045939182	-	early endosome	0	C
GO:0005887	37	44	0.8602824490659219	0.9998202045939182	-	integral component of plasma membrane	0	C
GO:0005905	3	5	0.8670124962520183	0.9998202045939182	-	clathrin-coated pit	0	C
GO:0019898	3	5	0.8670124962520183	0.9998202045939182	-	extrinsic component of membrane	0	C
GO:0000932	3	5	0.8670124962520183	0.9998202045939182	-	P-body	0	C
GO:0030175	3	5	0.8670124962520183	0.9998202045939182	-	filopodium	0	C
GO:0044297	3	5	0.8670124962520183	0.9998202045939182	-	cell body	0	C
GO:0030425	10	14	0.8676634964079808	0.9998202045939182	-	dendrite	0	C
GO:0032009	1	2	0.8814645562537212	0.9998202045939182	-	early phagosome	0	C
GO:0032432	1	2	0.8814645562537212	0.9998202045939182	-	actin filament bundle	0	C
GO:0005774	1	2	0.8814645562537212	0.9998202045939182	-	vacuolar membrane	0	C
GO:0005795	1	2	0.8814645562537212	0.9998202045939182	-	Golgi stack	0	C
GO:0016234	1	2	0.8814645562537212	0.9998202045939182	-	inclusion body	0	C
GO:0071986	1	2	0.8814645562537212	0.9998202045939182	-	Ragulator complex	0	C
GO:0098982	1	2	0.8814645562537212	0.9998202045939182	-	GABA-ergic synapse	0	C
GO:0000145	1	2	0.8814645562537212	0.9998202045939182	-	exocyst	0	C
GO:0071438	1	2	0.8814645562537212	0.9998202045939182	-	invadopodium membrane	0	C
GO:0001931	1	2	0.8814645562537212	0.9998202045939182	-	uropod	0	C
GO:0031305	1	2	0.8814645562537212	0.9998202045939182	-	integral component of mitochondrial inner membrane	0	C
GO:0005771	1	2	0.8814645562537212	0.9998202045939182	-	multivesicular body	0	C
GO:0031090	1	2	0.8814645562537212	0.9998202045939182	-	organelle membrane	0	C
GO:0031362	1	2	0.8814645562537212	0.9998202045939182	-	anchored component of external side of plasma membrane	0	C
GO:0005844	1	2	0.8814645562537212	0.9998202045939182	-	polysome	0	C
GO:0071944	1	2	0.8814645562537212	0.9998202045939182	-	cell periphery	0	C
GO:0045177	1	2	0.8814645562537212	0.9998202045939182	-	apical part of cell	0	C
GO:0030061	1	2	0.8814645562537212	0.9998202045939182	-	mitochondrial crista	0	C
GO:0097487	1	2	0.8814645562537212	0.9998202045939182	-	multivesicular body, internal vesicle	0	C
GO:0036195	1	2	0.8814645562537212	0.9998202045939182	-	muscle cell projection membrane	0	C
GO:0016235	1	2	0.8814645562537212	0.9998202045939182	-	aggresome	0	C
GO:0035577	8	12	0.88596766474305	0.9998202045939182	-	azurophil granule membrane	0	C
GO:0009897	12	17	0.8891182289277109	0.9998202045939182	-	external side of plasma membrane	0	C
GO:0001891	2	4	0.8988632423216547	0.9998202045939182	-	phagocytic cup	0	C
GO:0042734	2	4	0.8988632423216547	0.9998202045939182	-	presynaptic membrane	0	C
GO:0016021	70	81	0.8993557204446199	0.9998202045939182	-	integral component of membrane	0	C
GO:0045121	22	30	0.9199702119687302	0.9998202045939182	-	membrane raft	0	C
GO:0043235	5	9	0.9213480212416973	0.9998202045939182	-	receptor complex	0	C
GO:0030667	11	17	0.924120691321449	0.9998202045939182	-	secretory granule membrane	0	C
GO:0005576	70	83	0.9263743878039191	0.9998202045939182	-	extracellular region	0	C
GO:0005743	7	12	0.9289287798208101	0.9998202045939182	-	mitochondrial inner membrane	0	C
GO:0030426	4	8	0.9358009455020645	0.9998202045939182	-	growth cone	0	C
GO:1990904	4	8	0.9358009455020645	0.9998202045939182	-	ribonucleoprotein complex	0	C
GO:0031902	6	11	0.9387544119160759	0.9998202045939182	-	late endosome membrane	0	C
GO:0030670	6	11	0.9387544119160759	0.9998202045939182	-	phagocytic vesicle membrane	0	C
GO:0030134	1	3	0.9418898508196953	0.9998202045939182	-	COPII-coated ER to Golgi transport vesicle	0	C
GO:0005681	1	3	0.9418898508196953	0.9998202045939182	-	spliceosomal complex	0	C
GO:0009898	1	3	0.9418898508196953	0.9998202045939182	-	cytoplasmic side of plasma membrane	0	C
GO:0035861	2	5	0.9433670245011466	0.9998202045939182	-	site of double-strand break	0	C
GO:0033116	2	5	0.9433670245011466	0.9998202045939182	-	endoplasmic reticulum-Golgi intermediate compartment membrane	0	C
GO:0005788	8	14	0.9445664275527034	0.9998202045939182	-	endoplasmic reticulum lumen	0	C
GO:0043202	5	10	0.9492392681520511	0.9998202045939182	-	lysosomal lumen	0	C
GO:0005634	150	169	0.9559160591523517	0.9998202045939182	-	nucleus	0	C
GO:0016323	6	12	0.9598481625205934	0.9998202045939182	-	basolateral plasma membrane	0	C
GO:0030658	2	6	0.9688185339175227	0.9998202045939182	-	transport vesicle membrane	0	C
GO:0098794	2	6	0.9688185339175227	0.9998202045939182	-	postsynapse	0	C
GO:0034774	9	17	0.9703219962242077	0.9998202045939182	-	secretory granule lumen	0	C
GO:0031514	1	4	0.9715413638911704	0.9998202045939182	-	motile cilium	0	C
GO:0005758	1	4	0.9715413638911704	0.9998202045939182	-	mitochondrial intermembrane space	0	C
GO:0032588	3	8	0.9717471007450044	0.9998202045939182	-	trans-Golgi network membrane	0	C
GO:0005938	6	13	0.9741186468435702	0.9998202045939182	-	cell cortex	0	C
GO:0101003	6	13	0.9741186468435702	0.9998202045939182	-	ficolin-1-rich granule membrane	0	C
GO:1904813	13	23	0.976012751720208	0.9998202045939182	-	ficolin-1-rich granule lumen	0	C
GO:0035580	2	7	0.9830625302150375	0.9998202045939182	-	specific granule lumen	0	C
GO:0070821	6	14	0.9835716292309843	0.9998202045939182	-	tertiary granule membrane	0	C
GO:0030669	3	9	0.983729152492651	0.9998202045939182	-	clathrin-coated endocytic vesicle membrane	0	C
GO:0070062	133	159	0.9862051521815456	0.9998202045939182	-	extracellular exosome	0	C
GO:0062023	14	26	0.9867026264455738	0.9998202045939182	-	collagen-containing extracellular matrix	0	C
GO:0005730	18	32	0.9894799453639577	0.9998202045939182	-	nucleolus	0	C
GO:0031234	2	8	0.9909016254269066	0.9998202045939182	-	extrinsic component of cytoplasmic side of plasma membrane	0	C
GO:0012507	1	6	0.9931953154890615	0.9998202045939182	-	ER to Golgi transport vesicle membrane	0	C
GO:0005791	1	6	0.9931953154890615	0.9998202045939182	-	rough endoplasmic reticulum	0	C
GO:0030666	3	11	0.9948091223754919	0.9998202045939182	-	endocytic vesicle membrane	0	C
GO:1904724	4	13	0.9950471203373457	0.9998202045939182	-	tertiary granule lumen	0	C
GO:0030496	2	9	0.9951581864673293	0.9998202045939182	-	midbody	0	C
GO:0005615	43	66	0.9956222046114681	0.9998202045939182	-	extracellular space	0	C
GO:0016020	83	115	0.9979721843411526	0.9998202045939182	-	membrane	0	C
GO:0005764	5	20	0.999705828130743	0.9998202045939182	-	lysosome	0	C
GO:0005765	10	30	0.9998202045939182	0.9998202045939182	-	lysosomal membrane	0	C
GO:0006936	15	1	0.00029963738095750497	0.4102035745308243	-	muscle contraction	0	P
GO:0002576	38	14	0.0006823770461398712	0.4670870880827418	-	platelet degranulation	0	P
GO:0070527	20	5	0.002371495351253638	0.9999736573290944	-	platelet aggregation	0	P
GO:0006886	24	8	0.004088708428301748	0.9999736573290944	-	intracellular protein transport	0	P
GO:0000278	12	2	0.00733135739457189	0.9999736573290944	-	mitotic cell cycle	0	P
GO:0030168	20	8	0.020636185363494354	0.9999736573290944	-	platelet activation	0	P
GO:0007596	21	9	0.02475731476987943	0.9999736573290944	-	blood coagulation	0	P
GO:0006888	11	3	0.03206927847879207	0.9999736573290944	-	endoplasmic reticulum to Golgi vesicle-mediated transport	0	P
GO:0000079	7	1	0.038365710360789826	0.9999736573290944	-	regulation of cyclin-dependent protein serine/threonine kinase activity	0	P
GO:0030198	16	7	0.052854422820239236	0.9999736573290944	-	extracellular matrix organization	0	P
GO:0008150	20	10	0.05665024449447191	0.9999736573290944	-	biological_process	0	P
GO:0016197	6	1	0.06744776696746518	0.9999736573290944	-	endosomal transport	0	P
GO:0050714	6	1	0.06744776696746518	0.9999736573290944	-	positive regulation of protein secretion	0	P
GO:0007017	6	1	0.06744776696746518	0.9999736573290944	-	microtubule-based process	0	P
GO:0016236	6	1	0.06744776696746518	0.9999736573290944	-	macroautophagy	0	P
GO:0072659	12	5	0.07983308333025489	0.9999736573290944	-	protein localization to plasma membrane	0	P
GO:0030335	21	12	0.09308486252247104	0.9999736573290944	-	positive regulation of cell migration	0	P
GO:0000226	13	6	0.09316303388405328	0.9999736573290944	-	microtubule cytoskeleton organization	0	P
GO:0007265	7	2	0.09727185838451691	0.9999736573290944	-	Ras protein signal transduction	0	P
GO:0035329	5	1	0.11662770242832539	0.9999736573290944	-	hippo signaling	0	P
GO:0035307	5	1	0.11662770242832539	0.9999736573290944	-	positive regulation of protein dephosphorylation	0	P
GO:0006904	5	1	0.11662770242832539	0.9999736573290944	-	vesicle docking involved in exocytosis	0	P
GO:0031175	5	1	0.11662770242832539	0.9999736573290944	-	neuron projection development	0	P
GO:2000300	5	1	0.11662770242832539	0.9999736573290944	-	regulation of synaptic vesicle exocytosis	0	P
GO:0045736	5	1	0.11662770242832539	0.9999736573290944	-	negative regulation of cyclin-dependent protein serine/threonine kinase activity	0	P
GO:0006334	5	1	0.11662770242832539	0.9999736573290944	-	nucleosome assembly	0	P
GO:0042060	8	3	0.1230170146261113	0.9999736573290944	-	wound healing	0	P
GO:0007156	6	2	0.15469393678749122	0.9999736573290944	-	homophilic cell adhesion via plasma membrane adhesion molecules	0	P
GO:0097711	6	2	0.15469393678749122	0.9999736573290944	-	ciliary basal body-plasma membrane docking	0	P
GO:0016192	10	5	0.16474615990768413	0.9999736573290944	-	vesicle-mediated transport	0	P
GO:0030308	10	5	0.16474615990768413	0.9999736573290944	-	negative regulation of cell growth	0	P
GO:0007050	11	6	0.1818829521274304	0.9999736573290944	-	cell cycle arrest	0	P
GO:0001934	16	10	0.18219605561317745	0.9999736573290944	-	positive regulation of protein phosphorylation	0	P
GO:0000086	7	3	0.18458951788137756	0.9999736573290944	-	G2/M transition of mitotic cell cycle	0	P
GO:0042307	4	1	0.19740748702539465	0.9999736573290944	-	positive regulation of protein import into nucleus	0	P
GO:0008015	4	1	0.19740748702539465	0.9999736573290944	-	blood circulation	0	P
GO:0045214	4	1	0.19740748702539465	0.9999736573290944	-	sarcomere organization	0	P
GO:0070493	4	1	0.19740748702539465	0.9999736573290944	-	thrombin-activated receptor signaling pathway	0	P
GO:0008584	4	1	0.19740748702539465	0.9999736573290944	-	male gonad development	0	P
GO:0060021	4	1	0.19740748702539465	0.9999736573290944	-	roof of mouth development	0	P
GO:0006893	4	1	0.19740748702539465	0.9999736573290944	-	Golgi to plasma membrane transport	0	P
GO:0007205	4	1	0.19740748702539465	0.9999736573290944	-	protein kinase C-activating G protein-coupled receptor signaling pathway	0	P
GO:1900273	4	1	0.19740748702539465	0.9999736573290944	-	positive regulation of long-term synaptic potentiation	0	P
GO:0030155	4	1	0.19740748702539465	0.9999736573290944	-	regulation of cell adhesion	0	P
GO:0010613	4	1	0.19740748702539465	0.9999736573290944	-	positive regulation of cardiac muscle hypertrophy	0	P
GO:0036498	4	1	0.19740748702539465	0.9999736573290944	-	IRE1-mediated unfolded protein response	0	P
GO:0045785	4	1	0.19740748702539465	0.9999736573290944	-	positive regulation of cell adhesion	0	P
GO:0045665	4	1	0.19740748702539465	0.9999736573290944	-	negative regulation of neuron differentiation	0	P
GO:0001822	4	1	0.19740748702539465	0.9999736573290944	-	kidney development	0	P
GO:0034446	4	1	0.19740748702539465	0.9999736573290944	-	substrate adhesion-dependent cell spreading	0	P
GO:0090557	4	1	0.19740748702539465	0.9999736573290944	-	establishment of endothelial intestinal barrier	0	P
GO:0045599	4	1	0.19740748702539465	0.9999736573290944	-	negative regulation of fat cell differentiation	0	P
GO:0006325	8	4	0.20884581160964563	0.9999736573290944	-	chromatin organization	0	P
GO:0007015	9	5	0.22905054137018013	0.9999736573290944	-	actin filament organization	0	P
GO:0031954	5	2	0.23957754108047588	0.9999736573290944	-	positive regulation of protein autophosphorylation	0	P
GO:0010389	5	2	0.23957754108047588	0.9999736573290944	-	regulation of G2/M transition of mitotic cell cycle	0	P
GO:0045184	5	2	0.23957754108047588	0.9999736573290944	-	establishment of protein localization	0	P
GO:0043552	5	2	0.23957754108047588	0.9999736573290944	-	positive regulation of phosphatidylinositol 3-kinase activity	0	P
GO:0007605	5	2	0.23957754108047588	0.9999736573290944	-	sensory perception of sound	0	P
GO:0098869	5	2	0.23957754108047588	0.9999736573290944	-	cellular oxidant detoxification	0	P
GO:0010595	5	2	0.23957754108047588	0.9999736573290944	-	positive regulation of endothelial cell migration	0	P
GO:0030100	5	2	0.23957754108047588	0.9999736573290944	-	regulation of endocytosis	0	P
GO:0007189	5	2	0.23957754108047588	0.9999736573290944	-	adenylate cyclase-activating G protein-coupled receptor signaling pathway	0	P
GO:0030049	5	2	0.23957754108047588	0.9999736573290944	-	muscle filament sliding	0	P
GO:0030036	16	11	0.24467684393014374	0.9999736573290944	-	actin cytoskeleton organization	0	P
GO:0007160	11	7	0.2611123339983916	0.9999736573290944	-	cell-matrix adhesion	0	P
GO:0051291	6	3	0.2695380935934397	0.9999736573290944	-	protein heterooligomerization	0	P
GO:0009967	6	3	0.2695380935934397	0.9999736573290944	-	positive regulation of signal transduction	0	P
GO:0030220	6	3	0.2695380935934397	0.9999736573290944	-	platelet formation	0	P
GO:0043406	6	3	0.2695380935934397	0.9999736573290944	-	positive regulation of MAP kinase activity	0	P
GO:0000082	6	3	0.2695380935934397	0.9999736573290944	-	G1/S transition of mitotic cell cycle	0	P
GO:0009887	6	3	0.2695380935934397	0.9999736573290944	-	animal organ morphogenesis	0	P
GO:0007275	12	8	0.2741656137988785	0.9999736573290944	-	multicellular organism development	0	P
GO:0007204	7	4	0.2923413985780936	0.9999736573290944	-	positive regulation of cytosolic calcium ion concentration	0	P
GO:0055085	7	4	0.2923413985780936	0.9999736573290944	-	transmembrane transport	0	P
GO:0030032	7	4	0.2923413985780936	0.9999736573290944	-	lamellipodium assembly	0	P
GO:0007507	7	4	0.2923413985780936	0.9999736573290944	-	heart development	0	P
GO:0043154	8	5	0.3105212258884649	0.9999736573290944	-	negative regulation of cysteine-type endopeptidase activity involved in apoptotic process	0	P
GO:2000134	3	1	0.3246628856507586	0.9999736573290944	-	negative regulation of G1/S transition of mitotic cell cycle	0	P
GO:0002553	3	1	0.3246628856507586	0.9999736573290944	-	histamine secretion by mast cell	0	P
GO:0006351	3	1	0.3246628856507586	0.9999736573290944	-	transcription, DNA-templated	0	P
GO:0003151	3	1	0.3246628856507586	0.9999736573290944	-	outflow tract morphogenesis	0	P
GO:0060315	3	1	0.3246628856507586	0.9999736573290944	-	negative regulation of ryanodine-sensitive calcium-release channel activity	0	P
GO:0045596	3	1	0.3246628856507586	0.9999736573290944	-	negative regulation of cell differentiation	0	P
GO:0051968	3	1	0.3246628856507586	0.9999736573290944	-	positive regulation of synaptic transmission, glutamatergic	0	P
GO:0060048	3	1	0.3246628856507586	0.9999736573290944	-	cardiac muscle contraction	0	P
GO:0048333	3	1	0.3246628856507586	0.9999736573290944	-	mesodermal cell differentiation	0	P
GO:0030512	3	1	0.3246628856507586	0.9999736573290944	-	negative regulation of transforming growth factor beta receptor signaling pathway	0	P
GO:0046627	3	1	0.3246628856507586	0.9999736573290944	-	negative regulation of insulin receptor signaling pathway	0	P
GO:0010862	3	1	0.3246628856507586	0.9999736573290944	-	positive regulation of pathway-restricted SMAD protein phosphorylation	0	P
GO:1902004	3	1	0.3246628856507586	0.9999736573290944	-	positive regulation of amyloid-beta formation	0	P
GO:0042552	3	1	0.3246628856507586	0.9999736573290944	-	myelination	0	P
GO:0030194	3	1	0.3246628856507586	0.9999736573290944	-	positive regulation of blood coagulation	0	P
GO:0007599	3	1	0.3246628856507586	0.9999736573290944	-	hemostasis	0	P
GO:0035264	3	1	0.3246628856507586	0.9999736573290944	-	multicellular organism growth	0	P
GO:0035025	3	1	0.3246628856507586	0.9999736573290944	-	positive regulation of Rho protein signal transduction	0	P
GO:0043101	3	1	0.3246628856507586	0.9999736573290944	-	purine-containing compound salvage	0	P
GO:0035584	3	1	0.3246628856507586	0.9999736573290944	-	calcium-mediated signaling using intracellular calcium source	0	P
GO:0043297	3	1	0.3246628856507586	0.9999736573290944	-	apical junction assembly	0	P
GO:0042035	3	1	0.3246628856507586	0.9999736573290944	-	regulation of cytokine biosynthetic process	0	P
GO:0009952	3	1	0.3246628856507586	0.9999736573290944	-	anterior/posterior pattern specification	0	P
GO:0010759	3	1	0.3246628856507586	0.9999736573290944	-	positive regulation of macrophage chemotaxis	0	P
GO:0030010	3	1	0.3246628856507586	0.9999736573290944	-	establishment of cell polarity	0	P
GO:0021766	3	1	0.3246628856507586	0.9999736573290944	-	hippocampus development	0	P
GO:0071560	3	1	0.3246628856507586	0.9999736573290944	-	cellular response to transforming growth factor beta stimulus	0	P
GO:0071850	3	1	0.3246628856507586	0.9999736573290944	-	mitotic cell cycle arrest	0	P
GO:0007052	3	1	0.3246628856507586	0.9999736573290944	-	mitotic spindle organization	0	P
GO:0051016	3	1	0.3246628856507586	0.9999736573290944	-	barbed-end actin filament capping	0	P
GO:0060391	3	1	0.3246628856507586	0.9999736573290944	-	positive regulation of SMAD protein signal transduction	0	P
GO:0060325	3	1	0.3246628856507586	0.9999736573290944	-	face morphogenesis	0	P
GO:0048873	3	1	0.3246628856507586	0.9999736573290944	-	homeostasis of number of cells within a tissue	0	P
GO:0048821	3	1	0.3246628856507586	0.9999736573290944	-	erythrocyte development	0	P
GO:0051895	3	1	0.3246628856507586	0.9999736573290944	-	negative regulation of focal adhesion assembly	0	P
GO:0001973	3	1	0.3246628856507586	0.9999736573290944	-	adenosine receptor signaling pathway	0	P
GO:0007399	9	6	0.3255071135639243	0.9999736573290944	-	nervous system development	0	P
GO:0060271	9	6	0.3255071135639243	0.9999736573290944	-	cilium assembly	0	P
GO:0043547	19	15	0.33595124302273377	0.9999736573290944	-	positive regulation of GTPase activity	0	P
GO:0030334	10	7	0.33817528745881886	0.9999736573290944	-	regulation of cell migration	0	P
GO:0051216	4	2	0.3589670562195333	0.9999736573290944	-	cartilage development	0	P
GO:0050679	4	2	0.3589670562195333	0.9999736573290944	-	positive regulation of epithelial cell proliferation	0	P
GO:0030218	4	2	0.3589670562195333	0.9999736573290944	-	erythrocyte differentiation	0	P
GO:0090307	4	2	0.3589670562195333	0.9999736573290944	-	mitotic spindle assembly	0	P
GO:2000145	4	2	0.3589670562195333	0.9999736573290944	-	regulation of cell motility	0	P
GO:0007612	4	2	0.3589670562195333	0.9999736573290944	-	learning	0	P
GO:0008277	4	2	0.3589670562195333	0.9999736573290944	-	regulation of G protein-coupled receptor signaling pathway	0	P
GO:0033627	4	2	0.3589670562195333	0.9999736573290944	-	cell adhesion mediated by integrin	0	P
GO:0034504	4	2	0.3589670562195333	0.9999736573290944	-	protein localization to nucleus	0	P
GO:0006906	4	2	0.3589670562195333	0.9999736573290944	-	vesicle fusion	0	P
GO:0006687	4	2	0.3589670562195333	0.9999736573290944	-	glycosphingolipid metabolic process	0	P
GO:0017148	4	2	0.3589670562195333	0.9999736573290944	-	negative regulation of translation	0	P
GO:1903077	4	2	0.3589670562195333	0.9999736573290944	-	negative regulation of protein localization to plasma membrane	0	P
GO:0031647	4	2	0.3589670562195333	0.9999736573290944	-	regulation of protein stability	0	P
GO:0042632	4	2	0.3589670562195333	0.9999736573290944	-	cholesterol homeostasis	0	P
GO:0010811	4	2	0.3589670562195333	0.9999736573290944	-	positive regulation of cell-substrate adhesion	0	P
GO:0051924	4	2	0.3589670562195333	0.9999736573290944	-	regulation of calcium ion transport	0	P
GO:0006281	4	2	0.3589670562195333	0.9999736573290944	-	DNA repair	0	P
GO:0007229	13	10	0.367150464660051	0.9999736573290944	-	integrin-mediated signaling pathway	0	P
GO:0048208	5	3	0.3810502149021172	0.9999736573290944	-	COPII vesicle coating	0	P
GO:0043536	5	3	0.3810502149021172	0.9999736573290944	-	positive regulation of blood vessel endothelial cell migration	0	P
GO:0021762	5	3	0.3810502149021172	0.9999736573290944	-	substantia nigra development	0	P
GO:0045669	5	3	0.3810502149021172	0.9999736573290944	-	positive regulation of osteoblast differentiation	0	P
GO:0000209	5	3	0.3810502149021172	0.9999736573290944	-	protein polyubiquitination	0	P
GO:0007409	5	3	0.3810502149021172	0.9999736573290944	-	axonogenesis	0	P
GO:0006260	5	3	0.3810502149021172	0.9999736573290944	-	DNA replication	0	P
GO:0042113	5	3	0.3810502149021172	0.9999736573290944	-	B cell activation	0	P
GO:0043491	5	3	0.3810502149021172	0.9999736573290944	-	protein kinase B signaling	0	P
GO:0045860	5	3	0.3810502149021172	0.9999736573290944	-	positive regulation of protein kinase activity	0	P
GO:0051496	5	3	0.3810502149021172	0.9999736573290944	-	positive regulation of stress fiber assembly	0	P
GO:0001843	5	3	0.3810502149021172	0.9999736573290944	-	neural tube closure	0	P
GO:0016525	5	3	0.3810502149021172	0.9999736573290944	-	negative regulation of angiogenesis	0	P
GO:0030336	6	4	0.39696095716153285	0.9999736573290944	-	negative regulation of cell migration	0	P
GO:0030041	6	4	0.39696095716153285	0.9999736573290944	-	actin filament polymerization	0	P
GO:0032355	6	4	0.39696095716153285	0.9999736573290944	-	response to estradiol	0	P
GO:0071260	6	4	0.39696095716153285	0.9999736573290944	-	cellular response to mechanical stimulus	0	P
GO:0007173	6	4	0.39696095716153285	0.9999736573290944	-	epidermal growth factor receptor signaling pathway	0	P
GO:0065003	7	5	0.4092352203339207	0.9999736573290944	-	protein-containing complex assembly	0	P
GO:0006511	7	5	0.4092352203339207	0.9999736573290944	-	ubiquitin-dependent protein catabolic process	0	P
GO:0007155	21	18	0.41296782714245467	0.9999736573290944	-	cell adhesion	0	P
GO:0055114	21	18	0.41296782714245467	0.9999736573290944	-	oxidation-reduction process	0	P
GO:0043410	8	6	0.419148805246178	0.9999736573290944	-	positive regulation of MAPK cascade	0	P
GO:0007411	8	6	0.419148805246178	0.9999736573290944	-	axon guidance	0	P
GO:0014068	8	6	0.419148805246178	0.9999736573290944	-	positive regulation of phosphatidylinositol 3-kinase signaling	0	P
GO:0008284	26	23	0.43129229173658207	0.9999736573290944	-	positive regulation of cell population proliferation	0	P
GO:0001666	10	8	0.43450397928435297	0.9999736573290944	-	response to hypoxia	0	P
GO:0042493	11	9	0.44067979300319743	0.9999736573290944	-	response to drug	0	P
GO:0016477	15	13	0.4595728605878306	0.9999736573290944	-	cell migration	0	P
GO:0001523	2	1	0.5124257791032786	0.9999736573290944	-	retinoid metabolic process	0	P
GO:0002902	2	1	0.5124257791032786	0.9999736573290944	-	regulation of B cell apoptotic process	0	P
GO:0030517	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of axon extension	0	P
GO:0030325	2	1	0.5124257791032786	0.9999736573290944	-	adrenal gland development	0	P
GO:0006417	2	1	0.5124257791032786	0.9999736573290944	-	regulation of translation	0	P
GO:0050892	2	1	0.5124257791032786	0.9999736573290944	-	intestinal absorption	0	P
GO:0050732	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of peptidyl-tyrosine phosphorylation	0	P
GO:0050849	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of calcium-mediated signaling	0	P
GO:2000394	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of lamellipodium morphogenesis	0	P
GO:0000060	2	1	0.5124257791032786	0.9999736573290944	-	obsolete protein import into nucleus, translocation	0	P
GO:0050919	2	1	0.5124257791032786	0.9999736573290944	-	negative chemotaxis	0	P
GO:1903829	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of cellular protein localization	0	P
GO:1903955	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of protein targeting to mitochondrion	0	P
GO:2000036	2	1	0.5124257791032786	0.9999736573290944	-	regulation of stem cell population maintenance	0	P
GO:2001240	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of extrinsic apoptotic signaling pathway in absence of ligand	0	P
GO:0007528	2	1	0.5124257791032786	0.9999736573290944	-	neuromuscular junction development	0	P
GO:0032435	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of proteasomal ubiquitin-dependent protein catabolic process	0	P
GO:0032463	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of protein homooligomerization	0	P
GO:0032464	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of protein homooligomerization	0	P
GO:0032465	2	1	0.5124257791032786	0.9999736573290944	-	regulation of cytokinesis	0	P
GO:0031396	2	1	0.5124257791032786	0.9999736573290944	-	regulation of protein ubiquitination	0	P
GO:0031629	2	1	0.5124257791032786	0.9999736573290944	-	synaptic vesicle fusion to presynaptic active zone membrane	0	P
GO:0071243	2	1	0.5124257791032786	0.9999736573290944	-	cellular response to arsenic-containing substance	0	P
GO:0030949	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of vascular endothelial growth factor receptor signaling pathway	0	P
GO:0031032	2	1	0.5124257791032786	0.9999736573290944	-	actomyosin structure organization	0	P
GO:0031334	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of protein complex assembly	0	P
GO:0030511	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of transforming growth factor beta receptor signaling pathway	0	P
GO:0048167	2	1	0.5124257791032786	0.9999736573290944	-	regulation of synaptic plasticity	0	P
GO:0070560	2	1	0.5124257791032786	0.9999736573290944	-	protein secretion by platelet	0	P
GO:0070848	2	1	0.5124257791032786	0.9999736573290944	-	response to growth factor	0	P
GO:0033211	2	1	0.5124257791032786	0.9999736573290944	-	adiponectin-activated signaling pathway	0	P
GO:0019852	2	1	0.5124257791032786	0.9999736573290944	-	L-ascorbic acid metabolic process	0	P
GO:0019934	2	1	0.5124257791032786	0.9999736573290944	-	cGMP-mediated signaling	0	P
GO:0090263	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of canonical Wnt signaling pathway	0	P
GO:0090305	2	1	0.5124257791032786	0.9999736573290944	-	nucleic acid phosphodiester bond hydrolysis	0	P
GO:0019371	2	1	0.5124257791032786	0.9999736573290944	-	cyclooxygenase pathway	0	P
GO:0030206	2	1	0.5124257791032786	0.9999736573290944	-	chondroitin sulfate biosynthetic process	0	P
GO:0030208	2	1	0.5124257791032786	0.9999736573290944	-	dermatan sulfate biosynthetic process	0	P
GO:0033137	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of peptidyl-serine phosphorylation	0	P
GO:0030042	2	1	0.5124257791032786	0.9999736573290944	-	actin filament depolymerization	0	P
GO:0099133	2	1	0.5124257791032786	0.9999736573290944	-	ATP hydrolysis coupled anion transmembrane transport	0	P
GO:0099509	2	1	0.5124257791032786	0.9999736573290944	-	regulation of presynaptic cytosolic calcium ion concentration	0	P
GO:0098761	2	1	0.5124257791032786	0.9999736573290944	-	cellular response to interleukin-7	0	P
GO:0043393	2	1	0.5124257791032786	0.9999736573290944	-	regulation of protein binding	0	P
GO:0009743	2	1	0.5124257791032786	0.9999736573290944	-	response to carbohydrate	0	P
GO:0009755	2	1	0.5124257791032786	0.9999736573290944	-	hormone-mediated signaling pathway	0	P
GO:0042246	2	1	0.5124257791032786	0.9999736573290944	-	tissue regeneration	0	P
GO:0009306	2	1	0.5124257791032786	0.9999736573290944	-	protein secretion	0	P
GO:0009629	2	1	0.5124257791032786	0.9999736573290944	-	response to gravity	0	P
GO:0033591	2	1	0.5124257791032786	0.9999736573290944	-	response to L-ascorbic acid	0	P
GO:0051492	2	1	0.5124257791032786	0.9999736573290944	-	regulation of stress fiber assembly	0	P
GO:1901799	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of proteasomal protein catabolic process	0	P
GO:1902230	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of intrinsic apoptotic signaling pathway in response to DNA damage	0	P
GO:1902476	2	1	0.5124257791032786	0.9999736573290944	-	chloride transmembrane transport	0	P
GO:0048278	2	1	0.5124257791032786	0.9999736573290944	-	vesicle docking	0	P
GO:0048642	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of skeletal muscle tissue development	0	P
GO:0048675	2	1	0.5124257791032786	0.9999736573290944	-	axon extension	0	P
GO:0006521	2	1	0.5124257791032786	0.9999736573290944	-	regulation of cellular amino acid metabolic process	0	P
GO:0015871	2	1	0.5124257791032786	0.9999736573290944	-	choline transport	0	P
GO:0042059	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of epidermal growth factor receptor signaling pathway	0	P
GO:0012501	2	1	0.5124257791032786	0.9999736573290944	-	programmed cell death	0	P
GO:0035646	2	1	0.5124257791032786	0.9999736573290944	-	endosome to melanosome transport	0	P
GO:0035774	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of insulin secretion involved in cellular response to glucose stimulus	0	P
GO:0035987	2	1	0.5124257791032786	0.9999736573290944	-	endodermal cell differentiation	0	P
GO:0036297	2	1	0.5124257791032786	0.9999736573290944	-	interstrand cross-link repair	0	P
GO:0010880	2	1	0.5124257791032786	0.9999736573290944	-	regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum	0	P
GO:0035116	2	1	0.5124257791032786	0.9999736573290944	-	embryonic hindlimb morphogenesis	0	P
GO:0071711	2	1	0.5124257791032786	0.9999736573290944	-	basement membrane organization	0	P
GO:0090150	2	1	0.5124257791032786	0.9999736573290944	-	establishment of protein localization to membrane	0	P
GO:0090162	2	1	0.5124257791032786	0.9999736573290944	-	establishment of epithelial cell polarity	0	P
GO:0034765	2	1	0.5124257791032786	0.9999736573290944	-	regulation of ion transmembrane transport	0	P
GO:0018107	2	1	0.5124257791032786	0.9999736573290944	-	peptidyl-threonine phosphorylation	0	P
GO:0033628	2	1	0.5124257791032786	0.9999736573290944	-	regulation of cell adhesion mediated by integrin	0	P
GO:0008354	2	1	0.5124257791032786	0.9999736573290944	-	germ cell migration	0	P
GO:0008356	2	1	0.5124257791032786	0.9999736573290944	-	asymmetric cell division	0	P
GO:0008652	2	1	0.5124257791032786	0.9999736573290944	-	cellular amino acid biosynthetic process	0	P
GO:0007032	2	1	0.5124257791032786	0.9999736573290944	-	endosome organization	0	P
GO:0006997	2	1	0.5124257791032786	0.9999736573290944	-	nucleus organization	0	P
GO:0045921	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of exocytosis	0	P
GO:0046135	2	1	0.5124257791032786	0.9999736573290944	-	pyrimidine nucleoside catabolic process	0	P
GO:0045667	2	1	0.5124257791032786	0.9999736573290944	-	regulation of osteoblast differentiation	0	P
GO:0045777	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of blood pressure	0	P
GO:0010594	2	1	0.5124257791032786	0.9999736573290944	-	regulation of endothelial cell migration	0	P
GO:0010632	2	1	0.5124257791032786	0.9999736573290944	-	regulation of epithelial cell migration	0	P
GO:0010633	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of epithelial cell migration	0	P
GO:0010761	2	1	0.5124257791032786	0.9999736573290944	-	fibroblast migration	0	P
GO:0010762	2	1	0.5124257791032786	0.9999736573290944	-	regulation of fibroblast migration	0	P
GO:0042119	2	1	0.5124257791032786	0.9999736573290944	-	neutrophil activation	0	P
GO:0060347	2	1	0.5124257791032786	0.9999736573290944	-	heart trabecula formation	0	P
GO:0060369	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of Fc receptor mediated stimulatory signaling pathway	0	P
GO:0006930	2	1	0.5124257791032786	0.9999736573290944	-	substrate-dependent cell migration, cell extension	0	P
GO:0006970	2	1	0.5124257791032786	0.9999736573290944	-	response to osmotic stress	0	P
GO:0051865	2	1	0.5124257791032786	0.9999736573290944	-	protein autoubiquitination	0	P
GO:0051873	2	1	0.5124257791032786	0.9999736573290944	-	killing by host of symbiont cells	0	P
GO:0051881	2	1	0.5124257791032786	0.9999736573290944	-	regulation of mitochondrial membrane potential	0	P
GO:0014909	2	1	0.5124257791032786	0.9999736573290944	-	smooth muscle cell migration	0	P
GO:0010716	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of extracellular matrix disassembly	0	P
GO:0010718	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of epithelial to mesenchymal transition	0	P
GO:0010165	2	1	0.5124257791032786	0.9999736573290944	-	response to X-ray	0	P
GO:0010572	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of platelet activation	0	P
GO:0010591	2	1	0.5124257791032786	0.9999736573290944	-	regulation of lamellipodium assembly	0	P
GO:0043388	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of DNA binding	0	P
GO:0042744	2	1	0.5124257791032786	0.9999736573290944	-	hydrogen peroxide catabolic process	0	P
GO:0043029	2	1	0.5124257791032786	0.9999736573290944	-	T cell homeostasis	0	P
GO:0097194	2	1	0.5124257791032786	0.9999736573290944	-	execution phase of apoptosis	0	P
GO:0097352	2	1	0.5124257791032786	0.9999736573290944	-	autophagosome maturation	0	P
GO:0097368	2	1	0.5124257791032786	0.9999736573290944	-	establishment of Sertoli cell barrier	0	P
GO:0090398	2	1	0.5124257791032786	0.9999736573290944	-	cellular senescence	0	P
GO:0097193	2	1	0.5124257791032786	0.9999736573290944	-	intrinsic apoptotic signaling pathway	0	P
GO:0060586	2	1	0.5124257791032786	0.9999736573290944	-	multicellular organismal iron ion homeostasis	0	P
GO:0060996	2	1	0.5124257791032786	0.9999736573290944	-	dendritic spine development	0	P
GO:2000273	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of signaling receptor activity	0	P
GO:2000060	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of ubiquitin-dependent protein catabolic process	0	P
GO:0002513	2	1	0.5124257791032786	0.9999736573290944	-	tolerance induction to self antigen	0	P
GO:0001889	2	1	0.5124257791032786	0.9999736573290944	-	liver development	0	P
GO:0002040	2	1	0.5124257791032786	0.9999736573290944	-	sprouting angiogenesis	0	P
GO:0032026	2	1	0.5124257791032786	0.9999736573290944	-	response to magnesium ion	0	P
GO:0051044	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of membrane protein ectodomain proteolysis	0	P
GO:0051280	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of release of sequestered calcium ion into cytosol	0	P
GO:0001503	2	1	0.5124257791032786	0.9999736573290944	-	ossification	0	P
GO:0007190	2	1	0.5124257791032786	0.9999736573290944	-	activation of adenylate cyclase activity	0	P
GO:0060050	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of protein glycosylation	0	P
GO:0060100	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of phagocytosis, engulfment	0	P
GO:0060137	2	1	0.5124257791032786	0.9999736573290944	-	maternal process involved in parturition	0	P
GO:0007183	2	1	0.5124257791032786	0.9999736573290944	-	SMAD protein complex assembly	0	P
GO:1900024	2	1	0.5124257791032786	0.9999736573290944	-	regulation of substrate adhesion-dependent cell spreading	0	P
GO:1900025	2	1	0.5124257791032786	0.9999736573290944	-	negative regulation of substrate adhesion-dependent cell spreading	0	P
GO:0032332	2	1	0.5124257791032786	0.9999736573290944	-	positive regulation of chondrocyte differentiation	0	P
GO:0006486	3	2	0.5155459835888045	0.9999736573290944	-	protein glycosylation	0	P
GO:0090051	3	2	0.5155459835888045	0.9999736573290944	-	negative regulation of cell migration involved in sprouting angiogenesis	0	P
GO:0035162	3	2	0.5155459835888045	0.9999736573290944	-	embryonic hemopoiesis	0	P
GO:0035023	3	2	0.5155459835888045	0.9999736573290944	-	regulation of Rho protein signal transduction	0	P
GO:0051893	3	2	0.5155459835888045	0.9999736573290944	-	regulation of focal adhesion assembly	0	P
GO:0010507	3	2	0.5155459835888045	0.9999736573290944	-	negative regulation of autophagy	0	P
GO:0042593	3	2	0.5155459835888045	0.9999736573290944	-	glucose homeostasis	0	P
GO:0034613	3	2	0.5155459835888045	0.9999736573290944	-	cellular protein localization	0	P
GO:1903076	3	2	0.5155459835888045	0.9999736573290944	-	regulation of protein localization to plasma membrane	0	P
GO:0007626	3	2	0.5155459835888045	0.9999736573290944	-	locomotory behavior	0	P
GO:0097435	3	2	0.5155459835888045	0.9999736573290944	-	supramolecular fiber organization	0	P
GO:0061025	3	2	0.5155459835888045	0.9999736573290944	-	membrane fusion	0	P
GO:0051592	3	2	0.5155459835888045	0.9999736573290944	-	response to calcium ion	0	P
GO:0048488	3	2	0.5155459835888045	0.9999736573290944	-	synaptic vesicle endocytosis	0	P
GO:0007249	3	2	0.5155459835888045	0.9999736573290944	-	I-kappaB kinase/NF-kappaB signaling	0	P
GO:0016311	3	2	0.5155459835888045	0.9999736573290944	-	dephosphorylation	0	P
GO:0002248	3	2	0.5155459835888045	0.9999736573290944	-	connective tissue replacement involved in inflammatory response wound healing	0	P
GO:0006596	3	2	0.5155459835888045	0.9999736573290944	-	polyamine biosynthetic process	0	P
GO:0008154	3	2	0.5155459835888045	0.9999736573290944	-	actin polymerization or depolymerization	0	P
GO:0007059	3	2	0.5155459835888045	0.9999736573290944	-	chromosome segregation	0	P
GO:0045648	3	2	0.5155459835888045	0.9999736573290944	-	positive regulation of erythrocyte differentiation	0	P
GO:0050918	3	2	0.5155459835888045	0.9999736573290944	-	positive chemotaxis	0	P
GO:0007005	3	2	0.5155459835888045	0.9999736573290944	-	mitochondrion organization	0	P
GO:0006644	3	2	0.5155459835888045	0.9999736573290944	-	phospholipid metabolic process	0	P
GO:0006816	3	2	0.5155459835888045	0.9999736573290944	-	calcium ion transport	0	P
GO:1904706	3	2	0.5155459835888045	0.9999736573290944	-	negative regulation of vascular smooth muscle cell proliferation	0	P
GO:0001570	3	2	0.5155459835888045	0.9999736573290944	-	vasculogenesis	0	P
GO:2001046	3	2	0.5155459835888045	0.9999736573290944	-	positive regulation of integrin-mediated signaling pathway	0	P
GO:2000379	3	2	0.5155459835888045	0.9999736573290944	-	positive regulation of reactive oxygen species metabolic process	0	P
GO:0032981	3	2	0.5155459835888045	0.9999736573290944	-	mitochondrial respiratory chain complex I assembly	0	P
GO:2000178	3	2	0.5155459835888045	0.9999736573290944	-	negative regulation of neural precursor cell proliferation	0	P
GO:2000249	3	2	0.5155459835888045	0.9999736573290944	-	regulation of actin cytoskeleton reorganization	0	P
GO:2001237	3	2	0.5155459835888045	0.9999736573290944	-	negative regulation of extrinsic apoptotic signaling pathway	0	P
GO:0030836	3	2	0.5155459835888045	0.9999736573290944	-	positive regulation of actin filament depolymerization	0	P
GO:1901223	3	2	0.5155459835888045	0.9999736573290944	-	negative regulation of NIK/NF-kappaB signaling	0	P
GO:0033138	3	2	0.5155459835888045	0.9999736573290944	-	positive regulation of peptidyl-serine phosphorylation	0	P
GO:0090201	3	2	0.5155459835888045	0.9999736573290944	-	negative regulation of release of cytochrome c from mitochondria	0	P
GO:0051091	3	2	0.5155459835888045	0.9999736573290944	-	positive regulation of DNA-binding transcription factor activity	0	P
GO:0070268	3	2	0.5155459835888045	0.9999736573290944	-	cornification	0	P
GO:0019882	3	2	0.5155459835888045	0.9999736573290944	-	antigen processing and presentation	0	P
GO:0031589	3	2	0.5155459835888045	0.9999736573290944	-	cell-substrate adhesion	0	P
GO:0030889	3	2	0.5155459835888045	0.9999736573290944	-	negative regulation of B cell proliferation	0	P
GO:0030030	3	2	0.5155459835888045	0.9999736573290944	-	cell projection organization	0	P
GO:0030595	3	2	0.5155459835888045	0.9999736573290944	-	leukocyte chemotaxis	0	P
GO:0032456	3	2	0.5155459835888045	0.9999736573290944	-	endocytic recycling	0	P
GO:0032967	3	2	0.5155459835888045	0.9999736573290944	-	positive regulation of collagen biosynthetic process	0	P
GO:0055010	3	2	0.5155459835888045	0.9999736573290944	-	ventricular cardiac muscle tissue morphogenesis	0	P
GO:0007200	3	2	0.5155459835888045	0.9999736573290944	-	phospholipase C-activating G protein-coupled receptor signaling pathway	0	P
GO:1900034	3	2	0.5155459835888045	0.9999736573290944	-	regulation of cellular response to heat	0	P
GO:0050804	3	2	0.5155459835888045	0.9999736573290944	-	modulation of chemical synaptic transmission	0	P
GO:0002479	3	2	0.5155459835888045	0.9999736573290944	-	antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent	0	P
GO:0007595	3	2	0.5155459835888045	0.9999736573290944	-	lactation	0	P
GO:0042220	3	2	0.5155459835888045	0.9999736573290944	-	response to cocaine	0	P
GO:0007584	3	2	0.5155459835888045	0.9999736573290944	-	response to nutrient	0	P
GO:0042327	3	2	0.5155459835888045	0.9999736573290944	-	positive regulation of phosphorylation	0	P
GO:0001975	4	3	0.5181530764049431	0.9999736573290944	-	response to amphetamine	0	P
GO:0001649	4	3	0.5181530764049431	0.9999736573290944	-	osteoblast differentiation	0	P
GO:0006874	4	3	0.5181530764049431	0.9999736573290944	-	cellular calcium ion homeostasis	0	P
GO:0051028	4	3	0.5181530764049431	0.9999736573290944	-	mRNA transport	0	P
GO:0016485	4	3	0.5181530764049431	0.9999736573290944	-	protein processing	0	P
GO:0030866	4	3	0.5181530764049431	0.9999736573290944	-	cortical actin cytoskeleton organization	0	P
GO:0009749	4	3	0.5181530764049431	0.9999736573290944	-	response to glucose	0	P
GO:0045746	4	3	0.5181530764049431	0.9999736573290944	-	negative regulation of Notch signaling pathway	0	P
GO:0048678	4	3	0.5181530764049431	0.9999736573290944	-	response to axon injury	0	P
GO:0009653	4	3	0.5181530764049431	0.9999736573290944	-	anatomical structure morphogenesis	0	P
GO:0006338	4	3	0.5181530764049431	0.9999736573290944	-	chromatin remodeling	0	P
GO:0042110	4	3	0.5181530764049431	0.9999736573290944	-	T cell activation	0	P
GO:0051493	4	3	0.5181530764049431	0.9999736573290944	-	regulation of cytoskeleton organization	0	P
GO:0043434	4	3	0.5181530764049431	0.9999736573290944	-	response to peptide hormone	0	P
GO:0045807	4	3	0.5181530764049431	0.9999736573290944	-	positive regulation of endocytosis	0	P
GO:0014003	4	3	0.5181530764049431	0.9999736573290944	-	oligodendrocyte development	0	P
GO:0051894	4	3	0.5181530764049431	0.9999736573290944	-	positive regulation of focal adhesion assembly	0	P
GO:0070098	4	3	0.5181530764049431	0.9999736573290944	-	chemokine-mediated signaling pathway	0	P
GO:0007613	4	3	0.5181530764049431	0.9999736573290944	-	memory	0	P
GO:0009725	4	3	0.5181530764049431	0.9999736573290944	-	response to hormone	0	P
GO:0071277	4	3	0.5181530764049431	0.9999736573290944	-	cellular response to calcium ion	0	P
GO:0044319	4	3	0.5181530764049431	0.9999736573290944	-	wound healing, spreading of cells	0	P
GO:0035855	4	3	0.5181530764049431	0.9999736573290944	-	megakaryocyte development	0	P
GO:0046718	5	4	0.5204403665379638	0.9999736573290944	-	viral entry into host cell	0	P
GO:0009267	5	4	0.5204403665379638	0.9999736573290944	-	cellular response to starvation	0	P
GO:0033209	5	4	0.5204403665379638	0.9999736573290944	-	tumor necrosis factor-mediated signaling pathway	0	P
GO:0048146	5	4	0.5204403665379638	0.9999736573290944	-	positive regulation of fibroblast proliferation	0	P
GO:0043149	5	4	0.5204403665379638	0.9999736573290944	-	stress fiber assembly	0	P
GO:0032781	5	4	0.5204403665379638	0.9999736573290944	-	positive regulation of ATPase activity	0	P
GO:0009636	6	5	0.5225044274616604	0.9999736573290944	-	response to toxic substance	0	P
GO:0034329	6	5	0.5225044274616604	0.9999736573290944	-	cell junction assembly	0	P
GO:0032869	6	5	0.5225044274616604	0.9999736573290944	-	cellular response to insulin stimulus	0	P
GO:0051301	9	8	0.527828267466206	0.9999736573290944	-	cell division	0	P
GO:0007179	9	8	0.527828267466206	0.9999736573290944	-	transforming growth factor beta receptor signaling pathway	0	P
GO:0051897	11	10	0.5308983989355476	0.9999736573290944	-	positive regulation of protein kinase B signaling	0	P
GO:0010469	14	13	0.5350376256941394	0.9999736573290944	-	regulation of signaling receptor activity	0	P
GO:0015031	14	13	0.5350376256941394	0.9999736573290944	-	protein transport	0	P
GO:0008285	19	18	0.5410997114065627	0.9999736573290944	-	negative regulation of cell population proliferation	0	P
GO:0006357	20	20	0.6057992161569897	0.9999736573290944	-	regulation of transcription by RNA polymerase II	0	P
GO:0043687	14	14	0.6105023908004474	0.9999736573290944	-	post-translational protein modification	0	P
GO:0048013	12	12	0.6136065037229891	0.9999736573290944	-	ephrin receptor signaling pathway	0	P
GO:0044267	11	11	0.6156464929773675	0.9999736573290944	-	cellular protein metabolic process	0	P
GO:0032956	8	8	0.6249120142701703	0.9999736573290944	-	regulation of actin cytoskeleton organization	0	P
GO:0051056	8	8	0.6249120142701703	0.9999736573290944	-	regulation of small GTPase mediated signal transduction	0	P
GO:0016055	7	7	0.629654289127727	0.9999736573290944	-	Wnt signaling pathway	0	P
GO:0006897	7	7	0.629654289127727	0.9999736573290944	-	endocytosis	0	P
GO:0120162	7	7	0.629654289127727	0.9999736573290944	-	positive regulation of cold-induced thermogenesis	0	P
GO:0042127	7	7	0.629654289127727	0.9999736573290944	-	regulation of cell population proliferation	0	P
GO:0006914	6	6	0.6357736345893998	0.9999736573290944	-	autophagy	0	P
GO:0009617	6	6	0.6357736345893998	0.9999736573290944	-	response to bacterium	0	P
GO:0009611	5	5	0.6439197759143948	0.9999736573290944	-	response to wounding	0	P
GO:0006979	5	5	0.6439197759143948	0.9999736573290944	-	response to oxidative stress	0	P
GO:0007565	5	5	0.6439197759143948	0.9999736573290944	-	female pregnancy	0	P
GO:0043161	5	5	0.6439197759143948	0.9999736573290944	-	proteasome-mediated ubiquitin-dependent protein catabolic process	0	P
GO:0006470	5	5	0.6439197759143948	0.9999736573290944	-	protein dephosphorylation	0	P
GO:0034314	5	5	0.6439197759143948	0.9999736573290944	-	Arp2/3 complex-mediated actin nucleation	0	P
GO:0001938	5	5	0.6439197759143948	0.9999736573290944	-	positive regulation of endothelial cell proliferation	0	P
GO:0016567	5	5	0.6439197759143948	0.9999736573290944	-	protein ubiquitination	0	P
GO:0007159	5	5	0.6439197759143948	0.9999736573290944	-	leukocyte cell-cell adhesion	0	P
GO:0061844	4	4	0.6552559379077691	0.9999736573290944	-	antimicrobial humoral immune response mediated by antimicrobial peptide	0	P
GO:0050853	4	4	0.6552559379077691	0.9999736573290944	-	B cell receptor signaling pathway	0	P
GO:0032570	4	4	0.6552559379077691	0.9999736573290944	-	response to progesterone	0	P
GO:0030182	4	4	0.6552559379077691	0.9999736573290944	-	neuron differentiation	0	P
GO:0023014	4	4	0.6552559379077691	0.9999736573290944	-	signal transduction by protein phosphorylation	0	P
GO:0051726	4	4	0.6552559379077691	0.9999736573290944	-	regulation of cell cycle	0	P
GO:0002223	4	4	0.6552559379077691	0.9999736573290944	-	stimulatory C-type lectin receptor signaling pathway	0	P
GO:0032436	4	4	0.6552559379077691	0.9999736573290944	-	positive regulation of proteasomal ubiquitin-dependent protein catabolic process	0	P
GO:0048008	4	4	0.6552559379077691	0.9999736573290944	-	platelet-derived growth factor receptor signaling pathway	0	P
GO:0051384	4	4	0.6552559379077691	0.9999736573290944	-	response to glucocorticoid	0	P
GO:0070301	3	3	0.6721249109580756	0.9999736573290944	-	cellular response to hydrogen peroxide	0	P
GO:0034260	3	3	0.6721249109580756	0.9999736573290944	-	negative regulation of GTPase activity	0	P
GO:0070498	3	3	0.6721249109580756	0.9999736573290944	-	interleukin-1-mediated signaling pathway	0	P
GO:0001775	3	3	0.6721249109580756	0.9999736573290944	-	cell activation	0	P
GO:1900740	3	3	0.6721249109580756	0.9999736573290944	-	positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway	0	P
GO:0097067	3	3	0.6721249109580756	0.9999736573290944	-	cellular response to thyroid hormone stimulus	0	P
GO:0032233	3	3	0.6721249109580756	0.9999736573290944	-	positive regulation of actin filament bundle assembly	0	P
GO:0031100	3	3	0.6721249109580756	0.9999736573290944	-	animal organ regeneration	0	P
GO:0061418	3	3	0.6721249109580756	0.9999736573290944	-	regulation of transcription from RNA polymerase II promoter in response to hypoxia	0	P
GO:0032526	3	3	0.6721249109580756	0.9999736573290944	-	response to retinoic acid	0	P
GO:0043085	3	3	0.6721249109580756	0.9999736573290944	-	positive regulation of catalytic activity	0	P
GO:0048870	3	3	0.6721249109580756	0.9999736573290944	-	cell motility	0	P
GO:0001558	3	3	0.6721249109580756	0.9999736573290944	-	regulation of cell growth	0	P
GO:0045666	3	3	0.6721249109580756	0.9999736573290944	-	positive regulation of neuron differentiation	0	P
GO:0051402	3	3	0.6721249109580756	0.9999736573290944	-	neuron apoptotic process	0	P
GO:0090630	3	3	0.6721249109580756	0.9999736573290944	-	activation of GTPase activity	0	P
GO:0042026	3	3	0.6721249109580756	0.9999736573290944	-	protein refolding	0	P
GO:0031529	3	3	0.6721249109580756	0.9999736573290944	-	ruffle organization	0	P
GO:0034614	3	3	0.6721249109580756	0.9999736573290944	-	cellular response to reactive oxygen species	0	P
GO:0071230	3	3	0.6721249109580756	0.9999736573290944	-	cellular response to amino acid stimulus	0	P
GO:0030324	3	3	0.6721249109580756	0.9999736573290944	-	lung development	0	P
GO:0001764	3	3	0.6721249109580756	0.9999736573290944	-	neuron migration	0	P
GO:0001932	3	3	0.6721249109580756	0.9999736573290944	-	regulation of protein phosphorylation	0	P
GO:0008217	3	3	0.6721249109580756	0.9999736573290944	-	regulation of blood pressure	0	P
GO:0000902	3	3	0.6721249109580756	0.9999736573290944	-	cell morphogenesis	0	P
GO:0061077	3	3	0.6721249109580756	0.9999736573290944	-	chaperone-mediated protein folding	0	P
GO:0061024	13	14	0.6832984608735273	0.9999736573290944	-	membrane organization	0	P
GO:0045892	13	14	0.6832984608735273	0.9999736573290944	-	negative regulation of transcription, DNA-templated	0	P
GO:0030154	13	14	0.6832984608735273	0.9999736573290944	-	cell differentiation	0	P
GO:0050900	12	13	0.6873575992077777	0.9999736573290944	-	leukocyte migration	0	P
GO:0030509	2	2	0.7001886725557986	0.9999736573290944	-	BMP signaling pathway	0	P
GO:0006611	2	2	0.7001886725557986	0.9999736573290944	-	protein export from nucleus	0	P
GO:0030097	2	2	0.7001886725557986	0.9999736573290944	-	hemopoiesis	0	P
GO:0030705	2	2	0.7001886725557986	0.9999736573290944	-	cytoskeleton-dependent intracellular transport	0	P
GO:0030521	2	2	0.7001886725557986	0.9999736573290944	-	androgen receptor signaling pathway	0	P
GO:0043304	2	2	0.7001886725557986	0.9999736573290944	-	regulation of mast cell degranulation	0	P
GO:0018149	2	2	0.7001886725557986	0.9999736573290944	-	peptide cross-linking	0	P
GO:0008306	2	2	0.7001886725557986	0.9999736573290944	-	associative learning	0	P
GO:0050861	2	2	0.7001886725557986	0.9999736573290944	-	positive regulation of B cell receptor signaling pathway	0	P
GO:0006612	2	2	0.7001886725557986	0.9999736573290944	-	protein targeting to membrane	0	P
GO:0030950	2	2	0.7001886725557986	0.9999736573290944	-	establishment or maintenance of actin cytoskeleton polarity	0	P
GO:0030033	2	2	0.7001886725557986	0.9999736573290944	-	microvillus assembly	0	P
GO:0030031	2	2	0.7001886725557986	0.9999736573290944	-	cell projection assembly	0	P
GO:0021943	2	2	0.7001886725557986	0.9999736573290944	-	formation of radial glial scaffolds	0	P
GO:0061512	2	2	0.7001886725557986	0.9999736573290944	-	protein localization to cilium	0	P
GO:0032008	2	2	0.7001886725557986	0.9999736573290944	-	positive regulation of TOR signaling	0	P
GO:0090316	2	2	0.7001886725557986	0.9999736573290944	-	positive regulation of intracellular protein transport	0	P
GO:0010917	2	2	0.7001886725557986	0.9999736573290944	-	negative regulation of mitochondrial membrane potential	0	P
GO:0001782	2	2	0.7001886725557986	0.9999736573290944	-	B cell homeostasis	0	P
GO:0006880	2	2	0.7001886725557986	0.9999736573290944	-	intracellular sequestering of iron ion	0	P
GO:0051764	2	2	0.7001886725557986	0.9999736573290944	-	actin crosslink formation	0	P
GO:0006120	2	2	0.7001886725557986	0.9999736573290944	-	mitochondrial electron transport, NADH to ubiquinone	0	P
GO:0051930	2	2	0.7001886725557986	0.9999736573290944	-	regulation of sensory perception of pain	0	P
GO:0050901	2	2	0.7001886725557986	0.9999736573290944	-	leukocyte tethering or rolling	0	P
GO:0003007	2	2	0.7001886725557986	0.9999736573290944	-	heart morphogenesis	0	P
GO:2001235	2	2	0.7001886725557986	0.9999736573290944	-	positive regulation of apoptotic signaling pathway	0	P
GO:0000045	2	2	0.7001886725557986	0.9999736573290944	-	autophagosome assembly	0	P
GO:0002931	2	2	0.7001886725557986	0.9999736573290944	-	response to ischemia	0	P
GO:2000811	2	2	0.7001886725557986	0.9999736573290944	-	negative regulation of anoikis	0	P
GO:0032486	2	2	0.7001886725557986	0.9999736573290944	-	Rap protein signal transduction	0	P
GO:0007030	2	2	0.7001886725557986	0.9999736573290944	-	Golgi organization	0	P
GO:0046007	2	2	0.7001886725557986	0.9999736573290944	-	negative regulation of activated T cell proliferation	0	P
GO:0007029	2	2	0.7001886725557986	0.9999736573290944	-	endoplasmic reticulum organization	0	P
GO:0006977	2	2	0.7001886725557986	0.9999736573290944	-	DNA damage response, signal transduction by p53 class mediator resulting in cell cycle arrest	0	P
GO:0006631	2	2	0.7001886725557986	0.9999736573290944	-	fatty acid metabolic process	0	P
GO:0001817	2	2	0.7001886725557986	0.9999736573290944	-	regulation of cytokine production	0	P
GO:0009612	2	2	0.7001886725557986	0.9999736573290944	-	response to mechanical stimulus	0	P
GO:0000413	2	2	0.7001886725557986	0.9999736573290944	-	protein peptidyl-prolyl isomerization	0	P
GO:0097190	2	2	0.7001886725557986	0.9999736573290944	-	apoptotic signaling pathway	0	P
GO:0120163	2	2	0.7001886725557986	0.9999736573290944	-	negative regulation of cold-induced thermogenesis	0	P
GO:0060079	2	2	0.7001886725557986	0.9999736573290944	-	excitatory postsynaptic potential	0	P
GO:1990000	2	2	0.7001886725557986	0.9999736573290944	-	amyloid fibril formation	0	P
GO:2000009	2	2	0.7001886725557986	0.9999736573290944	-	negative regulation of protein localization to cell surface	0	P
GO:0050730	2	2	0.7001886725557986	0.9999736573290944	-	regulation of peptidyl-tyrosine phosphorylation	0	P
GO:0050765	2	2	0.7001886725557986	0.9999736573290944	-	negative regulation of phagocytosis	0	P
GO:0050790	2	2	0.7001886725557986	0.9999736573290944	-	regulation of catalytic activity	0	P
GO:0051412	2	2	0.7001886725557986	0.9999736573290944	-	response to corticosterone	0	P
GO:0051639	2	2	0.7001886725557986	0.9999736573290944	-	actin filament network formation	0	P
GO:0034612	2	2	0.7001886725557986	0.9999736573290944	-	response to tumor necrosis factor	0	P
GO:0016575	2	2	0.7001886725557986	0.9999736573290944	-	histone deacetylation	0	P
GO:0001836	2	2	0.7001886725557986	0.9999736573290944	-	release of cytochrome c from mitochondria	0	P
GO:1902961	2	2	0.7001886725557986	0.9999736573290944	-	positive regulation of aspartic-type endopeptidase activity involved in amyloid precursor protein catabolic process	0	P
GO:0035690	2	2	0.7001886725557986	0.9999736573290944	-	cellular response to drug	0	P
GO:0042472	2	2	0.7001886725557986	0.9999736573290944	-	inner ear morphogenesis	0	P
GO:0006283	2	2	0.7001886725557986	0.9999736573290944	-	transcription-coupled nucleotide-excision repair	0	P
GO:0090303	2	2	0.7001886725557986	0.9999736573290944	-	positive regulation of wound healing	0	P
GO:0030216	2	2	0.7001886725557986	0.9999736573290944	-	keratinocyte differentiation	0	P
GO:0097192	2	2	0.7001886725557986	0.9999736573290944	-	extrinsic apoptotic signaling pathway in absence of ligand	0	P
GO:0043408	2	2	0.7001886725557986	0.9999736573290944	-	regulation of MAPK cascade	0	P
GO:0048469	2	2	0.7001886725557986	0.9999736573290944	-	cell maturation	0	P
GO:0048260	2	2	0.7001886725557986	0.9999736573290944	-	positive regulation of receptor-mediated endocytosis	0	P
GO:0097028	2	2	0.7001886725557986	0.9999736573290944	-	dendritic cell differentiation	0	P
GO:0098586	2	2	0.7001886725557986	0.9999736573290944	-	cellular response to virus	0	P
GO:0001837	2	2	0.7001886725557986	0.9999736573290944	-	epithelial to mesenchymal transition	0	P
GO:0098974	2	2	0.7001886725557986	0.9999736573290944	-	postsynaptic actin cytoskeleton organization	0	P
GO:0010951	8	9	0.7112086780959164	0.9999736573290944	-	negative regulation of endopeptidase activity	0	P
GO:0050821	8	9	0.7112086780959164	0.9999736573290944	-	protein stabilization	0	P
GO:0006468	19	21	0.7225691315348293	0.9999736573290944	-	protein phosphorylation	0	P
GO:0006879	6	7	0.7312354237914973	0.9999736573290944	-	cellular iron ion homeostasis	0	P
GO:0001701	6	7	0.7312354237914973	0.9999736573290944	-	in utero embryonic development	0	P
GO:0002244	5	6	0.7450992329583398	0.9999736573290944	-	hematopoietic progenitor cell differentiation	0	P
GO:0050727	5	6	0.7450992329583398	0.9999736573290944	-	regulation of inflammatory response	0	P
GO:0046854	5	6	0.7450992329583398	0.9999736573290944	-	phosphatidylinositol phosphorylation	0	P
GO:0018105	5	6	0.7450992329583398	0.9999736573290944	-	peptidyl-serine phosphorylation	0	P
GO:0032147	5	6	0.7450992329583398	0.9999736573290944	-	activation of protein kinase activity	0	P
GO:0035722	5	6	0.7450992329583398	0.9999736573290944	-	interleukin-12-mediated signaling pathway	0	P
GO:0001525	13	15	0.7463883882701964	0.9999736573290944	-	angiogenesis	0	P
GO:0060041	1	1	0.758451630536907	0.9999736573290944	-	retina development in camera-type eye	0	P
GO:0051292	1	1	0.758451630536907	0.9999736573290944	-	nuclear pore complex assembly	0	P
GO:0046326	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of glucose import	0	P
GO:0045947	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of translational initiation	0	P
GO:0007004	1	1	0.758451630536907	0.9999736573290944	-	telomere maintenance via telomerase	0	P
GO:0006991	1	1	0.758451630536907	0.9999736573290944	-	response to sterol depletion	0	P
GO:0007034	1	1	0.758451630536907	0.9999736573290944	-	vacuolar transport	0	P
GO:0007016	1	1	0.758451630536907	0.9999736573290944	-	cytoskeletal anchoring at plasma membrane	0	P
GO:0007093	1	1	0.758451630536907	0.9999736573290944	-	mitotic cell cycle checkpoint	0	P
GO:0007175	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of epidermal growth factor-activated receptor activity	0	P
GO:0007161	1	1	0.758451630536907	0.9999736573290944	-	calcium-independent cell-matrix adhesion	0	P
GO:0055007	1	1	0.758451630536907	0.9999736573290944	-	cardiac muscle cell differentiation	0	P
GO:0052501	1	1	0.758451630536907	0.9999736573290944	-	positive regulation by organism of apoptotic process in other organism involved in symbiotic interaction	0	P
GO:0051964	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of synapse assembly	0	P
GO:0045776	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of blood pressure	0	P
GO:0060038	1	1	0.758451630536907	0.9999736573290944	-	cardiac muscle cell proliferation	0	P
GO:0055119	1	1	0.758451630536907	0.9999736573290944	-	relaxation of cardiac muscle	0	P
GO:0055088	1	1	0.758451630536907	0.9999736573290944	-	lipid homeostasis	0	P
GO:0060242	1	1	0.758451630536907	0.9999736573290944	-	contact inhibition	0	P
GO:0060193	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of lipase activity	0	P
GO:0060161	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of dopamine receptor signaling pathway	0	P
GO:0060075	1	1	0.758451630536907	0.9999736573290944	-	regulation of resting membrane potential	0	P
GO:0045428	1	1	0.758451630536907	0.9999736573290944	-	regulation of nitric oxide biosynthetic process	0	P
GO:0045347	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of MHC class II biosynthetic process	0	P
GO:0045332	1	1	0.758451630536907	0.9999736573290944	-	phospholipid translocation	0	P
GO:0045217	1	1	0.758451630536907	0.9999736573290944	-	cell-cell junction maintenance	0	P
GO:0060263	1	1	0.758451630536907	0.9999736573290944	-	regulation of respiratory burst	0	P
GO:0007213	1	1	0.758451630536907	0.9999736573290944	-	G protein-coupled acetylcholine receptor signaling pathway	0	P
GO:0006605	1	1	0.758451630536907	0.9999736573290944	-	protein targeting	0	P
GO:0051451	1	1	0.758451630536907	0.9999736573290944	-	myoblast migration	0	P
GO:0051403	1	1	0.758451630536907	0.9999736573290944	-	stress-activated MAPK cascade	0	P
GO:0051668	1	1	0.758451630536907	0.9999736573290944	-	localization within membrane	0	P
GO:0051660	1	1	0.758451630536907	0.9999736573290944	-	establishment of centrosome localization	0	P
GO:0051623	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of norepinephrine uptake	0	P
GO:0051621	1	1	0.758451630536907	0.9999736573290944	-	regulation of norepinephrine uptake	0	P
GO:0051604	1	1	0.758451630536907	0.9999736573290944	-	protein maturation	0	P
GO:0046827	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of protein export from nucleus	0	P
GO:0046686	1	1	0.758451630536907	0.9999736573290944	-	response to cadmium ion	0	P
GO:0046676	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of insulin secretion	0	P
GO:0046636	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of alpha-beta T cell activation	0	P
GO:0046628	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of insulin receptor signaling pathway	0	P
GO:0006672	1	1	0.758451630536907	0.9999736573290944	-	ceramide metabolic process	0	P
GO:0007195	1	1	0.758451630536907	0.9999736573290944	-	adenylate cyclase-inhibiting dopamine receptor signaling pathway	0	P
GO:0006865	1	1	0.758451630536907	0.9999736573290944	-	amino acid transport	0	P
GO:0006813	1	1	0.758451630536907	0.9999736573290944	-	potassium ion transport	0	P
GO:0006811	1	1	0.758451630536907	0.9999736573290944	-	ion transport	0	P
GO:0006749	1	1	0.758451630536907	0.9999736573290944	-	glutathione metabolic process	0	P
GO:0006694	1	1	0.758451630536907	0.9999736573290944	-	steroid biosynthetic process	0	P
GO:0006690	1	1	0.758451630536907	0.9999736573290944	-	icosanoid metabolic process	0	P
GO:0006913	1	1	0.758451630536907	0.9999736573290944	-	nucleocytoplasmic transport	0	P
GO:0045664	1	1	0.758451630536907	0.9999736573290944	-	regulation of neuron differentiation	0	P
GO:0045636	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of melanocyte differentiation	0	P
GO:0045579	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of B cell differentiation	0	P
GO:0045762	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of adenylate cyclase activity	0	P
GO:0045740	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of DNA replication	0	P
GO:0042699	1	1	0.758451630536907	0.9999736573290944	-	follicle-stimulating hormone signaling pathway	0	P
GO:0008210	1	1	0.758451630536907	0.9999736573290944	-	estrogen metabolic process	0	P
GO:0008045	1	1	0.758451630536907	0.9999736573290944	-	motor neuron axon guidance	0	P
GO:0008016	1	1	0.758451630536907	0.9999736573290944	-	regulation of heart contraction	0	P
GO:0008347	1	1	0.758451630536907	0.9999736573290944	-	glial cell migration	0	P
GO:0009117	1	1	0.758451630536907	0.9999736573290944	-	nucleotide metabolic process	0	P
GO:0008626	1	1	0.758451630536907	0.9999736573290944	-	granzyme-mediated apoptotic signaling pathway	0	P
GO:0008625	1	1	0.758451630536907	0.9999736573290944	-	extrinsic apoptotic signaling pathway via death domain receptors	0	P
GO:0009566	1	1	0.758451630536907	0.9999736573290944	-	fertilization	0	P
GO:2000353	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of endothelial cell apoptotic process	0	P
GO:0009314	1	1	0.758451630536907	0.9999736573290944	-	response to radiation	0	P
GO:0009792	1	1	0.758451630536907	0.9999736573290944	-	embryo development ending in birth or egg hatching	0	P
GO:0042325	1	1	0.758451630536907	0.9999736573290944	-	regulation of phosphorylation	0	P
GO:0042759	1	1	0.758451630536907	0.9999736573290944	-	long-chain fatty acid biosynthetic process	0	P
GO:1901381	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of potassium ion transmembrane transport	0	P
GO:0042594	1	1	0.758451630536907	0.9999736573290944	-	response to starvation	0	P
GO:0043044	1	1	0.758451630536907	0.9999736573290944	-	ATP-dependent chromatin remodeling	0	P
GO:0042982	1	1	0.758451630536907	0.9999736573290944	-	amyloid precursor protein metabolic process	0	P
GO:0042771	1	1	0.758451630536907	0.9999736573290944	-	intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator	0	P
GO:0043249	1	1	0.758451630536907	0.9999736573290944	-	erythrocyte maturation	0	P
GO:0010324	1	1	0.758451630536907	0.9999736573290944	-	membrane invagination	0	P
GO:0010310	1	1	0.758451630536907	0.9999736573290944	-	regulation of hydrogen peroxide metabolic process	0	P
GO:0010288	1	1	0.758451630536907	0.9999736573290944	-	response to lead ion	0	P
GO:0010043	1	1	0.758451630536907	0.9999736573290944	-	response to zinc ion	0	P
GO:0010508	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of autophagy	0	P
GO:0010710	1	1	0.758451630536907	0.9999736573290944	-	regulation of collagen catabolic process	0	P
GO:0010621	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of transcription by transcription factor localization	0	P
GO:0010757	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of plasminogen activation	0	P
GO:0043922	1	1	0.758451630536907	0.9999736573290944	-	negative regulation by host of viral transcription	0	P
GO:0007268	1	1	0.758451630536907	0.9999736573290944	-	chemical synaptic transmission	0	P
GO:0007257	1	1	0.758451630536907	0.9999736573290944	-	activation of JUN kinase activity	0	P
GO:0007254	1	1	0.758451630536907	0.9999736573290944	-	JNK cascade	0	P
GO:0007219	1	1	0.758451630536907	0.9999736573290944	-	Notch signaling pathway	0	P
GO:0007520	1	1	0.758451630536907	0.9999736573290944	-	myoblast fusion	0	P
GO:0007519	1	1	0.758451630536907	0.9999736573290944	-	skeletal muscle tissue development	0	P
GO:0043484	1	1	0.758451630536907	0.9999736573290944	-	regulation of RNA splicing	0	P
GO:0043589	1	1	0.758451630536907	0.9999736573290944	-	skin morphogenesis	0	P
GO:0043551	1	1	0.758451630536907	0.9999736573290944	-	regulation of phosphatidylinositol 3-kinase activity	0	P
GO:0043518	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of DNA damage response, signal transduction by p53 class mediator	0	P
GO:0043507	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of JUN kinase activity	0	P
GO:0044257	1	1	0.758451630536907	0.9999736573290944	-	cellular protein catabolic process	0	P
GO:0044130	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of growth of symbiont in host	0	P
GO:0051453	1	1	0.758451630536907	0.9999736573290944	-	regulation of intracellular pH	0	P
GO:0043654	1	1	0.758451630536907	0.9999736573290944	-	recognition of apoptotic cell	0	P
GO:0045124	1	1	0.758451630536907	0.9999736573290944	-	regulation of bone resorption	0	P
GO:0045109	1	1	0.758451630536907	0.9999736573290944	-	intermediate filament organization	0	P
GO:0045055	1	1	0.758451630536907	0.9999736573290944	-	regulated exocytosis	0	P
GO:0045019	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of nitric oxide biosynthetic process	0	P
GO:0044387	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of protein kinase activity by regulation of protein phosphorylation	0	P
GO:1900029	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of ruffle assembly	0	P
GO:1901164	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of trophoblast cell migration	0	P
GO:1900746	1	1	0.758451630536907	0.9999736573290944	-	regulation of vascular endothelial growth factor signaling pathway	0	P
GO:1900425	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of defense response to bacterium	0	P
GO:1900126	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of hyaluronan biosynthetic process	0	P
GO:1901687	1	1	0.758451630536907	0.9999736573290944	-	glutathione derivative biosynthetic process	0	P
GO:0051247	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of protein metabolic process	0	P
GO:1990441	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of transcription from RNA polymerase II promoter in response to endoplasmic reticulum stress	0	P
GO:1990266	1	1	0.758451630536907	0.9999736573290944	-	neutrophil migration	0	P
GO:0002762	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of myeloid leukocyte differentiation	0	P
GO:0002756	1	1	0.758451630536907	0.9999736573290944	-	MyD88-independent toll-like receptor signaling pathway	0	P
GO:0002551	1	1	0.758451630536907	0.9999736573290944	-	mast cell chemotaxis	0	P
GO:0051000	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of nitric-oxide synthase activity	0	P
GO:0050965	1	1	0.758451630536907	0.9999736573290944	-	detection of temperature stimulus involved in sensory perception of pain	0	P
GO:0050930	1	1	0.758451630536907	0.9999736573290944	-	induction of positive chemotaxis	0	P
GO:0050904	1	1	0.758451630536907	0.9999736573290944	-	diapedesis	0	P
GO:0051224	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of protein transport	0	P
GO:0051131	1	1	0.758451630536907	0.9999736573290944	-	chaperone-mediated protein complex assembly	0	P
GO:0051281	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of release of sequestered calcium ion into cytosol	0	P
GO:0051262	1	1	0.758451630536907	0.9999736573290944	-	protein tetramerization	0	P
GO:1990786	1	1	0.758451630536907	0.9999736573290944	-	cellular response to dsDNA	0	P
GO:0051246	1	1	0.758451630536907	0.9999736573290944	-	regulation of protein metabolic process	0	P
GO:0001516	1	1	0.758451630536907	0.9999736573290944	-	prostaglandin biosynthetic process	0	P
GO:0001504	1	1	0.758451630536907	0.9999736573290944	-	neurotransmitter uptake	0	P
GO:0050482	1	1	0.758451630536907	0.9999736573290944	-	arachidonic acid secretion	0	P
GO:0048856	1	1	0.758451630536907	0.9999736573290944	-	anatomical structure development	0	P
GO:0048854	1	1	0.758451630536907	0.9999736573290944	-	brain morphogenesis	0	P
GO:0048757	1	1	0.758451630536907	0.9999736573290944	-	pigment granule maturation	0	P
GO:0050710	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of cytokine secretion	0	P
GO:0050707	1	1	0.758451630536907	0.9999736573290944	-	regulation of cytokine secretion	0	P
GO:0050796	1	1	0.758451630536907	0.9999736573290944	-	regulation of insulin secretion	0	P
GO:0050777	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of immune response	0	P
GO:0050885	1	1	0.758451630536907	0.9999736573290944	-	neuromuscular process controlling balance	0	P
GO:0050855	1	1	0.758451630536907	0.9999736573290944	-	regulation of B cell receptor signaling pathway	0	P
GO:1903898	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of PERK-mediated unfolded protein response	0	P
GO:2001269	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of cysteine-type endopeptidase activity involved in apoptotic signaling pathway	0	P
GO:2001044	1	1	0.758451630536907	0.9999736573290944	-	regulation of integrin-mediated signaling pathway	0	P
GO:2001224	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of neuron migration	0	P
GO:2000810	1	1	0.758451630536907	0.9999736573290944	-	regulation of bicellular tight junction assembly	0	P
GO:2000786	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of autophagosome assembly	0	P
GO:2000767	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of cytoplasmic translation	0	P
GO:2000727	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of cardiac muscle cell differentiation	0	P
GO:2000666	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of interleukin-13 secretion	0	P
GO:2000643	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of early endosome to late endosome transport	0	P
GO:2000642	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of early endosome to late endosome transport	0	P
GO:2000615	1	1	0.758451630536907	0.9999736573290944	-	regulation of histone H3-K9 acetylation	0	P
GO:1904659	1	1	0.758451630536907	0.9999736573290944	-	glucose transmembrane transport	0	P
GO:1904385	1	1	0.758451630536907	0.9999736573290944	-	cellular response to angiotensin	0	P
GO:0050851	1	1	0.758451630536907	0.9999736573290944	-	antigen receptor-mediated signaling pathway	0	P
GO:1903672	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of sprouting angiogenesis	0	P
GO:1903670	1	1	0.758451630536907	0.9999736573290944	-	regulation of sprouting angiogenesis	0	P
GO:1903589	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of blood vessel endothelial cell proliferation involved in sprouting angiogenesis	0	P
GO:1905908	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of amyloid fibril formation	0	P
GO:1905602	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of receptor-mediated endocytosis involved in cholesterol transport	0	P
GO:1905581	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of low-density lipoprotein particle clearance	0	P
GO:1905451	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of Fc-gamma receptor signaling pathway involved in phagocytosis	0	P
GO:1905031	1	1	0.758451630536907	0.9999736573290944	-	regulation of membrane repolarization during cardiac muscle cell action potential	0	P
GO:1905000	1	1	0.758451630536907	0.9999736573290944	-	regulation of membrane repolarization during atrial cardiac muscle cell action potential	0	P
GO:0002544	1	1	0.758451630536907	0.9999736573290944	-	chronic inflammatory response	0	P
GO:0002526	1	1	0.758451630536907	0.9999736573290944	-	acute inflammatory response	0	P
GO:2000010	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of protein localization to cell surface	0	P
GO:1902268	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of polyamine transmembrane transport	0	P
GO:0048668	1	1	0.758451630536907	0.9999736573290944	-	collateral sprouting	0	P
GO:0048667	1	1	0.758451630536907	0.9999736573290944	-	cell morphogenesis involved in neuron differentiation	0	P
GO:1902237	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of endoplasmic reticulum stress-induced intrinsic apoptotic signaling pathway	0	P
GO:1902219	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of intrinsic apoptotic signaling pathway in response to osmotic stress	0	P
GO:1902043	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of extrinsic apoptotic signaling pathway via death domain receptors	0	P
GO:1902042	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of extrinsic apoptotic signaling pathway via death domain receptors	0	P
GO:1901888	1	1	0.758451630536907	0.9999736573290944	-	regulation of cell junction assembly	0	P
GO:1902766	1	1	0.758451630536907	0.9999736573290944	-	skeletal muscle satellite cell migration	0	P
GO:1902748	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of lens fiber cell differentiation	0	P
GO:1902622	1	1	0.758451630536907	0.9999736573290944	-	regulation of neutrophil migration	0	P
GO:1902532	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of intracellular signal transduction	0	P
GO:1902430	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of amyloid-beta formation	0	P
GO:1902396	1	1	0.758451630536907	0.9999736573290944	-	protein localization to bicellular tight junction	0	P
GO:0048679	1	1	0.758451630536907	0.9999736573290944	-	regulation of axon regeneration	0	P
GO:1903056	1	1	0.758451630536907	0.9999736573290944	-	regulation of melanosome organization	0	P
GO:1903026	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of RNA polymerase II regulatory region sequence-specific DNA binding	0	P
GO:1902991	1	1	0.758451630536907	0.9999736573290944	-	regulation of amyloid precursor protein catabolic process	0	P
GO:1902966	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of protein localization to early endosome	0	P
GO:1902949	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of tau-protein kinase activity	0	P
GO:1902896	1	1	0.758451630536907	0.9999736573290944	-	terminal web assembly	0	P
GO:0006542	1	1	0.758451630536907	0.9999736573290944	-	glutamine biosynthetic process	0	P
GO:1903445	1	1	0.758451630536907	0.9999736573290944	-	protein transport from ciliary membrane to plasma membrane	0	P
GO:1903393	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of adherens junction organization	0	P
GO:1903364	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of cellular protein catabolic process	0	P
GO:1903146	1	1	0.758451630536907	0.9999736573290944	-	regulation of autophagy of mitochondrion	0	P
GO:0051489	1	1	0.758451630536907	0.9999736573290944	-	regulation of filopodium assembly	0	P
GO:0048011	1	1	0.758451630536907	0.9999736573290944	-	neurotrophin TRK receptor signaling pathway	0	P
GO:0006122	1	1	0.758451630536907	0.9999736573290944	-	mitochondrial electron transport, ubiquinol to cytochrome c	0	P
GO:0006105	1	1	0.758451630536907	0.9999736573290944	-	succinate metabolic process	0	P
GO:0006099	1	1	0.758451630536907	0.9999736573290944	-	tricarboxylic acid cycle	0	P
GO:0006002	1	1	0.758451630536907	0.9999736573290944	-	fructose 6-phosphate metabolic process	0	P
GO:0003382	1	1	0.758451630536907	0.9999736573290944	-	epithelial cell morphogenesis	0	P
GO:0006306	1	1	0.758451630536907	0.9999736573290944	-	DNA methylation	0	P
GO:0006275	1	1	0.758451630536907	0.9999736573290944	-	regulation of DNA replication	0	P
GO:0006213	1	1	0.758451630536907	0.9999736573290944	-	pyrimidine nucleoside metabolic process	0	P
GO:0006164	1	1	0.758451630536907	0.9999736573290944	-	purine nucleotide biosynthetic process	0	P
GO:0006405	1	1	0.758451630536907	0.9999736573290944	-	RNA export from nucleus	0	P
GO:1901741	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of myoblast fusion	0	P
GO:0006538	1	1	0.758451630536907	0.9999736573290944	-	glutamate catabolic process	0	P
GO:0048012	1	1	0.758451630536907	0.9999736573290944	-	hepatocyte growth factor receptor signaling pathway	0	P
GO:0009447	1	1	0.758451630536907	0.9999736573290944	-	putrescine catabolic process	0	P
GO:0046903	1	1	0.758451630536907	0.9999736573290944	-	secretion	0	P
GO:0046902	1	1	0.758451630536907	0.9999736573290944	-	regulation of mitochondrial membrane permeability	0	P
GO:0046887	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of hormone secretion	0	P
GO:0048247	1	1	0.758451630536907	0.9999736573290944	-	lymphocyte chemotaxis	0	P
GO:0048143	1	1	0.758451630536907	0.9999736573290944	-	astrocyte activation	0	P
GO:0048066	1	1	0.758451630536907	0.9999736573290944	-	developmental pigmentation	0	P
GO:0048026	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of mRNA splicing, via spliceosome	0	P
GO:0048023	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of melanin biosynthetic process	0	P
GO:0048557	1	1	0.758451630536907	0.9999736573290944	-	embryonic digestive tract morphogenesis	0	P
GO:0048298	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of isotype switching to IgA isotypes	0	P
GO:0048266	1	1	0.758451630536907	0.9999736573290944	-	behavioral response to pain	0	P
GO:0048714	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of oligodendrocyte differentiation	0	P
GO:0032532	1	1	0.758451630536907	0.9999736573290944	-	regulation of microvillus length	0	P
GO:0021895	1	1	0.758451630536907	0.9999736573290944	-	cerebral cortex neuron differentiation	0	P
GO:0021846	1	1	0.758451630536907	0.9999736573290944	-	cell proliferation in forebrain	0	P
GO:0021769	1	1	0.758451630536907	0.9999736573290944	-	orbitofrontal cortex development	0	P
GO:0021540	1	1	0.758451630536907	0.9999736573290944	-	corpus callosum morphogenesis	0	P
GO:0022898	1	1	0.758451630536907	0.9999736573290944	-	regulation of transmembrane transporter activity	0	P
GO:0022615	1	1	0.758451630536907	0.9999736573290944	-	protein to membrane docking	0	P
GO:0022604	1	1	0.758451630536907	0.9999736573290944	-	regulation of cell morphogenesis	0	P
GO:0022417	1	1	0.758451630536907	0.9999736573290944	-	protein maturation by protein folding	0	P
GO:0021983	1	1	0.758451630536907	0.9999736573290944	-	pituitary gland development	0	P
GO:0021957	1	1	0.758451630536907	0.9999736573290944	-	corticospinal tract morphogenesis	0	P
GO:0021954	1	1	0.758451630536907	0.9999736573290944	-	central nervous system neuron development	0	P
GO:0030011	1	1	0.758451630536907	0.9999736573290944	-	maintenance of cell polarity	0	P
GO:0022904	1	1	0.758451630536907	0.9999736573290944	-	respiratory electron transport chain	0	P
GO:0032364	1	1	0.758451630536907	0.9999736573290944	-	oxygen homeostasis	0	P
GO:0021915	1	1	0.758451630536907	0.9999736573290944	-	neural tube development	0	P
GO:0032516	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of phosphoprotein phosphatase activity	0	P
GO:0032508	1	1	0.758451630536907	0.9999736573290944	-	DNA duplex unwinding	0	P
GO:0032467	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of cytokinesis	0	P
GO:0032804	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of low-density lipoprotein particle receptor catabolic process	0	P
GO:0032757	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of interleukin-8 production	0	P
GO:0032707	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of interleukin-23 production	0	P
GO:0032700	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of interleukin-17 production	0	P
GO:0032613	1	1	0.758451630536907	0.9999736573290944	-	interleukin-10 production	0	P
GO:0032925	1	1	0.758451630536907	0.9999736573290944	-	regulation of activin receptor signaling pathway	0	P
GO:0032918	1	1	0.758451630536907	0.9999736573290944	-	spermidine acetylation	0	P
GO:0030099	1	1	0.758451630536907	0.9999736573290944	-	myeloid cell differentiation	0	P
GO:0030043	1	1	0.758451630536907	0.9999736573290944	-	actin filament fragmentation	0	P
GO:0030178	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of Wnt signaling pathway	0	P
GO:0070244	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of thymocyte apoptotic process	0	P
GO:0072011	1	1	0.758451630536907	0.9999736573290944	-	glomerular endothelium development	0	P
GO:0071801	1	1	0.758451630536907	0.9999736573290944	-	regulation of podosome assembly	0	P
GO:0090140	1	1	0.758451630536907	0.9999736573290944	-	regulation of mitochondrial fission	0	P
GO:0090102	1	1	0.758451630536907	0.9999736573290944	-	cochlea development	0	P
GO:0090025	1	1	0.758451630536907	0.9999736573290944	-	regulation of monocyte chemotaxis	0	P
GO:0080182	1	1	0.758451630536907	0.9999736573290944	-	histone H3-K4 trimethylation	0	P
GO:0019432	1	1	0.758451630536907	0.9999736573290944	-	triglyceride biosynthetic process	0	P
GO:0019430	1	1	0.758451630536907	0.9999736573290944	-	removal of superoxide radicals	0	P
GO:0090166	1	1	0.758451630536907	0.9999736573290944	-	Golgi disassembly	0	P
GO:0061842	1	1	0.758451630536907	0.9999736573290944	-	microtubule organizing center localization	0	P
GO:0033198	1	1	0.758451630536907	0.9999736573290944	-	response to ATP	0	P
GO:0010744	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of macrophage derived foam cell differentiation	0	P
GO:0070266	1	1	0.758451630536907	0.9999736573290944	-	necroptotic process	0	P
GO:0030111	1	1	0.758451630536907	0.9999736573290944	-	regulation of Wnt signaling pathway	0	P
GO:0070234	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of T cell apoptotic process	0	P
GO:0070528	1	1	0.758451630536907	0.9999736573290944	-	protein kinase C signaling	0	P
GO:0070447	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of oligodendrocyte progenitor proliferation	0	P
GO:0070365	1	1	0.758451630536907	0.9999736573290944	-	hepatocyte differentiation	0	P
GO:0070304	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of stress-activated protein kinase signaling cascade	0	P
GO:0070723	1	1	0.758451630536907	0.9999736573290944	-	response to cholesterol	0	P
GO:0070676	1	1	0.758451630536907	0.9999736573290944	-	intralumenal vesicle formation	0	P
GO:0070668	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of mast cell proliferation	0	P
GO:0070667	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of mast cell proliferation	0	P
GO:0019915	1	1	0.758451630536907	0.9999736573290944	-	lipid storage	0	P
GO:0019827	1	1	0.758451630536907	0.9999736573290944	-	stem cell population maintenance	0	P
GO:0019676	1	1	0.758451630536907	0.9999736573290944	-	ammonia assimilation cycle	0	P
GO:0019563	1	1	0.758451630536907	0.9999736573290944	-	glycerol catabolic process	0	P
GO:0002467	1	1	0.758451630536907	0.9999736573290944	-	germinal center formation	0	P
GO:0032024	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of insulin secretion	0	P
GO:0001886	1	1	0.758451630536907	0.9999736573290944	-	endothelial cell morphogenesis	0	P
GO:2001171	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of ATP biosynthetic process	0	P
GO:0000723	1	1	0.758451630536907	0.9999736573290944	-	telomere maintenance	0	P
GO:2000587	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of platelet-derived growth factor receptor-beta signaling pathway	0	P
GO:0001961	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of cytokine-mediated signaling pathway	0	P
GO:0001953	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of cell-matrix adhesion	0	P
GO:0001951	1	1	0.758451630536907	0.9999736573290944	-	intestinal D-glucose absorption	0	P
GO:0002084	1	1	0.758451630536907	0.9999736573290944	-	protein depalmitoylation	0	P
GO:0002062	1	1	0.758451630536907	0.9999736573290944	-	chondrocyte differentiation	0	P
GO:0002031	1	1	0.758451630536907	0.9999736573290944	-	G protein-coupled receptor internalization	0	P
GO:1903553	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of extracellular exosome assembly	0	P
GO:1903543	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of exosomal secretion	0	P
GO:1903507	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of nucleic acid-templated transcription	0	P
GO:0032060	1	1	0.758451630536907	0.9999736573290944	-	bleb assembly	0	P
GO:0002407	1	1	0.758451630536907	0.9999736573290944	-	dendritic cell chemotaxis	0	P
GO:0001780	1	1	0.758451630536907	0.9999736573290944	-	neutrophil homeostasis	0	P
GO:0001709	1	1	0.758451630536907	0.9999736573290944	-	cell fate determination	0	P
GO:0001654	1	1	0.758451630536907	0.9999736573290944	-	eye development	0	P
GO:0001825	1	1	0.758451630536907	0.9999736573290944	-	blastocyst formation	0	P
GO:0001818	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of cytokine production	0	P
GO:0001812	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of type I hypersensitivity	0	P
GO:0001805	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of type III hypersensitivity	0	P
GO:2000172	1	1	0.758451630536907	0.9999736573290944	-	regulation of branching morphogenesis of a nerve	0	P
GO:2000562	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of CD4-positive, alpha-beta T cell proliferation	0	P
GO:2000503	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of natural killer cell chemotaxis	0	P
GO:2000483	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of interleukin-8 secretion	0	P
GO:2000370	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of clathrin-dependent endocytosis	0	P
GO:0071226	1	1	0.758451630536907	0.9999736573290944	-	cellular response to molecule of fungal origin	0	P
GO:0030214	1	1	0.758451630536907	0.9999736573290944	-	hyaluronan catabolic process	0	P
GO:0030433	1	1	0.758451630536907	0.9999736573290944	-	ubiquitin-dependent ERAD pathway	0	P
GO:0030431	1	1	0.758451630536907	0.9999736573290944	-	sleep	0	P
GO:0030326	1	1	0.758451630536907	0.9999736573290944	-	embryonic limb morphogenesis	0	P
GO:0030490	1	1	0.758451630536907	0.9999736573290944	-	maturation of SSU-rRNA	0	P
GO:0030948	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of vascular endothelial growth factor receptor signaling pathway	0	P
GO:0030845	1	1	0.758451630536907	0.9999736573290944	-	phospholipase C-inhibiting G protein-coupled receptor signaling pathway	0	P
GO:0030837	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of actin filament polymerization	0	P
GO:0030833	1	1	0.758451630536907	0.9999736573290944	-	regulation of actin filament polymerization	0	P
GO:0031146	1	1	0.758451630536907	0.9999736573290944	-	SCF-dependent proteasomal ubiquitin-dependent protein catabolic process	0	P
GO:0031116	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of microtubule polymerization	0	P
GO:0030953	1	1	0.758451630536907	0.9999736573290944	-	astral microtubule organization	0	P
GO:0031638	1	1	0.758451630536907	0.9999736573290944	-	zymogen activation	0	P
GO:0070294	1	1	0.758451630536907	0.9999736573290944	-	renal sodium ion absorption	0	P
GO:0071158	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of cell cycle arrest	0	P
GO:0071103	1	1	0.758451630536907	0.9999736573290944	-	DNA conformation change	0	P
GO:0070972	1	1	0.758451630536907	0.9999736573290944	-	protein localization to endoplasmic reticulum	0	P
GO:0070932	1	1	0.758451630536907	0.9999736573290944	-	histone H3 deacetylation	0	P
GO:0071375	1	1	0.758451630536907	0.9999736573290944	-	cellular response to peptide hormone stimulus	0	P
GO:0071359	1	1	0.758451630536907	0.9999736573290944	-	cellular response to dsRNA	0	P
GO:0071353	1	1	0.758451630536907	0.9999736573290944	-	cellular response to interleukin-4	0	P
GO:0031666	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of lipopolysaccharide-mediated signaling pathway	0	P
GO:0071539	1	1	0.758451630536907	0.9999736573290944	-	protein localization to centrosome	0	P
GO:0071476	1	1	0.758451630536907	0.9999736573290944	-	cellular hypotonic response	0	P
GO:0071474	1	1	0.758451630536907	0.9999736573290944	-	cellular hyperosmotic response	0	P
GO:0032270	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of cellular protein metabolic process	0	P
GO:0032264	1	1	0.758451630536907	0.9999736573290944	-	IMP salvage	0	P
GO:0060411	1	1	0.758451630536907	0.9999736573290944	-	cardiac septum morphogenesis	0	P
GO:0015853	1	1	0.758451630536907	0.9999736573290944	-	adenine transport	0	P
GO:0015721	1	1	0.758451630536907	0.9999736573290944	-	bile acid and bile salt transport	0	P
GO:0014839	1	1	0.758451630536907	0.9999736573290944	-	myoblast migration involved in skeletal muscle regeneration	0	P
GO:0060574	1	1	0.758451630536907	0.9999736573290944	-	intestinal epithelial cell maturation	0	P
GO:0035810	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of urine volume	0	P
GO:0016266	1	1	0.758451630536907	0.9999736573290944	-	O-glycan processing	0	P
GO:0090399	1	1	0.758451630536907	0.9999736573290944	-	replicative senescence	0	P
GO:0097345	1	1	0.758451630536907	0.9999736573290944	-	mitochondrial outer membrane permeabilization	0	P
GO:0016573	1	1	0.758451630536907	0.9999736573290944	-	histone acetylation	0	P
GO:0060710	1	1	0.758451630536907	0.9999736573290944	-	chorio-allantoic fusion	0	P
GO:0016479	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of transcription by RNA polymerase I	0	P
GO:0016358	1	1	0.758451630536907	0.9999736573290944	-	dendrite development	0	P
GO:0033631	1	1	0.758451630536907	0.9999736573290944	-	cell-cell adhesion mediated by integrin	0	P
GO:0035815	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of renal sodium excretion	0	P
GO:0090673	1	1	0.758451630536907	0.9999736573290944	-	endothelial cell-matrix adhesion	0	P
GO:0033625	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of integrin activation	0	P
GO:0061097	1	1	0.758451630536907	0.9999736573290944	-	regulation of protein tyrosine kinase activity	0	P
GO:0033373	1	1	0.758451630536907	0.9999736573290944	-	maintenance of protease location in mast cell secretory granule	0	P
GO:0014066	1	1	0.758451630536907	0.9999736573290944	-	regulation of phosphatidylinositol 3-kinase signaling	0	P
GO:0060766	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of androgen receptor signaling pathway	0	P
GO:0061003	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of dendritic spine morphogenesis	0	P
GO:0097494	1	1	0.758451630536907	0.9999736573290944	-	regulation of vesicle size	0	P
GO:0097581	1	1	0.758451630536907	0.9999736573290944	-	lamellipodium organization	0	P
GO:0038027	1	1	0.758451630536907	0.9999736573290944	-	apolipoprotein A-I-mediated signaling pathway	0	P
GO:0036152	1	1	0.758451630536907	0.9999736573290944	-	phosphatidylethanolamine acyl-chain remodeling	0	P
GO:0038166	1	1	0.758451630536907	0.9999736573290944	-	angiotensin-activated signaling pathway	0	P
GO:0042100	1	1	0.758451630536907	0.9999736573290944	-	B cell proliferation	0	P
GO:0040029	1	1	0.758451630536907	0.9999736573290944	-	regulation of gene expression, epigenetic	0	P
GO:0033382	1	1	0.758451630536907	0.9999736573290944	-	maintenance of granzyme B location in T cell secretory granule	0	P
GO:0097211	1	1	0.758451630536907	0.9999736573290944	-	cellular response to gonadotropin-releasing hormone	0	P
GO:0097324	1	1	0.758451630536907	0.9999736573290944	-	melanocyte migration	0	P
GO:0090324	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of oxidative phosphorylation	0	P
GO:0010936	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of macrophage cytokine production	0	P
GO:0039534	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of MDA-5 signaling pathway	0	P
GO:0035729	1	1	0.758451630536907	0.9999736573290944	-	cellular response to hepatocyte growth factor stimulus	0	P
GO:0010925	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of inositol-polyphosphate 5-phosphatase activity	0	P
GO:0060397	1	1	0.758451630536907	0.9999736573290944	-	growth hormone receptor signaling pathway via JAK-STAT	0	P
GO:0010886	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of cholesterol storage	0	P
GO:0010815	1	1	0.758451630536907	0.9999736573290944	-	bradykinin catabolic process	0	P
GO:0034356	1	1	0.758451630536907	0.9999736573290944	-	NAD biosynthesis via nicotinamide riboside salvage pathway	0	P
GO:0035020	1	1	0.758451630536907	0.9999736573290944	-	regulation of Rac protein signal transduction	0	P
GO:0060314	1	1	0.758451630536907	0.9999736573290944	-	regulation of ryanodine-sensitive calcium-release channel activity	0	P
GO:0061298	1	1	0.758451630536907	0.9999736573290944	-	retina vasculature development in camera-type eye	0	P
GO:0071677	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of mononuclear cell migration	0	P
GO:0010863	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of phospholipase C activity	0	P
GO:0010711	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of collagen catabolic process	0	P
GO:0033371	1	1	0.758451630536907	0.9999736573290944	-	T cell secretory granule organization	0	P
GO:0071559	1	1	0.758451630536907	0.9999736573290944	-	response to transforming growth factor beta	0	P
GO:0072749	1	1	0.758451630536907	0.9999736573290944	-	cellular response to cytochalasin B	0	P
GO:0072672	1	1	0.758451630536907	0.9999736573290944	-	neutrophil extravasation	0	P
GO:0072656	1	1	0.758451630536907	0.9999736573290944	-	maintenance of protein location in mitochondrion	0	P
GO:0061373	1	1	0.758451630536907	0.9999736573290944	-	mammillary axonal complex development	0	P
GO:0042178	1	1	0.758451630536907	0.9999736573290944	-	xenobiotic catabolic process	0	P
GO:0010920	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of inositol phosphate biosynthetic process	0	P
GO:0060546	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of necroptotic process	0	P
GO:0036149	1	1	0.758451630536907	0.9999736573290944	-	phosphatidylinositol acyl-chain remodeling	0	P
GO:0060305	1	1	0.758451630536907	0.9999736573290944	-	regulation of cell diameter	0	P
GO:0034198	1	1	0.758451630536907	0.9999736573290944	-	cellular response to amino acid starvation	0	P
GO:0034136	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of toll-like receptor 2 signaling pathway	0	P
GO:0034103	1	1	0.758451630536907	0.9999736573290944	-	regulation of tissue remodeling	0	P
GO:0060547	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of necrotic cell death	0	P
GO:0033993	1	1	0.758451630536907	0.9999736573290944	-	response to lipid	0	P
GO:0033674	1	1	0.758451630536907	0.9999736573290944	-	positive regulation of kinase activity	0	P
GO:0060311	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of elastin catabolic process	0	P
GO:0034395	1	1	0.758451630536907	0.9999736573290944	-	regulation of transcription from RNA polymerase II promoter in response to iron	0	P
GO:0060313	1	1	0.758451630536907	0.9999736573290944	-	negative regulation of blood vessel remodeling	0	P
GO:0034599	4	5	0.7631083950036134	0.9999736573290944	-	cellular response to oxidative stress	0	P
GO:0034383	4	5	0.7631083950036134	0.9999736573290944	-	low-density lipoprotein particle clearance	0	P
GO:0010634	4	5	0.7631083950036134	0.9999736573290944	-	positive regulation of epithelial cell migration	0	P
GO:0070588	4	5	0.7631083950036134	0.9999736573290944	-	calcium ion transmembrane transport	0	P
GO:1900026	4	5	0.7631083950036134	0.9999736573290944	-	positive regulation of substrate adhesion-dependent cell spreading	0	P
GO:0007186	20	23	0.7684937974537605	0.9999736573290944	-	G protein-coupled receptor signaling pathway	0	P
GO:0000165	9	11	0.7737720548797316	0.9999736573290944	-	MAPK cascade	0	P
GO:0006355	9	11	0.7737720548797316	0.9999736573290944	-	regulation of transcription, DNA-templated	0	P
GO:0007267	8	10	0.7832535760542025	0.9999736573290944	-	cell-cell signaling	0	P
GO:0006096	3	4	0.7876037868729249	0.9999736573290944	-	glycolytic process	0	P
GO:0006406	3	4	0.7876037868729249	0.9999736573290944	-	mRNA export from nucleus	0	P
GO:0042542	3	4	0.7876037868729249	0.9999736573290944	-	response to hydrogen peroxide	0	P
GO:0031398	3	4	0.7876037868729249	0.9999736573290944	-	positive regulation of protein ubiquitination	0	P
GO:0038083	3	4	0.7876037868729249	0.9999736573290944	-	peptidyl-tyrosine autophosphorylation	0	P
GO:0042531	3	4	0.7876037868729249	0.9999736573290944	-	positive regulation of tyrosine phosphorylation of STAT protein	0	P
GO:0061098	3	4	0.7876037868729249	0.9999736573290944	-	positive regulation of protein tyrosine kinase activity	0	P
GO:0006606	3	4	0.7876037868729249	0.9999736573290944	-	protein import into nucleus	0	P
GO:0007010	3	4	0.7876037868729249	0.9999736573290944	-	cytoskeleton organization	0	P
GO:0034605	3	4	0.7876037868729249	0.9999736573290944	-	cellular response to heat	0	P
GO:0045444	3	4	0.7876037868729249	0.9999736573290944	-	fat cell differentiation	0	P
GO:0071479	3	4	0.7876037868729249	0.9999736573290944	-	cellular response to ionizing radiation	0	P
GO:0007188	3	4	0.7876037868729249	0.9999736573290944	-	adenylate cyclase-modulating G protein-coupled receptor signaling pathway	0	P
GO:0007417	3	4	0.7876037868729249	0.9999736573290944	-	central nervous system development	0	P
GO:0048812	3	4	0.7876037868729249	0.9999736573290944	-	neuron projection morphogenesis	0	P
GO:0032880	3	4	0.7876037868729249	0.9999736573290944	-	regulation of protein localization	0	P
GO:0043627	3	4	0.7876037868729249	0.9999736573290944	-	response to estrogen	0	P
GO:0006909	7	9	0.7943215815086736	0.9999736573290944	-	phagocytosis	0	P
GO:0006898	6	8	0.8074212747893253	0.9999736573290944	-	receptor-mediated endocytosis	0	P
GO:0008360	11	14	0.8141591751740763	0.9999736573290944	-	regulation of cell shape	0	P
GO:0008283	18	22	0.8194140216696246	0.9999736573290944	-	cell population proliferation	0	P
GO:0050690	5	7	0.8231889460790113	0.9999736573290944	-	regulation of defense response to virus by virus	0	P
GO:0050731	5	7	0.8231889460790113	0.9999736573290944	-	positive regulation of peptidyl-tyrosine phosphorylation	0	P
GO:0010976	5	7	0.8231889460790113	0.9999736573290944	-	positive regulation of neuron projection development	0	P
GO:0016241	5	7	0.8231889460790113	0.9999736573290944	-	regulation of macroautophagy	0	P
GO:0030183	5	7	0.8231889460790113	0.9999736573290944	-	B cell differentiation	0	P
GO:0010592	2	3	0.823283798533795	0.9999736573290944	-	positive regulation of lamellipodium assembly	0	P
GO:0002092	2	3	0.823283798533795	0.9999736573290944	-	positive regulation of receptor internalization	0	P
GO:0014823	2	3	0.823283798533795	0.9999736573290944	-	response to activity	0	P
GO:0071526	2	3	0.823283798533795	0.9999736573290944	-	semaphorin-plexin signaling pathway	0	P
GO:0016239	2	3	0.823283798533795	0.9999736573290944	-	positive regulation of macroautophagy	0	P
GO:0042177	2	3	0.823283798533795	0.9999736573290944	-	negative regulation of protein catabolic process	0	P
GO:1902895	2	3	0.823283798533795	0.9999736573290944	-	positive regulation of pri-miRNA transcription by RNA polymerase II	0	P
GO:0008542	2	3	0.823283798533795	0.9999736573290944	-	visual learning	0	P
GO:0016042	2	3	0.823283798533795	0.9999736573290944	-	lipid catabolic process	0	P
GO:0010506	2	3	0.823283798533795	0.9999736573290944	-	regulation of autophagy	0	P
GO:0006986	2	3	0.823283798533795	0.9999736573290944	-	response to unfolded protein	0	P
GO:0035019	2	3	0.823283798533795	0.9999736573290944	-	somatic stem cell population maintenance	0	P
GO:0072583	2	3	0.823283798533795	0.9999736573290944	-	clathrin-dependent endocytosis	0	P
GO:0045732	2	3	0.823283798533795	0.9999736573290944	-	positive regulation of protein catabolic process	0	P
GO:0045727	2	3	0.823283798533795	0.9999736573290944	-	positive regulation of translation	0	P
GO:0045815	2	3	0.823283798533795	0.9999736573290944	-	positive regulation of gene expression, epigenetic	0	P
GO:0050771	2	3	0.823283798533795	0.9999736573290944	-	negative regulation of axonogenesis	0	P
GO:0051279	2	3	0.823283798533795	0.9999736573290944	-	regulation of release of sequestered calcium ion into cytosol	0	P
GO:0032868	2	3	0.823283798533795	0.9999736573290944	-	response to insulin	0	P
GO:0051259	2	3	0.823283798533795	0.9999736573290944	-	protein complex oligomerization	0	P
GO:0071300	2	3	0.823283798533795	0.9999736573290944	-	cellular response to retinoic acid	0	P
GO:0032802	2	3	0.823283798533795	0.9999736573290944	-	low-density lipoprotein particle receptor catabolic process	0	P
GO:0060395	2	3	0.823283798533795	0.9999736573290944	-	SMAD protein signal transduction	0	P
GO:0071363	2	3	0.823283798533795	0.9999736573290944	-	cellular response to growth factor stimulus	0	P
GO:0008037	2	3	0.823283798533795	0.9999736573290944	-	cell recognition	0	P
GO:0070935	2	3	0.823283798533795	0.9999736573290944	-	3'-UTR-mediated mRNA stabilization	0	P
GO:0043542	2	3	0.823283798533795	0.9999736573290944	-	endothelial cell migration	0	P
GO:0042789	2	3	0.823283798533795	0.9999736573290944	-	mRNA transcription by RNA polymerase II	0	P
GO:0051898	2	3	0.823283798533795	0.9999736573290944	-	negative regulation of protein kinase B signaling	0	P
GO:0030855	2	3	0.823283798533795	0.9999736573290944	-	epithelial cell differentiation	0	P
GO:1901214	2	3	0.823283798533795	0.9999736573290944	-	regulation of neuron death	0	P
GO:0045893	17	21	0.8239302536749207	0.9999736573290944	-	positive regulation of transcription, DNA-templated	0	P
GO:0006974	9	12	0.8310397246610084	0.9999736573290944	-	cellular response to DNA damage stimulus	0	P
GO:0007049	9	12	0.8310397246610084	0.9999736573290944	-	cell cycle	0	P
GO:0002250	8	11	0.8410894207977226	0.9999736573290944	-	adaptive immune response	0	P
GO:0051017	4	6	0.8425674743964261	0.9999736573290944	-	actin filament bundle assembly	0	P
GO:0006887	4	6	0.8425674743964261	0.9999736573290944	-	exocytosis	0	P
GO:0034097	4	6	0.8425674743964261	0.9999736573290944	-	response to cytokine	0	P
GO:0019722	4	6	0.8425674743964261	0.9999736573290944	-	calcium-mediated signaling	0	P
GO:0060071	4	6	0.8425674743964261	0.9999736573290944	-	Wnt signaling pathway, planar cell polarity pathway	0	P
GO:0048661	4	6	0.8425674743964261	0.9999736573290944	-	positive regulation of smooth muscle cell proliferation	0	P
GO:0043407	4	6	0.8425674743964261	0.9999736573290944	-	negative regulation of MAP kinase activity	0	P
GO:0043086	4	6	0.8425674743964261	0.9999736573290944	-	negative regulation of catalytic activity	0	P
GO:0051289	4	6	0.8425674743964261	0.9999736573290944	-	protein homotetramerization	0	P
GO:0048010	6	9	0.86556552742099	0.9999736573290944	-	vascular endothelial growth factor receptor signaling pathway	0	P
GO:0038095	6	9	0.86556552742099	0.9999736573290944	-	Fc-epsilon receptor signaling pathway	0	P
GO:0043280	3	5	0.8670124962520183	0.9999736573290944	-	positive regulation of cysteine-type endopeptidase activity involved in apoptotic process	0	P
GO:0060326	3	5	0.8670124962520183	0.9999736573290944	-	cell chemotaxis	0	P
GO:0022617	3	5	0.8670124962520183	0.9999736573290944	-	extracellular matrix disassembly	0	P
GO:0006469	3	5	0.8670124962520183	0.9999736573290944	-	negative regulation of protein kinase activity	0	P
GO:0031098	3	5	0.8670124962520183	0.9999736573290944	-	stress-activated protein kinase signaling cascade	0	P
GO:0007163	3	5	0.8670124962520183	0.9999736573290944	-	establishment or maintenance of cell polarity	0	P
GO:0031663	3	5	0.8670124962520183	0.9999736573290944	-	lipopolysaccharide-mediated signaling pathway	0	P
GO:0061621	3	5	0.8670124962520183	0.9999736573290944	-	canonical glycolysis	0	P
GO:0008361	3	5	0.8670124962520183	0.9999736573290944	-	regulation of cell size	0	P
GO:0090090	3	5	0.8670124962520183	0.9999736573290944	-	negative regulation of canonical Wnt signaling pathway	0	P
GO:0070374	10	14	0.8676634964079808	0.9999736573290944	-	positive regulation of ERK1 and ERK2 cascade	0	P
GO:0034144	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of toll-like receptor 4 signaling pathway	0	P
GO:0010737	1	2	0.8814645562537212	0.9999736573290944	-	protein kinase A signaling	0	P
GO:0009968	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of signal transduction	0	P
GO:0009052	1	2	0.8814645562537212	0.9999736573290944	-	pentose-phosphate shunt, non-oxidative branch	0	P
GO:0009060	1	2	0.8814645562537212	0.9999736573290944	-	aerobic respiration	0	P
GO:0036151	1	2	0.8814645562537212	0.9999736573290944	-	phosphatidylcholine acyl-chain remodeling	0	P
GO:0035767	1	2	0.8814645562537212	0.9999736573290944	-	endothelial cell chemotaxis	0	P
GO:0035902	1	2	0.8814645562537212	0.9999736573290944	-	response to immobilization stress	0	P
GO:0035606	1	2	0.8814645562537212	0.9999736573290944	-	peptidyl-cysteine S-trans-nitrosylation	0	P
GO:0035666	1	2	0.8814645562537212	0.9999736573290944	-	TRIF-dependent toll-like receptor signaling pathway	0	P
GO:0035024	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of Rho protein signal transduction	0	P
GO:0090330	1	2	0.8814645562537212	0.9999736573290944	-	regulation of platelet aggregation	0	P
GO:0033630	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of cell adhesion mediated by integrin	0	P
GO:0033364	1	2	0.8814645562537212	0.9999736573290944	-	mast cell secretory granule organization	0	P
GO:0033574	1	2	0.8814645562537212	0.9999736573290944	-	response to testosterone	0	P
GO:0070885	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of calcineurin-NFAT signaling cascade	0	P
GO:0033003	1	2	0.8814645562537212	0.9999736573290944	-	regulation of mast cell activation	0	P
GO:0032728	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of interferon-beta production	0	P
GO:0032695	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of interleukin-12 production	0	P
GO:0030449	1	2	0.8814645562537212	0.9999736573290944	-	regulation of complement activation	0	P
GO:0030217	1	2	0.8814645562537212	0.9999736573290944	-	T cell differentiation	0	P
GO:0051480	1	2	0.8814645562537212	0.9999736573290944	-	regulation of cytosolic calcium ion concentration	0	P
GO:0042535	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of tumor necrosis factor biosynthetic process	0	P
GO:0042391	1	2	0.8814645562537212	0.9999736573290944	-	regulation of membrane potential	0	P
GO:0061028	1	2	0.8814645562537212	0.9999736573290944	-	establishment of endothelial barrier	0	P
GO:0060999	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of dendritic spine development	0	P
GO:0040008	1	2	0.8814645562537212	0.9999736573290944	-	regulation of growth	0	P
GO:0038128	1	2	0.8814645562537212	0.9999736573290944	-	ERBB2 signaling pathway	0	P
GO:0035914	1	2	0.8814645562537212	0.9999736573290944	-	skeletal muscle cell differentiation	0	P
GO:0016064	1	2	0.8814645562537212	0.9999736573290944	-	immunoglobulin mediated immune response	0	P
GO:0014850	1	2	0.8814645562537212	0.9999736573290944	-	response to muscle activity	0	P
GO:0030225	1	2	0.8814645562537212	0.9999736573290944	-	macrophage differentiation	0	P
GO:0014065	1	2	0.8814645562537212	0.9999736573290944	-	phosphatidylinositol 3-kinase signaling	0	P
GO:0010923	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of phosphatase activity	0	P
GO:0010942	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of cell death	0	P
GO:0010944	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of transcription by competitive promoter binding	0	P
GO:0010812	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of cell-substrate adhesion	0	P
GO:0010818	1	2	0.8814645562537212	0.9999736573290944	-	T cell chemotaxis	0	P
GO:0097421	1	2	0.8814645562537212	0.9999736573290944	-	liver regeneration	0	P
GO:0097284	1	2	0.8814645562537212	0.9999736573290944	-	hepatocyte apoptotic process	0	P
GO:0097191	1	2	0.8814645562537212	0.9999736573290944	-	extrinsic apoptotic signaling pathway	0	P
GO:1990314	1	2	0.8814645562537212	0.9999736573290944	-	cellular response to insulin-like growth factor stimulus	0	P
GO:0051220	1	2	0.8814645562537212	0.9999736573290944	-	cytoplasmic sequestering of protein	0	P
GO:0051045	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of membrane protein ectodomain proteolysis	0	P
GO:0050807	1	2	0.8814645562537212	0.9999736573290944	-	regulation of synapse organization	0	P
GO:0005977	1	2	0.8814645562537212	0.9999736573290944	-	glycogen metabolic process	0	P
GO:0005999	1	2	0.8814645562537212	0.9999736573290944	-	xylulose biosynthetic process	0	P
GO:0002768	1	2	0.8814645562537212	0.9999736573290944	-	immune response-regulating cell surface receptor signaling pathway	0	P
GO:0002554	1	2	0.8814645562537212	0.9999736573290944	-	serotonin secretion by platelet	0	P
GO:1990535	1	2	0.8814645562537212	0.9999736573290944	-	neuron projection maintenance	0	P
GO:0001892	1	2	0.8814645562537212	0.9999736573290944	-	embryonic placenta development	0	P
GO:0051209	1	2	0.8814645562537212	0.9999736573290944	-	release of sequestered calcium ion into cytosol	0	P
GO:1904707	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of vascular smooth muscle cell proliferation	0	P
GO:1905049	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of metallopeptidase activity	0	P
GO:0002431	1	2	0.8814645562537212	0.9999736573290944	-	Fc receptor mediated stimulatory signaling pathway	0	P
GO:0002052	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of neuroblast proliferation	0	P
GO:0002218	1	2	0.8814645562537212	0.9999736573290944	-	activation of innate immune response	0	P
GO:0001954	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of cell-matrix adhesion	0	P
GO:0000281	1	2	0.8814645562537212	0.9999736573290944	-	mitotic cytokinesis	0	P
GO:0050663	1	2	0.8814645562537212	0.9999736573290944	-	cytokine secretion	0	P
GO:1990776	1	2	0.8814645562537212	0.9999736573290944	-	response to angiotensin	0	P
GO:0031214	1	2	0.8814645562537212	0.9999736573290944	-	biomineral tissue development	0	P
GO:0030048	1	2	0.8814645562537212	0.9999736573290944	-	actin filament-based movement	0	P
GO:0090026	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of monocyte chemotaxis	0	P
GO:0072673	1	2	0.8814645562537212	0.9999736573290944	-	lamellipodium morphogenesis	0	P
GO:0016973	1	2	0.8814645562537212	0.9999736573290944	-	poly(A)+ mRNA export from nucleus	0	P
GO:0018279	1	2	0.8814645562537212	0.9999736573290944	-	protein N-linked glycosylation via asparagine	0	P
GO:0018345	1	2	0.8814645562537212	0.9999736573290944	-	protein palmitoylation	0	P
GO:0015914	1	2	0.8814645562537212	0.9999736573290944	-	phospholipid transport	0	P
GO:0032259	1	2	0.8814645562537212	0.9999736573290944	-	methylation	0	P
GO:0071447	1	2	0.8814645562537212	0.9999736573290944	-	cellular response to hydroperoxide	0	P
GO:0014912	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of smooth muscle cell migration	0	P
GO:0031293	1	2	0.8814645562537212	0.9999736573290944	-	membrane protein intracellular domain proteolysis	0	P
GO:0030522	1	2	0.8814645562537212	0.9999736573290944	-	intracellular receptor signaling pathway	0	P
GO:2000670	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of dendritic cell apoptotic process	0	P
GO:0001816	1	2	0.8814645562537212	0.9999736573290944	-	cytokine production	0	P
GO:0001819	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of cytokine production	0	P
GO:0001756	1	2	0.8814645562537212	0.9999736573290944	-	somitogenesis	0	P
GO:0001667	1	2	0.8814645562537212	0.9999736573290944	-	ameboidal-type cell migration	0	P
GO:0001708	1	2	0.8814645562537212	0.9999736573290944	-	cell fate specification	0	P
GO:0051272	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of cellular component movement	0	P
GO:0042770	1	2	0.8814645562537212	0.9999736573290944	-	signal transduction in response to DNA damage	0	P
GO:0006412	1	2	0.8814645562537212	0.9999736573290944	-	translation	0	P
GO:0006413	1	2	0.8814645562537212	0.9999736573290944	-	translational initiation	0	P
GO:0006458	1	2	0.8814645562537212	0.9999736573290944	-	'de novo' protein folding	0	P
GO:0006465	1	2	0.8814645562537212	0.9999736573290944	-	signal peptide processing	0	P
GO:0006368	1	2	0.8814645562537212	0.9999736573290944	-	transcription elongation from RNA polymerase II promoter	0	P
GO:0006383	1	2	0.8814645562537212	0.9999736573290944	-	transcription by RNA polymerase III	0	P
GO:0045616	1	2	0.8814645562537212	0.9999736573290944	-	regulation of keratinocyte differentiation	0	P
GO:0045646	1	2	0.8814645562537212	0.9999736573290944	-	regulation of erythrocyte differentiation	0	P
GO:0006826	1	2	0.8814645562537212	0.9999736573290944	-	iron ion transport	0	P
GO:0045453	1	2	0.8814645562537212	0.9999736573290944	-	bone resorption	0	P
GO:0045591	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of regulatory T cell differentiation	0	P
GO:0045198	1	2	0.8814645562537212	0.9999736573290944	-	establishment of epithelial cell apical/basal polarity	0	P
GO:0060252	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of glial cell proliferation	0	P
GO:0051966	1	2	0.8814645562537212	0.9999736573290944	-	regulation of synaptic transmission, glutamatergic	0	P
GO:0044794	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation by host of viral process	0	P
GO:0044849	1	2	0.8814645562537212	0.9999736573290944	-	estrous cycle	0	P
GO:0045651	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of macrophage differentiation	0	P
GO:0006821	1	2	0.8814645562537212	0.9999736573290944	-	chloride transport	0	P
GO:0045742	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of epidermal growth factor receptor signaling pathway	0	P
GO:0045814	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of gene expression, epigenetic	0	P
GO:0045806	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of endocytosis	0	P
GO:0045780	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of bone resorption	0	P
GO:0045930	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of mitotic cell cycle	0	P
GO:0046622	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of organ growth	0	P
GO:0046488	1	2	0.8814645562537212	0.9999736573290944	-	phosphatidylinositol metabolic process	0	P
GO:0051549	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of keratinocyte migration	0	P
GO:0046638	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of alpha-beta T cell differentiation	0	P
GO:0048246	1	2	0.8814645562537212	0.9999736573290944	-	macrophage chemotaxis	0	P
GO:0048245	1	2	0.8814645562537212	0.9999736573290944	-	eosinophil chemotaxis	0	P
GO:0048147	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of fibroblast proliferation	0	P
GO:0048538	1	2	0.8814645562537212	0.9999736573290944	-	thymus development	0	P
GO:0048268	1	2	0.8814645562537212	0.9999736573290944	-	clathrin coat assembly	0	P
GO:1900087	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of G1/S transition of mitotic cell cycle	0	P
GO:0007346	1	2	0.8814645562537212	0.9999736573290944	-	regulation of mitotic cell cycle	0	P
GO:0007498	1	2	0.8814645562537212	0.9999736573290944	-	mesoderm development	0	P
GO:0007566	1	2	0.8814645562537212	0.9999736573290944	-	embryo implantation	0	P
GO:0099132	1	2	0.8814645562537212	0.9999736573290944	-	ATP hydrolysis coupled cation transmembrane transport	0	P
GO:0007368	1	2	0.8814645562537212	0.9999736573290944	-	determination of left/right symmetry	0	P
GO:0098742	1	2	0.8814645562537212	0.9999736573290944	-	cell-cell adhesion via plasma-membrane adhesion molecules	0	P
GO:0007187	1	2	0.8814645562537212	0.9999736573290944	-	G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger	0	P
GO:0043113	1	2	0.8814645562537212	0.9999736573290944	-	receptor clustering	0	P
GO:0043032	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of macrophage activation	0	P
GO:1900027	1	2	0.8814645562537212	0.9999736573290944	-	regulation of ruffle assembly	0	P
GO:1900745	1	2	0.8814645562537212	0.9999736573290944	-	positive regulation of p38MAPK cascade	0	P
GO:1901379	1	2	0.8814645562537212	0.9999736573290944	-	regulation of potassium ion transmembrane transport	0	P
GO:0043087	1	2	0.8814645562537212	0.9999736573290944	-	regulation of GTPase activity	0	P
GO:0043277	1	2	0.8814645562537212	0.9999736573290944	-	apoptotic cell clearance	0	P
GO:0042953	1	2	0.8814645562537212	0.9999736573290944	-	lipoprotein transport	0	P
GO:0043011	1	2	0.8814645562537212	0.9999736573290944	-	myeloid dendritic cell differentiation	0	P
GO:0043473	1	2	0.8814645562537212	0.9999736573290944	-	pigmentation	0	P
GO:1901222	1	2	0.8814645562537212	0.9999736573290944	-	regulation of NIK/NF-kappaB signaling	0	P
GO:0042730	1	2	0.8814645562537212	0.9999736573290944	-	fibrinolysis	0	P
GO:1901215	1	2	0.8814645562537212	0.9999736573290944	-	negative regulation of neuron death	0	P
GO:0000122	26	33	0.8857564306103228	0.9999736573290944	-	negative regulation of transcription by RNA polymerase II	0	P
GO:0007166	11	16	0.8959706577116138	0.9999736573290944	-	cell surface receptor signaling pathway	0	P
GO:0051260	7	11	0.8965669107070411	0.9999736573290944	-	protein homooligomerization	0	P
GO:0051928	4	7	0.8982636123610467	0.9999736573290944	-	positive regulation of calcium ion transport	0	P
GO:0043488	4	7	0.8982636123610467	0.9999736573290944	-	regulation of mRNA stability	0	P
GO:0007169	4	7	0.8982636123610467	0.9999736573290944	-	transmembrane receptor protein tyrosine kinase signaling pathway	0	P
GO:1903078	4	7	0.8982636123610467	0.9999736573290944	-	positive regulation of protein localization to plasma membrane	0	P
GO:0000187	4	7	0.8982636123610467	0.9999736573290944	-	activation of MAPK activity	0	P
GO:0035335	4	7	0.8982636123610467	0.9999736573290944	-	peptidyl-tyrosine dephosphorylation	0	P
GO:0001933	4	7	0.8982636123610467	0.9999736573290944	-	negative regulation of protein phosphorylation	0	P
GO:0034394	2	4	0.8988632423216547	0.9999736573290944	-	protein localization to cell surface	0	P
GO:0001755	2	4	0.8988632423216547	0.9999736573290944	-	neural crest cell migration	0	P
GO:0006911	2	4	0.8988632423216547	0.9999736573290944	-	phagocytosis, engulfment	0	P
GO:0045744	2	4	0.8988632423216547	0.9999736573290944	-	negative regulation of G protein-coupled receptor signaling pathway	0	P
GO:0007193	2	4	0.8988632423216547	0.9999736573290944	-	adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway	0	P
GO:0005975	2	4	0.8988632423216547	0.9999736573290944	-	carbohydrate metabolic process	0	P
GO:0046597	2	4	0.8988632423216547	0.9999736573290944	-	negative regulation of viral entry into host cell	0	P
GO:0043200	2	4	0.8988632423216547	0.9999736573290944	-	response to amino acid	0	P
GO:0042752	2	4	0.8988632423216547	0.9999736573290944	-	regulation of circadian rhythm	0	P
GO:0019730	2	4	0.8988632423216547	0.9999736573290944	-	antimicrobial humoral response	0	P
GO:2000179	2	4	0.8988632423216547	0.9999736573290944	-	positive regulation of neural precursor cell proliferation	0	P
GO:0045089	2	4	0.8988632423216547	0.9999736573290944	-	positive regulation of innate immune response	0	P
GO:0032870	2	4	0.8988632423216547	0.9999736573290944	-	cellular response to hormone stimulus	0	P
GO:0071320	2	4	0.8988632423216547	0.9999736573290944	-	cellular response to cAMP	0	P
GO:0055072	2	4	0.8988632423216547	0.9999736573290944	-	iron ion homeostasis	0	P
GO:0031668	2	4	0.8988632423216547	0.9999736573290944	-	cellular response to extracellular stimulus	0	P
GO:0006397	2	4	0.8988632423216547	0.9999736573290944	-	mRNA processing	0	P
GO:0030502	2	4	0.8988632423216547	0.9999736573290944	-	negative regulation of bone mineralization	0	P
GO:0032515	2	4	0.8988632423216547	0.9999736573290944	-	negative regulation of phosphoprotein phosphatase activity	0	P
GO:0006098	2	4	0.8988632423216547	0.9999736573290944	-	pentose-phosphate shunt	0	P
GO:0045672	2	4	0.8988632423216547	0.9999736573290944	-	positive regulation of osteoclast differentiation	0	P
GO:0016310	2	4	0.8988632423216547	0.9999736573290944	-	phosphorylation	0	P
GO:0018108	6	10	0.9083118949683799	0.9999736573290944	-	peptidyl-tyrosine phosphorylation	0	P
GO:0048015	3	6	0.9189645468762208	0.9999736573290944	-	phosphatidylinositol-mediated signaling	0	P
GO:0043124	3	6	0.9189645468762208	0.9999736573290944	-	negative regulation of I-kappaB kinase/NF-kappaB signaling	0	P
GO:0045454	3	6	0.9189645468762208	0.9999736573290944	-	cell redox homeostasis	0	P
GO:0007283	3	6	0.9189645468762208	0.9999736573290944	-	spermatogenesis	0	P
GO:0043524	5	9	0.9213480212416973	0.9999736573290944	-	negative regulation of neuron apoptotic process	0	P
GO:0032091	5	9	0.9213480212416973	0.9999736573290944	-	negative regulation of protein binding	0	P
GO:0043065	11	17	0.924120691321449	0.9999736573290944	-	positive regulation of apoptotic process	0	P
GO:0010628	11	17	0.924120691321449	0.9999736573290944	-	positive regulation of gene expression	0	P
GO:0007266	7	12	0.9289287798208101	0.9999736573290944	-	Rho protein signal transduction	0	P
GO:0045944	31	41	0.9320584653432252	0.9999736573290944	-	positive regulation of transcription by RNA polymerase II	0	P
GO:0071356	4	8	0.9358009455020645	0.9999736573290944	-	cellular response to tumor necrosis factor	0	P
GO:0006366	9	15	0.9374033719084438	0.9999736573290944	-	transcription by RNA polymerase II	0	P
GO:0035556	12	19	0.9402625063177401	0.9999736573290944	-	intracellular signal transduction	0	P
GO:0010466	1	3	0.9418898508196953	0.9999736573290944	-	negative regulation of peptidase activity	0	P
GO:0007202	1	3	0.9418898508196953	0.9999736573290944	-	activation of phospholipase C activity	0	P
GO:0007517	1	3	0.9418898508196953	0.9999736573290944	-	muscle organ development	0	P
GO:0042832	1	3	0.9418898508196953	0.9999736573290944	-	defense response to protozoan	0	P
GO:0031640	1	3	0.9418898508196953	0.9999736573290944	-	killing of cells of other organism	0	P
GO:0040018	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of multicellular organism growth	0	P
GO:0042267	1	3	0.9418898508196953	0.9999736573290944	-	natural killer cell mediated cytotoxicity	0	P
GO:0060964	1	3	0.9418898508196953	0.9999736573290944	-	regulation of gene silencing by miRNA	0	P
GO:0090314	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of protein targeting to membrane	0	P
GO:0010803	1	3	0.9418898508196953	0.9999736573290944	-	regulation of tumor necrosis factor-mediated signaling pathway	0	P
GO:0010950	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of endopeptidase activity	0	P
GO:0097009	1	3	0.9418898508196953	0.9999736573290944	-	energy homeostasis	0	P
GO:0035970	1	3	0.9418898508196953	0.9999736573290944	-	peptidyl-threonine dephosphorylation	0	P
GO:0010575	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of vascular endothelial growth factor production	0	P
GO:0006956	1	3	0.9418898508196953	0.9999736573290944	-	complement activation	0	P
GO:0090050	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of cell migration involved in sprouting angiogenesis	0	P
GO:0071803	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of podosome assembly	0	P
GO:0070371	1	3	0.9418898508196953	0.9999736573290944	-	ERK1 and ERK2 cascade	0	P
GO:0000188	1	3	0.9418898508196953	0.9999736573290944	-	inactivation of MAPK activity	0	P
GO:0002774	1	3	0.9418898508196953	0.9999736573290944	-	Fc receptor mediated inhibitory signaling pathway	0	P
GO:0002523	1	3	0.9418898508196953	0.9999736573290944	-	leukocyte migration involved in inflammatory response	0	P
GO:0050772	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of axonogenesis	0	P
GO:0045786	1	3	0.9418898508196953	0.9999736573290944	-	negative regulation of cell cycle	0	P
GO:0045765	1	3	0.9418898508196953	0.9999736573290944	-	regulation of angiogenesis	0	P
GO:0051591	1	3	0.9418898508196953	0.9999736573290944	-	response to cAMP	0	P
GO:0045861	1	3	0.9418898508196953	0.9999736573290944	-	negative regulation of proteolysis	0	P
GO:0048261	1	3	0.9418898508196953	0.9999736573290944	-	negative regulation of receptor-mediated endocytosis	0	P
GO:0001895	1	3	0.9418898508196953	0.9999736573290944	-	retina homeostasis	0	P
GO:0048813	1	3	0.9418898508196953	0.9999736573290944	-	dendrite morphogenesis	0	P
GO:0006900	1	3	0.9418898508196953	0.9999736573290944	-	vesicle budding from membrane	0	P
GO:0007040	1	3	0.9418898508196953	0.9999736573290944	-	lysosome organization	0	P
GO:0050890	1	3	0.9418898508196953	0.9999736573290944	-	cognition	0	P
GO:0035094	1	3	0.9418898508196953	0.9999736573290944	-	response to nicotine	0	P
GO:0030574	1	3	0.9418898508196953	0.9999736573290944	-	collagen catabolic process	0	P
GO:0030301	1	3	0.9418898508196953	0.9999736573290944	-	cholesterol transport	0	P
GO:0002042	1	3	0.9418898508196953	0.9999736573290944	-	cell migration involved in sprouting angiogenesis	0	P
GO:0031065	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of histone deacetylation	0	P
GO:0046579	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of Ras protein signal transduction	0	P
GO:0050829	1	3	0.9418898508196953	0.9999736573290944	-	defense response to Gram-negative bacterium	0	P
GO:0045637	1	3	0.9418898508196953	0.9999736573290944	-	regulation of myeloid cell differentiation	0	P
GO:0045647	1	3	0.9418898508196953	0.9999736573290944	-	negative regulation of erythrocyte differentiation	0	P
GO:0006805	1	3	0.9418898508196953	0.9999736573290944	-	xenobiotic metabolic process	0	P
GO:0006139	1	3	0.9418898508196953	0.9999736573290944	-	nucleobase-containing compound metabolic process	0	P
GO:0001568	1	3	0.9418898508196953	0.9999736573290944	-	blood vessel development	0	P
GO:0007601	1	3	0.9418898508196953	0.9999736573290944	-	visual perception	0	P
GO:0008203	1	3	0.9418898508196953	0.9999736573290944	-	cholesterol metabolic process	0	P
GO:0043306	1	3	0.9418898508196953	0.9999736573290944	-	positive regulation of mast cell degranulation	0	P
GO:0008344	1	3	0.9418898508196953	0.9999736573290944	-	adult locomotory behavior	0	P
GO:0071345	2	5	0.9433670245011466	0.9999736573290944	-	cellular response to cytokine stimulus	0	P
GO:0050728	2	5	0.9433670245011466	0.9999736573290944	-	negative regulation of inflammatory response	0	P
GO:1901224	2	5	0.9433670245011466	0.9999736573290944	-	positive regulation of NIK/NF-kappaB signaling	0	P
GO:0001501	2	5	0.9433670245011466	0.9999736573290944	-	skeletal system development	0	P
GO:0007264	2	5	0.9433670245011466	0.9999736573290944	-	small GTPase mediated signal transduction	0	P
GO:0021987	2	5	0.9433670245011466	0.9999736573290944	-	cerebral cortex development	0	P
GO:0030177	2	5	0.9433670245011466	0.9999736573290944	-	positive regulation of Wnt signaling pathway	0	P
GO:0032930	2	5	0.9433670245011466	0.9999736573290944	-	positive regulation of superoxide anion generation	0	P
GO:0048662	2	5	0.9433670245011466	0.9999736573290944	-	negative regulation of smooth muscle cell proliferation	0	P
GO:0009409	2	5	0.9433670245011466	0.9999736573290944	-	response to cold	0	P
GO:0010033	2	5	0.9433670245011466	0.9999736573290944	-	response to organic substance	0	P
GO:0071407	2	5	0.9433670245011466	0.9999736573290944	-	cellular response to organic cyclic compound	0	P
GO:0071404	2	5	0.9433670245011466	0.9999736573290944	-	cellular response to low-density lipoprotein particle stimulus	0	P
GO:0034113	2	5	0.9433670245011466	0.9999736573290944	-	heterotypic cell-cell adhesion	0	P
GO:0006629	2	5	0.9433670245011466	0.9999736573290944	-	lipid metabolic process	0	P
GO:0042981	5	10	0.9492392681520511	0.9999736573290944	-	regulation of apoptotic process	0	P
GO:0050852	5	10	0.9492392681520511	0.9999736573290944	-	T cell receptor signaling pathway	0	P
GO:0006919	5	10	0.9492392681520511	0.9999736573290944	-	activation of cysteine-type endopeptidase activity involved in apoptotic process	0	P
GO:0006915	23	33	0.9496921159188183	0.9999736573290944	-	apoptotic process	0	P
GO:0022900	3	7	0.9517061493675614	0.9999736573290944	-	electron transport chain	0	P
GO:0042130	3	7	0.9517061493675614	0.9999736573290944	-	negative regulation of T cell proliferation	0	P
GO:0006457	3	7	0.9517061493675614	0.9999736573290944	-	protein folding	0	P
GO:0030307	3	7	0.9517061493675614	0.9999736573290944	-	positive regulation of cell growth	0	P
GO:0050776	7	13	0.952061687939604	0.9999736573290944	-	regulation of immune response	0	P
GO:0019886	9	16	0.9565679314147401	0.9999736573290944	-	antigen processing and presentation of exogenous peptide antigen via MHC class II	0	P
GO:0032496	6	12	0.9598481625205934	0.9999736573290944	-	response to lipopolysaccharide	0	P
GO:0031623	6	12	0.9598481625205934	0.9999736573290944	-	receptor internalization	0	P
GO:0045766	6	12	0.9598481625205934	0.9999736573290944	-	positive regulation of angiogenesis	0	P
GO:0098609	4	9	0.9603080779435565	0.9999736573290944	-	cell-cell adhesion	0	P
GO:0016579	5	11	0.9678426195991743	0.9999736573290944	-	protein deubiquitination	0	P
GO:0071346	5	11	0.9678426195991743	0.9999736573290944	-	cellular response to interferon-gamma	0	P
GO:0043123	7	14	0.9682077948972527	0.9999736573290944	-	positive regulation of I-kappaB kinase/NF-kappaB signaling	0	P
GO:0071364	2	6	0.9688185339175227	0.9999736573290944	-	cellular response to epidermal growth factor stimulus	0	P
GO:0031532	2	6	0.9688185339175227	0.9999736573290944	-	actin cytoskeleton reorganization	0	P
GO:0007623	2	6	0.9688185339175227	0.9999736573290944	-	circadian rhythm	0	P
GO:1990830	2	6	0.9688185339175227	0.9999736573290944	-	cellular response to leukemia inhibitory factor	0	P
GO:0032092	2	6	0.9688185339175227	0.9999736573290944	-	positive regulation of protein binding	0	P
GO:0071347	2	6	0.9688185339175227	0.9999736573290944	-	cellular response to interleukin-1	0	P
GO:0071456	2	6	0.9688185339175227	0.9999736573290944	-	cellular response to hypoxia	0	P
GO:0038096	9	17	0.9703219962242077	0.9999736573290944	-	Fc-gamma receptor signaling pathway involved in phagocytosis	0	P
GO:0046330	1	4	0.9715413638911704	0.9999736573290944	-	positive regulation of JNK cascade	0	P
GO:0002548	1	4	0.9715413638911704	0.9999736573290944	-	monocyte chemotaxis	0	P
GO:0050766	1	4	0.9715413638911704	0.9999736573290944	-	positive regulation of phagocytosis	0	P
GO:1990090	1	4	0.9715413638911704	0.9999736573290944	-	cellular response to nerve growth factor stimulus	0	P
GO:0050715	1	4	0.9715413638911704	0.9999736573290944	-	positive regulation of cytokine secretion	0	P
GO:0006367	1	4	0.9715413638911704	0.9999736573290944	-	transcription initiation from RNA polymerase II promoter	0	P
GO:0090200	1	4	0.9715413638911704	0.9999736573290944	-	positive regulation of release of cytochrome c from mitochondria	0	P
GO:0050860	1	4	0.9715413638911704	0.9999736573290944	-	negative regulation of T cell receptor signaling pathway	0	P
GO:0031397	1	4	0.9715413638911704	0.9999736573290944	-	negative regulation of protein ubiquitination	0	P
GO:0032715	1	4	0.9715413638911704	0.9999736573290944	-	negative regulation of interleukin-6 production	0	P
GO:0035924	1	4	0.9715413638911704	0.9999736573290944	-	cellular response to vascular endothelial growth factor stimulus	0	P
GO:0038066	1	4	0.9715413638911704	0.9999736573290944	-	p38MAPK cascade	0	P
GO:0016601	1	4	0.9715413638911704	0.9999736573290944	-	Rac protein signal transduction	0	P
GO:0031648	1	4	0.9715413638911704	0.9999736573290944	-	protein destabilization	0	P
GO:0097178	1	4	0.9715413638911704	0.9999736573290944	-	ruffle assembly	0	P
GO:0090023	1	4	0.9715413638911704	0.9999736573290944	-	positive regulation of neutrophil chemotaxis	0	P
GO:0043525	1	4	0.9715413638911704	0.9999736573290944	-	positive regulation of neuron apoptotic process	0	P
GO:0070372	1	4	0.9715413638911704	0.9999736573290944	-	regulation of ERK1 and ERK2 cascade	0	P
GO:0006801	1	4	0.9715413638911704	0.9999736573290944	-	superoxide metabolic process	0	P
GO:0006878	1	4	0.9715413638911704	0.9999736573290944	-	cellular copper ion homeostasis	0	P
GO:0008380	1	4	0.9715413638911704	0.9999736573290944	-	RNA splicing	0	P
GO:2000406	1	4	0.9715413638911704	0.9999736573290944	-	positive regulation of T cell migration	0	P
GO:0001774	1	4	0.9715413638911704	0.9999736573290944	-	microglial cell activation	0	P
GO:0030316	1	4	0.9715413638911704	0.9999736573290944	-	osteoclast differentiation	0	P
GO:0045429	1	4	0.9715413638911704	0.9999736573290944	-	positive regulation of nitric oxide biosynthetic process	0	P
GO:0032760	3	8	0.9717471007450044	0.9999736573290944	-	positive regulation of tumor necrosis factor production	0	P
GO:0007420	3	8	0.9717471007450044	0.9999736573290944	-	brain development	0	P
GO:0019216	3	8	0.9717471007450044	0.9999736573290944	-	regulation of lipid metabolic process	0	P
GO:0051092	6	13	0.9741186468435702	0.9999736573290944	-	positive regulation of NF-kappaB transcription factor activity	0	P
GO:0046777	6	13	0.9741186468435702	0.9999736573290944	-	protein autophosphorylation	0	P
GO:0043066	20	32	0.9761080703584478	0.9999736573290944	-	negative regulation of apoptotic process	0	P
GO:0009615	5	12	0.9799627061337985	0.9999736573290944	-	response to virus	0	P
GO:0045600	2	7	0.9830625302150375	0.9999736573290944	-	positive regulation of fat cell differentiation	0	P
GO:0071902	2	7	0.9830625302150375	0.9999736573290944	-	positive regulation of protein serine/threonine kinase activity	0	P
GO:0014070	2	7	0.9830625302150375	0.9999736573290944	-	response to organic cyclic compound	0	P
GO:0045471	2	7	0.9830625302150375	0.9999736573290944	-	response to ethanol	0	P
GO:0007568	6	14	0.9835716292309843	0.9999736573290944	-	aging	0	P
GO:0031295	3	9	0.983729152492651	0.9999736573290944	-	T cell costimulation	0	P
GO:0032088	3	9	0.983729152492651	0.9999736573290944	-	negative regulation of NF-kappaB transcription factor activity	0	P
GO:0030593	4	11	0.985584039705118	0.9999736573290944	-	neutrophil chemotaxis	0	P
GO:0044344	1	5	0.9860769882050736	0.9999736573290944	-	cellular response to fibroblast growth factor stimulus	0	P
GO:0000186	1	5	0.9860769882050736	0.9999736573290944	-	activation of MAPKK activity	0	P
GO:0071276	1	5	0.9860769882050736	0.9999736573290944	-	cellular response to cadmium ion	0	P
GO:1901216	1	5	0.9860769882050736	0.9999736573290944	-	positive regulation of neuron death	0	P
GO:0045071	1	5	0.9860769882050736	0.9999736573290944	-	negative regulation of viral genome replication	0	P
GO:0043433	1	5	0.9860769882050736	0.9999736573290944	-	negative regulation of DNA-binding transcription factor activity	0	P
GO:2000484	1	5	0.9860769882050736	0.9999736573290944	-	positive regulation of interleukin-8 secretion	0	P
GO:2000778	1	5	0.9860769882050736	0.9999736573290944	-	positive regulation of interleukin-6 secretion	0	P
GO:0034976	1	5	0.9860769882050736	0.9999736573290944	-	response to endoplasmic reticulum stress	0	P
GO:0007611	1	5	0.9860769882050736	0.9999736573290944	-	learning or memory	0	P
GO:0034341	1	5	0.9860769882050736	0.9999736573290944	-	response to interferon-gamma	0	P
GO:2001243	1	5	0.9860769882050736	0.9999736573290944	-	negative regulation of intrinsic apoptotic signaling pathway	0	P
GO:0006094	1	5	0.9860769882050736	0.9999736573290944	-	gluconeogenesis	0	P
GO:0006954	22	37	0.9888804032556696	0.9999736573290944	-	inflammatory response	0	P
GO:0006935	3	10	0.9907554748573794	0.9999736573290944	-	chemotaxis	0	P
GO:0070373	2	8	0.9909016254269066	0.9999736573290944	-	negative regulation of ERK1 and ERK2 cascade	0	P
GO:0030838	2	8	0.9909016254269066	0.9999736573290944	-	positive regulation of actin filament polymerization	0	P
GO:0042742	4	12	0.9914978464070993	0.9999736573290944	-	defense response to bacterium	0	P
GO:0051607	4	12	0.9914978464070993	0.9999736573290944	-	defense response to virus	0	P
GO:0050832	1	6	0.9931953154890615	0.9999736573290944	-	defense response to fungus	0	P
GO:0002755	1	6	0.9931953154890615	0.9999736573290944	-	MyD88-dependent toll-like receptor signaling pathway	0	P
GO:0032922	1	6	0.9931953154890615	0.9999736573290944	-	circadian regulation of gene expression	0	P
GO:0007165	38	59	0.9942329568753095	0.9999736573290944	-	signal transduction	0	P
GO:0010629	3	11	0.9948091223754919	0.9999736573290944	-	negative regulation of gene expression	0	P
GO:0016032	12	26	0.9954781680832236	0.9999736573290944	-	viral process	0	P
GO:0032720	1	7	0.9966777128564241	0.9999736573290944	-	negative regulation of tumor necrosis factor production	0	P
GO:0051603	1	7	0.9966777128564241	0.9999736573290944	-	proteolysis involved in cellular protein catabolic process	0	P
GO:0050718	1	7	0.9966777128564241	0.9999736573290944	-	positive regulation of interleukin-1 beta secretion	0	P
GO:0042102	3	12	0.9971153930430854	0.9999736573290944	-	positive regulation of T cell proliferation	0	P
GO:0006952	2	10	0.997443993262265	0.9999736573290944	-	defense response	0	P
GO:0006968	2	10	0.997443993262265	0.9999736573290944	-	cellular defense response	0	P
GO:0032755	1	8	0.9983796106865974	0.9999736573290944	-	positive regulation of interleukin-6 production	0	P
GO:0050729	1	8	0.9983796106865974	0.9999736573290944	-	positive regulation of inflammatory response	0	P
GO:0043312	52	81	0.9986572213336987	0.9999736573290944	-	neutrophil degranulation	0	P
GO:0034220	2	11	0.9986600875176987	0.9999736573290944	-	ion transmembrane transport	0	P
GO:0006508	5	18	0.9991035087338992	0.9999736573290944	-	proteolysis	0	P
GO:0060333	1	9	0.9992104979376742	0.9999736573290944	-	interferon-gamma-mediated signaling pathway	0	P
GO:0032729	1	9	0.9992104979376742	0.9999736573290944	-	positive regulation of interferon-gamma production	0	P
GO:0071222	5	19	0.9994843726571336	0.9999736573290944	-	cellular response to lipopolysaccharide	0	P
GO:0045087	12	32	0.99966891550062	0.9999736573290944	-	innate immune response	0	P
GO:0008286	1	11	0.9998131596140217	0.9999736573290944	-	insulin receptor signaling pathway	0	P
GO:0019221	10	32	0.9999337519079993	0.9999736573290944	-	cytokine-mediated signaling pathway	0	P
GO:0006955	9	32	0.9999736573290944	0.9999736573290944	-	immune response	0	P
GO:0008201	1	10	0.011213047910295617	0.022426095820591234	*	heparin binding	1	F
GO:0005509	1	52	0.05402650356778797	0.05402650356778797	-	calcium ion binding	1	F
GO:0005925	1	78	0.08053007135575943	0.14475025484199797	-	focal adhesion	1	C
GO:0005576	1	141	0.14475025484199797	0.14475025484199797	-	extracellular region	1	C
GO:0007155	1	37	0.038735983690112136	0.038735983690112136	*	cell adhesion	1	P
GO:0031072	2	4	0.0031621087182835406	0.09277964280838723	-	heat shock protein binding	10	F
GO:0051082	2	5	0.004387818780022021	0.09277964280838723	-	unfolded protein binding	10	F
GO:0051087	2	6	0.005798727675524202	0.09277964280838723	-	chaperone binding	10	F
GO:0004222	1	1	0.03036260375709917	0.1121080754108277	-	metalloendopeptidase activity	10	F
GO:0005528	1	1	0.03036260375709917	0.1121080754108277	-	FK506 binding	10	F
GO:0008035	1	1	0.03036260375709917	0.1121080754108277	-	high-density lipoprotein particle binding	10	F
GO:0019838	1	1	0.03036260375709917	0.1121080754108277	-	growth factor binding	10	F
GO:0005138	1	1	0.03036260375709917	0.1121080754108277	-	interleukin-6 receptor binding	10	F
GO:0036459	1	1	0.03036260375709917	0.1121080754108277	-	thiol-dependent ubiquitinyl hydrolase activity	10	F
GO:0043559	1	1	0.03036260375709917	0.1121080754108277	-	insulin binding	10	F
GO:0051213	1	1	0.03036260375709917	0.1121080754108277	-	dioxygenase activity	10	F
GO:0050811	1	1	0.03036260375709917	0.1121080754108277	-	GABA receptor binding	10	F
GO:0034185	1	1	0.03036260375709917	0.1121080754108277	-	apolipoprotein binding	10	F
GO:0003755	1	2	0.04521915221843064	0.15123291853109125	-	peptidyl-prolyl cis-trans isomerase activity	10	F
GO:0000976	1	3	0.05986303025189029	0.15123291853109125	-	transcription regulatory region sequence-specific DNA binding	10	F
GO:0001530	1	3	0.05986303025189029	0.15123291853109125	-	lipopolysaccharide binding	10	F
GO:0004843	1	3	0.05986303025189029	0.15123291853109125	-	thiol-dependent ubiquitin-specific protease activity	10	F
GO:0005507	1	3	0.05986303025189029	0.15123291853109125	-	copper ion binding	10	F
GO:0019955	1	3	0.05986303025189029	0.15123291853109125	-	cytokine binding	10	F
GO:0004896	1	4	0.07429706765846311	0.1573761056826649	-	cytokine receptor activity	10	F
GO:0004435	1	4	0.07429706765846311	0.1573761056826649	-	phosphatidylinositol phospholipase C activity	10	F
GO:0003697	1	5	0.08852405944649902	0.1573761056826649	-	single-stranded DNA binding	10	F
GO:0001671	1	5	0.08852405944649902	0.1573761056826649	-	ATPase activator activity	10	F
GO:0050681	1	5	0.08852405944649902	0.1573761056826649	-	androgen receptor binding	10	F
GO:0016887	1	5	0.08852405944649902	0.1573761056826649	-	ATPase activity	10	F
GO:0008234	1	5	0.08852405944649902	0.1573761056826649	-	cysteine-type peptidase activity	10	F
GO:0002039	1	5	0.08852405944649902	0.1573761056826649	-	p53 binding	10	F
GO:0000979	1	6	0.1025467662242452	0.17480592152563662	-	RNA polymerase II core promoter sequence-specific DNA binding	10	F
GO:0003725	1	7	0.11636791458834816	0.17480592152563662	-	double-stranded RNA binding	10	F
GO:0005506	1	7	0.11636791458834816	0.17480592152563662	-	iron ion binding	10	F
GO:0031490	1	7	0.11636791458834816	0.17480592152563662	-	chromatin DNA binding	10	F
GO:0005524	3	77	0.11653728101709107	0.17480592152563662	-	ATP binding	10	F
GO:0004197	1	9	0.14341627470730847	0.20860549048335778	-	cysteine-type endopeptidase activity	10	F
GO:0004252	1	10	0.15664877303829752	0.22115120899524357	-	serine-type endopeptidase activity	10	F
GO:0047485	1	12	0.18254337839200957	0.25034520465189886	-	protein N-terminus binding	10	F
GO:0002020	1	17	0.24406997648006623	0.325426635306755	-	protease binding	10	F
GO:0004672	1	19	0.2674478423119434	0.34695936299927793	-	protein kinase activity	10	F
GO:0004674	1	24	0.32297389935933263	0.40796703076968327	-	protein serine/threonine kinase activity	10	F
GO:0031625	1	28	0.3645391777788338	0.4486636034201032	-	ubiquitin protein ligase binding	10	F
GO:0019899	1	31	0.3941365406638629	0.47296384879663544	-	enzyme binding	10	F
GO:0005515	11	645	0.4095764189970067	0.47950410028917856	-	protein binding	10	F
GO:0003700	1	41	0.48376383557823377	0.5528729549465529	-	DNA-binding transcription factor activity	10	F
GO:0003677	1	49	0.5463842858434426	0.6099173423368661	-	DNA binding	10	F
GO:0046982	1	52	0.5679813346318819	0.6196160014165984	-	protein heterodimerization activity	10	F
GO:0003723	1	64	0.6451339918528537	0.6881429246430439	-	RNA binding	10	F
GO:0042803	1	78	0.7188427010524024	0.7500967315329417	-	protein homodimerization activity	10	F
GO:0042802	1	91	0.7742486502796946	0.789128152330689	-	identical protein binding	10	F
GO:0046872	1	95	0.789128152330689	0.789128152330689	-	metal ion binding	10	F
GO:0000785	1	1	0.03036260375709917	0.28841278000568965	-	chromatin	10	C
GO:0030061	1	1	0.03036260375709917	0.28841278000568965	-	mitochondrial crista	10	C
GO:0046696	1	1	0.03036260375709917	0.28841278000568965	-	lipopolysaccharide receptor complex	10	C
GO:0042588	1	1	0.03036260375709917	0.28841278000568965	-	zymogen granule	10	C
GO:0005795	1	2	0.04521915221843064	0.28841278000568965	-	Golgi stack	10	C
GO:0031305	1	2	0.04521915221843064	0.28841278000568965	-	integral component of mitochondrial inner membrane	10	C
GO:0030016	1	3	0.05986303025189029	0.28841278000568965	-	myofibril	10	C
GO:0005694	1	3	0.05986303025189029	0.28841278000568965	-	chromosome	10	C
GO:0031514	1	4	0.07429706765846311	0.28841278000568965	-	motile cilium	10	C
GO:0097546	1	4	0.07429706765846311	0.28841278000568965	-	ciliary base	10	C
GO:0000784	1	5	0.08852405944649902	0.28841278000568965	-	nuclear chromosome, telomeric region	10	C
GO:0005791	1	5	0.08852405944649902	0.28841278000568965	-	rough endoplasmic reticulum	10	C
GO:0030141	1	6	0.1025467662242452	0.28841278000568965	-	secretory granule	10	C
GO:0005905	1	6	0.1025467662242452	0.28841278000568965	-	clathrin-coated pit	10	C
GO:0016363	1	6	0.1025467662242452	0.28841278000568965	-	nuclear matrix	10	C
GO:0016605	1	6	0.1025467662242452	0.28841278000568965	-	PML body	10	C
GO:0005794	3	77	0.11653728101709107	0.3084810379864175	-	Golgi apparatus	10	C
GO:0016020	5	179	0.13204844212853167	0.3301211053213292	-	membrane	10	C
GO:0015630	1	11	0.16969028685729287	0.4018980478199041	-	microtubule cytoskeleton	10	C
GO:0043235	1	12	0.18254337839200957	0.41072260138202155	-	receptor complex	10	C
GO:0005759	1	15	0.21999726728450597	0.44574640385323905	-	mitochondrial matrix	10	C
GO:0005654	4	164	0.24620982872633768	0.44574640385323905	-	nucleoplasm	10	C
GO:0005743	1	18	0.2558445874383207	0.44574640385323905	-	mitochondrial inner membrane	10	C
GO:0043209	1	18	0.2558445874383207	0.44574640385323905	-	myelin sheath	10	C
GO:0005739	2	64	0.2674108936612954	0.44574640385323905	-	mitochondrion	10	C
GO:0031410	1	19	0.2674478423119434	0.44574640385323905	-	cytoplasmic vesicle	10	C
GO:0005788	1	19	0.2674478423119434	0.44574640385323905	-	endoplasmic reticulum lumen	10	C
GO:0005769	1	23	0.3121932289669652	0.4844377690866702	-	early endosome	10	C
GO:0030425	1	23	0.3121932289669652	0.4844377690866702	-	dendrite	10	C
GO:0009897	1	26	0.3440637614617014	0.5160956421925521	-	external side of plasma membrane	10	C
GO:0005737	6	316	0.3646446950129718	0.5293229443736688	-	cytoplasm	10	C
GO:0043025	1	32	0.4037128861749714	0.5677212461835536	-	neuronal cell body	10	C
GO:0005789	1	44	0.5081360454991279	0.6832683211618693	-	endoplasmic reticulum membrane	10	C
GO:0005730	1	47	0.5314309164592317	0.6832683211618693	-	nucleolus	10	C
GO:0045121	1	47	0.5314309164592317	0.6832683211618693	-	membrane raft	10	C
GO:0032991	1	50	0.553692809702033	0.6921160121275414	-	protein-containing complex	10	C
GO:0000139	1	57	0.6018567306914743	0.7319879157058471	-	Golgi membrane	10	C
GO:0070062	4	265	0.6240896905460872	0.7390535809098401	-	extracellular exosome	10	C
GO:0009986	1	68	0.6678500518207966	0.7705962136393806	-	cell surface	10	C
GO:0005634	4	298	0.7273672940728579	0.8182882058319652	-	nucleus	10	C
GO:0005615	1	100	0.8064408910045056	0.8851180511025062	-	extracellular space	10	C
GO:0005576	1	141	0.9059320247580107	0.9205261403615792	-	extracellular region	10	C
GO:0016021	1	144	0.9108935332857022	0.9205261403615792	-	integral component of membrane	10	C
GO:0005886	3	322	0.9198451872030246	0.9205261403615792	-	plasma membrane	10	C
GO:0005829	4	396	0.9205261403615792	0.9205261403615792	-	cytosol	10	C
GO:0042026	2	3	0.002126880625599014	0.145249398034959	-	protein refolding	10	P
GO:0016579	2	12	0.017869782097011636	0.145249398034959	-	protein deubiquitination	10	P
GO:0051604	1	1	0.03036260375709917	0.145249398034959	-	protein maturation	10	P
GO:0006458	1	1	0.03036260375709917	0.145249398034959	-	'de novo' protein folding	10	P
GO:0050859	1	1	0.03036260375709917	0.145249398034959	-	negative regulation of B cell receptor signaling pathway	10	P
GO:0048711	1	1	0.03036260375709917	0.145249398034959	-	positive regulation of astrocyte differentiation	10	P
GO:0043517	1	1	0.03036260375709917	0.145249398034959	-	positive regulation of DNA damage response, signal transduction by p53 class mediator	10	P
GO:0051131	1	1	0.03036260375709917	0.145249398034959	-	chaperone-mediated protein complex assembly	10	P
GO:0033617	1	1	0.03036260375709917	0.145249398034959	-	mitochondrial respiratory chain complex IV assembly	10	P
GO:0033198	1	1	0.03036260375709917	0.145249398034959	-	response to ATP	10	P
GO:0042100	1	1	0.03036260375709917	0.145249398034959	-	B cell proliferation	10	P
GO:0006825	1	1	0.03036260375709917	0.145249398034959	-	copper ion transport	10	P
GO:0008156	1	1	0.03036260375709917	0.145249398034959	-	negative regulation of DNA replication	10	P
GO:0008637	1	1	0.03036260375709917	0.145249398034959	-	apoptotic mitochondrial changes	10	P
GO:0007214	1	1	0.03036260375709917	0.145249398034959	-	gamma-aminobutyric acid signaling pathway	10	P
GO:0090399	1	1	0.03036260375709917	0.145249398034959	-	replicative senescence	10	P
GO:0009408	1	1	0.03036260375709917	0.145249398034959	-	response to heat	10	P
GO:0000723	1	1	0.03036260375709917	0.145249398034959	-	telomere maintenance	10	P
GO:1903427	1	1	0.03036260375709917	0.145249398034959	-	negative regulation of reactive oxygen species biosynthetic process	10	P
GO:0071480	1	2	0.04521915221843064	0.145249398034959	-	cellular response to gamma radiation	10	P
GO:0036297	1	2	0.04521915221843064	0.145249398034959	-	interstrand cross-link repair	10	P
GO:0006997	1	2	0.04521915221843064	0.145249398034959	-	nucleus organization	10	P
GO:0032735	1	2	0.04521915221843064	0.145249398034959	-	positive regulation of interleukin-12 production	10	P
GO:0043032	1	2	0.04521915221843064	0.145249398034959	-	positive regulation of macrophage activation	10	P
GO:0032959	1	2	0.04521915221843064	0.145249398034959	-	inositol trisphosphate biosynthetic process	10	P
GO:0009755	1	2	0.04521915221843064	0.145249398034959	-	hormone-mediated signaling pathway	10	P
GO:2000036	1	2	0.04521915221843064	0.145249398034959	-	regulation of stem cell population maintenance	10	P
GO:0000413	1	2	0.04521915221843064	0.145249398034959	-	protein peptidyl-prolyl isomerization	10	P
GO:0002821	1	2	0.04521915221843064	0.145249398034959	-	positive regulation of adaptive immune response	10	P
GO:0005977	1	2	0.04521915221843064	0.145249398034959	-	glycogen metabolic process	10	P
GO:0098761	1	2	0.04521915221843064	0.145249398034959	-	cellular response to interleukin-7	10	P
GO:0031293	1	2	0.04521915221843064	0.145249398034959	-	membrane protein intracellular domain proteolysis	10	P
GO:1901799	1	2	0.04521915221843064	0.145249398034959	-	negative regulation of proteasomal protein catabolic process	10	P
GO:0008203	1	3	0.05986303025189029	0.15145171484225173	-	cholesterol metabolic process	10	P
GO:0010575	1	3	0.05986303025189029	0.15145171484225173	-	positive regulation of vascular endothelial growth factor production	10	P
GO:0030509	1	3	0.05986303025189029	0.15145171484225173	-	BMP signaling pathway	10	P
GO:0030521	1	3	0.05986303025189029	0.15145171484225173	-	androgen receptor signaling pathway	10	P
GO:0007601	1	3	0.05986303025189029	0.15145171484225173	-	visual perception	10	P
GO:0007040	1	3	0.05986303025189029	0.15145171484225173	-	lysosome organization	10	P
GO:0002931	1	3	0.05986303025189029	0.15145171484225173	-	response to ischemia	10	P
GO:0120163	1	3	0.05986303025189029	0.15145171484225173	-	negative regulation of cold-induced thermogenesis	10	P
GO:0032733	1	4	0.07429706765846311	0.15145171484225173	-	positive regulation of interleukin-10 production	10	P
GO:0042531	1	4	0.07429706765846311	0.15145171484225173	-	positive regulation of tyrosine phosphorylation of STAT protein	10	P
GO:0007005	1	4	0.07429706765846311	0.15145171484225173	-	mitochondrion organization	10	P
GO:0006986	1	4	0.07429706765846311	0.15145171484225173	-	response to unfolded protein	10	P
GO:0006878	1	4	0.07429706765846311	0.15145171484225173	-	cellular copper ion homeostasis	10	P
GO:0014823	1	4	0.07429706765846311	0.15145171484225173	-	response to activity	10	P
GO:0050870	1	4	0.07429706765846311	0.15145171484225173	-	positive regulation of T cell activation	10	P
GO:0061077	1	4	0.07429706765846311	0.15145171484225173	-	chaperone-mediated protein folding	10	P
GO:0042220	1	4	0.07429706765846311	0.15145171484225173	-	response to cocaine	10	P
GO:1900034	1	4	0.07429706765846311	0.15145171484225173	-	regulation of cellular response to heat	10	P
GO:0010613	1	4	0.07429706765846311	0.15145171484225173	-	positive regulation of cardiac muscle hypertrophy	10	P
GO:0042110	1	5	0.08852405944649902	0.15639250502214827	-	T cell activation	10	P
GO:0034976	1	5	0.08852405944649902	0.15639250502214827	-	response to endoplasmic reticulum stress	10	P
GO:0009409	1	5	0.08852405944649902	0.15639250502214827	-	response to cold	10	P
GO:0034605	1	5	0.08852405944649902	0.15639250502214827	-	cellular response to heat	10	P
GO:0043627	1	5	0.08852405944649902	0.15639250502214827	-	response to estrogen	10	P
GO:0006281	1	5	0.08852405944649902	0.15639250502214827	-	DNA repair	10	P
GO:2000778	1	5	0.08852405944649902	0.15639250502214827	-	positive regulation of interleukin-6 secretion	10	P
GO:1904469	1	5	0.08852405944649902	0.15639250502214827	-	positive regulation of tumor necrosis factor secretion	10	P
GO:0016032	2	34	0.10191756968289578	0.16723011107338448	-	viral process	10	P
GO:0002755	1	6	0.1025467662242452	0.16723011107338448	-	MyD88-dependent toll-like receptor signaling pathway	10	P
GO:0042542	1	6	0.1025467662242452	0.16723011107338448	-	response to hydrogen peroxide	10	P
GO:0006629	1	6	0.1025467662242452	0.16723011107338448	-	lipid metabolic process	10	P
GO:0006606	1	6	0.1025467662242452	0.16723011107338448	-	protein import into nucleus	10	P
GO:0032755	1	7	0.11636791458834816	0.16897258830636858	-	positive regulation of interleukin-6 production	10	P
GO:1990830	1	7	0.11636791458834816	0.16897258830636858	-	cellular response to leukemia inhibitory factor	10	P
GO:0006260	1	7	0.11636791458834816	0.16897258830636858	-	DNA replication	10	P
GO:0042113	1	7	0.11636791458834816	0.16897258830636858	-	B cell activation	10	P
GO:0045669	1	7	0.11636791458834816	0.16897258830636858	-	positive regulation of osteoblast differentiation	10	P
GO:0051384	1	7	0.11636791458834816	0.16897258830636858	-	response to glucocorticoid	10	P
GO:0034097	1	7	0.11636791458834816	0.16897258830636858	-	response to cytokine	10	P
GO:0007283	1	7	0.11636791458834816	0.16897258830636858	-	spermatogenesis	10	P
GO:0050729	1	8	0.1299901975083634	0.1813021175774542	-	positive regulation of inflammatory response	10	P
GO:0032781	1	8	0.1299901975083634	0.1813021175774542	-	positive regulation of ATPase activity	10	P
GO:0006457	1	8	0.1299901975083634	0.1813021175774542	-	protein folding	10	P
GO:0032729	1	9	0.14341627470730847	0.19743019635032075	-	positive regulation of interferon-gamma production	10	P
GO:0008284	2	43	0.14804041720402034	0.2011831310721302	-	positive regulation of cell population proliferation	10	P
GO:0018105	1	10	0.15664877303829752	0.2075596242757442	-	peptidyl-serine phosphorylation	10	P
GO:0009617	1	10	0.15664877303829752	0.2075596242757442	-	response to bacterium	10	P
GO:0006511	1	11	0.16969028685729287	0.22206383218361783	-	ubiquitin-dependent protein catabolic process	10	P
GO:0043066	2	48	0.1753790670676312	0.22670952572157205	-	negative regulation of apoptotic process	10	P
GO:0120162	1	13	0.19521057810700954	0.2463371580874168	-	positive regulation of cold-induced thermogenesis	10	P
GO:0043524	1	13	0.19521057810700954	0.2463371580874168	-	negative regulation of neuron apoptotic process	10	P
GO:0042102	1	14	0.20769438506501867	0.2501773274646816	-	positive regulation of T cell proliferation	10	P
GO:0007179	1	14	0.20769438506501867	0.2501773274646816	-	transforming growth factor beta receptor signaling pathway	10	P
GO:0006919	1	14	0.20769438506501867	0.2501773274646816	-	activation of cysteine-type endopeptidase activity involved in apoptotic process	10	P
GO:0050821	1	14	0.20769438506501867	0.2501773274646816	-	protein stabilization	10	P
GO:0032496	1	16	0.2321216620935552	0.27645950766198707	-	response to lipopolysaccharide	10	P
GO:0001666	1	17	0.24406997648006623	0.28121105985746764	-	response to hypoxia	10	P
GO:0046777	1	17	0.24406997648006623	0.28121105985746764	-	protein autophosphorylation	10	P
GO:0000226	1	17	0.24406997648006623	0.28121105985746764	-	microtubule cytoskeleton organization	10	P
GO:0042493	1	18	0.2558445874383207	0.29160780933830105	-	response to drug	10	P
GO:0007275	1	19	0.2674478423119434	0.2953069925527709	-	multicellular organism development	10	P
GO:0006974	1	19	0.2674478423119434	0.2953069925527709	-	cellular response to DNA damage stimulus	10	P
GO:0044267	1	19	0.2674478423119434	0.2953069925527709	-	cellular protein metabolic process	10	P
GO:0006508	1	22	0.30125250620397953	0.3292037696662044	-	proteolysis	10	P
GO:0043687	1	24	0.32297389935933263	0.3493391156335639	-	post-translational protein modification	10	P
GO:0043065	1	26	0.3440637614617014	0.36839150217111466	-	positive regulation of apoptotic process	10	P
GO:0035556	1	29	0.3745516907339993	0.393093853641623	-	intracellular signal transduction	10	P
GO:0008150	1	29	0.3745516907339993	0.393093853641623	-	biological_process	10	P
GO:0055114	1	35	0.43160112669937195	0.44852666107973954	-	oxidation-reduction process	10	P
GO:0045893	1	36	0.4406233310374771	0.4534570202909958	-	positive regulation of transcription, DNA-templated	10	P
GO:0006357	1	38	0.4582681726721669	0.4670810221466316	-	regulation of transcription by RNA polymerase II	10	P
GO:0019221	1	39	0.46689447565509395	0.471341089708952	-	cytokine-mediated signaling pathway	10	P
GO:0045944	1	68	0.6678500518207966	0.6678500518207966	-	positive regulation of transcription by RNA polymerase II	10	P
GO:0030246	10	1	0.0007456245442497631	0.2893023231689081	-	carbohydrate binding	11	F
GO:0001228	17	9	0.007903445129147259	0.8954691190585615	-	DNA-binding transcription activator activity, RNA polymerase II-specific	11	F
GO:0004197	8	2	0.012603650929812193	0.8954691190585615	-	cysteine-type endopeptidase activity	11	F
GO:0046961	6	1	0.01931746453269835	0.8954691190585615	-	proton-transporting ATPase activity, rotational mechanism	11	F
GO:0001664	7	2	0.025684296641731077	0.8954691190585615	-	G protein-coupled receptor binding	11	F
GO:0003700	24	19	0.026062256157111517	0.8954691190585615	-	DNA-binding transcription factor activity	11	F
GO:0004252	8	3	0.030052961908567037	0.8954691190585615	-	serine-type endopeptidase activity	11	F
GO:0000978	20	15	0.03043976787568557	0.8954691190585615	-	RNA polymerase II proximal promoter sequence-specific DNA binding	11	F
GO:0038023	11	6	0.035718085130336835	0.8954691190585615	-	signaling receptor activity	11	F
GO:0004888	6	2	0.0510697365619902	0.8954691190585615	-	transmembrane signaling receptor activity	11	F
GO:0001784	6	2	0.0510697365619902	0.8954691190585615	-	phosphotyrosine residue binding	11	F
GO:0008233	6	2	0.0510697365619902	0.8954691190585615	-	peptidase activity	11	F
GO:0042826	7	3	0.05620580330287516	0.8954691190585615	-	histone deacetylase binding	11	F
GO:0048365	7	3	0.05620580330287516	0.8954691190585615	-	Rac GTPase binding	11	F
GO:0001227	12	8	0.05807966386037942	0.8954691190585615	-	DNA-binding transcription repressor activity, RNA polymerase II-specific	11	F
GO:0017124	12	8	0.05807966386037942	0.8954691190585615	-	SH3 domain binding	11	F
GO:0001540	11	7	0.05924656278126912	0.8954691190585615	-	amyloid-beta binding	11	F
GO:0019864	4	1	0.08889826176013384	0.8954691190585615	-	IgG binding	11	F
GO:0035035	4	1	0.08889826176013384	0.8954691190585615	-	histone acetyltransferase binding	11	F
GO:0051019	4	1	0.08889826176013384	0.8954691190585615	-	mitogen-activated protein kinase binding	11	F
GO:0004869	4	1	0.08889826176013384	0.8954691190585615	-	cysteine-type endopeptidase inhibitor activity	11	F
GO:0050786	5	2	0.09850686585876353	0.8954691190585615	-	RAGE receptor binding	11	F
GO:0042605	6	3	0.10184061640250841	0.8954691190585615	-	peptide antigen binding	11	F
GO:0046934	6	3	0.10184061640250841	0.8954691190585615	-	phosphatidylinositol-4,5-bisphosphate 3-kinase activity	11	F
GO:0004175	6	3	0.10184061640250841	0.8954691190585615	-	endopeptidase activity	11	F
GO:0000981	32	35	0.12399216475895199	0.8954691190585615	-	DNA-binding transcription factor activity, RNA polymerase II-specific	11	F
GO:0000977	12	10	0.12435189113292845	0.8954691190585615	-	RNA polymerase II regulatory region sequence-specific DNA binding	11	F
GO:0005102	12	10	0.12435189113292845	0.8954691190585615	-	signaling receptor binding	11	F
GO:0044877	14	13	0.149219134855131	0.8954691190585615	-	protein-containing complex binding	11	F
GO:0009055	8	6	0.15424665509514018	0.8954691190585615	-	electron transfer activity	11	F
GO:0008134	13	12	0.15858870198695282	0.8954691190585615	-	transcription factor binding	11	F
GO:0019899	17	17	0.16010721874507472	0.8954691190585615	-	enzyme binding	11	F
GO:0042803	39	46	0.16394380495782518	0.8954691190585615	-	protein homodimerization activity	11	F
GO:0005215	6	4	0.17029283605195825	0.8954691190585615	-	transporter activity	11	F
GO:0019900	6	4	0.17029283605195825	0.8954691190585615	-	kinase binding	11	F
GO:0005125	6	4	0.17029283605195825	0.8954691190585615	-	cytokine activity	11	F
GO:0044548	5	3	0.1775687480200524	0.8954691190585615	-	S100 protein binding	11	F
GO:0019903	5	3	0.1775687480200524	0.8954691190585615	-	protein phosphatase binding	11	F
GO:0043565	11	10	0.17965470107221398	0.8954691190585615	-	sequence-specific DNA binding	11	F
GO:0002020	11	10	0.17965470107221398	0.8954691190585615	-	protease binding	11	F
GO:0004708	3	1	0.1823133186217152	0.8954691190585615	-	MAP kinase kinase activity	11	F
GO:0005507	3	1	0.1823133186217152	0.8954691190585615	-	copper ion binding	11	F
GO:0005044	3	1	0.1823133186217152	0.8954691190585615	-	scavenger receptor activity	11	F
GO:0032395	3	1	0.1823133186217152	0.8954691190585615	-	MHC class II receptor activity	11	F
GO:0005041	3	1	0.1823133186217152	0.8954691190585615	-	low-density lipoprotein particle receptor activity	11	F
GO:1990841	3	1	0.1823133186217152	0.8954691190585615	-	promoter-specific chromatin binding	11	F
GO:0042288	3	1	0.1823133186217152	0.8954691190585615	-	MHC class I protein binding	11	F
GO:0071889	3	1	0.1823133186217152	0.8954691190585615	-	14-3-3 protein binding	11	F
GO:0030276	3	1	0.1823133186217152	0.8954691190585615	-	clathrin binding	11	F
GO:0004177	3	1	0.1823133186217152	0.8954691190585615	-	aminopeptidase activity	11	F
GO:0005154	3	1	0.1823133186217152	0.8954691190585615	-	epidermal growth factor receptor binding	11	F
GO:0004896	4	2	0.18280876974296753	0.8954691190585615	-	cytokine receptor activity	11	F
GO:0031267	4	2	0.18280876974296753	0.8954691190585615	-	small GTPase binding	11	F
GO:0043325	4	2	0.18280876974296753	0.8954691190585615	-	phosphatidylinositol-3,4-bisphosphate binding	11	F
GO:0008234	4	2	0.18280876974296753	0.8954691190585615	-	cysteine-type peptidase activity	11	F
GO:0051020	4	2	0.18280876974296753	0.8954691190585615	-	GTPase binding	11	F
GO:0003723	32	38	0.20305206998301284	0.8954691190585615	-	RNA binding	11	F
GO:0005515	290	413	0.20402578523201584	0.8954691190585615	-	protein binding	11	F
GO:0044212	7	6	0.23463591011498824	0.8954691190585615	-	transcription regulatory region DNA binding	11	F
GO:0004713	7	6	0.23463591011498824	0.8954691190585615	-	protein tyrosine kinase activity	11	F
GO:0005516	10	10	0.25179243143552615	0.8954691190585615	-	calmodulin binding	11	F
GO:0005546	6	5	0.2522763084228111	0.8954691190585615	-	phosphatidylinositol-4,5-bisphosphate binding	11	F
GO:0008270	21	25	0.2717923735487593	0.8954691190585615	-	zinc ion binding	11	F
GO:0023026	5	4	0.2722289125419825	0.8954691190585615	-	MHC class II protein complex binding	11	F
GO:0020037	5	4	0.2722289125419825	0.8954691190585615	-	heme binding	11	F
GO:0008289	5	4	0.2722289125419825	0.8954691190585615	-	lipid binding	11	F
GO:0032947	4	3	0.29521130825523967	0.8954691190585615	-	protein-containing complex scaffold activity	11	F
GO:0003729	4	3	0.29521130825523967	0.8954691190585615	-	mRNA binding	11	F
GO:0000149	4	3	0.29521130825523967	0.8954691190585615	-	SNARE binding	11	F
GO:0030674	4	3	0.29521130825523967	0.8954691190585615	-	protein binding, bridging	11	F
GO:0070491	4	3	0.29521130825523967	0.8954691190585615	-	repressing transcription factor binding	11	F
GO:0003714	7	7	0.3150251651348364	0.8954691190585615	-	transcription corepressor activity	11	F
GO:0017137	7	7	0.3150251651348364	0.8954691190585615	-	Rab GTPase binding	11	F
GO:0015459	3	2	0.3224359039140872	0.8954691190585615	-	potassium channel regulator activity	11	F
GO:0008191	3	2	0.3224359039140872	0.8954691190585615	-	metalloendopeptidase inhibitor activity	11	F
GO:0019902	3	2	0.3224359039140872	0.8954691190585615	-	phosphatase binding	11	F
GO:0004866	3	2	0.3224359039140872	0.8954691190585615	-	endopeptidase inhibitor activity	11	F
GO:0008656	3	2	0.3224359039140872	0.8954691190585615	-	cysteine-type endopeptidase activator activity involved in apoptotic process	11	F
GO:0016301	3	2	0.3224359039140872	0.8954691190585615	-	kinase activity	11	F
GO:0019955	3	2	0.3224359039140872	0.8954691190585615	-	cytokine binding	11	F
GO:0008144	3	2	0.3224359039140872	0.8954691190585615	-	drug binding	11	F
GO:0003677	22	28	0.34286218374094596	0.8954691190585615	-	DNA binding	11	F
GO:0031625	14	17	0.3510404263813546	0.8954691190585615	-	ubiquitin protein ligase binding	11	F
GO:0019904	14	17	0.3510404263813546	0.8954691190585615	-	protein domain specific binding	11	F
GO:0017056	2	1	0.3562932750014786	0.8954691190585615	-	structural constituent of nuclear pore	11	F
GO:0035662	2	1	0.3562932750014786	0.8954691190585615	-	Toll-like receptor 4 binding	11	F
GO:0050661	2	1	0.3562932750014786	0.8954691190585615	-	NADP binding	11	F
GO:0050544	2	1	0.3562932750014786	0.8954691190585615	-	arachidonic acid binding	11	F
GO:0031683	2	1	0.3562932750014786	0.8954691190585615	-	G-protein beta/gamma-subunit complex binding	11	F
GO:0031996	2	1	0.3562932750014786	0.8954691190585615	-	thioesterase binding	11	F
GO:0017017	2	1	0.3562932750014786	0.8954691190585615	-	MAP kinase tyrosine/serine/threonine phosphatase activity	11	F
GO:0004860	2	1	0.3562932750014786	0.8954691190585615	-	protein kinase inhibitor activity	11	F
GO:0004864	2	1	0.3562932750014786	0.8954691190585615	-	protein phosphatase inhibitor activity	11	F
GO:0017025	2	1	0.3562932750014786	0.8954691190585615	-	TBP-class protein binding	11	F
GO:0051022	2	1	0.3562932750014786	0.8954691190585615	-	Rho GDP-dissociation inhibitor binding	11	F
GO:0016810	2	1	0.3562932750014786	0.8954691190585615	-	hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds	11	F
GO:0051018	2	1	0.3562932750014786	0.8954691190585615	-	protein kinase A binding	11	F
GO:0019863	2	1	0.3562932750014786	0.8954691190585615	-	IgE binding	11	F
GO:0015631	2	1	0.3562932750014786	0.8954691190585615	-	tubulin binding	11	F
GO:0046875	2	1	0.3562932750014786	0.8954691190585615	-	ephrin receptor binding	11	F
GO:0046914	2	1	0.3562932750014786	0.8954691190585615	-	transition metal ion binding	11	F
GO:0015179	2	1	0.3562932750014786	0.8954691190585615	-	L-amino acid transmembrane transporter activity	11	F
GO:0042834	2	1	0.3562932750014786	0.8954691190585615	-	peptidoglycan binding	11	F
GO:0043394	2	1	0.3562932750014786	0.8954691190585615	-	proteoglycan binding	11	F
GO:0015035	2	1	0.3562932750014786	0.8954691190585615	-	protein disulfide oxidoreductase activity	11	F
GO:0034988	2	1	0.3562932750014786	0.8954691190585615	-	Fc-gamma receptor I complex binding	11	F
GO:0005262	2	1	0.3562932750014786	0.8954691190585615	-	calcium channel activity	11	F
GO:0140031	2	1	0.3562932750014786	0.8954691190585615	-	phosphorylation-dependent protein binding	11	F
GO:0089720	2	1	0.3562932750014786	0.8954691190585615	-	caspase binding	11	F
GO:0004185	2	1	0.3562932750014786	0.8954691190585615	-	serine-type carboxypeptidase activity	11	F
GO:0004190	2	1	0.3562932750014786	0.8954691190585615	-	aspartic-type endopeptidase activity	11	F
GO:0051219	2	1	0.3562932750014786	0.8954691190585615	-	phosphoprotein binding	11	F
GO:0097153	2	1	0.3562932750014786	0.8954691190585615	-	cysteine-type endopeptidase activity involved in apoptotic process	11	F
GO:0050700	2	1	0.3562932750014786	0.8954691190585615	-	CARD domain binding	11	F
GO:0035259	2	1	0.3562932750014786	0.8954691190585615	-	glucocorticoid receptor binding	11	F
GO:0030414	2	1	0.3562932750014786	0.8954691190585615	-	peptidase inhibitor activity	11	F
GO:0046983	5	5	0.3741649890320067	0.8954691190585615	-	protein dimerization activity	11	F
GO:0050839	5	5	0.3741649890320067	0.8954691190585615	-	cell adhesion molecule binding	11	F
GO:0004715	4	4	0.4128538684904268	0.8954691190585615	-	non-membrane spanning protein tyrosine kinase activity	11	F
GO:0008017	8	10	0.44697133472163975	0.8954691190585615	-	microtubule binding	11	F
GO:0042802	43	61	0.44921534999174434	0.8954691190585615	-	identical protein binding	11	F
GO:0019901	23	32	0.4595597560734912	0.8954691190585615	-	protein kinase binding	11	F
GO:0004674	12	16	0.46200688919745825	0.8954691190585615	-	protein serine/threonine kinase activity	11	F
GO:0002039	3	3	0.46206303808520693	0.8954691190585615	-	p53 binding	11	F
GO:0043236	3	3	0.46206303808520693	0.8954691190585615	-	laminin binding	11	F
GO:0031072	3	3	0.46206303808520693	0.8954691190585615	-	heat shock protein binding	11	F
GO:0004930	3	3	0.46206303808520693	0.8954691190585615	-	G protein-coupled receptor activity	11	F
GO:0030165	3	3	0.46206303808520693	0.8954691190585615	-	PDZ domain binding	11	F
GO:0051287	3	3	0.46206303808520693	0.8954691190585615	-	NAD binding	11	F
GO:0008198	3	3	0.46206303808520693	0.8954691190585615	-	ferrous iron binding	11	F
GO:0048306	6	8	0.524008101644768	0.8954691190585615	-	calcium-dependent protein binding	11	F
GO:0000287	6	8	0.524008101644768	0.8954691190585615	-	magnesium ion binding	11	F
GO:0004725	4	5	0.525353833249182	0.8954691190585615	-	protein tyrosine phosphatase activity	11	F
GO:0098641	4	5	0.525353833249182	0.8954691190585615	-	cadherin binding involved in cell-cell adhesion	11	F
GO:0051959	2	2	0.530273231381242	0.8954691190585615	-	dynein light intermediate chain binding	11	F
GO:0070412	2	2	0.530273231381242	0.8954691190585615	-	R-SMAD binding	11	F
GO:0003727	2	2	0.530273231381242	0.8954691190585615	-	single-stranded RNA binding	11	F
GO:0005246	2	2	0.530273231381242	0.8954691190585615	-	calcium channel regulator activity	11	F
GO:0003823	2	2	0.530273231381242	0.8954691190585615	-	antigen binding	11	F
GO:0042609	2	2	0.530273231381242	0.8954691190585615	-	CD4 receptor binding	11	F
GO:0008199	2	2	0.530273231381242	0.8954691190585615	-	ferric iron binding	11	F
GO:0019834	2	2	0.530273231381242	0.8954691190585615	-	phospholipase A2 inhibitor activity	11	F
GO:0004722	2	2	0.530273231381242	0.8954691190585615	-	protein serine/threonine phosphatase activity	11	F
GO:0008138	2	2	0.530273231381242	0.8954691190585615	-	protein tyrosine/serine/threonine phosphatase activity	11	F
GO:0001530	2	2	0.530273231381242	0.8954691190585615	-	lipopolysaccharide binding	11	F
GO:0004322	2	2	0.530273231381242	0.8954691190585615	-	ferroxidase activity	11	F
GO:0004407	2	2	0.530273231381242	0.8954691190585615	-	histone deacetylase activity	11	F
GO:0005523	2	2	0.530273231381242	0.8954691190585615	-	tropomyosin binding	11	F
GO:0005522	2	2	0.530273231381242	0.8954691190585615	-	profilin binding	11	F
GO:0043539	2	2	0.530273231381242	0.8954691190585615	-	protein serine/threonine kinase activator activity	11	F
GO:0042056	2	2	0.530273231381242	0.8954691190585615	-	chemoattractant activity	11	F
GO:0017048	5	7	0.5711060434595705	0.8954691190585615	-	Rho GTPase binding	11	F
GO:0042169	3	4	0.5872018354576825	0.8954691190585615	-	SH2 domain binding	11	F
GO:0005518	3	4	0.5872018354576825	0.8954691190585615	-	collagen binding	11	F
GO:0005088	3	4	0.5872018354576825	0.8954691190585615	-	Ras guanyl-nucleotide exchange factor activity	11	F
GO:0001085	3	4	0.5872018354576825	0.8954691190585615	-	RNA polymerase II transcription factor binding	11	F
GO:0000979	3	4	0.5872018354576825	0.8954691190585615	-	RNA polymerase II core promoter sequence-specific DNA binding	11	F
GO:0005544	3	4	0.5872018354576825	0.8954691190585615	-	calcium-dependent phospholipid binding	11	F
GO:0046332	3	4	0.5872018354576825	0.8954691190585615	-	SMAD binding	11	F
GO:0005509	23	35	0.5925313064395671	0.8954691190585615	-	calcium ion binding	11	F
GO:0005200	8	12	0.5962420267724065	0.8954691190585615	-	structural constituent of cytoskeleton	11	F
GO:0045296	15	23	0.6045459319333096	0.8954691190585615	-	cadherin binding	11	F
GO:0030145	4	6	0.6261463960606322	0.8954691190585615	-	manganese ion binding	11	F
GO:0051117	4	6	0.6261463960606322	0.8954691190585615	-	ATPase binding	11	F
GO:0005085	4	6	0.6261463960606322	0.8954691190585615	-	guanyl-nucleotide exchange factor activity	11	F
GO:0003730	4	6	0.6261463960606322	0.8954691190585615	-	mRNA 3'-UTR binding	11	F
GO:0005095	1	1	0.6439069180859244	0.8954691190585615	-	GTPase inhibitor activity	11	F
GO:0030395	1	1	0.6439069180859244	0.8954691190585615	-	lactose binding	11	F
GO:0030023	1	1	0.6439069180859244	0.8954691190585615	-	extracellular matrix constituent conferring elasticity	11	F
GO:0005355	1	1	0.6439069180859244	0.8954691190585615	-	glucose transmembrane transporter activity	11	F
GO:0005138	1	1	0.6439069180859244	0.8954691190585615	-	interleukin-6 receptor binding	11	F
GO:0050699	1	1	0.6439069180859244	0.8954691190585615	-	WW domain binding	11	F
GO:0048185	1	1	0.6439069180859244	0.8954691190585615	-	activin binding	11	F
GO:0005168	1	1	0.6439069180859244	0.8954691190585615	-	neurotrophin TRKA receptor binding	11	F
GO:0005086	1	1	0.6439069180859244	0.8954691190585615	-	ARF guanyl-nucleotide exchange factor activity	11	F
GO:0030332	1	1	0.6439069180859244	0.8954691190585615	-	cyclin binding	11	F
GO:0042277	1	1	0.6439069180859244	0.8954691190585615	-	peptide binding	11	F
GO:0042289	1	1	0.6439069180859244	0.8954691190585615	-	MHC class II protein binding	11	F
GO:0035605	1	1	0.6439069180859244	0.8954691190585615	-	peptidyl-cysteine S-nitrosylase activity	11	F
GO:0035718	1	1	0.6439069180859244	0.8954691190585615	-	macrophage migration inhibitory factor binding	11	F
GO:0036134	1	1	0.6439069180859244	0.8954691190585615	-	12-hydroxyheptadecatrienoic acid synthase activity	11	F
GO:0033558	1	1	0.6439069180859244	0.8954691190585615	-	protein deacetylase activity	11	F
GO:0033691	1	1	0.6439069180859244	0.8954691190585615	-	sialic acid binding	11	F
GO:0031691	1	1	0.6439069180859244	0.8954691190585615	-	alpha-1A adrenergic receptor binding	11	F
GO:0008190	1	1	0.6439069180859244	0.8954691190585615	-	eukaryotic initiation factor 4E binding	11	F
GO:0008035	1	1	0.6439069180859244	0.8954691190585615	-	high-density lipoprotein particle binding	11	F
GO:0008073	1	1	0.6439069180859244	0.8954691190585615	-	ornithine decarboxylase inhibitor activity	11	F
GO:0008080	1	1	0.6439069180859244	0.8954691190585615	-	N-acetyltransferase activity	11	F
GO:0008121	1	1	0.6439069180859244	0.8954691190585615	-	ubiquinol-cytochrome-c reductase activity	11	F
GO:0005521	1	1	0.6439069180859244	0.8954691190585615	-	lamin binding	11	F
GO:0005527	1	1	0.6439069180859244	0.8954691190585615	-	macrolide binding	11	F
GO:0031434	1	1	0.6439069180859244	0.8954691190585615	-	mitogen-activated protein kinase kinase binding	11	F
GO:0034185	1	1	0.6439069180859244	0.8954691190585615	-	apolipoprotein binding	11	F
GO:0031692	1	1	0.6439069180859244	0.8954691190585615	-	alpha-1B adrenergic receptor binding	11	F
GO:0031701	1	1	0.6439069180859244	0.8954691190585615	-	angiotensin receptor binding	11	F
GO:0030957	1	1	0.6439069180859244	0.8954691190585615	-	Tat protein binding	11	F
GO:0031404	1	1	0.6439069180859244	0.8954691190585615	-	chloride ion binding	11	F
GO:0030169	1	1	0.6439069180859244	0.8954691190585615	-	low-density lipoprotein particle binding	11	F
GO:0030170	1	1	0.6439069180859244	0.8954691190585615	-	pyridoxal phosphate binding	11	F
GO:0030296	1	1	0.6439069180859244	0.8954691190585615	-	protein tyrosine kinase activator activity	11	F
GO:0072542	1	1	0.6439069180859244	0.8954691190585615	-	protein phosphatase activator activity	11	F
GO:0044183	1	1	0.6439069180859244	0.8954691190585615	-	protein folding chaperone	11	F
GO:0044730	1	1	0.6439069180859244	0.8954691190585615	-	bone sialoprotein binding	11	F
GO:0043395	1	1	0.6439069180859244	0.8954691190585615	-	heparan sulfate proteoglycan binding	11	F
GO:0042658	1	1	0.6439069180859244	0.8954691190585615	-	MHC class II protein binding, via antigen binding groove	11	F
GO:0043015	1	1	0.6439069180859244	0.8954691190585615	-	gamma-tubulin binding	11	F
GO:0043120	1	1	0.6439069180859244	0.8954691190585615	-	tumor necrosis factor binding	11	F
GO:0051059	1	1	0.6439069180859244	0.8954691190585615	-	NF-kappaB binding	11	F
GO:0072320	1	1	0.6439069180859244	0.8954691190585615	-	volume-sensitive chloride channel activity	11	F
GO:0046790	1	1	0.6439069180859244	0.8954691190585615	-	virion binding	11	F
GO:0097199	1	1	0.6439069180859244	0.8954691190585615	-	cysteine-type endopeptidase activity involved in apoptotic signaling pathway	11	F
GO:0098973	1	1	0.6439069180859244	0.8954691190585615	-	structural constituent of postsynaptic actin cytoskeleton	11	F
GO:0099511	1	1	0.6439069180859244	0.8954691190585615	-	voltage-gated calcium channel activity involved in regulation of cytosolic calcium levels	11	F
GO:0102121	1	1	0.6439069180859244	0.8954691190585615	-	ceramidase activity	11	F
GO:1905538	1	1	0.6439069180859244	0.8954691190585615	-	polysome binding	11	F
GO:0070324	1	1	0.6439069180859244	0.8954691190585615	-	thyroid hormone binding	11	F
GO:0070700	1	1	0.6439069180859244	0.8954691190585615	-	BMP receptor binding	11	F
GO:0004796	1	1	0.6439069180859244	0.8954691190585615	-	thromboxane-A synthase activity	11	F
GO:0031726	1	1	0.6439069180859244	0.8954691190585615	-	CCR1 chemokine receptor binding	11	F
GO:0031730	1	1	0.6439069180859244	0.8954691190585615	-	CCR5 chemokine receptor binding	11	F
GO:0031748	1	1	0.6439069180859244	0.8954691190585615	-	D1 dopamine receptor binding	11	F
GO:0031762	1	1	0.6439069180859244	0.8954691190585615	-	follicle-stimulating hormone receptor binding	11	F
GO:0031802	1	1	0.6439069180859244	0.8954691190585615	-	type 5 metabotropic glutamate receptor binding	11	F
GO:0032041	1	1	0.6439069180859244	0.8954691190585615	-	NAD-dependent histone deacetylase activity (H3-K14 specific)	11	F
GO:0005031	1	1	0.6439069180859244	0.8954691190585615	-	tumor necrosis factor-activated receptor activity	11	F
GO:0005068	1	1	0.6439069180859244	0.8954691190585615	-	transmembrane receptor protein tyrosine kinase adaptor activity	11	F
GO:0030955	1	1	0.6439069180859244	0.8954691190585615	-	potassium ion binding	11	F
GO:0004801	1	1	0.6439069180859244	0.8954691190585615	-	sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase activity	11	F
GO:0004727	1	1	0.6439069180859244	0.8954691190585615	-	prenylated protein tyrosine phosphatase activity	11	F
GO:0004497	1	1	0.6439069180859244	0.8954691190585615	-	monooxygenase activity	11	F
GO:0004623	1	1	0.6439069180859244	0.8954691190585615	-	phospholipase A2 activity	11	F
GO:0004629	1	1	0.6439069180859244	0.8954691190585615	-	phospholipase C activity	11	F
GO:0004698	1	1	0.6439069180859244	0.8954691190585615	-	calcium-dependent protein kinase C activity	11	F
GO:0045503	1	1	0.6439069180859244	0.8954691190585615	-	dynein light chain binding	11	F
GO:0016004	1	1	0.6439069180859244	0.8954691190585615	-	phospholipase activator activity	11	F
GO:0004356	1	1	0.6439069180859244	0.8954691190585615	-	glutamate-ammonia ligase activity	11	F
GO:0004364	1	1	0.6439069180859244	0.8954691190585615	-	glutathione transferase activity	11	F
GO:0004365	1	1	0.6439069180859244	0.8954691190585615	-	glyceraldehyde-3-phosphate dehydrogenase (NAD+) (phosphorylating) activity	11	F
GO:0016505	1	1	0.6439069180859244	0.8954691190585615	-	peptidase activator activity involved in apoptotic process	11	F
GO:0016595	1	1	0.6439069180859244	0.8954691190585615	-	glutamate binding	11	F
GO:0016705	1	1	0.6439069180859244	0.8954691190585615	-	oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen	11	F
GO:0016787	1	1	0.6439069180859244	0.8954691190585615	-	hydrolase activity	11	F
GO:0015293	1	1	0.6439069180859244	0.8954691190585615	-	symporter activity	11	F
GO:0048037	1	1	0.6439069180859244	0.8954691190585615	-	cofactor binding	11	F
GO:0016209	1	1	0.6439069180859244	0.8954691190585615	-	antioxidant activity	11	F
GO:0008432	1	1	0.6439069180859244	0.8954691190585615	-	JUN kinase binding	11	F
GO:0008474	1	1	0.6439069180859244	0.8954691190585615	-	palmitoyl-(protein) hydrolase activity	11	F
GO:0008553	1	1	0.6439069180859244	0.8954691190585615	-	proton-exporting ATPase activity, phosphorylative mechanism	11	F
GO:0010314	1	1	0.6439069180859244	0.8954691190585615	-	phosphatidylinositol-5-phosphate binding	11	F
GO:0008301	1	1	0.6439069180859244	0.8954691190585615	-	DNA binding, bending	11	F
GO:0008330	1	1	0.6439069180859244	0.8954691190585615	-	protein tyrosine/threonine phosphatase activity	11	F
GO:0004198	1	1	0.6439069180859244	0.8954691190585615	-	calcium-dependent cysteine-type endopeptidase activity	11	F
GO:0070006	1	1	0.6439069180859244	0.8954691190585615	-	metalloaminopeptidase activity	11	F
GO:0070034	1	1	0.6439069180859244	0.8954691190585615	-	telomerase RNA binding	11	F
GO:0070053	1	1	0.6439069180859244	0.8954691190585615	-	thrombospondin receptor activity	11	F
GO:0004145	1	1	0.6439069180859244	0.8954691190585615	-	diamine N-acetyltransferase activity	11	F
GO:0004180	1	1	0.6439069180859244	0.8954691190585615	-	carboxypeptidase activity	11	F
GO:0070063	1	1	0.6439069180859244	0.8954691190585615	-	RNA polymerase binding	11	F
GO:0070273	1	1	0.6439069180859244	0.8954691190585615	-	phosphatidylinositol-4-phosphate binding	11	F
GO:0003796	1	1	0.6439069180859244	0.8954691190585615	-	lysozyme activity	11	F
GO:0015171	1	1	0.6439069180859244	0.8954691190585615	-	amino acid transmembrane transporter activity	11	F
GO:0001882	1	1	0.6439069180859244	0.8954691190585615	-	nucleoside binding	11	F
GO:0001055	1	1	0.6439069180859244	0.8954691190585615	-	RNA polymerase II activity	11	F
GO:0001222	1	1	0.6439069180859244	0.8954691190585615	-	transcription corepressor binding	11	F
GO:0000339	1	1	0.6439069180859244	0.8954691190585615	-	RNA cap binding	11	F
GO:0000987	1	1	0.6439069180859244	0.8954691190585615	-	proximal promoter sequence-specific DNA binding	11	F
GO:0048020	1	1	0.6439069180859244	0.8954691190585615	-	CCR chemokine receptor binding	11	F
GO:0048029	1	1	0.6439069180859244	0.8954691190585615	-	monosaccharide binding	11	F
GO:0019894	1	1	0.6439069180859244	0.8954691190585615	-	kinesin binding	11	F
GO:0019706	1	1	0.6439069180859244	0.8954691190585615	-	protein-cysteine S-palmitoyltransferase activity	11	F
GO:0017040	1	1	0.6439069180859244	0.8954691190585615	-	N-acylsphingosine amidohydrolase activity	11	F
GO:0019767	1	1	0.6439069180859244	0.8954691190585615	-	IgE receptor activity	11	F
GO:0016936	1	1	0.6439069180859244	0.8954691190585615	-	galactoside binding	11	F
GO:0019208	1	1	0.6439069180859244	0.8954691190585615	-	phosphatase regulator activity	11	F
GO:0019828	1	1	0.6439069180859244	0.8954691190585615	-	aspartic-type endopeptidase inhibitor activity	11	F
GO:0019869	1	1	0.6439069180859244	0.8954691190585615	-	chloride channel inhibitor activity	11	F
GO:0019855	1	1	0.6439069180859244	0.8954691190585615	-	calcium channel inhibitor activity	11	F
GO:0019838	1	1	0.6439069180859244	0.8954691190585615	-	growth factor binding	11	F
GO:0019809	1	1	0.6439069180859244	0.8954691190585615	-	spermidine binding	11	F
GO:0008022	5	8	0.6565198293732681	0.9042041898284293	-	protein C-terminus binding	11	F
GO:0005096	8	13	0.6628628039068198	0.9042041898284293	-	GTPase activator activity	11	F
GO:0070888	2	3	0.6688314496926784	0.9042041898284293	-	E-box binding	11	F
GO:0008013	2	3	0.6688314496926784	0.9042041898284293	-	beta-catenin binding	11	F
GO:0043531	2	3	0.6688314496926784	0.9042041898284293	-	ADP binding	11	F
GO:0004435	2	3	0.6688314496926784	0.9042041898284293	-	phosphatidylinositol phospholipase C activity	11	F
GO:0030971	2	3	0.6688314496926784	0.9042041898284293	-	receptor tyrosine kinase binding	11	F
GO:0050750	2	3	0.6688314496926784	0.9042041898284293	-	low-density lipoprotein particle receptor binding	11	F
GO:0005524	33	53	0.6885192305109964	0.9096218183801222	-	ATP binding	11	F
GO:0003725	3	5	0.6918106156380359	0.9096218183801222	-	double-stranded RNA binding	11	F
GO:0003690	3	5	0.6918106156380359	0.9096218183801222	-	double-stranded DNA binding	11	F
GO:0008009	3	5	0.6918106156380359	0.9096218183801222	-	chemokine activity	11	F
GO:0031490	3	5	0.6918106156380359	0.9096218183801222	-	chromatin DNA binding	11	F
GO:0005070	3	5	0.6918106156380359	0.9096218183801222	-	SH3/SH2 adaptor activity	11	F
GO:0005506	4	7	0.7120935426440396	0.9096218183801222	-	iron ion binding	11	F
GO:0019003	5	9	0.7301374558891018	0.9096218183801222	-	GDP binding	11	F
GO:0004672	7	13	0.761100221037226	0.9096218183801222	-	protein kinase activity	11	F
GO:0004867	2	4	0.7722156554964141	0.9096218183801222	-	serine-type endopeptidase inhibitor activity	11	F
GO:0005201	2	4	0.7722156554964141	0.9096218183801222	-	extracellular matrix structural constituent	11	F
GO:0048156	2	4	0.7722156554964141	0.9096218183801222	-	tau protein binding	11	F
GO:0031492	2	4	0.7722156554964141	0.9096218183801222	-	nucleosomal DNA binding	11	F
GO:0050998	2	4	0.7722156554964141	0.9096218183801222	-	nitric-oxide synthase binding	11	F
GO:0003774	2	4	0.7722156554964141	0.9096218183801222	-	motor activity	11	F
GO:0005547	3	6	0.7750390068324627	0.9096218183801222	-	phosphatidylinositol-3,4,5-trisphosphate binding	11	F
GO:0005543	3	6	0.7750390068324627	0.9096218183801222	-	phospholipid binding	11	F
GO:0008083	4	8	0.7825872922362739	0.9096218183801222	-	growth factor activity	11	F
GO:0043560	1	2	0.7877137396281472	0.9096218183801222	-	insulin receptor substrate binding	11	F
GO:0008047	1	2	0.7877137396281472	0.9096218183801222	-	enzyme activator activity	11	F
GO:0080025	1	2	0.7877137396281472	0.9096218183801222	-	phosphatidylinositol-3,5-bisphosphate binding	11	F
GO:0003785	1	2	0.7877137396281472	0.9096218183801222	-	actin monomer binding	11	F
GO:0008179	1	2	0.7877137396281472	0.9096218183801222	-	adenylate cyclase binding	11	F
GO:0004622	1	2	0.7877137396281472	0.9096218183801222	-	lysophospholipase activity	11	F
GO:0004602	1	2	0.7877137396281472	0.9096218183801222	-	glutathione peroxidase activity	11	F
GO:0003743	1	2	0.7877137396281472	0.9096218183801222	-	translation initiation factor activity	11	F
GO:0043208	1	2	0.7877137396281472	0.9096218183801222	-	glycosphingolipid binding	11	F
GO:0004721	1	2	0.7877137396281472	0.9096218183801222	-	phosphoprotein phosphatase activity	11	F
GO:0004714	1	2	0.7877137396281472	0.9096218183801222	-	transmembrane receptor protein tyrosine kinase activity	11	F
GO:0003824	1	2	0.7877137396281472	0.9096218183801222	-	catalytic activity	11	F
GO:0098772	1	2	0.7877137396281472	0.9096218183801222	-	molecular function regulator	11	F
GO:0017080	1	2	0.7877137396281472	0.9096218183801222	-	sodium channel regulator activity	11	F
GO:0004386	1	2	0.7877137396281472	0.9096218183801222	-	helicase activity	11	F
GO:0017046	1	2	0.7877137396281472	0.9096218183801222	-	peptide hormone binding	11	F
GO:0016791	1	2	0.7877137396281472	0.9096218183801222	-	phosphatase activity	11	F
GO:0005123	1	2	0.7877137396281472	0.9096218183801222	-	death receptor binding	11	F
GO:0030159	1	2	0.7877137396281472	0.9096218183801222	-	receptor signaling complex scaffold activity	11	F
GO:0017075	1	2	0.7877137396281472	0.9096218183801222	-	syntaxin-1 binding	11	F
GO:0005536	1	2	0.7877137396281472	0.9096218183801222	-	glucose binding	11	F
GO:0005528	1	2	0.7877137396281472	0.9096218183801222	-	FK506 binding	11	F
GO:0034713	1	2	0.7877137396281472	0.9096218183801222	-	type I transforming growth factor beta receptor binding	11	F
GO:1990763	1	2	0.7877137396281472	0.9096218183801222	-	arrestin family protein binding	11	F
GO:0017160	1	2	0.7877137396281472	0.9096218183801222	-	Ral GTPase binding	11	F
GO:0005254	1	2	0.7877137396281472	0.9096218183801222	-	chloride channel activity	11	F
GO:0031852	1	2	0.7877137396281472	0.9096218183801222	-	mu-type opioid receptor binding	11	F
GO:0005244	1	2	0.7877137396281472	0.9096218183801222	-	voltage-gated ion channel activity	11	F
GO:0017022	1	2	0.7877137396281472	0.9096218183801222	-	myosin binding	11	F
GO:0050431	1	2	0.7877137396281472	0.9096218183801222	-	transforming growth factor beta binding	11	F
GO:0001102	1	2	0.7877137396281472	0.9096218183801222	-	RNA polymerase II activating transcription factor binding	11	F
GO:0003682	5	10	0.7915952837839347	0.9112421171647552	-	chromatin binding	11	F
GO:0046982	19	34	0.7938140092826991	0.9112421171647552	-	protein heterodimerization activity	11	F
GO:0051015	14	26	0.8048616250737455	0.9211985561316024	-	actin filament binding	11	F
GO:0003779	15	28	0.8151524330638141	0.9302327765551761	-	actin binding	11	F
GO:0044325	9	18	0.8278225943888747	0.9419213097445261	-	ion channel binding	11	F
GO:0005080	3	7	0.8388501257346759	0.946727849280226	-	protein kinase C binding	11	F
GO:0035615	2	5	0.8462211835119069	0.946727849280226	-	clathrin adaptor activity	11	F
GO:0097718	1	3	0.8735272423771157	0.946727849280226	-	disordered domain specific binding	11	F
GO:0015026	1	3	0.8735272423771157	0.946727849280226	-	coreceptor activity	11	F
GO:0008137	1	3	0.8735272423771157	0.946727849280226	-	NADH dehydrogenase (ubiquinone) activity	11	F
GO:0005484	1	3	0.8735272423771157	0.946727849280226	-	SNAP receptor activity	11	F
GO:0008093	1	3	0.8735272423771157	0.946727849280226	-	cytoskeletal adaptor activity	11	F
GO:0043621	1	3	0.8735272423771157	0.946727849280226	-	protein self-association	11	F
GO:0003755	1	3	0.8735272423771157	0.946727849280226	-	peptidyl-prolyl cis-trans isomerase activity	11	F
GO:0004843	1	3	0.8735272423771157	0.946727849280226	-	thiol-dependent ubiquitin-specific protease activity	11	F
GO:0000976	1	3	0.8735272423771157	0.946727849280226	-	transcription regulatory region sequence-specific DNA binding	11	F
GO:0032266	1	3	0.8735272423771157	0.946727849280226	-	phosphatidylinositol-3-phosphate binding	11	F
GO:0005160	1	3	0.8735272423771157	0.946727849280226	-	transforming growth factor beta receptor binding	11	F
GO:0030544	1	3	0.8735272423771157	0.946727849280226	-	Hsp70 protein binding	11	F
GO:0050660	1	3	0.8735272423771157	0.946727849280226	-	flavin adenine dinucleotide binding	11	F
GO:0048487	1	3	0.8735272423771157	0.946727849280226	-	beta-tubulin binding	11	F
GO:0050840	1	3	0.8735272423771157	0.946727849280226	-	extracellular matrix binding	11	F
GO:0000980	3	8	0.8863838443936645	0.9553248100687272	-	RNA polymerase II distal enhancer sequence-specific DNA binding	11	F
GO:0008201	3	8	0.8863838443936645	0.9553248100687272	-	heparin binding	11	F
GO:0051087	2	6	0.8976913728031972	0.9621664437780124	-	chaperone binding	11	F
GO:0035091	2	6	0.8976913728031972	0.9621664437780124	-	phosphatidylinositol binding	11	F
GO:0046872	34	65	0.9170461752712985	0.9695498649051758	-	metal ion binding	11	F
GO:0005198	3	9	0.9209826951127948	0.9695498649051758	-	structural molecule activity	11	F
GO:0097110	1	4	0.9247011905482249	0.9695498649051758	-	scaffold protein binding	11	F
GO:0042393	1	4	0.9247011905482249	0.9695498649051758	-	histone binding	11	F
GO:0017112	1	4	0.9247011905482249	0.9695498649051758	-	Rab guanyl-nucleotide exchange factor activity	11	F
GO:0001968	1	4	0.9247011905482249	0.9695498649051758	-	fibronectin binding	11	F
GO:0043130	1	4	0.9247011905482249	0.9695498649051758	-	ubiquitin binding	11	F
GO:0003676	1	4	0.9247011905482249	0.9695498649051758	-	nucleic acid binding	11	F
GO:0003924	14	31	0.9270695873191243	0.9695498649051758	-	GTPase activity	11	F
GO:0008092	2	7	0.9327349059376928	0.97122504498303	-	cytoskeletal protein binding	11	F
GO:0005178	6	16	0.9337239175532193	0.97122504498303	-	integrin binding	11	F
GO:0047485	3	10	0.9456923209273503	0.97122504498303	-	protein N-terminus binding	11	F
GO:0003697	1	5	0.955198297558587	0.97122504498303	-	single-stranded DNA binding	11	F
GO:0001618	1	5	0.955198297558587	0.97122504498303	-	virus receptor activity	11	F
GO:0001786	1	5	0.955198297558587	0.97122504498303	-	phosphatidylserine binding	11	F
GO:0001965	1	5	0.955198297558587	0.97122504498303	-	G-protein alpha-subunit binding	11	F
GO:0016887	1	5	0.955198297558587	0.97122504498303	-	ATPase activity	11	F
GO:0050681	1	5	0.955198297558587	0.97122504498303	-	androgen receptor binding	11	F
GO:0008565	1	5	0.955198297558587	0.97122504498303	-	protein transporter activity	11	F
GO:0004842	2	8	0.9562061009884469	0.97122504498303	-	ubiquitin-protein transferase activity	11	F
GO:0005525	12	31	0.9703044096361046	0.9829715690308318	-	GTP binding	11	F
GO:0005161	1	6	0.9733611498997004	0.983500328544489	-	platelet-derived growth factor receptor binding	11	F
GO:0003674	10	29	0.9830232580326724	0.990683179523836	-	molecular_function	11	F
GO:0003713	1	9	0.9944222605248115	0.9966922707763417	-	transcription coactivator activity	11	F
GO:0061630	1	9	0.9944222605248115	0.9966922707763417	-	ubiquitin protein ligase activity	11	F
GO:0000166	1	10	0.9966922707763417	0.9966922707763417	-	nucleotide binding	11	F
GO:0005764	20	6	0.0001369770726834335	0.04218893838649752	*	lysosome	11	C
GO:0070062	146	154	0.0003203644771253942	0.04933612947731071	*	extracellular exosome	11	C
GO:0005765	29	17	0.001225925202310387	0.1258616541038664	-	lysosomal membrane	11	C
GO:0005615	61	53	0.0017586377419161413	0.13541510612754287	-	extracellular space	11	C
GO:1904813	23	13	0.003128191904963193	0.1926966213457327	-	ficolin-1-rich granule lumen	11	C
GO:1904724	13	5	0.005833515634653725	0.2994538025788912	-	tertiary granule lumen	11	C
GO:0062023	25	17	0.008047067224290308	0.3540709578687736	-	collagen-containing extracellular matrix	11	C
GO:0005576	78	81	0.009591060425003978	0.3692558263626532	-	extracellular region	11	C
GO:0070821	14	7	0.012495646725285497	0.4122896428429071	-	tertiary granule membrane	11	C
GO:0005886	159	193	0.013386027365029451	0.4122896428429071	-	plasma membrane	11	C
GO:0030496	9	3	0.015647971795286035	0.4240324127366463	-	midbody	11	C
GO:0005938	13	7	0.02148575137356023	0.4240324127366463	-	cell cortex	11	C
GO:0101003	13	7	0.02148575137356023	0.4240324127366463	-	ficolin-1-rich granule membrane	11	C
GO:0030667	17	11	0.021830492961457792	0.4240324127366463	-	secretory granule membrane	11	C
GO:0034774	17	11	0.021830492961457792	0.4240324127366463	-	secretory granule lumen	11	C
GO:0016020	95	108	0.022027657804501107	0.4240324127366463	-	membrane	11	C
GO:0005634	145	181	0.03743918541658382	0.6593243451280855	-	nucleus	11	C
GO:0016021	72	82	0.047301256592038377	0.6593243451280855	-	integral component of membrane	11	C
GO:0005887	41	42	0.05100844236872503	0.6593243451280855	-	integral component of plasma membrane	11	C
GO:0031234	7	3	0.05620580330287516	0.6593243451280855	-	extrinsic component of cytoplasmic side of plasma membrane	11	C
GO:0035580	7	3	0.05620580330287516	0.6593243451280855	-	specific granule lumen	11	C
GO:0005788	13	9	0.05659390041483318	0.6593243451280855	-	endoplasmic reticulum lumen	11	C
GO:0035577	12	8	0.05807966386037942	0.6593243451280855	-	azurophil granule membrane	11	C
GO:0042613	8	4	0.058862942135129034	0.6593243451280855	-	MHC class II protein complex	11	C
GO:0030426	8	4	0.058862942135129034	0.6593243451280855	-	growth cone	11	C
GO:0030666	11	7	0.05924656278126912	0.6593243451280855	-	endocytic vesicle membrane	11	C
GO:0030670	11	7	0.05924656278126912	0.6593243451280855	-	phagocytic vesicle membrane	11	C
GO:0043202	10	6	0.05993857682982595	0.6593243451280855	-	lysosomal lumen	11	C
GO:0043231	33	33	0.06402683274706962	0.6800091202102567	-	intracellular membrane-bounded organelle	11	C
GO:0045121	27	26	0.07084666544924027	0.7273590986122002	-	membrane raft	11	C
GO:0005730	26	25	0.0748909432917957	0.744077759157196	-	nucleolus	11	C
GO:0005791	5	2	0.09850686585876353	0.8923563142499755	-	rough endoplasmic reticulum	11	C
GO:0033116	5	2	0.09850686585876353	0.8923563142499755	-	endoplasmic reticulum-Golgi intermediate compartment membrane	11	C
GO:0035861	5	2	0.09850686585876353	0.8923563142499755	-	site of double-strand break	11	C
GO:0071556	7	4	0.10197327574291436	0.8973648265376464	-	integral component of lumenal side of endoplasmic reticulum membrane	11	C
GO:0035579	13	11	0.11773265660386127	0.9748057632324335	-	specific granule membrane	11	C
GO:0009986	36	40	0.11902484026352159	0.9748057632324335	-	cell surface	11	C
GO:0031902	11	9	0.13132713661313253	0.9748057632324335	-	late endosome membrane	11	C
GO:0030669	9	7	0.14631405296356192	0.9748057632324335	-	clathrin-coated endocytic vesicle membrane	11	C
GO:0098794	5	3	0.1775687480200524	0.9748057632324335	-	postsynapse	11	C
GO:0009897	15	15	0.18111307948188457	0.9748057632324335	-	external side of plasma membrane	11	C
GO:0030134	3	1	0.1823133186217152	0.9748057632324335	-	COPII-coated ER to Golgi transport vesicle	11	C
GO:0016471	3	1	0.1823133186217152	0.9748057632324335	-	vacuolar proton-transporting V-type ATPase complex	11	C
GO:0033179	3	1	0.1823133186217152	0.9748057632324335	-	proton-transporting V-type ATPase, V0 domain	11	C
GO:0042734	4	2	0.18280876974296753	0.9748057632324335	-	presynaptic membrane	11	C
GO:0005769	14	14	0.19290888303543816	0.9748057632324335	-	early endosome	11	C
GO:0005667	9	8	0.204560673382693	0.9748057632324335	-	transcription factor complex	11	C
GO:0005789	22	25	0.2174510949398906	0.9748057632324335	-	endoplasmic reticulum membrane	11	C
GO:0032588	8	7	0.21883957790291697	0.9748057632324335	-	trans-Golgi network membrane	11	C
GO:0030425	12	12	0.21967249532932034	0.9748057632324335	-	dendrite	11	C
GO:0035578	12	12	0.21967249532932034	0.9748057632324335	-	azurophil granule lumen	11	C
GO:0005783	26	31	0.24043510133379045	0.9748057632324335	-	endoplasmic reticulum	11	C
GO:0005884	10	10	0.25179243143552615	0.9748057632324335	-	actin filament	11	C
GO:0012507	6	5	0.2522763084228111	0.9748057632324335	-	ER to Golgi transport vesicle membrane	11	C
GO:0000790	9	9	0.2704661719122204	0.9748057632324335	-	nuclear chromatin	11	C
GO:0036464	5	4	0.2722289125419825	0.9748057632324335	-	cytoplasmic ribonucleoprotein granule	11	C
GO:0005885	5	4	0.2722289125419825	0.9748057632324335	-	Arp2/3 protein complex	11	C
GO:0005737	145	203	0.28517455881810505	0.9748057632324335	-	cytoplasm	11	C
GO:0010008	14	16	0.2949183611721088	0.9748057632324335	-	endosome membrane	11	C
GO:0001891	4	3	0.29521130825523967	0.9748057632324335	-	phagocytic cup	11	C
GO:0098793	4	3	0.29521130825523967	0.9748057632324335	-	presynapse	11	C
GO:0016328	3	2	0.3224359039140872	0.9748057632324335	-	lateral plasma membrane	11	C
GO:0045334	3	2	0.3224359039140872	0.9748057632324335	-	clathrin-coated endocytic vesicle	11	C
GO:0030122	3	2	0.3224359039140872	0.9748057632324335	-	AP-2 adaptor complex	11	C
GO:0005758	3	2	0.3224359039140872	0.9748057632324335	-	mitochondrial intermembrane space	11	C
GO:0031225	3	2	0.3224359039140872	0.9748057632324335	-	anchored component of membrane	11	C
GO:0005834	3	2	0.3224359039140872	0.9748057632324335	-	heterotrimeric G-protein complex	11	C
GO:0000502	3	2	0.3224359039140872	0.9748057632324335	-	proteasome complex	11	C
GO:0030027	12	14	0.3363512728097037	0.9748057632324335	-	lamellipodium	11	C
GO:0005743	9	10	0.34148192101132263	0.9748057632324335	-	mitochondrial inner membrane	11	C
GO:0016607	9	10	0.34148192101132263	0.9748057632324335	-	nuclear speck	11	C
GO:0016323	9	10	0.34148192101132263	0.9748057632324335	-	basolateral plasma membrane	11	C
GO:0030658	6	6	0.34222487405180846	0.9748057632324335	-	transport vesicle membrane	11	C
GO:0043204	6	6	0.34222487405180846	0.9748057632324335	-	perikaryon	11	C
GO:1990904	6	6	0.34222487405180846	0.9748057632324335	-	ribonucleoprotein complex	11	C
GO:0071986	2	1	0.3562932750014786	0.9748057632324335	-	Ragulator complex	11	C
GO:0097487	2	1	0.3562932750014786	0.9748057632324335	-	multivesicular body, internal vesicle	11	C
GO:0032009	2	1	0.3562932750014786	0.9748057632324335	-	early phagosome	11	C
GO:0098982	2	1	0.3562932750014786	0.9748057632324335	-	GABA-ergic synapse	11	C
GO:0031362	2	1	0.3562932750014786	0.9748057632324335	-	anchored component of external side of plasma membrane	11	C
GO:0016234	2	1	0.3562932750014786	0.9748057632324335	-	inclusion body	11	C
GO:0031090	2	1	0.3562932750014786	0.9748057632324335	-	organelle membrane	11	C
GO:0016235	2	1	0.3562932750014786	0.9748057632324335	-	aggresome	11	C
GO:0019897	2	1	0.3562932750014786	0.9748057632324335	-	extrinsic component of plasma membrane	11	C
GO:0033181	2	1	0.3562932750014786	0.9748057632324335	-	plasma membrane proton-transporting V-type ATPase complex	11	C
GO:0032432	2	1	0.3562932750014786	0.9748057632324335	-	actin filament bundle	11	C
GO:0036195	2	1	0.3562932750014786	0.9748057632324335	-	muscle cell projection membrane	11	C
GO:0032991	23	30	0.3692022791830768	0.9748057632324335	-	protein-containing complex	11	C
GO:0005925	37	51	0.4053614952493939	0.9748057632324335	-	focal adhesion	11	C
GO:0019898	4	4	0.4128538684904268	0.9748057632324335	-	extrinsic component of membrane	11	C
GO:0005905	4	4	0.4128538684904268	0.9748057632324335	-	clathrin-coated pit	11	C
GO:1903561	4	4	0.4128538684904268	0.9748057632324335	-	extracellular vesicle	11	C
GO:0044297	4	4	0.4128538684904268	0.9748057632324335	-	cell body	11	C
GO:0015629	20	27	0.4295237404031602	0.9748057632324335	-	actin cytoskeleton	11	C
GO:0043235	6	7	0.43444398599765444	0.9748057632324335	-	receptor complex	11	C
GO:0016324	8	10	0.44697133472163975	0.9748057632324335	-	apical plasma membrane	11	C
GO:0044291	3	3	0.46206303808520693	0.9748057632324335	-	cell-cell contact zone	11	C
GO:0005741	7	9	0.4830494603898167	0.9748057632324335	-	mitochondrial outer membrane	11	C
GO:0042470	8	11	0.5236909083291426	0.9748057632324335	-	melanosome	11	C
GO:0032587	10	14	0.5238043695750283	0.9748057632324335	-	ruffle membrane	11	C
GO:0005768	12	17	0.5241330298148692	0.9748057632324335	-	endosome	11	C
GO:0031982	14	20	0.5245806219553533	0.9748057632324335	-	vesicle	11	C
GO:0030139	2	2	0.530273231381242	0.9748057632324335	-	endocytic vesicle	11	C
GO:0032153	2	2	0.530273231381242	0.9748057632324335	-	cell division site	11	C
GO:0090543	2	2	0.530273231381242	0.9748057632324335	-	Flemming body	11	C
GO:0031527	2	2	0.530273231381242	0.9748057632324335	-	filopodium membrane	11	C
GO:0031941	2	2	0.530273231381242	0.9748057632324335	-	filamentous actin	11	C
GO:0031258	2	2	0.530273231381242	0.9748057632324335	-	lamellipodium membrane	11	C
GO:0090575	2	2	0.530273231381242	0.9748057632324335	-	RNA polymerase II transcription factor complex	11	C
GO:0030665	2	2	0.530273231381242	0.9748057632324335	-	clathrin-coated vesicle membrane	11	C
GO:0030672	2	2	0.530273231381242	0.9748057632324335	-	synaptic vesicle membrane	11	C
GO:0005681	2	2	0.530273231381242	0.9748057632324335	-	spliceosomal complex	11	C
GO:0008043	2	2	0.530273231381242	0.9748057632324335	-	intracellular ferritin complex	11	C
GO:0036020	2	2	0.530273231381242	0.9748057632324335	-	endolysosome membrane	11	C
GO:0016459	2	2	0.530273231381242	0.9748057632324335	-	myosin complex	11	C
GO:0043195	2	2	0.530273231381242	0.9748057632324335	-	terminal bouton	11	C
GO:0005771	2	2	0.530273231381242	0.9748057632324335	-	multivesicular body	11	C
GO:0005829	174	258	0.5317324217399537	0.9748057632324335	-	cytosol	11	C
GO:0031965	7	10	0.563216865676027	0.9748057632324335	-	nuclear membrane	11	C
GO:0005794	34	51	0.5672445067404912	0.9748057632324335	-	Golgi apparatus	11	C
GO:0030659	5	7	0.5711060434595705	0.9748057632324335	-	cytoplasmic vesicle membrane	11	C
GO:0031012	5	7	0.5711060434595705	0.9748057632324335	-	extracellular matrix	11	C
GO:0043025	14	21	0.580125582884546	0.9748057632324335	-	neuronal cell body	11	C
GO:0030175	3	4	0.5872018354576825	0.9748057632324335	-	filopodium	11	C
GO:0016363	3	4	0.5872018354576825	0.9748057632324335	-	nuclear matrix	11	C
GO:0005604	3	4	0.5872018354576825	0.9748057632324335	-	basement membrane	11	C
GO:0001772	3	4	0.5872018354576825	0.9748057632324335	-	immunological synapse	11	C
GO:0044754	3	4	0.5872018354576825	0.9748057632324335	-	autolysosome	11	C
GO:0048471	26	40	0.6098106522948453	0.9748057632324335	-	perinuclear region of cytoplasm	11	C
GO:0042383	4	6	0.6261463960606322	0.9748057632324335	-	sarcolemma	11	C
GO:0035749	1	1	0.6439069180859244	0.9748057632324335	-	myelin sheath adaxonal region	11	C
GO:0008385	1	1	0.6439069180859244	0.9748057632324335	-	IkappaB kinase complex	11	C
GO:0010494	1	1	0.6439069180859244	0.9748057632324335	-	cytoplasmic stress granule	11	C
GO:0016281	1	1	0.6439069180859244	0.9748057632324335	-	eukaryotic translation initiation factor 4F complex	11	C
GO:0005882	1	1	0.6439069180859244	0.9748057632324335	-	intermediate filament	11	C
GO:0042588	1	1	0.6439069180859244	0.9748057632324335	-	zymogen granule	11	C
GO:0016600	1	1	0.6439069180859244	0.9748057632324335	-	flotillin complex	11	C
GO:0045111	1	1	0.6439069180859244	0.9748057632324335	-	intermediate filament cytoskeleton	11	C
GO:0044753	1	1	0.6439069180859244	0.9748057632324335	-	amphisome	11	C
GO:0043196	1	1	0.6439069180859244	0.9748057632324335	-	varicosity	11	C
GO:0019005	1	1	0.6439069180859244	0.9748057632324335	-	SCF ubiquitin ligase complex	11	C
GO:0030132	1	1	0.6439069180859244	0.9748057632324335	-	clathrin coat of coated pit	11	C
GO:0030119	1	1	0.6439069180859244	0.9748057632324335	-	AP-type membrane coat adaptor complex	11	C
GO:0030660	1	1	0.6439069180859244	0.9748057632324335	-	Golgi-associated vesicle membrane	11	C
GO:0030485	1	1	0.6439069180859244	0.9748057632324335	-	smooth muscle contractile fiber	11	C
GO:0030688	1	1	0.6439069180859244	0.9748057632324335	-	preribosome, small subunit precursor	11	C
GO:0035693	1	1	0.6439069180859244	0.9748057632324335	-	NOS2-CD74 complex	11	C
GO:0046696	1	1	0.6439069180859244	0.9748057632324335	-	lipopolysaccharide receptor complex	11	C
GO:0031904	1	1	0.6439069180859244	0.9748057632324335	-	endosome lumen	11	C
GO:0031430	1	1	0.6439069180859244	0.9748057632324335	-	M band	11	C
GO:0031523	1	1	0.6439069180859244	0.9748057632324335	-	Myb complex	11	C
GO:0031312	1	1	0.6439069180859244	0.9748057632324335	-	extrinsic component of organelle membrane	11	C
GO:0030904	1	1	0.6439069180859244	0.9748057632324335	-	retromer complex	11	C
GO:0071438	1	1	0.6439069180859244	0.9748057632324335	-	invadopodium membrane	11	C
GO:1990425	1	1	0.6439069180859244	0.9748057632324335	-	ryanodine receptor complex	11	C
GO:0005671	1	1	0.6439069180859244	0.9748057632324335	-	Ada2/Gcn5/Ada3 transcription activator complex	11	C
GO:0032998	1	1	0.6439069180859244	0.9748057632324335	-	Fc-epsilon receptor I complex	11	C
GO:0034707	1	1	0.6439069180859244	0.9748057632324335	-	chloride channel complex	11	C
GO:0032433	1	1	0.6439069180859244	0.9748057632324335	-	filopodium tip	11	C
GO:0032592	1	1	0.6439069180859244	0.9748057632324335	-	integral component of mitochondrial membrane	11	C
GO:0032809	1	1	0.6439069180859244	0.9748057632324335	-	neuronal cell body membrane	11	C
GO:0044354	1	1	0.6439069180859244	0.9748057632324335	-	macropinosome	11	C
GO:0005665	1	1	0.6439069180859244	0.9748057632324335	-	RNA polymerase II, core complex	11	C
GO:0070469	1	1	0.6439069180859244	0.9748057632324335	-	respirasome	11	C
GO:0001669	1	1	0.6439069180859244	0.9748057632324335	-	acrosomal vesicle	11	C
GO:0005577	1	1	0.6439069180859244	0.9748057632324335	-	fibrinogen complex	11	C
GO:0002947	1	1	0.6439069180859244	0.9748057632324335	-	tumor necrosis factor receptor superfamily complex	11	C
GO:0097452	1	1	0.6439069180859244	0.9748057632324335	-	GAIT complex	11	C
GO:0005640	1	1	0.6439069180859244	0.9748057632324335	-	nuclear outer membrane	11	C
GO:0099523	1	1	0.6439069180859244	0.9748057632324335	-	presynaptic cytosol	11	C
GO:0099091	1	1	0.6439069180859244	0.9748057632324335	-	postsynaptic specialization, intracellular component	11	C
GO:0071437	1	1	0.6439069180859244	0.9748057632324335	-	invadopodium	11	C
GO:0070578	1	1	0.6439069180859244	0.9748057632324335	-	RISC-loading complex	11	C
GO:0035267	1	1	0.6439069180859244	0.9748057632324335	-	NuA4 histone acetyltransferase complex	11	C
GO:1990667	1	1	0.6439069180859244	0.9748057632324335	-	PCSK9-AnxA2 complex	11	C
GO:0097433	1	1	0.6439069180859244	0.9748057632324335	-	dense body	11	C
GO:0000178	1	1	0.6439069180859244	0.9748057632324335	-	exosome (RNase complex)	11	C
GO:0097418	1	1	0.6439069180859244	0.9748057632324335	-	neurofibrillary tangle	11	C
GO:0098871	1	1	0.6439069180859244	0.9748057632324335	-	postsynaptic actin cytoskeleton	11	C
GO:0098562	1	1	0.6439069180859244	0.9748057632324335	-	cytoplasmic side of membrane	11	C
GO:0005750	1	1	0.6439069180859244	0.9748057632324335	-	mitochondrial respiratory chain complex III	11	C
GO:0048787	1	1	0.6439069180859244	0.9748057632324335	-	presynaptic active zone membrane	11	C
GO:0000176	1	1	0.6439069180859244	0.9748057632324335	-	nuclear exosome (RNase complex)	11	C
GO:0035692	1	1	0.6439069180859244	0.9748057632324335	-	macrophage migration inhibitory factor receptor complex	11	C
GO:0036284	1	1	0.6439069180859244	0.9748057632324335	-	tubulobulbar complex	11	C
GO:0036057	1	1	0.6439069180859244	0.9748057632324335	-	slit diaphragm	11	C
GO:0043296	1	1	0.6439069180859244	0.9748057632324335	-	apical junction complex	11	C
GO:0005790	1	1	0.6439069180859244	0.9748057632324335	-	smooth endoplasmic reticulum	11	C
GO:0001740	1	1	0.6439069180859244	0.9748057632324335	-	Barr body	11	C
GO:0005778	1	1	0.6439069180859244	0.9748057632324335	-	peroxisomal membrane	11	C
GO:0005773	1	1	0.6439069180859244	0.9748057632324335	-	vacuole	11	C
GO:0000777	1	1	0.6439069180859244	0.9748057632324335	-	condensed chromosome kinetochore	11	C
GO:0008021	5	8	0.6565198293732681	0.9748057632324335	-	synaptic vesicle	11	C
GO:0015630	5	8	0.6565198293732681	0.9748057632324335	-	microtubule cytoskeleton	11	C
GO:0045211	5	8	0.6565198293732681	0.9748057632324335	-	postsynaptic membrane	11	C
GO:0042995	8	13	0.6628628039068198	0.9748057632324335	-	cell projection	11	C
GO:0000776	2	3	0.6688314496926784	0.9748057632324335	-	kinetochore	11	C
GO:0031594	2	3	0.6688314496926784	0.9748057632324335	-	neuromuscular junction	11	C
GO:0031514	2	3	0.6688314496926784	0.9748057632324335	-	motile cilium	11	C
GO:0031201	2	3	0.6688314496926784	0.9748057632324335	-	SNARE complex	11	C
GO:0030133	2	3	0.6688314496926784	0.9748057632324335	-	transport vesicle	11	C
GO:0009898	2	3	0.6688314496926784	0.9748057632324335	-	cytoplasmic side of plasma membrane	11	C
GO:0005814	2	3	0.6688314496926784	0.9748057632324335	-	centriole	11	C
GO:0032154	3	5	0.6918106156380359	0.9748057632324335	-	cleavage furrow	11	C
GO:0017053	3	5	0.6918106156380359	0.9748057632324335	-	transcriptional repressor complex	11	C
GO:0030136	3	5	0.6918106156380359	0.9748057632324335	-	clathrin-coated vesicle	11	C
GO:0042629	3	5	0.6918106156380359	0.9748057632324335	-	mast cell granule	11	C
GO:0000932	3	5	0.6918106156380359	0.9748057632324335	-	P-body	11	C
GO:0005813	10	17	0.7053195422089882	0.9748057632324335	-	centrosome	11	C
GO:0030054	12	21	0.7396265061169586	0.9748057632324335	-	cell junction	11	C
GO:0043209	7	13	0.761100221037226	0.9748057632324335	-	myelin sheath	11	C
GO:0001726	7	13	0.761100221037226	0.9748057632324335	-	ruffle	11	C
GO:0031226	2	4	0.7722156554964141	0.9748057632324335	-	intrinsic component of plasma membrane	11	C
GO:0043679	2	4	0.7722156554964141	0.9748057632324335	-	axon terminus	11	C
GO:0005901	2	4	0.7722156554964141	0.9748057632324335	-	caveola	11	C
GO:0043198	2	4	0.7722156554964141	0.9748057632324335	-	dendritic shaft	11	C
GO:0000784	2	4	0.7722156554964141	0.9748057632324335	-	nuclear chromosome, telomeric region	11	C
GO:0030864	2	4	0.7722156554964141	0.9748057632324335	-	cortical actin cytoskeleton	11	C
GO:0005811	3	6	0.7750390068324627	0.9748057632324335	-	lipid droplet	11	C
GO:0000407	1	2	0.7877137396281472	0.9748057632324335	-	phagophore assembly site	11	C
GO:0008290	1	2	0.7877137396281472	0.9748057632324335	-	F-actin capping protein complex	11	C
GO:0031305	1	2	0.7877137396281472	0.9748057632324335	-	integral component of mitochondrial inner membrane	11	C
GO:0031224	1	2	0.7877137396281472	0.9748057632324335	-	intrinsic component of membrane	11	C
GO:0005793	1	2	0.7877137396281472	0.9748057632324335	-	endoplasmic reticulum-Golgi intermediate compartment	11	C
GO:0014802	1	2	0.7877137396281472	0.9748057632324335	-	terminal cisterna	11	C
GO:0099056	1	2	0.7877137396281472	0.9748057632324335	-	integral component of presynaptic membrane	11	C
GO:0005747	1	2	0.7877137396281472	0.9748057632324335	-	mitochondrial respiratory chain complex I	11	C
GO:0071944	1	2	0.7877137396281472	0.9748057632324335	-	cell periphery	11	C
GO:0005881	1	2	0.7877137396281472	0.9748057632324335	-	cytoplasmic microtubule	11	C
GO:0005844	1	2	0.7877137396281472	0.9748057632324335	-	polysome	11	C
GO:0016529	1	2	0.7877137396281472	0.9748057632324335	-	sarcoplasmic reticulum	11	C
GO:0012506	1	2	0.7877137396281472	0.9748057632324335	-	vesicle membrane	11	C
GO:0005795	1	2	0.7877137396281472	0.9748057632324335	-	Golgi stack	11	C
GO:0001931	1	2	0.7877137396281472	0.9748057632324335	-	uropod	11	C
GO:0005774	1	2	0.7877137396281472	0.9748057632324335	-	vacuolar membrane	11	C
GO:0032585	1	2	0.7877137396281472	0.9748057632324335	-	multivesicular body membrane	11	C
GO:0043220	1	2	0.7877137396281472	0.9748057632324335	-	Schmidt-Lanterman incisure	11	C
GO:0000421	1	2	0.7877137396281472	0.9748057632324335	-	autophagosome membrane	11	C
GO:0030478	1	2	0.7877137396281472	0.9748057632324335	-	actin cap	11	C
GO:0016461	1	2	0.7877137396281472	0.9748057632324335	-	unconventional myosin complex	11	C
GO:0045177	1	2	0.7877137396281472	0.9748057632324335	-	apical part of cell	11	C
GO:0030061	1	2	0.7877137396281472	0.9748057632324335	-	mitochondrial crista	11	C
GO:0031252	1	2	0.7877137396281472	0.9748057632324335	-	cell leading edge	11	C
GO:0000145	1	2	0.7877137396281472	0.9748057632324335	-	exocyst	11	C
GO:0000118	1	2	0.7877137396281472	0.9748057632324335	-	histone deacetylase complex	11	C
GO:0097440	1	2	0.7877137396281472	0.9748057632324335	-	apical dendrite	11	C
GO:0000228	1	2	0.7877137396281472	0.9748057632324335	-	nuclear chromosome	11	C
GO:0043197	5	10	0.7915952837839347	0.9748057632324335	-	dendritic spine	11	C
GO:0055038	5	10	0.7915952837839347	0.9748057632324335	-	recycling endosome membrane	11	C
GO:0016604	6	12	0.8009535263136393	0.9748057632324335	-	nuclear body	11	C
GO:0014069	6	12	0.8009535263136393	0.9748057632324335	-	postsynaptic density	11	C
GO:0043005	7	14	0.8102189296024289	0.9748057632324335	-	neuron projection	11	C
GO:0031410	7	14	0.8102189296024289	0.9748057632324335	-	cytoplasmic vesicle	11	C
GO:0098978	14	27	0.837084529981141	0.9748057632324335	-	glutamatergic synapse	11	C
GO:0005902	4	9	0.838617275730943	0.9748057632324335	-	microvillus	11	C
GO:0031901	4	9	0.838617275730943	0.9748057632324335	-	early endosome membrane	11	C
GO:0005802	4	9	0.838617275730943	0.9748057632324335	-	trans-Golgi network	11	C
GO:0005635	3	7	0.8388501257346759	0.9748057632324335	-	nuclear envelope	11	C
GO:0031966	3	7	0.8388501257346759	0.9748057632324335	-	mitochondrial membrane	11	C
GO:0005913	6	13	0.8459782985916855	0.9748057632324335	-	cell-cell adherens junction	11	C
GO:0008305	2	5	0.8462211835119069	0.9748057632324335	-	integrin complex	11	C
GO:0036064	2	5	0.8462211835119069	0.9748057632324335	-	ciliary basal body	11	C
GO:0005770	2	5	0.8462211835119069	0.9748057632324335	-	late endosome	11	C
GO:0030141	2	5	0.8462211835119069	0.9748057632324335	-	secretory granule	11	C
GO:0032993	2	5	0.8462211835119069	0.9748057632324335	-	protein-DNA complex	11	C
GO:1904115	1	3	0.8735272423771157	0.9748057632324335	-	axon cytoplasm	11	C
GO:0044853	1	3	0.8735272423771157	0.9748057632324335	-	plasma membrane raft	11	C
GO:0034666	1	3	0.8735272423771157	0.9748057632324335	-	integrin alpha2-beta1 complex	11	C
GO:0097386	1	3	0.8735272423771157	0.9748057632324335	-	glial cell projection	11	C
GO:0005862	1	3	0.8735272423771157	0.9748057632324335	-	muscle thin filament tropomyosin	11	C
GO:0030315	1	3	0.8735272423771157	0.9748057632324335	-	T-tubule	11	C
GO:0008180	1	3	0.8735272423771157	0.9748057632324335	-	COP9 signalosome	11	C
GO:0043034	1	3	0.8735272423771157	0.9748057632324335	-	costamere	11	C
GO:0030016	1	3	0.8735272423771157	0.9748057632324335	-	myofibril	11	C
GO:0005719	1	3	0.8735272423771157	0.9748057632324335	-	nuclear euchromatin	11	C
GO:0099059	1	3	0.8735272423771157	0.9748057632324335	-	integral component of presynaptic active zone membrane	11	C
GO:0005911	6	14	0.8823546175435968	0.975022228833031	-	cell-cell junction	11	C
GO:0030424	6	14	0.8823546175435968	0.975022228833031	-	axon	11	C
GO:0031092	3	8	0.8863838443936645	0.975022228833031	-	platelet alpha granule membrane	11	C
GO:0045202	3	8	0.8863838443936645	0.975022228833031	-	synapse	11	C
GO:0000139	22	43	0.8910077440591804	0.9766205877944041	-	Golgi membrane	11	C
GO:0012505	2	6	0.8976913728031972	0.9804572440545558	-	endomembrane system	11	C
GO:0045335	3	9	0.9209826951127948	0.9926994508910852	-	phagocytic vesicle	11	C
GO:0072562	3	9	0.9209826951127948	0.9926994508910852	-	blood microparticle	11	C
GO:0032580	1	4	0.9247011905482249	0.9926994508910852	-	Golgi cisterna membrane	11	C
GO:0031088	1	4	0.9247011905482249	0.9926994508910852	-	platelet dense granule membrane	11	C
GO:0005856	15	33	0.9297315620516167	0.9926994508910852	-	cytoskeleton	11	C
GO:0001725	4	12	0.9393372472303301	0.9926994508910852	-	stress fiber	11	C
GO:0005654	61	112	0.9421476823148898	0.9926994508910852	-	nucleoplasm	11	C
GO:0055037	3	10	0.9456923209273503	0.9926994508910852	-	recycling endosome	11	C
GO:0001650	1	5	0.955198297558587	0.9926994508910852	-	fibrillar center	11	C
GO:0000786	1	5	0.955198297558587	0.9926994508910852	-	nucleosome	11	C
GO:0005796	1	5	0.955198297558587	0.9926994508910852	-	Golgi lumen	11	C
GO:0005912	1	5	0.955198297558587	0.9926994508910852	-	adherens junction	11	C
GO:0030863	1	5	0.955198297558587	0.9926994508910852	-	cortical cytoskeleton	11	C
GO:0030018	2	8	0.9562061009884469	0.9926994508910852	-	Z disc	11	C
GO:0005923	3	11	0.9630604438163515	0.9926994508910852	-	bicellular tight junction	11	C
GO:0005874	4	14	0.9701365075222683	0.9926994508910852	-	microtubule	11	C
GO:0005929	2	9	0.9717221580095097	0.9926994508910852	-	cilium	11	C
GO:0072686	1	6	0.9733611498997004	0.9926994508910852	-	mitotic spindle	11	C
GO:0030017	1	6	0.9733611498997004	0.9926994508910852	-	sarcomere	11	C
GO:0016605	1	6	0.9733611498997004	0.9926994508910852	-	PML body	11	C
GO:0031093	5	18	0.9832811785260016	0.995912531162562	-	platelet alpha granule lumen	11	C
GO:0005759	3	13	0.9833576279823679	0.995912531162562	-	mitochondrial matrix	11	C
GO:0005739	19	49	0.9898738520947613	0.995912531162562	-	mitochondrion	11	C
GO:0002102	1	8	0.9906006445711751	0.995912531162562	-	podosome	11	C
GO:0005575	2	12	0.9926790489185277	0.995912531162562	-	cellular_component	11	C
GO:0005815	1	13	0.9993129204486644	0.9993129204486644	-	microtubule organizing center	11	C
GO:0043312	81	58	3.3810210803615722e-06	0.004905861587604641	**	neutrophil degranulation	11	P
GO:0019221	29	13	0.00011593952495974062	0.06579743759781788	-	cytokine-mediated signaling pathway	11	P
GO:0045087	31	15	0.00013603880964400662	0.06579743759781788	-	innate immune response	11	P
GO:0006955	29	14	0.0002231022729694826	0.08093034951967981	-	immune response	11	P
GO:0007165	55	44	0.0009237105051801207	0.26806078860327104	-	signal transduction	11	P
GO:0071222	18	7	0.001184366539429893	0.2864193081187958	-	cellular response to lipopolysaccharide	11	P
GO:1902600	8	1	0.003890253060421065	0.528299892274	-	proton transmembrane transport	11	P
GO:0002224	8	1	0.003890253060421065	0.528299892274	-	toll-like receptor signaling pathway	11	P
GO:0034220	11	3	0.003976150068513271	0.528299892274	-	ion transmembrane transport	11	P
GO:0006508	16	7	0.003981865172630845	0.528299892274	-	proteolysis	11	P
GO:0006954	36	27	0.004005030196425913	0.528299892274	-	inflammatory response	11	P
GO:0006968	9	2	0.00606015931277913	0.666953971865499	-	cellular defense response	11	P
GO:0006935	10	3	0.007965310401536163	0.666953971865499	-	chemotaxis	11	P
GO:0008286	10	3	0.007965310401536163	0.666953971865499	-	insulin receptor signaling pathway	11	P
GO:0050830	7	1	0.008733373856267733	0.666953971865499	-	defense response to Gram-positive bacterium	11	P
GO:0032720	7	1	0.008733373856267733	0.666953971865499	-	negative regulation of tumor necrosis factor production	11	P
GO:0051603	7	1	0.008733373856267733	0.666953971865499	-	proteolysis involved in cellular protein catabolic process	11	P
GO:0050718	7	1	0.008733373856267733	0.666953971865499	-	positive regulation of interleukin-1 beta secretion	11	P
GO:0032753	7	1	0.008733373856267733	0.666953971865499	-	positive regulation of interleukin-4 production	11	P
GO:0009615	12	5	0.010804149601198819	0.7838410535669743	-	response to virus	11	P
GO:0030838	8	2	0.012603650929812193	0.8708522618646426	-	positive regulation of actin filament polymerization	11	P
GO:0031295	9	3	0.015647971795286035	0.8826043522783333	-	T cell costimulation	11	P
GO:0006915	31	26	0.019100142553630232	0.8826043522783333	-	apoptotic process	11	P
GO:0015991	6	1	0.01931746453269835	0.8826043522783333	-	ATP hydrolysis coupled proton transport	11	P
GO:0033572	6	1	0.01931746453269835	0.8826043522783333	-	transferrin transport	11	P
GO:0032922	6	1	0.01931746453269835	0.8826043522783333	-	circadian regulation of gene expression	11	P
GO:0051607	11	5	0.019597303317365186	0.8826043522783333	-	defense response to virus	11	P
GO:0051092	13	7	0.02148575137356023	0.8826043522783333	-	positive regulation of NF-kappaB transcription factor activity	11	P
GO:0038096	16	10	0.022026312028118736	0.8826043522783333	-	Fc-gamma receptor signaling pathway involved in phagocytosis	11	P
GO:0014070	7	2	0.025684296641731077	0.8826043522783333	-	response to organic cyclic compound	11	P
GO:0071902	7	2	0.025684296641731077	0.8826043522783333	-	positive regulation of protein serine/threonine kinase activity	11	P
GO:0016032	22	17	0.028247638881132603	0.8826043522783333	-	viral process	11	P
GO:0043066	29	25	0.028450419880925625	0.8826043522783333	-	negative regulation of apoptotic process	11	P
GO:0032729	8	3	0.030052961908567037	0.8826043522783333	-	positive regulation of interferon-gamma production	11	P
GO:0019216	8	3	0.030052961908567037	0.8826043522783333	-	regulation of lipid metabolic process	11	P
GO:0006952	9	4	0.03293395993122323	0.8826043522783333	-	defense response	11	P
GO:0060333	9	4	0.03293395993122323	0.8826043522783333	-	interferon-gamma-mediated signaling pathway	11	P
GO:0007166	16	11	0.03440024838054615	0.8826043522783333	-	cell surface receptor signaling pathway	11	P
GO:0010629	10	5	0.03472528088453797	0.8826043522783333	-	negative regulation of gene expression	11	P
GO:0042102	11	6	0.035718085130336835	0.8826043522783333	-	positive regulation of T cell proliferation	11	P
GO:0030593	11	6	0.035718085130336835	0.8826043522783333	-	neutrophil chemotaxis	11	P
GO:0043123	14	9	0.03574528788618846	0.8826043522783333	-	positive regulation of I-kappaB kinase/NF-kappaB signaling	11	P
GO:0007266	12	7	0.03612331636828793	0.8826043522783333	-	Rho protein signal transduction	11	P
GO:0042742	12	7	0.03612331636828793	0.8826043522783333	-	defense response to bacterium	11	P
GO:1901216	5	1	0.04194300776871698	0.8826043522783333	-	positive regulation of neuron death	11	P
GO:2000484	5	1	0.04194300776871698	0.8826043522783333	-	positive regulation of interleukin-8 secretion	11	P
GO:0071276	5	1	0.04194300776871698	0.8826043522783333	-	cellular response to cadmium ion	11	P
GO:0044344	5	1	0.04194300776871698	0.8826043522783333	-	cellular response to fibroblast growth factor stimulus	11	P
GO:1904646	5	1	0.04194300776871698	0.8826043522783333	-	cellular response to amyloid-beta	11	P
GO:1904469	5	1	0.04194300776871698	0.8826043522783333	-	positive regulation of tumor necrosis factor secretion	11	P
GO:0034341	5	1	0.04194300776871698	0.8826043522783333	-	response to interferon-gamma	11	P
GO:0007611	5	1	0.04194300776871698	0.8826043522783333	-	learning or memory	11	P
GO:0008630	5	1	0.04194300776871698	0.8826043522783333	-	intrinsic apoptotic signaling pathway in response to DNA damage	11	P
GO:0000186	5	1	0.04194300776871698	0.8826043522783333	-	activation of MAPKK activity	11	P
GO:0043433	5	1	0.04194300776871698	0.8826043522783333	-	negative regulation of DNA-binding transcription factor activity	11	P
GO:0010977	5	1	0.04194300776871698	0.8826043522783333	-	negative regulation of neuron projection development	11	P
GO:0045071	5	1	0.04194300776871698	0.8826043522783333	-	negative regulation of viral genome replication	11	P
GO:0006094	5	1	0.04194300776871698	0.8826043522783333	-	gluconeogenesis	11	P
GO:0071347	6	2	0.0510697365619902	0.8826043522783333	-	cellular response to interleukin-1	11	P
GO:0050832	6	2	0.0510697365619902	0.8826043522783333	-	defense response to fungus	11	P
GO:0050729	7	3	0.05620580330287516	0.8826043522783333	-	positive regulation of inflammatory response	11	P
GO:0032755	7	3	0.05620580330287516	0.8826043522783333	-	positive regulation of interleukin-6 production	11	P
GO:0022900	7	3	0.05620580330287516	0.8826043522783333	-	electron transport chain	11	P
GO:0042130	7	3	0.05620580330287516	0.8826043522783333	-	negative regulation of T cell proliferation	11	P
GO:0032088	8	4	0.058862942135129034	0.8826043522783333	-	negative regulation of NF-kappaB transcription factor activity	11	P
GO:0071356	8	4	0.058862942135129034	0.8826043522783333	-	cellular response to tumor necrosis factor	11	P
GO:0032760	8	4	0.058862942135129034	0.8826043522783333	-	positive regulation of tumor necrosis factor production	11	P
GO:0032496	11	7	0.05924656278126912	0.8826043522783333	-	response to lipopolysaccharide	11	P
GO:0070374	14	11	0.08001938394254589	0.8826043522783333	-	positive regulation of ERK1 and ERK2 cascade	11	P
GO:0007568	12	9	0.0873929871048765	0.8826043522783333	-	aging	11	P
GO:0032715	4	1	0.08889826176013384	0.8826043522783333	-	negative regulation of interleukin-6 production	11	P
GO:0046330	4	1	0.08889826176013384	0.8826043522783333	-	positive regulation of JNK cascade	11	P
GO:0007035	4	1	0.08889826176013384	0.8826043522783333	-	vacuolar acidification	11	P
GO:0038066	4	1	0.08889826176013384	0.8826043522783333	-	p38MAPK cascade	11	P
GO:0043525	4	1	0.08889826176013384	0.8826043522783333	-	positive regulation of neuron apoptotic process	11	P
GO:0035924	4	1	0.08889826176013384	0.8826043522783333	-	cellular response to vascular endothelial growth factor stimulus	11	P
GO:0070372	4	1	0.08889826176013384	0.8826043522783333	-	regulation of ERK1 and ERK2 cascade	11	P
GO:0030890	4	1	0.08889826176013384	0.8826043522783333	-	positive regulation of B cell proliferation	11	P
GO:0006801	4	1	0.08889826176013384	0.8826043522783333	-	superoxide metabolic process	11	P
GO:0050715	4	1	0.08889826176013384	0.8826043522783333	-	positive regulation of cytokine secretion	11	P
GO:0031397	4	1	0.08889826176013384	0.8826043522783333	-	negative regulation of protein ubiquitination	11	P
GO:0002548	4	1	0.08889826176013384	0.8826043522783333	-	monocyte chemotaxis	11	P
GO:0051260	11	8	0.09104102659911828	0.8826043522783333	-	protein homooligomerization	11	P
GO:0002250	11	8	0.09104102659911828	0.8826043522783333	-	adaptive immune response	11	P
GO:0050776	11	8	0.09104102659911828	0.8826043522783333	-	regulation of immune response	11	P
GO:0071346	10	7	0.09453927925766752	0.8826043522783333	-	cellular response to interferon-gamma	11	P
GO:0098609	9	6	0.09772784763833547	0.8826043522783333	-	cell-cell adhesion	11	P
GO:0048010	9	6	0.09772784763833547	0.8826043522783333	-	vascular endothelial growth factor receptor signaling pathway	11	P
GO:0042981	9	6	0.09772784763833547	0.8826043522783333	-	regulation of apoptotic process	11	P
GO:1901224	5	2	0.09850686585876353	0.8826043522783333	-	positive regulation of NIK/NF-kappaB signaling	11	P
GO:0002755	5	2	0.09850686585876353	0.8826043522783333	-	MyD88-dependent toll-like receptor signaling pathway	11	P
GO:0021987	5	2	0.09850686585876353	0.8826043522783333	-	cerebral cortex development	11	P
GO:0071407	5	2	0.09850686585876353	0.8826043522783333	-	cellular response to organic cyclic compound	11	P
GO:0032930	5	2	0.09850686585876353	0.8826043522783333	-	positive regulation of superoxide anion generation	11	P
GO:0019886	16	14	0.09991822941492179	0.8826043522783333	-	antigen processing and presentation of exogenous peptide antigen via MHC class II	11	P
GO:0007420	8	5	0.10034931366137831	0.8826043522783333	-	brain development	11	P
GO:0045600	6	3	0.10184061640250841	0.8826043522783333	-	positive regulation of fat cell differentiation	11	P
GO:0043124	6	3	0.10184061640250841	0.8826043522783333	-	negative regulation of I-kappaB kinase/NF-kappaB signaling	11	P
GO:0030307	7	4	0.10197327574291436	0.8826043522783333	-	positive regulation of cell growth	11	P
GO:0001933	7	4	0.10197327574291436	0.8826043522783333	-	negative regulation of protein phosphorylation	11	P
GO:0010628	15	13	0.10552938667482395	0.8826043522783333	-	positive regulation of gene expression	11	P
GO:0006366	13	11	0.11773265660386127	0.8826043522783333	-	transcription by RNA polymerase II	11	P
GO:0007049	12	10	0.12435189113292845	0.8826043522783333	-	cell cycle	11	P
GO:0000165	11	9	0.13132713661313253	0.8826043522783333	-	MAPK cascade	11	P
GO:0045944	34	38	0.13263942049238675	0.8826043522783333	-	positive regulation of transcription by RNA polymerase II	11	P
GO:0031623	10	8	0.1386551748531656	0.8826043522783333	-	receptor internalization	11	P
GO:0018108	9	7	0.14631405296356192	0.8826043522783333	-	peptidyl-tyrosine phosphorylation	11	P
GO:0016579	9	7	0.14631405296356192	0.8826043522783333	-	protein deubiquitination	11	P
GO:0045892	14	13	0.149219134855131	0.8826043522783333	-	negative regulation of transcription, DNA-templated	11	P
GO:0061024	14	13	0.149219134855131	0.8826043522783333	-	membrane organization	11	P
GO:0006898	8	6	0.15424665509514018	0.8826043522783333	-	receptor-mediated endocytosis	11	P
GO:0032091	8	6	0.15424665509514018	0.8826043522783333	-	negative regulation of protein binding	11	P
GO:0043524	8	6	0.15424665509514018	0.8826043522783333	-	negative regulation of neuron apoptotic process	11	P
GO:0000122	28	31	0.15479326605796256	0.8826043522783333	-	negative regulation of transcription by RNA polymerase II	11	P
GO:0007186	21	22	0.15723043848255577	0.8826043522783333	-	G protein-coupled receptor signaling pathway	11	P
GO:0050690	7	5	0.16232774279381376	0.8826043522783333	-	regulation of defense response to virus by virus	11	P
GO:0051928	7	5	0.16232774279381376	0.8826043522783333	-	positive regulation of calcium ion transport	11	P
GO:0000187	7	5	0.16232774279381376	0.8826043522783333	-	activation of MAPK activity	11	P
GO:0008283	20	21	0.16683389903297008	0.8826043522783333	-	cell population proliferation	11	P
GO:0060071	6	4	0.17029283605195825	0.8826043522783333	-	Wnt signaling pathway, planar cell polarity pathway	11	P
GO:0045471	6	4	0.17029283605195825	0.8826043522783333	-	response to ethanol	11	P
GO:0070373	6	4	0.17029283605195825	0.8826043522783333	-	negative regulation of ERK1 and ERK2 cascade	11	P
GO:0022617	5	3	0.1775687480200524	0.8826043522783333	-	extracellular matrix disassembly	11	P
GO:0007623	5	3	0.1775687480200524	0.8826043522783333	-	circadian rhythm	11	P
GO:0071456	5	3	0.1775687480200524	0.8826043522783333	-	cellular response to hypoxia	11	P
GO:0006469	5	3	0.1775687480200524	0.8826043522783333	-	negative regulation of protein kinase activity	11	P
GO:0031663	5	3	0.1775687480200524	0.8826043522783333	-	lipopolysaccharide-mediated signaling pathway	11	P
GO:0048662	5	3	0.1775687480200524	0.8826043522783333	-	negative regulation of smooth muscle cell proliferation	11	P
GO:0071364	5	3	0.1775687480200524	0.8826043522783333	-	cellular response to epidermal growth factor stimulus	11	P
GO:0032092	5	3	0.1775687480200524	0.8826043522783333	-	positive regulation of protein binding	11	P
GO:0031532	5	3	0.1775687480200524	0.8826043522783333	-	actin cytoskeleton reorganization	11	P
GO:0061621	5	3	0.1775687480200524	0.8826043522783333	-	canonical glycolysis	11	P
GO:0031098	5	3	0.1775687480200524	0.8826043522783333	-	stress-activated protein kinase signaling cascade	11	P
GO:0007163	5	3	0.1775687480200524	0.8826043522783333	-	establishment or maintenance of cell polarity	11	P
GO:0050772	3	1	0.1823133186217152	0.8826043522783333	-	positive regulation of axonogenesis	11	P
GO:0008344	3	1	0.1823133186217152	0.8826043522783333	-	adult locomotory behavior	11	P
GO:0002774	3	1	0.1823133186217152	0.8826043522783333	-	Fc receptor mediated inhibitory signaling pathway	11	P
GO:0030574	3	1	0.1823133186217152	0.8826043522783333	-	collagen catabolic process	11	P
GO:0071803	3	1	0.1823133186217152	0.8826043522783333	-	positive regulation of podosome assembly	11	P
GO:0007517	3	1	0.1823133186217152	0.8826043522783333	-	muscle organ development	11	P
GO:0050873	3	1	0.1823133186217152	0.8826043522783333	-	brown fat cell differentiation	11	P
GO:0042832	3	1	0.1823133186217152	0.8826043522783333	-	defense response to protozoan	11	P
GO:0048261	3	1	0.1823133186217152	0.8826043522783333	-	negative regulation of receptor-mediated endocytosis	11	P
GO:0021772	3	1	0.1823133186217152	0.8826043522783333	-	olfactory bulb development	11	P
GO:0045647	3	1	0.1823133186217152	0.8826043522783333	-	negative regulation of erythrocyte differentiation	11	P
GO:0001568	3	1	0.1823133186217152	0.8826043522783333	-	blood vessel development	11	P
GO:0046579	3	1	0.1823133186217152	0.8826043522783333	-	positive regulation of Ras protein signal transduction	11	P
GO:0031065	3	1	0.1823133186217152	0.8826043522783333	-	positive regulation of histone deacetylation	11	P
GO:0006805	3	1	0.1823133186217152	0.8826043522783333	-	xenobiotic metabolic process	11	P
GO:0008543	3	1	0.1823133186217152	0.8826043522783333	-	fibroblast growth factor receptor signaling pathway	11	P
GO:0060135	3	1	0.1823133186217152	0.8826043522783333	-	maternal process involved in female pregnancy	11	P
GO:0090314	3	1	0.1823133186217152	0.8826043522783333	-	positive regulation of protein targeting to membrane	11	P
GO:0090050	3	1	0.1823133186217152	0.8826043522783333	-	positive regulation of cell migration involved in sprouting angiogenesis	11	P
GO:0051591	3	1	0.1823133186217152	0.8826043522783333	-	response to cAMP	11	P
GO:0045861	3	1	0.1823133186217152	0.8826043522783333	-	negative regulation of proteolysis	11	P
GO:0045786	3	1	0.1823133186217152	0.8826043522783333	-	negative regulation of cell cycle	11	P
GO:0045765	3	1	0.1823133186217152	0.8826043522783333	-	regulation of angiogenesis	11	P
GO:0046598	3	1	0.1823133186217152	0.8826043522783333	-	positive regulation of viral entry into host cell	11	P
GO:0010950	3	1	0.1823133186217152	0.8826043522783333	-	positive regulation of endopeptidase activity	11	P
GO:0060907	3	1	0.1823133186217152	0.8826043522783333	-	positive regulation of macrophage cytokine production	11	P
GO:0060828	3	1	0.1823133186217152	0.8826043522783333	-	regulation of canonical Wnt signaling pathway	11	P
GO:0006956	3	1	0.1823133186217152	0.8826043522783333	-	complement activation	11	P
GO:0010466	3	1	0.1823133186217152	0.8826043522783333	-	negative regulation of peptidase activity	11	P
GO:0030301	3	1	0.1823133186217152	0.8826043522783333	-	cholesterol transport	11	P
GO:0070371	3	1	0.1823133186217152	0.8826043522783333	-	ERK1 and ERK2 cascade	11	P
GO:0002283	3	1	0.1823133186217152	0.8826043522783333	-	neutrophil activation involved in immune response	11	P
GO:0038156	3	1	0.1823133186217152	0.8826043522783333	-	interleukin-3-mediated signaling pathway	11	P
GO:0002503	3	1	0.1823133186217152	0.8826043522783333	-	peptide antigen assembly with MHC class II protein complex	11	P
GO:2000778	4	2	0.18280876974296753	0.8826043522783333	-	positive regulation of interleukin-6 secretion	11	P
GO:0016310	4	2	0.18280876974296753	0.8826043522783333	-	phosphorylation	11	P
GO:0055072	4	2	0.18280876974296753	0.8826043522783333	-	iron ion homeostasis	11	P
GO:0046597	4	2	0.18280876974296753	0.8826043522783333	-	negative regulation of viral entry into host cell	11	P
GO:0050766	4	2	0.18280876974296753	0.8826043522783333	-	positive regulation of phagocytosis	11	P
GO:0045429	4	2	0.18280876974296753	0.8826043522783333	-	positive regulation of nitric oxide biosynthetic process	11	P
GO:0050870	4	2	0.18280876974296753	0.8826043522783333	-	positive regulation of T cell activation	11	P
GO:0030316	4	2	0.18280876974296753	0.8826043522783333	-	osteoclast differentiation	11	P
GO:0030502	4	2	0.18280876974296753	0.8826043522783333	-	negative regulation of bone mineralization	11	P
GO:2000406	4	2	0.18280876974296753	0.8826043522783333	-	positive regulation of T cell migration	11	P
GO:0090023	4	2	0.18280876974296753	0.8826043522783333	-	positive regulation of neutrophil chemotaxis	11	P
GO:2001243	4	2	0.18280876974296753	0.8826043522783333	-	negative regulation of intrinsic apoptotic signaling pathway	11	P
GO:0006098	4	2	0.18280876974296753	0.8826043522783333	-	pentose-phosphate shunt	11	P
GO:0005975	4	2	0.18280876974296753	0.8826043522783333	-	carbohydrate metabolic process	11	P
GO:0016601	4	2	0.18280876974296753	0.8826043522783333	-	Rac protein signal transduction	11	P
GO:0034113	4	2	0.18280876974296753	0.8826043522783333	-	heterotypic cell-cell adhesion	11	P
GO:0043200	4	2	0.18280876974296753	0.8826043522783333	-	response to amino acid	11	P
GO:0001774	4	2	0.18280876974296753	0.8826043522783333	-	microglial cell activation	11	P
GO:0001755	4	2	0.18280876974296753	0.8826043522783333	-	neural crest cell migration	11	P
GO:0097178	4	2	0.18280876974296753	0.8826043522783333	-	ruffle assembly	11	P
GO:0032733	4	2	0.18280876974296753	0.8826043522783333	-	positive regulation of interleukin-10 production	11	P
GO:2000179	4	2	0.18280876974296753	0.8826043522783333	-	positive regulation of neural precursor cell proliferation	11	P
GO:0046777	10	9	0.19155978402432933	0.8826043522783333	-	protein autophosphorylation	11	P
GO:0007267	10	9	0.19155978402432933	0.8826043522783333	-	cell-cell signaling	11	P
GO:0010951	9	8	0.204560673382693	0.8826043522783333	-	negative regulation of endopeptidase activity	11	P
GO:0008360	13	13	0.20571484162027467	0.8826043522783333	-	regulation of cell shape	11	P
GO:0006468	19	21	0.2163085726306884	0.8826043522783333	-	protein phosphorylation	11	P
GO:0006909	8	7	0.21883957790291697	0.8826043522783333	-	phagocytosis	11	P
GO:0048013	12	12	0.21967249532932034	0.8826043522783333	-	ephrin receptor signaling pathway	11	P
GO:0035556	15	16	0.22677013913231048	0.8826043522783333	-	intracellular signal transduction	11	P
GO:0016241	7	6	0.23463591011498824	0.8826043522783333	-	regulation of macroautophagy	11	P
GO:0050731	7	6	0.23463591011498824	0.8826043522783333	-	positive regulation of peptidyl-tyrosine phosphorylation	11	P
GO:0001525	14	15	0.24180922971667068	0.8826043522783333	-	angiogenesis	11	P
GO:0006355	10	10	0.25179243143552615	0.8826043522783333	-	regulation of transcription, DNA-templated	11	P
GO:0046854	6	5	0.2522763084228111	0.8826043522783333	-	phosphatidylinositol phosphorylation	11	P
GO:0043488	6	5	0.2522763084228111	0.8826043522783333	-	regulation of mRNA stability	11	P
GO:0007169	6	5	0.2522763084228111	0.8826043522783333	-	transmembrane receptor protein tyrosine kinase signaling pathway	11	P
GO:0043086	6	5	0.2522763084228111	0.8826043522783333	-	negative regulation of catalytic activity	11	P
GO:0048661	6	5	0.2522763084228111	0.8826043522783333	-	positive regulation of smooth muscle cell proliferation	11	P
GO:0050727	6	5	0.2522763084228111	0.8826043522783333	-	regulation of inflammatory response	11	P
GO:0030183	6	5	0.2522763084228111	0.8826043522783333	-	B cell differentiation	11	P
GO:0043407	6	5	0.2522763084228111	0.8826043522783333	-	negative regulation of MAP kinase activity	11	P
GO:0032147	6	5	0.2522763084228111	0.8826043522783333	-	activation of protein kinase activity	11	P
GO:0030154	13	14	0.2581751407593369	0.8826043522783333	-	cell differentiation	11	P
GO:0043065	13	14	0.2581751407593369	0.8826043522783333	-	positive regulation of apoptotic process	11	P
GO:0050852	9	9	0.2704661719122204	0.8826043522783333	-	T cell receptor signaling pathway	11	P
GO:0034383	5	4	0.2722289125419825	0.8826043522783333	-	low-density lipoprotein particle clearance	11	P
GO:1900026	5	4	0.2722289125419825	0.8826043522783333	-	positive regulation of substrate adhesion-dependent cell spreading	11	P
GO:0045893	18	21	0.2748843708864943	0.8826043522783333	-	positive regulation of transcription, DNA-templated	11	P
GO:0038095	8	8	0.291365102842272	0.8826043522783333	-	Fc-epsilon receptor signaling pathway	11	P
GO:0051056	8	8	0.291365102842272	0.8826043522783333	-	regulation of small GTPase mediated signal transduction	11	P
GO:0010033	4	3	0.29521130825523967	0.8826043522783333	-	response to organic substance	11	P
GO:0071345	4	3	0.29521130825523967	0.8826043522783333	-	cellular response to cytokine stimulus	11	P
GO:0007264	4	3	0.29521130825523967	0.8826043522783333	-	small GTPase mediated signal transduction	11	P
GO:0061098	4	3	0.29521130825523967	0.8826043522783333	-	positive regulation of protein tyrosine kinase activity	11	P
GO:0048812	4	3	0.29521130825523967	0.8826043522783333	-	neuron projection morphogenesis	11	P
GO:0045744	4	3	0.29521130825523967	0.8826043522783333	-	negative regulation of G protein-coupled receptor signaling pathway	11	P
GO:0006096	4	3	0.29521130825523967	0.8826043522783333	-	glycolytic process	11	P
GO:0031668	4	3	0.29521130825523967	0.8826043522783333	-	cellular response to extracellular stimulus	11	P
GO:0006629	4	3	0.29521130825523967	0.8826043522783333	-	lipid metabolic process	11	P
GO:0007188	4	3	0.29521130825523967	0.8826043522783333	-	adenylate cyclase-modulating G protein-coupled receptor signaling pathway	11	P
GO:0071479	4	3	0.29521130825523967	0.8826043522783333	-	cellular response to ionizing radiation	11	P
GO:0009409	4	3	0.29521130825523967	0.8826043522783333	-	response to cold	11	P
GO:0032880	4	3	0.29521130825523967	0.8826043522783333	-	regulation of protein localization	11	P
GO:0071404	4	3	0.29521130825523967	0.8826043522783333	-	cellular response to low-density lipoprotein particle stimulus	11	P
GO:0007010	4	3	0.29521130825523967	0.8826043522783333	-	cytoskeleton organization	11	P
GO:0045444	4	3	0.29521130825523967	0.8826043522783333	-	fat cell differentiation	11	P
GO:0030177	4	3	0.29521130825523967	0.8826043522783333	-	positive regulation of Wnt signaling pathway	11	P
GO:0032870	4	3	0.29521130825523967	0.8826043522783333	-	cellular response to hormone stimulus	11	P
GO:0031398	4	3	0.29521130825523967	0.8826043522783333	-	positive regulation of protein ubiquitination	11	P
GO:0006897	7	7	0.3150251651348364	0.8826043522783333	-	endocytosis	11	P
GO:0010976	7	7	0.3150251651348364	0.8826043522783333	-	positive regulation of neuron projection development	11	P
GO:0042127	7	7	0.3150251651348364	0.8826043522783333	-	regulation of cell population proliferation	11	P
GO:0006879	7	7	0.3150251651348364	0.8826043522783333	-	cellular iron ion homeostasis	11	P
GO:0006974	10	11	0.3173721863112643	0.8826043522783333	-	cellular response to DNA damage stimulus	11	P
GO:2001244	3	2	0.3224359039140872	0.8826043522783333	-	positive regulation of intrinsic apoptotic signaling pathway	11	P
GO:0050771	3	2	0.3224359039140872	0.8826043522783333	-	negative regulation of axonogenesis	11	P
GO:0051259	3	2	0.3224359039140872	0.8826043522783333	-	protein complex oligomerization	11	P
GO:0050860	3	2	0.3224359039140872	0.8826043522783333	-	negative regulation of T cell receptor signaling pathway	11	P
GO:0002523	3	2	0.3224359039140872	0.8826043522783333	-	leukocyte migration involved in inflammatory response	11	P
GO:1902895	3	2	0.3224359039140872	0.8826043522783333	-	positive regulation of pri-miRNA transcription by RNA polymerase II	11	P
GO:1990090	3	2	0.3224359039140872	0.8826043522783333	-	cellular response to nerve growth factor stimulus	11	P
GO:0031648	3	2	0.3224359039140872	0.8826043522783333	-	protein destabilization	11	P
GO:1901214	3	2	0.3224359039140872	0.8826043522783333	-	regulation of neuron death	11	P
GO:0071526	3	2	0.3224359039140872	0.8826043522783333	-	semaphorin-plexin signaling pathway	11	P
GO:0072583	3	2	0.3224359039140872	0.8826043522783333	-	clathrin-dependent endocytosis	11	P
GO:0006367	3	2	0.3224359039140872	0.8826043522783333	-	transcription initiation from RNA polymerase II promoter	11	P
GO:0016042	3	2	0.3224359039140872	0.8826043522783333	-	lipid catabolic process	11	P
GO:0042177	3	2	0.3224359039140872	0.8826043522783333	-	negative regulation of protein catabolic process	11	P
GO:0016239	3	2	0.3224359039140872	0.8826043522783333	-	positive regulation of macroautophagy	11	P
GO:0042789	3	2	0.3224359039140872	0.8826043522783333	-	mRNA transcription by RNA polymerase II	11	P
GO:0008380	3	2	0.3224359039140872	0.8826043522783333	-	RNA splicing	11	P
GO:0008037	3	2	0.3224359039140872	0.8826043522783333	-	cell recognition	11	P
GO:0043306	3	2	0.3224359039140872	0.8826043522783333	-	positive regulation of mast cell degranulation	11	P
GO:0006900	3	2	0.3224359039140872	0.8826043522783333	-	vesicle budding from membrane	11	P
GO:0006878	3	2	0.3224359039140872	0.8826043522783333	-	cellular copper ion homeostasis	11	P
GO:0043542	3	2	0.3224359039140872	0.8826043522783333	-	endothelial cell migration	11	P
GO:0045815	3	2	0.3224359039140872	0.8826043522783333	-	positive regulation of gene expression, epigenetic	11	P
GO:0010592	3	2	0.3224359039140872	0.8826043522783333	-	positive regulation of lamellipodium assembly	11	P
GO:0035019	3	2	0.3224359039140872	0.8826043522783333	-	somatic stem cell population maintenance	11	P
GO:0060395	3	2	0.3224359039140872	0.8826043522783333	-	SMAD protein signal transduction	11	P
GO:0051898	3	2	0.3224359039140872	0.8826043522783333	-	negative regulation of protein kinase B signaling	11	P
GO:0031640	3	2	0.3224359039140872	0.8826043522783333	-	killing of cells of other organism	11	P
GO:0010506	3	2	0.3224359039140872	0.8826043522783333	-	regulation of autophagy	11	P
GO:0090200	3	2	0.3224359039140872	0.8826043522783333	-	positive regulation of release of cytochrome c from mitochondria	11	P
GO:0051279	3	2	0.3224359039140872	0.8826043522783333	-	regulation of release of sequestered calcium ion into cytosol	11	P
GO:0051290	3	2	0.3224359039140872	0.8826043522783333	-	protein heterotetramerization	11	P
GO:0002092	3	2	0.3224359039140872	0.8826043522783333	-	positive regulation of receptor internalization	11	P
GO:0032868	3	2	0.3224359039140872	0.8826043522783333	-	response to insulin	11	P
GO:0030855	3	2	0.3224359039140872	0.8826043522783333	-	epithelial cell differentiation	11	P
GO:0050829	3	2	0.3224359039140872	0.8826043522783333	-	defense response to Gram-negative bacterium	11	P
GO:0035722	6	6	0.34222487405180846	0.8826043522783333	-	interleukin-12-mediated signaling pathway	11	P
GO:0045591	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of regulatory T cell differentiation	11	P
GO:0044849	2	1	0.3562932750014786	0.8826043522783333	-	estrous cycle	11	P
GO:0045058	2	1	0.3562932750014786	0.8826043522783333	-	T cell selection	11	P
GO:0045646	2	1	0.3562932750014786	0.8826043522783333	-	regulation of erythrocyte differentiation	11	P
GO:0045651	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of macrophage differentiation	11	P
GO:0045657	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of monocyte differentiation	11	P
GO:0045453	2	1	0.3562932750014786	0.8826043522783333	-	bone resorption	11	P
GO:0032259	2	1	0.3562932750014786	0.8826043522783333	-	methylation	11	P
GO:0045576	2	1	0.3562932750014786	0.8826043522783333	-	mast cell activation	11	P
GO:0032469	2	1	0.3562932750014786	0.8826043522783333	-	endoplasmic reticulum calcium ion homeostasis	11	P
GO:0045616	2	1	0.3562932750014786	0.8826043522783333	-	regulation of keratinocyte differentiation	11	P
GO:0032119	2	1	0.3562932750014786	0.8826043522783333	-	sequestering of zinc ion	11	P
GO:0051220	2	1	0.3562932750014786	0.8826043522783333	-	cytoplasmic sequestering of protein	11	P
GO:0042590	2	1	0.3562932750014786	0.8826043522783333	-	antigen processing and presentation of exogenous peptide antigen via MHC class I	11	P
GO:0042535	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of tumor necrosis factor biosynthetic process	11	P
GO:0043011	2	1	0.3562932750014786	0.8826043522783333	-	myeloid dendritic cell differentiation	11	P
GO:0006821	2	1	0.3562932750014786	0.8826043522783333	-	chloride transport	11	P
GO:0006826	2	1	0.3562932750014786	0.8826043522783333	-	iron ion transport	11	P
GO:0045806	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of endocytosis	11	P
GO:0009052	2	1	0.3562932750014786	0.8826043522783333	-	pentose-phosphate shunt, non-oxidative branch	11	P
GO:0048538	2	1	0.3562932750014786	0.8826043522783333	-	thymus development	11	P
GO:0051966	2	1	0.3562932750014786	0.8826043522783333	-	regulation of synaptic transmission, glutamatergic	11	P
GO:0048268	2	1	0.3562932750014786	0.8826043522783333	-	clathrin coat assembly	11	P
GO:0007566	2	1	0.3562932750014786	0.8826043522783333	-	embryo implantation	11	P
GO:0010812	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of cell-substrate adhesion	11	P
GO:0010818	2	1	0.3562932750014786	0.8826043522783333	-	T cell chemotaxis	11	P
GO:0010942	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of cell death	11	P
GO:0060999	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of dendritic spine development	11	P
GO:0045814	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of gene expression, epigenetic	11	P
GO:0045742	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of epidermal growth factor receptor signaling pathway	11	P
GO:0045780	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of bone resorption	11	P
GO:0010543	2	1	0.3562932750014786	0.8826043522783333	-	regulation of platelet activation	11	P
GO:0007498	2	1	0.3562932750014786	0.8826043522783333	-	mesoderm development	11	P
GO:0007346	2	1	0.3562932750014786	0.8826043522783333	-	regulation of mitotic cell cycle	11	P
GO:0031641	2	1	0.3562932750014786	0.8826043522783333	-	regulation of myelination	11	P
GO:0043966	2	1	0.3562932750014786	0.8826043522783333	-	histone H3 acetylation	11	P
GO:0036151	2	1	0.3562932750014786	0.8826043522783333	-	phosphatidylcholine acyl-chain remodeling	11	P
GO:0016973	2	1	0.3562932750014786	0.8826043522783333	-	poly(A)+ mRNA export from nucleus	11	P
GO:0035666	2	1	0.3562932750014786	0.8826043522783333	-	TRIF-dependent toll-like receptor signaling pathway	11	P
GO:0035767	2	1	0.3562932750014786	0.8826043522783333	-	endothelial cell chemotaxis	11	P
GO:0035902	2	1	0.3562932750014786	0.8826043522783333	-	response to immobilization stress	11	P
GO:0035914	2	1	0.3562932750014786	0.8826043522783333	-	skeletal muscle cell differentiation	11	P
GO:0040008	2	1	0.3562932750014786	0.8826043522783333	-	regulation of growth	11	P
GO:0042129	2	1	0.3562932750014786	0.8826043522783333	-	regulation of T cell proliferation	11	P
GO:0070488	2	1	0.3562932750014786	0.8826043522783333	-	neutrophil aggregation	11	P
GO:0006383	2	1	0.3562932750014786	0.8826043522783333	-	transcription by RNA polymerase III	11	P
GO:0036295	2	1	0.3562932750014786	0.8826043522783333	-	cellular response to increased oxygen levels	11	P
GO:0038128	2	1	0.3562932750014786	0.8826043522783333	-	ERBB2 signaling pathway	11	P
GO:0006465	2	1	0.3562932750014786	0.8826043522783333	-	signal peptide processing	11	P
GO:0014912	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of smooth muscle cell migration	11	P
GO:0015914	2	1	0.3562932750014786	0.8826043522783333	-	phospholipid transport	11	P
GO:0016070	2	1	0.3562932750014786	0.8826043522783333	-	RNA metabolic process	11	P
GO:0070885	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of calcineurin-NFAT signaling cascade	11	P
GO:0071294	2	1	0.3562932750014786	0.8826043522783333	-	cellular response to zinc ion	11	P
GO:0030449	2	1	0.3562932750014786	0.8826043522783333	-	regulation of complement activation	11	P
GO:0006691	2	1	0.3562932750014786	0.8826043522783333	-	leukotriene metabolic process	11	P
GO:0043277	2	1	0.3562932750014786	0.8826043522783333	-	apoptotic cell clearance	11	P
GO:0042953	2	1	0.3562932750014786	0.8826043522783333	-	lipoprotein transport	11	P
GO:0042795	2	1	0.3562932750014786	0.8826043522783333	-	snRNA transcription by RNA polymerase II	11	P
GO:0043087	2	1	0.3562932750014786	0.8826043522783333	-	regulation of GTPase activity	11	P
GO:0048245	2	1	0.3562932750014786	0.8826043522783333	-	eosinophil chemotaxis	11	P
GO:0048246	2	1	0.3562932750014786	0.8826043522783333	-	macrophage chemotaxis	11	P
GO:0071672	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of smooth muscle cell chemotaxis	11	P
GO:0060252	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of glial cell proliferation	11	P
GO:0033630	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of cell adhesion mediated by integrin	11	P
GO:0071480	2	1	0.3562932750014786	0.8826043522783333	-	cellular response to gamma radiation	11	P
GO:0043113	2	1	0.3562932750014786	0.8826043522783333	-	receptor clustering	11	P
GO:0014850	2	1	0.3562932750014786	0.8826043522783333	-	response to muscle activity	11	P
GO:0042770	2	1	0.3562932750014786	0.8826043522783333	-	signal transduction in response to DNA damage	11	P
GO:0018279	2	1	0.3562932750014786	0.8826043522783333	-	protein N-linked glycosylation via asparagine	11	P
GO:0018345	2	1	0.3562932750014786	0.8826043522783333	-	protein palmitoylation	11	P
GO:0097242	2	1	0.3562932750014786	0.8826043522783333	-	amyloid-beta clearance	11	P
GO:0051549	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of keratinocyte migration	11	P
GO:1902166	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator	11	P
GO:0097284	2	1	0.3562932750014786	0.8826043522783333	-	hepatocyte apoptotic process	11	P
GO:0097421	2	1	0.3562932750014786	0.8826043522783333	-	liver regeneration	11	P
GO:0098742	2	1	0.3562932750014786	0.8826043522783333	-	cell-cell adhesion via plasma-membrane adhesion molecules	11	P
GO:0099132	2	1	0.3562932750014786	0.8826043522783333	-	ATP hydrolysis coupled cation transmembrane transport	11	P
GO:0002606	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of dendritic cell antigen processing and presentation	11	P
GO:0051602	2	1	0.3562932750014786	0.8826043522783333	-	response to electrical stimulus	11	P
GO:0002042	2	1	0.3562932750014786	0.8826043522783333	-	cell migration involved in sprouting angiogenesis	11	P
GO:0051272	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of cellular component movement	11	P
GO:0001921	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of receptor recycling	11	P
GO:0007187	2	1	0.3562932750014786	0.8826043522783333	-	G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger	11	P
GO:0048147	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of fibroblast proliferation	11	P
GO:0010498	2	1	0.3562932750014786	0.8826043522783333	-	proteasomal protein catabolic process	11	P
GO:0046622	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of organ growth	11	P
GO:0046638	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of alpha-beta T cell differentiation	11	P
GO:0001913	2	1	0.3562932750014786	0.8826043522783333	-	T cell mediated cytotoxicity	11	P
GO:0032695	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of interleukin-12 production	11	P
GO:0097066	2	1	0.3562932750014786	0.8826043522783333	-	response to thyroid hormone	11	P
GO:0001947	2	1	0.3562932750014786	0.8826043522783333	-	heart looping	11	P
GO:0051045	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of membrane protein ectodomain proteolysis	11	P
GO:0001765	2	1	0.3562932750014786	0.8826043522783333	-	membrane raft assembly	11	P
GO:2000670	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of dendritic cell apoptotic process	11	P
GO:0032735	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of interleukin-12 production	11	P
GO:0032728	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of interferon-beta production	11	P
GO:0002218	2	1	0.3562932750014786	0.8826043522783333	-	activation of innate immune response	11	P
GO:0033364	2	1	0.3562932750014786	0.8826043522783333	-	mast cell secretory granule organization	11	P
GO:0032623	2	1	0.3562932750014786	0.8826043522783333	-	interleukin-2 production	11	P
GO:0001819	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of cytokine production	11	P
GO:0000281	2	1	0.3562932750014786	0.8826043522783333	-	mitotic cytokinesis	11	P
GO:0001667	2	1	0.3562932750014786	0.8826043522783333	-	ameboidal-type cell migration	11	P
GO:0001756	2	1	0.3562932750014786	0.8826043522783333	-	somitogenesis	11	P
GO:0032602	2	1	0.3562932750014786	0.8826043522783333	-	chemokine production	11	P
GO:0050807	2	1	0.3562932750014786	0.8826043522783333	-	regulation of synapse organization	11	P
GO:0001954	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of cell-matrix adhesion	11	P
GO:1902475	2	1	0.3562932750014786	0.8826043522783333	-	L-alpha-amino acid transmembrane transport	11	P
GO:0050663	2	1	0.3562932750014786	0.8826043522783333	-	cytokine secretion	11	P
GO:0033003	2	1	0.3562932750014786	0.8826043522783333	-	regulation of mast cell activation	11	P
GO:1900027	2	1	0.3562932750014786	0.8826043522783333	-	regulation of ruffle assembly	11	P
GO:1900153	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of nuclear-transcribed mRNA catabolic process, deadenylation-dependent decay	11	P
GO:0031340	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of vesicle fusion	11	P
GO:0033574	2	1	0.3562932750014786	0.8826043522783333	-	response to testosterone	11	P
GO:0030225	2	1	0.3562932750014786	0.8826043522783333	-	macrophage differentiation	11	P
GO:0046488	2	1	0.3562932750014786	0.8826043522783333	-	phosphatidylinositol metabolic process	11	P
GO:0009791	2	1	0.3562932750014786	0.8826043522783333	-	post-embryonic development	11	P
GO:1901379	2	1	0.3562932750014786	0.8826043522783333	-	regulation of potassium ion transmembrane transport	11	P
GO:1902236	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of endoplasmic reticulum stress-induced intrinsic apoptotic signaling pathway	11	P
GO:0030048	2	1	0.3562932750014786	0.8826043522783333	-	actin filament-based movement	11	P
GO:0034144	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of toll-like receptor 4 signaling pathway	11	P
GO:0030522	2	1	0.3562932750014786	0.8826043522783333	-	intracellular receptor signaling pathway	11	P
GO:1904707	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of vascular smooth muscle cell proliferation	11	P
GO:0036120	2	1	0.3562932750014786	0.8826043522783333	-	cellular response to platelet-derived growth factor stimulus	11	P
GO:1905049	2	1	0.3562932750014786	0.8826043522783333	-	negative regulation of metallopeptidase activity	11	P
GO:0001892	2	1	0.3562932750014786	0.8826043522783333	-	embryonic placenta development	11	P
GO:0005999	2	1	0.3562932750014786	0.8826043522783333	-	xylulose biosynthetic process	11	P
GO:0002830	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of type 2 immune response	11	P
GO:0090330	2	1	0.3562932750014786	0.8826043522783333	-	regulation of platelet aggregation	11	P
GO:1990776	2	1	0.3562932750014786	0.8826043522783333	-	response to angiotensin	11	P
GO:0071333	2	1	0.3562932750014786	0.8826043522783333	-	cellular response to glucose stimulus	11	P
GO:0002821	2	1	0.3562932750014786	0.8826043522783333	-	positive regulation of adaptive immune response	11	P
GO:0072673	2	1	0.3562932750014786	0.8826043522783333	-	lamellipodium morphogenesis	11	P
GO:1990535	2	1	0.3562932750014786	0.8826043522783333	-	neuron projection maintenance	11	P
GO:0002768	2	1	0.3562932750014786	0.8826043522783333	-	immune response-regulating cell surface receptor signaling pathway	11	P
GO:0045766	8	9	0.3685245956952311	0.8826043522783333	-	positive regulation of angiogenesis	11	P
GO:0034097	5	5	0.3741649890320067	0.8826043522783333	-	response to cytokine	11	P
GO:0007565	5	5	0.3741649890320067	0.8826043522783333	-	female pregnancy	11	P
GO:0070588	5	5	0.3741649890320067	0.8826043522783333	-	calcium ion transmembrane transport	11	P
GO:0034314	5	5	0.3741649890320067	0.8826043522783333	-	Arp2/3 complex-mediated actin nucleation	11	P
GO:0019722	5	5	0.3741649890320067	0.8826043522783333	-	calcium-mediated signaling	11	P
GO:0007283	5	5	0.3741649890320067	0.8826043522783333	-	spermatogenesis	11	P
GO:0006887	5	5	0.3741649890320067	0.8826043522783333	-	exocytosis	11	P
GO:0006457	5	5	0.3741649890320067	0.8826043522783333	-	protein folding	11	P
GO:0009611	5	5	0.3741649890320067	0.8826043522783333	-	response to wounding	11	P
GO:1903078	5	5	0.3741649890320067	0.8826043522783333	-	positive regulation of protein localization to plasma membrane	11	P
GO:0010469	12	15	0.3988921784499973	0.8826043522783333	-	regulation of signaling receptor activity	11	P
GO:1990830	4	4	0.4128538684904268	0.8826043522783333	-	cellular response to leukemia inhibitory factor	11	P
GO:0032570	4	4	0.4128538684904268	0.8826043522783333	-	response to progesterone	11	P
GO:0006911	4	4	0.4128538684904268	0.8826043522783333	-	phagocytosis, engulfment	11	P
GO:0050853	4	4	0.4128538684904268	0.8826043522783333	-	B cell receptor signaling pathway	11	P
GO:0090090	4	4	0.4128538684904268	0.8826043522783333	-	negative regulation of canonical Wnt signaling pathway	11	P
GO:0048008	4	4	0.4128538684904268	0.8826043522783333	-	platelet-derived growth factor receptor signaling pathway	11	P
GO:0043280	4	4	0.4128538684904268	0.8826043522783333	-	positive regulation of cysteine-type endopeptidase activity involved in apoptotic process	11	P
GO:0030182	4	4	0.4128538684904268	0.8826043522783333	-	neuron differentiation	11	P
GO:0019730	4	4	0.4128538684904268	0.8826043522783333	-	antimicrobial humoral response	11	P
GO:0060326	4	4	0.4128538684904268	0.8826043522783333	-	cell chemotaxis	11	P
GO:0023014	4	4	0.4128538684904268	0.8826043522783333	-	signal transduction by protein phosphorylation	11	P
GO:0050728	4	4	0.4128538684904268	0.8826043522783333	-	negative regulation of inflammatory response	11	P
GO:0006357	17	23	0.4469027713175669	0.8826043522783333	-	regulation of transcription by RNA polymerase II	11	P
GO:0044267	10	13	0.4555586831059928	0.8826043522783333	-	cellular protein metabolic process	11	P
GO:0043687	12	16	0.46200688919745825	0.8826043522783333	-	post-translational protein modification	11	P
GO:0034394	3	3	0.46206303808520693	0.8826043522783333	-	protein localization to cell surface	11	P
GO:0034260	3	3	0.46206303808520693	0.8826043522783333	-	negative regulation of GTPase activity	11	P
GO:0031529	3	3	0.46206303808520693	0.8826043522783333	-	ruffle organization	11	P
GO:0034976	3	3	0.46206303808520693	0.8826043522783333	-	response to endoplasmic reticulum stress	11	P
GO:0034614	3	3	0.46206303808520693	0.8826043522783333	-	cellular response to reactive oxygen species	11	P
GO:0071320	3	3	0.46206303808520693	0.8826043522783333	-	cellular response to cAMP	11	P
GO:0090630	3	3	0.46206303808520693	0.8826043522783333	-	activation of GTPase activity	11	P
GO:0071363	3	3	0.46206303808520693	0.8826043522783333	-	cellular response to growth factor stimulus	11	P
GO:0001932	3	3	0.46206303808520693	0.8826043522783333	-	regulation of protein phosphorylation	11	P
GO:0007193	3	3	0.46206303808520693	0.8826043522783333	-	adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway	11	P
GO:0031100	3	3	0.46206303808520693	0.8826043522783333	-	animal organ regeneration	11	P
GO:0051402	3	3	0.46206303808520693	0.8826043522783333	-	neuron apoptotic process	11	P
GO:0032515	3	3	0.46206303808520693	0.8826043522783333	-	negative regulation of phosphoprotein phosphatase activity	11	P
GO:0032526	3	3	0.46206303808520693	0.8826043522783333	-	response to retinoic acid	11	P
GO:0097067	3	3	0.46206303808520693	0.8826043522783333	-	cellular response to thyroid hormone stimulus	11	P
GO:0001775	3	3	0.46206303808520693	0.8826043522783333	-	cell activation	11	P
GO:0048870	3	3	0.46206303808520693	0.8826043522783333	-	cell motility	11	P
GO:0071230	3	3	0.46206303808520693	0.8826043522783333	-	cellular response to amino acid stimulus	11	P
GO:0070498	3	3	0.46206303808520693	0.8826043522783333	-	interleukin-1-mediated signaling pathway	11	P
GO:0006397	3	3	0.46206303808520693	0.8826043522783333	-	mRNA processing	11	P
GO:0042752	3	3	0.46206303808520693	0.8826043522783333	-	regulation of circadian rhythm	11	P
GO:0008217	3	3	0.46206303808520693	0.8826043522783333	-	regulation of blood pressure	11	P
GO:0045672	3	3	0.46206303808520693	0.8826043522783333	-	positive regulation of osteoclast differentiation	11	P
GO:0043085	3	3	0.46206303808520693	0.8826043522783333	-	positive regulation of catalytic activity	11	P
GO:0045666	3	3	0.46206303808520693	0.8826043522783333	-	positive regulation of neuron differentiation	11	P
GO:0070301	3	3	0.46206303808520693	0.8826043522783333	-	cellular response to hydrogen peroxide	11	P
GO:0061418	3	3	0.46206303808520693	0.8826043522783333	-	regulation of transcription from RNA polymerase II promoter in response to hypoxia	11	P
GO:0032233	3	3	0.46206303808520693	0.8826043522783333	-	positive regulation of actin filament bundle assembly	11	P
GO:0045089	3	3	0.46206303808520693	0.8826043522783333	-	positive regulation of innate immune response	11	P
GO:0008285	16	22	0.4713812786142479	0.8826043522783333	-	negative regulation of cell population proliferation	11	P
GO:0032869	5	6	0.4757388895396697	0.8826043522783333	-	cellular response to insulin stimulus	11	P
GO:0034329	5	6	0.4757388895396697	0.8826043522783333	-	cell junction assembly	11	P
GO:0009636	5	6	0.4757388895396697	0.8826043522783333	-	response to toxic substance	11	P
GO:0018105	5	6	0.4757388895396697	0.8826043522783333	-	peptidyl-serine phosphorylation	11	P
GO:0035335	5	6	0.4757388895396697	0.8826043522783333	-	peptidyl-tyrosine dephosphorylation	11	P
GO:0002244	5	6	0.4757388895396697	0.8826043522783333	-	hematopoietic progenitor cell differentiation	11	P
GO:0051017	5	6	0.4757388895396697	0.8826043522783333	-	actin filament bundle assembly	11	P
GO:0006919	7	9	0.4830494603898167	0.8826043522783333	-	activation of cysteine-type endopeptidase activity involved in apoptotic process	11	P
GO:0032956	7	9	0.4830494603898167	0.8826043522783333	-	regulation of actin cytoskeleton organization	11	P
GO:0007155	16	23	0.5250980959391686	0.8826043522783333	-	cell adhesion	11	P
GO:0045454	4	5	0.525353833249182	0.8826043522783333	-	cell redox homeostasis	11	P
GO:0061844	4	5	0.525353833249182	0.8826043522783333	-	antimicrobial humoral immune response mediated by antimicrobial peptide	11	P
GO:0034599	4	5	0.525353833249182	0.8826043522783333	-	cellular response to oxidative stress	11	P
GO:0010634	4	5	0.525353833249182	0.8826043522783333	-	positive regulation of epithelial cell migration	11	P
GO:0009267	4	5	0.525353833249182	0.8826043522783333	-	cellular response to starvation	11	P
GO:0002223	4	5	0.525353833249182	0.8826043522783333	-	stimulatory C-type lectin receptor signaling pathway	11	P
GO:0010614	2	2	0.530273231381242	0.8826043522783333	-	negative regulation of cardiac muscle hypertrophy	11	P
GO:0010575	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of vascular endothelial growth factor production	11	P
GO:0009612	2	2	0.530273231381242	0.8826043522783333	-	response to mechanical stimulus	11	P
GO:0090316	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of intracellular protein transport	11	P
GO:0090303	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of wound healing	11	P
GO:0071447	2	2	0.530273231381242	0.8826043522783333	-	cellular response to hydroperoxide	11	P
GO:0030097	2	2	0.530273231381242	0.8826043522783333	-	hemopoiesis	11	P
GO:0090026	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of monocyte chemotaxis	11	P
GO:0006139	2	2	0.530273231381242	0.8826043522783333	-	nucleobase-containing compound metabolic process	11	P
GO:0097009	2	2	0.530273231381242	0.8826043522783333	-	energy homeostasis	11	P
GO:0046007	2	2	0.530273231381242	0.8826043522783333	-	negative regulation of activated T cell proliferation	11	P
GO:0044794	2	2	0.530273231381242	0.8826043522783333	-	positive regulation by host of viral process	11	P
GO:0045637	2	2	0.530273231381242	0.8826043522783333	-	regulation of myeloid cell differentiation	11	P
GO:0050790	2	2	0.530273231381242	0.8826043522783333	-	regulation of catalytic activity	11	P
GO:0051480	2	2	0.530273231381242	0.8826043522783333	-	regulation of cytosolic calcium ion concentration	11	P
GO:0051412	2	2	0.530273231381242	0.8826043522783333	-	response to corticosterone	11	P
GO:0007202	2	2	0.530273231381242	0.8826043522783333	-	activation of phospholipase C activity	11	P
GO:0006612	2	2	0.530273231381242	0.8826043522783333	-	protein targeting to membrane	11	P
GO:0050730	2	2	0.530273231381242	0.8826043522783333	-	regulation of peptidyl-tyrosine phosphorylation	11	P
GO:0030216	2	2	0.530273231381242	0.8826043522783333	-	keratinocyte differentiation	11	P
GO:0050861	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of B cell receptor signaling pathway	11	P
GO:0008306	2	2	0.530273231381242	0.8826043522783333	-	associative learning	11	P
GO:0048469	2	2	0.530273231381242	0.8826043522783333	-	cell maturation	11	P
GO:0097190	2	2	0.530273231381242	0.8826043522783333	-	apoptotic signaling pathway	11	P
GO:1990000	2	2	0.530273231381242	0.8826043522783333	-	amyloid fibril formation	11	P
GO:0060079	2	2	0.530273231381242	0.8826043522783333	-	excitatory postsynaptic potential	11	P
GO:0008203	2	2	0.530273231381242	0.8826043522783333	-	cholesterol metabolic process	11	P
GO:0007601	2	2	0.530273231381242	0.8826043522783333	-	visual perception	11	P
GO:0048260	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of receptor-mediated endocytosis	11	P
GO:0006611	2	2	0.530273231381242	0.8826043522783333	-	protein export from nucleus	11	P
GO:0035970	2	2	0.530273231381242	0.8826043522783333	-	peptidyl-threonine dephosphorylation	11	P
GO:0030950	2	2	0.530273231381242	0.8826043522783333	-	establishment or maintenance of actin cytoskeleton polarity	11	P
GO:0032008	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of TOR signaling	11	P
GO:0050765	2	2	0.530273231381242	0.8826043522783333	-	negative regulation of phagocytosis	11	P
GO:0007030	2	2	0.530273231381242	0.8826043522783333	-	Golgi organization	11	P
GO:0006977	2	2	0.530273231381242	0.8826043522783333	-	DNA damage response, signal transduction by p53 class mediator resulting in cell cycle arrest	11	P
GO:0051930	2	2	0.530273231381242	0.8826043522783333	-	regulation of sensory perception of pain	11	P
GO:0010803	2	2	0.530273231381242	0.8826043522783333	-	regulation of tumor necrosis factor-mediated signaling pathway	11	P
GO:0000045	2	2	0.530273231381242	0.8826043522783333	-	autophagosome assembly	11	P
GO:0006880	2	2	0.530273231381242	0.8826043522783333	-	intracellular sequestering of iron ion	11	P
GO:0032486	2	2	0.530273231381242	0.8826043522783333	-	Rap protein signal transduction	11	P
GO:2000009	2	2	0.530273231381242	0.8826043522783333	-	negative regulation of protein localization to cell surface	11	P
GO:1900087	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of G1/S transition of mitotic cell cycle	11	P
GO:1902961	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of aspartic-type endopeptidase activity involved in amyloid precursor protein catabolic process	11	P
GO:0002506	2	2	0.530273231381242	0.8826043522783333	-	polysaccharide assembly with MHC class II protein complex	11	P
GO:0002431	2	2	0.530273231381242	0.8826043522783333	-	Fc receptor mediated stimulatory signaling pathway	11	P
GO:0097192	2	2	0.530273231381242	0.8826043522783333	-	extrinsic apoptotic signaling pathway in absence of ligand	11	P
GO:0097191	2	2	0.530273231381242	0.8826043522783333	-	extrinsic apoptotic signaling pathway	11	P
GO:0030031	2	2	0.530273231381242	0.8826043522783333	-	cell projection assembly	11	P
GO:2001235	2	2	0.530273231381242	0.8826043522783333	-	positive regulation of apoptotic signaling pathway	11	P
GO:0001782	2	2	0.530273231381242	0.8826043522783333	-	B cell homeostasis	11	P
GO:0097028	2	2	0.530273231381242	0.8826043522783333	-	dendritic cell differentiation	11	P
GO:0002554	2	2	0.530273231381242	0.8826043522783333	-	serotonin secretion by platelet	11	P
GO:0001816	2	2	0.530273231381242	0.8826043522783333	-	cytokine production	11	P
GO:0001817	2	2	0.530273231381242	0.8826043522783333	-	regulation of cytokine production	11	P
GO:0000188	2	2	0.530273231381242	0.8826043522783333	-	inactivation of MAPK activity	11	P
GO:0034612	2	2	0.530273231381242	0.8826043522783333	-	response to tumor necrosis factor	11	P
GO:0007040	2	2	0.530273231381242	0.8826043522783333	-	lysosome organization	11	P
GO:0035606	2	2	0.530273231381242	0.8826043522783333	-	peptidyl-cysteine S-trans-nitrosylation	11	P
GO:0043408	2	2	0.530273231381242	0.8826043522783333	-	regulation of MAPK cascade	11	P
GO:0043304	2	2	0.530273231381242	0.8826043522783333	-	regulation of mast cell degranulation	11	P
GO:0006631	2	2	0.530273231381242	0.8826043522783333	-	fatty acid metabolic process	11	P
GO:0016575	2	2	0.530273231381242	0.8826043522783333	-	histone deacetylation	11	P
GO:0042730	2	2	0.530273231381242	0.8826043522783333	-	fibrinolysis	11	P
GO:0035690	2	2	0.530273231381242	0.8826043522783333	-	cellular response to drug	11	P
GO:0018149	2	2	0.530273231381242	0.8826043522783333	-	peptide cross-linking	11	P
GO:0030705	2	2	0.530273231381242	0.8826043522783333	-	cytoskeleton-dependent intracellular transport	11	P
GO:0031214	2	2	0.530273231381242	0.8826043522783333	-	biomineral tissue development	11	P
GO:0098974	2	2	0.530273231381242	0.8826043522783333	-	postsynaptic actin cytoskeleton organization	11	P
GO:0098586	2	2	0.530273231381242	0.8826043522783333	-	cellular response to virus	11	P
GO:0051764	2	2	0.530273231381242	0.8826043522783333	-	actin crosslink formation	11	P
GO:0051639	2	2	0.530273231381242	0.8826043522783333	-	actin filament network formation	11	P
GO:0042267	2	2	0.530273231381242	0.8826043522783333	-	natural killer cell mediated cytotoxicity	11	P
GO:0030036	11	16	0.5555215950971202	0.8826043522783333	-	actin cytoskeleton organization	11	P
GO:0016477	11	16	0.5555215950971202	0.8826043522783333	-	cell migration	11	P
GO:0015031	11	16	0.5555215950971202	0.8826043522783333	-	protein transport	11	P
GO:0050821	7	10	0.563216865676027	0.8826043522783333	-	protein stabilization	11	P
GO:0051301	7	10	0.563216865676027	0.8826043522783333	-	cell division	11	P
GO:0051028	3	4	0.5872018354576825	0.8826043522783333	-	mRNA transport	11	P
GO:0007417	3	4	0.5872018354576825	0.8826043522783333	-	central nervous system development	11	P
GO:0000902	3	4	0.5872018354576825	0.8826043522783333	-	cell morphogenesis	11	P
GO:0001501	3	4	0.5872018354576825	0.8826043522783333	-	skeletal system development	11	P
GO:0009725	3	4	0.5872018354576825	0.8826043522783333	-	response to hormone	11	P
GO:0051894	3	4	0.5872018354576825	0.8826043522783333	-	positive regulation of focal adhesion assembly	11	P
GO:0034605	3	4	0.5872018354576825	0.8826043522783333	-	cellular response to heat	11	P
GO:0009749	3	4	0.5872018354576825	0.8826043522783333	-	response to glucose	11	P
GO:0043627	3	4	0.5872018354576825	0.8826043522783333	-	response to estrogen	11	P
GO:0014003	3	4	0.5872018354576825	0.8826043522783333	-	oligodendrocyte development	11	P
GO:0045807	3	4	0.5872018354576825	0.8826043522783333	-	positive regulation of endocytosis	11	P
GO:0070098	3	4	0.5872018354576825	0.8826043522783333	-	chemokine-mediated signaling pathway	11	P
GO:0044319	3	4	0.5872018354576825	0.8826043522783333	-	wound healing, spreading of cells	11	P
GO:0016485	3	4	0.5872018354576825	0.8826043522783333	-	protein processing	11	P
GO:0006874	3	4	0.5872018354576825	0.8826043522783333	-	cellular calcium ion homeostasis	11	P
GO:0045746	3	4	0.5872018354576825	0.8826043522783333	-	negative regulation of Notch signaling pathway	11	P
GO:0006406	3	4	0.5872018354576825	0.8826043522783333	-	mRNA export from nucleus	11	P
GO:0042531	3	4	0.5872018354576825	0.8826043522783333	-	positive regulation of tyrosine phosphorylation of STAT protein	11	P
GO:0043434	3	4	0.5872018354576825	0.8826043522783333	-	response to peptide hormone	11	P
GO:0006606	3	4	0.5872018354576825	0.8826043522783333	-	protein import into nucleus	11	P
GO:0051726	3	4	0.5872018354576825	0.8826043522783333	-	regulation of cell cycle	11	P
GO:0035855	3	4	0.5872018354576825	0.8826043522783333	-	megakaryocyte development	11	P
GO:0038083	3	4	0.5872018354576825	0.8826043522783333	-	peptidyl-tyrosine autophosphorylation	11	P
GO:0030866	3	4	0.5872018354576825	0.8826043522783333	-	cortical actin cytoskeleton organization	11	P
GO:0071277	3	4	0.5872018354576825	0.8826043522783333	-	cellular response to calcium ion	11	P
GO:0051897	8	12	0.5962420267724065	0.8826043522783333	-	positive regulation of protein kinase B signaling	11	P
GO:0008284	19	29	0.5973685161145801	0.8826043522783333	-	positive regulation of cell population proliferation	11	P
GO:0007229	9	14	0.6247614921374947	0.8826043522783333	-	integrin-mediated signaling pathway	11	P
GO:0043161	4	6	0.6261463960606322	0.8826043522783333	-	proteasome-mediated ubiquitin-dependent protein catabolic process	11	P
GO:0051289	4	6	0.6261463960606322	0.8826043522783333	-	protein homotetramerization	11	P
GO:0071260	4	6	0.6261463960606322	0.8826043522783333	-	cellular response to mechanical stimulus	11	P
GO:0006979	4	6	0.6261463960606322	0.8826043522783333	-	response to oxidative stress	11	P
GO:0032355	4	6	0.6261463960606322	0.8826043522783333	-	response to estradiol	11	P
GO:0006470	4	6	0.6261463960606322	0.8826043522783333	-	protein dephosphorylation	11	P
GO:0033209	4	6	0.6261463960606322	0.8826043522783333	-	tumor necrosis factor-mediated signaling pathway	11	P
GO:0030336	4	6	0.6261463960606322	0.8826043522783333	-	negative regulation of cell migration	11	P
GO:0001938	4	6	0.6261463960606322	0.8826043522783333	-	positive regulation of endothelial cell proliferation	11	P
GO:0070667	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of mast cell proliferation	11	P
GO:0046651	1	1	0.6439069180859244	0.8826043522783333	-	lymphocyte proliferation	11	P
GO:0010757	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of plasminogen activation	11	P
GO:0010755	1	1	0.6439069180859244	0.8826043522783333	-	regulation of plasminogen activation	11	P
GO:0010744	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of macrophage derived foam cell differentiation	11	P
GO:0045762	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of adenylate cyclase activity	11	P
GO:0045740	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of DNA replication	11	P
GO:0007093	1	1	0.6439069180859244	0.8826043522783333	-	mitotic cell cycle checkpoint	11	P
GO:0007175	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of epidermal growth factor-activated receptor activity	11	P
GO:0060075	1	1	0.6439069180859244	0.8826043522783333	-	regulation of resting membrane potential	11	P
GO:0055088	1	1	0.6439069180859244	0.8826043522783333	-	lipid homeostasis	11	P
GO:0060087	1	1	0.6439069180859244	0.8826043522783333	-	relaxation of vascular smooth muscle	11	P
GO:0052501	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation by organism of apoptotic process in other organism involved in symbiotic interaction	11	P
GO:0046326	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of glucose import	11	P
GO:0046676	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of insulin secretion	11	P
GO:0071226	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to molecule of fungal origin	11	P
GO:0006913	1	1	0.6439069180859244	0.8826043522783333	-	nucleocytoplasmic transport	11	P
GO:0070206	1	1	0.6439069180859244	0.8826043522783333	-	protein trimerization	11	P
GO:0070365	1	1	0.6439069180859244	0.8826043522783333	-	hepatocyte differentiation	11	P
GO:0070304	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of stress-activated protein kinase signaling cascade	11	P
GO:0070294	1	1	0.6439069180859244	0.8826043522783333	-	renal sodium ion absorption	11	P
GO:0010920	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of inositol phosphate biosynthetic process	11	P
GO:0014066	1	1	0.6439069180859244	0.8826043522783333	-	regulation of phosphatidylinositol 3-kinase signaling	11	P
GO:0045124	1	1	0.6439069180859244	0.8826043522783333	-	regulation of bone resorption	11	P
GO:0010886	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of cholesterol storage	11	P
GO:0010815	1	1	0.6439069180859244	0.8826043522783333	-	bradykinin catabolic process	11	P
GO:0070723	1	1	0.6439069180859244	0.8826043522783333	-	response to cholesterol	11	P
GO:0070676	1	1	0.6439069180859244	0.8826043522783333	-	intralumenal vesicle formation	11	P
GO:0060193	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of lipase activity	11	P
GO:0007219	1	1	0.6439069180859244	0.8826043522783333	-	Notch signaling pathway	11	P
GO:0048023	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of melanin biosynthetic process	11	P
GO:0010324	1	1	0.6439069180859244	0.8826043522783333	-	membrane invagination	11	P
GO:0010310	1	1	0.6439069180859244	0.8826043522783333	-	regulation of hydrogen peroxide metabolic process	11	P
GO:0010288	1	1	0.6439069180859244	0.8826043522783333	-	response to lead ion	11	P
GO:0046636	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of alpha-beta T cell activation	11	P
GO:0046628	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of insulin receptor signaling pathway	11	P
GO:0048012	1	1	0.6439069180859244	0.8826043522783333	-	hepatocyte growth factor receptor signaling pathway	11	P
GO:0048011	1	1	0.6439069180859244	0.8826043522783333	-	neurotrophin TRK receptor signaling pathway	11	P
GO:0060547	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of necrotic cell death	11	P
GO:0060546	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of necroptotic process	11	P
GO:0060452	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of cardiac muscle contraction	11	P
GO:0060397	1	1	0.6439069180859244	0.8826043522783333	-	growth hormone receptor signaling pathway via JAK-STAT	11	P
GO:0070668	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of mast cell proliferation	11	P
GO:0060161	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of dopamine receptor signaling pathway	11	P
GO:0060324	1	1	0.6439069180859244	0.8826043522783333	-	face development	11	P
GO:0060314	1	1	0.6439069180859244	0.8826043522783333	-	regulation of ryanodine-sensitive calcium-release channel activity	11	P
GO:0010621	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of transcription by transcription factor localization	11	P
GO:0045776	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of blood pressure	11	P
GO:0061003	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of dendritic spine morphogenesis	11	P
GO:0061842	1	1	0.6439069180859244	0.8826043522783333	-	microtubule organizing center localization	11	P
GO:0060956	1	1	0.6439069180859244	0.8826043522783333	-	endocardial cell differentiation	11	P
GO:0060766	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of androgen receptor signaling pathway	11	P
GO:0060710	1	1	0.6439069180859244	0.8826043522783333	-	chorio-allantoic fusion	11	P
GO:0060574	1	1	0.6439069180859244	0.8826043522783333	-	intestinal epithelial cell maturation	11	P
GO:0046686	1	1	0.6439069180859244	0.8826043522783333	-	response to cadmium ion	11	P
GO:0061373	1	1	0.6439069180859244	0.8826043522783333	-	mammillary axonal complex development	11	P
GO:0043951	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of cAMP-mediated signaling	11	P
GO:0045059	1	1	0.6439069180859244	0.8826043522783333	-	positive thymic T cell selection	11	P
GO:0045055	1	1	0.6439069180859244	0.8826043522783333	-	regulated exocytosis	11	P
GO:0045019	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of nitric oxide biosynthetic process	11	P
GO:0044387	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of protein kinase activity by regulation of protein phosphorylation	11	P
GO:0010936	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of macrophage cytokine production	11	P
GO:0010925	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of inositol-polyphosphate 5-phosphatase activity	11	P
GO:0015853	1	1	0.6439069180859244	0.8826043522783333	-	adenine transport	11	P
GO:0016266	1	1	0.6439069180859244	0.8826043522783333	-	O-glycan processing	11	P
GO:0016479	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of transcription by RNA polymerase I	11	P
GO:0015721	1	1	0.6439069180859244	0.8826043522783333	-	bile acid and bile salt transport	11	P
GO:0014839	1	1	0.6439069180859244	0.8826043522783333	-	myoblast migration involved in skeletal muscle regeneration	11	P
GO:0014739	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of muscle hyperplasia	11	P
GO:0044090	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of vacuole organization	11	P
GO:0061298	1	1	0.6439069180859244	0.8826043522783333	-	retina vasculature development in camera-type eye	11	P
GO:0061097	1	1	0.6439069180859244	0.8826043522783333	-	regulation of protein tyrosine kinase activity	11	P
GO:0070266	1	1	0.6439069180859244	0.8826043522783333	-	necroptotic process	11	P
GO:0070244	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of thymocyte apoptotic process	11	P
GO:0070234	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of T cell apoptotic process	11	P
GO:0006865	1	1	0.6439069180859244	0.8826043522783333	-	amino acid transport	11	P
GO:0061052	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of cell growth involved in cardiac muscle cell development	11	P
GO:0071225	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to muramyl dipeptide	11	P
GO:0071169	1	1	0.6439069180859244	0.8826043522783333	-	establishment of protein localization to chromatin	11	P
GO:0071158	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of cell cycle arrest	11	P
GO:0071103	1	1	0.6439069180859244	0.8826043522783333	-	DNA conformation change	11	P
GO:0032700	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of interleukin-17 production	11	P
GO:0045579	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of B cell differentiation	11	P
GO:0043507	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of JUN kinase activity	11	P
GO:0045109	1	1	0.6439069180859244	0.8826043522783333	-	intermediate filament organization	11	P
GO:0045060	1	1	0.6439069180859244	0.8826043522783333	-	negative thymic T cell selection	11	P
GO:0045428	1	1	0.6439069180859244	0.8826043522783333	-	regulation of nitric oxide biosynthetic process	11	P
GO:0045347	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of MHC class II biosynthetic process	11	P
GO:0045217	1	1	0.6439069180859244	0.8826043522783333	-	cell-cell junction maintenance	11	P
GO:0045582	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of T cell differentiation	11	P
GO:0045581	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of T cell differentiation	11	P
GO:0060313	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of blood vessel remodeling	11	P
GO:0060311	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of elastin catabolic process	11	P
GO:0007004	1	1	0.6439069180859244	0.8826043522783333	-	telomere maintenance via telomerase	11	P
GO:0006991	1	1	0.6439069180859244	0.8826043522783333	-	response to sterol depletion	11	P
GO:0010711	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of collagen catabolic process	11	P
GO:0007223	1	1	0.6439069180859244	0.8826043522783333	-	Wnt signaling pathway, calcium modulating pathway	11	P
GO:0045664	1	1	0.6439069180859244	0.8826043522783333	-	regulation of neuron differentiation	11	P
GO:0045636	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of melanocyte differentiation	11	P
GO:0043922	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation by host of viral transcription	11	P
GO:0043654	1	1	0.6439069180859244	0.8826043522783333	-	recognition of apoptotic cell	11	P
GO:0043647	1	1	0.6439069180859244	0.8826043522783333	-	inositol phosphate metabolic process	11	P
GO:0043589	1	1	0.6439069180859244	0.8826043522783333	-	skin morphogenesis	11	P
GO:0043551	1	1	0.6439069180859244	0.8826043522783333	-	regulation of phosphatidylinositol 3-kinase activity	11	P
GO:0044257	1	1	0.6439069180859244	0.8826043522783333	-	cellular protein catabolic process	11	P
GO:0019827	1	1	0.6439069180859244	0.8826043522783333	-	stem cell population maintenance	11	P
GO:0021769	1	1	0.6439069180859244	0.8826043522783333	-	orbitofrontal cortex development	11	P
GO:0044147	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of development of symbiont involved in interaction with host	11	P
GO:0044130	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of growth of symbiont in host	11	P
GO:2000535	1	1	0.6439069180859244	0.8826043522783333	-	regulation of entry of bacterium into host cell	11	P
GO:0001798	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of type IIa hypersensitivity	11	P
GO:2001269	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of cysteine-type endopeptidase activity involved in apoptotic signaling pathway	11	P
GO:0001953	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of cell-matrix adhesion	11	P
GO:0032707	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of interleukin-23 production	11	P
GO:2000727	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of cardiac muscle cell differentiation	11	P
GO:2000666	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of interleukin-13 secretion	11	P
GO:2000642	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of early endosome to late endosome transport	11	P
GO:2000615	1	1	0.6439069180859244	0.8826043522783333	-	regulation of histone H3-K9 acetylation	11	P
GO:2000587	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of platelet-derived growth factor receptor-beta signaling pathway	11	P
GO:2000562	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of CD4-positive, alpha-beta T cell proliferation	11	P
GO:2000172	1	1	0.6439069180859244	0.8826043522783333	-	regulation of branching morphogenesis of a nerve	11	P
GO:2000503	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of natural killer cell chemotaxis	11	P
GO:0032765	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of mast cell cytokine production	11	P
GO:0001886	1	1	0.6439069180859244	0.8826043522783333	-	endothelial cell morphogenesis	11	P
GO:2001171	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of ATP biosynthetic process	11	P
GO:2001044	1	1	0.6439069180859244	0.8826043522783333	-	regulation of integrin-mediated signaling pathway	11	P
GO:2000810	1	1	0.6439069180859244	0.8826043522783333	-	regulation of bicellular tight junction assembly	11	P
GO:2000786	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of autophagosome assembly	11	P
GO:2000370	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of clathrin-dependent endocytosis	11	P
GO:2000343	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of chemokine (C-X-C motif) ligand 2 production	11	P
GO:1903393	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of adherens junction organization	11	P
GO:1903146	1	1	0.6439069180859244	0.8826043522783333	-	regulation of autophagy of mitochondrion	11	P
GO:1903056	1	1	0.6439069180859244	0.8826043522783333	-	regulation of melanosome organization	11	P
GO:1903026	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of RNA polymerase II regulatory region sequence-specific DNA binding	11	P
GO:1902430	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of amyloid-beta formation	11	P
GO:1902396	1	1	0.6439069180859244	0.8826043522783333	-	protein localization to bicellular tight junction	11	P
GO:1902339	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of apoptotic process involved in morphogenesis	11	P
GO:0031116	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of microtubule polymerization	11	P
GO:0032364	1	1	0.6439069180859244	0.8826043522783333	-	oxygen homeostasis	11	P
GO:0032270	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of cellular protein metabolic process	11	P
GO:0001654	1	1	0.6439069180859244	0.8826043522783333	-	eye development	11	P
GO:2000279	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of DNA biosynthetic process	11	P
GO:1905599	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of low-density lipoprotein receptor activity	11	P
GO:2000121	1	1	0.6439069180859244	0.8826043522783333	-	regulation of removal of superoxide radicals	11	P
GO:1990786	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to dsDNA	11	P
GO:1902532	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of intracellular signal transduction	11	P
GO:1905451	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of Fc-gamma receptor signaling pathway involved in phagocytosis	11	P
GO:1905031	1	1	0.6439069180859244	0.8826043522783333	-	regulation of membrane repolarization during cardiac muscle cell action potential	11	P
GO:1905000	1	1	0.6439069180859244	0.8826043522783333	-	regulation of membrane repolarization during atrial cardiac muscle cell action potential	11	P
GO:0033373	1	1	0.6439069180859244	0.8826043522783333	-	maintenance of protease location in mast cell secretory granule	11	P
GO:0050904	1	1	0.6439069180859244	0.8826043522783333	-	diapedesis	11	P
GO:0032516	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of phosphoprotein phosphatase activity	11	P
GO:1903427	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of reactive oxygen species biosynthetic process	11	P
GO:1905597	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of low-density lipoprotein particle receptor binding	11	P
GO:2000767	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of cytoplasmic translation	11	P
GO:0001825	1	1	0.6439069180859244	0.8826043522783333	-	blastocyst formation	11	P
GO:0001818	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of cytokine production	11	P
GO:0001504	1	1	0.6439069180859244	0.8826043522783333	-	neurotransmitter uptake	11	P
GO:0032613	1	1	0.6439069180859244	0.8826043522783333	-	interleukin-10 production	11	P
GO:0032757	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of interleukin-8 production	11	P
GO:0000184	1	1	0.6439069180859244	0.8826043522783333	-	nuclear-transcribed mRNA catabolic process, nonsense-mediated decay	11	P
GO:2000483	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of interleukin-8 secretion	11	P
GO:0001709	1	1	0.6439069180859244	0.8826043522783333	-	cell fate determination	11	P
GO:0001708	1	1	0.6439069180859244	0.8826043522783333	-	cell fate specification	11	P
GO:0007213	1	1	0.6439069180859244	0.8826043522783333	-	G protein-coupled acetylcholine receptor signaling pathway	11	P
GO:0050965	1	1	0.6439069180859244	0.8826043522783333	-	detection of temperature stimulus involved in sensory perception of pain	11	P
GO:0050930	1	1	0.6439069180859244	0.8826043522783333	-	induction of positive chemotaxis	11	P
GO:0050796	1	1	0.6439069180859244	0.8826043522783333	-	regulation of insulin secretion	11	P
GO:0050779	1	1	0.6439069180859244	0.8826043522783333	-	RNA destabilization	11	P
GO:0007254	1	1	0.6439069180859244	0.8826043522783333	-	JNK cascade	11	P
GO:0051453	1	1	0.6439069180859244	0.8826043522783333	-	regulation of intracellular pH	11	P
GO:0051451	1	1	0.6439069180859244	0.8826043522783333	-	myoblast migration	11	P
GO:0051403	1	1	0.6439069180859244	0.8826043522783333	-	stress-activated MAPK cascade	11	P
GO:0060305	1	1	0.6439069180859244	0.8826043522783333	-	regulation of cell diameter	11	P
GO:0060263	1	1	0.6439069180859244	0.8826043522783333	-	regulation of respiratory burst	11	P
GO:0060242	1	1	0.6439069180859244	0.8826043522783333	-	contact inhibition	11	P
GO:0060213	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of nuclear-transcribed mRNA poly(A) tail shortening	11	P
GO:0007214	1	1	0.6439069180859244	0.8826043522783333	-	gamma-aminobutyric acid signaling pathway	11	P
GO:0051000	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of nitric-oxide synthase activity	11	P
GO:0007268	1	1	0.6439069180859244	0.8826043522783333	-	chemical synaptic transmission	11	P
GO:0051489	1	1	0.6439069180859244	0.8826043522783333	-	regulation of filopodium assembly	11	P
GO:0051668	1	1	0.6439069180859244	0.8826043522783333	-	localization within membrane	11	P
GO:0051623	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of norepinephrine uptake	11	P
GO:0051621	1	1	0.6439069180859244	0.8826043522783333	-	regulation of norepinephrine uptake	11	P
GO:0007257	1	1	0.6439069180859244	0.8826043522783333	-	activation of JUN kinase activity	11	P
GO:0046827	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of protein export from nucleus	11	P
GO:0046328	1	1	0.6439069180859244	0.8826043522783333	-	regulation of JNK cascade	11	P
GO:0052405	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation by host of symbiont molecular function	11	P
GO:0052362	1	1	0.6439069180859244	0.8826043522783333	-	catabolism by host of symbiont protein	11	P
GO:0051964	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of synapse assembly	11	P
GO:0060038	1	1	0.6439069180859244	0.8826043522783333	-	cardiac muscle cell proliferation	11	P
GO:0008626	1	1	0.6439069180859244	0.8826043522783333	-	granzyme-mediated apoptotic signaling pathway	11	P
GO:0048714	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of oligodendrocyte differentiation	11	P
GO:0048711	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of astrocyte differentiation	11	P
GO:0048679	1	1	0.6439069180859244	0.8826043522783333	-	regulation of axon regeneration	11	P
GO:0010043	1	1	0.6439069180859244	0.8826043522783333	-	response to zinc ion	11	P
GO:0010519	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of phospholipase activity	11	P
GO:0010508	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of autophagy	11	P
GO:0048143	1	1	0.6439069180859244	0.8826043522783333	-	astrocyte activation	11	P
GO:0048066	1	1	0.6439069180859244	0.8826043522783333	-	developmental pigmentation	11	P
GO:0009314	1	1	0.6439069180859244	0.8826043522783333	-	response to radiation	11	P
GO:0048668	1	1	0.6439069180859244	0.8826043522783333	-	collateral sprouting	11	P
GO:0048667	1	1	0.6439069180859244	0.8826043522783333	-	cell morphogenesis involved in neuron differentiation	11	P
GO:0050482	1	1	0.6439069180859244	0.8826043522783333	-	arachidonic acid secretion	11	P
GO:0048854	1	1	0.6439069180859244	0.8826043522783333	-	brain morphogenesis	11	P
GO:0070972	1	1	0.6439069180859244	0.8826043522783333	-	protein localization to endoplasmic reticulum	11	P
GO:0048298	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of isotype switching to IgA isotypes	11	P
GO:0048757	1	1	0.6439069180859244	0.8826043522783333	-	pigment granule maturation	11	P
GO:0050777	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of immune response	11	P
GO:0050855	1	1	0.6439069180859244	0.8826043522783333	-	regulation of B cell receptor signaling pathway	11	P
GO:0008637	1	1	0.6439069180859244	0.8826043522783333	-	apoptotic mitochondrial changes	11	P
GO:0009566	1	1	0.6439069180859244	0.8826043522783333	-	fertilization	11	P
GO:0009447	1	1	0.6439069180859244	0.8826043522783333	-	putrescine catabolic process	11	P
GO:0009792	1	1	0.6439069180859244	0.8826043522783333	-	embryo development ending in birth or egg hatching	11	P
GO:0008625	1	1	0.6439069180859244	0.8826043522783333	-	extrinsic apoptotic signaling pathway via death domain receptors	11	P
GO:0009408	1	1	0.6439069180859244	0.8826043522783333	-	response to heat	11	P
GO:0050885	1	1	0.6439069180859244	0.8826043522783333	-	neuromuscular process controlling balance	11	P
GO:0050859	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of B cell receptor signaling pathway	11	P
GO:0030845	1	1	0.6439069180859244	0.8826043522783333	-	phospholipase C-inhibiting G protein-coupled receptor signaling pathway	11	P
GO:0002062	1	1	0.6439069180859244	0.8826043522783333	-	chondrocyte differentiation	11	P
GO:0002031	1	1	0.6439069180859244	0.8826043522783333	-	G protein-coupled receptor internalization	11	P
GO:0002292	1	1	0.6439069180859244	0.8826043522783333	-	T cell differentiation involved in immune response	11	P
GO:0034356	1	1	0.6439069180859244	0.8826043522783333	-	NAD biosynthesis via nicotinamide riboside salvage pathway	11	P
GO:0030490	1	1	0.6439069180859244	0.8826043522783333	-	maturation of SSU-rRNA	11	P
GO:0030431	1	1	0.6439069180859244	0.8826043522783333	-	sleep	11	P
GO:1902268	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of polyamine transmembrane transport	11	P
GO:1902991	1	1	0.6439069180859244	0.8826043522783333	-	regulation of amyloid precursor protein catabolic process	11	P
GO:1902949	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of tau-protein kinase activity	11	P
GO:1902766	1	1	0.6439069180859244	0.8826043522783333	-	skeletal muscle satellite cell migration	11	P
GO:1902748	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of lens fiber cell differentiation	11	P
GO:0030948	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of vascular endothelial growth factor receptor signaling pathway	11	P
GO:0034629	1	1	0.6439069180859244	0.8826043522783333	-	cellular protein-containing complex localization	11	P
GO:0002544	1	1	0.6439069180859244	0.8826043522783333	-	chronic inflammatory response	11	P
GO:0030837	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of actin filament polymerization	11	P
GO:0033625	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of integrin activation	11	P
GO:0034136	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of toll-like receptor 2 signaling pathway	11	P
GO:0034103	1	1	0.6439069180859244	0.8826043522783333	-	regulation of tissue remodeling	11	P
GO:0034395	1	1	0.6439069180859244	0.8826043522783333	-	regulation of transcription from RNA polymerase II promoter in response to iron	11	P
GO:0051292	1	1	0.6439069180859244	0.8826043522783333	-	nuclear pore complex assembly	11	P
GO:0051247	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of protein metabolic process	11	P
GO:0051246	1	1	0.6439069180859244	0.8826043522783333	-	regulation of protein metabolic process	11	P
GO:0051224	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of protein transport	11	P
GO:0050710	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of cytokine secretion	11	P
GO:0050707	1	1	0.6439069180859244	0.8826043522783333	-	regulation of cytokine secretion	11	P
GO:0050702	1	1	0.6439069180859244	0.8826043522783333	-	interleukin-1 beta secretion	11	P
GO:2001224	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of neuron migration	11	P
GO:0035909	1	1	0.6439069180859244	0.8826043522783333	-	aorta morphogenesis	11	P
GO:1900227	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of NLRP3 inflammasome complex assembly	11	P
GO:1901687	1	1	0.6439069180859244	0.8826043522783333	-	glutathione derivative biosynthetic process	11	P
GO:1901381	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of potassium ion transmembrane transport	11	P
GO:0002906	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of mature B cell apoptotic process	11	P
GO:0002792	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of peptide secretion	11	P
GO:0003417	1	1	0.6439069180859244	0.8826043522783333	-	growth plate cartilage development	11	P
GO:0030099	1	1	0.6439069180859244	0.8826043522783333	-	myeloid cell differentiation	11	P
GO:0030326	1	1	0.6439069180859244	0.8826043522783333	-	embryonic limb morphogenesis	11	P
GO:0030178	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of Wnt signaling pathway	11	P
GO:0030214	1	1	0.6439069180859244	0.8826043522783333	-	hyaluronan catabolic process	11	P
GO:0035810	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of urine volume	11	P
GO:0035691	1	1	0.6439069180859244	0.8826043522783333	-	macrophage migration inhibitory factor signaling pathway	11	P
GO:0036035	1	1	0.6439069180859244	0.8826043522783333	-	osteoclast development	11	P
GO:0007628	1	1	0.6439069180859244	0.8826043522783333	-	adult walking behavior	11	P
GO:1902219	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of intrinsic apoptotic signaling pathway in response to osmotic stress	11	P
GO:1902065	1	1	0.6439069180859244	0.8826043522783333	-	response to L-glutamate	11	P
GO:1902042	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of extrinsic apoptotic signaling pathway via death domain receptors	11	P
GO:1901888	1	1	0.6439069180859244	0.8826043522783333	-	regulation of cell junction assembly	11	P
GO:1901741	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of myoblast fusion	11	P
GO:0002084	1	1	0.6439069180859244	0.8826043522783333	-	protein depalmitoylation	11	P
GO:1900425	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of defense response to bacterium	11	P
GO:0034101	1	1	0.6439069180859244	0.8826043522783333	-	erythrocyte homeostasis	11	P
GO:0002504	1	1	0.6439069180859244	0.8826043522783333	-	antigen processing and presentation of peptide or polysaccharide antigen via MHC class II	11	P
GO:0002467	1	1	0.6439069180859244	0.8826043522783333	-	germinal center formation	11	P
GO:0002407	1	1	0.6439069180859244	0.8826043522783333	-	dendritic cell chemotaxis	11	P
GO:0002551	1	1	0.6439069180859244	0.8826043522783333	-	mast cell chemotaxis	11	P
GO:0032925	1	1	0.6439069180859244	0.8826043522783333	-	regulation of activin receptor signaling pathway	11	P
GO:1903507	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of nucleic acid-templated transcription	11	P
GO:1903445	1	1	0.6439069180859244	0.8826043522783333	-	protein transport from ciliary membrane to plasma membrane	11	P
GO:1902622	1	1	0.6439069180859244	0.8826043522783333	-	regulation of neutrophil migration	11	P
GO:0032508	1	1	0.6439069180859244	0.8826043522783333	-	DNA duplex unwinding	11	P
GO:1900126	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of hyaluronan biosynthetic process	11	P
GO:1900029	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of ruffle assembly	11	P
GO:1900017	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of cytokine production involved in inflammatory response	11	P
GO:1901164	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of trophoblast cell migration	11	P
GO:1900924	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of glycine import across plasma membrane	11	P
GO:1900746	1	1	0.6439069180859244	0.8826043522783333	-	regulation of vascular endothelial growth factor signaling pathway	11	P
GO:1903753	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of p38MAPK cascade	11	P
GO:1903672	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of sprouting angiogenesis	11	P
GO:0033371	1	1	0.6439069180859244	0.8826043522783333	-	T cell secretory granule organization	11	P
GO:1903543	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of exosomal secretion	11	P
GO:0032918	1	1	0.6439069180859244	0.8826043522783333	-	spermidine acetylation	11	P
GO:0032872	1	1	0.6439069180859244	0.8826043522783333	-	regulation of stress-activated MAPK cascade	11	P
GO:0033026	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of mast cell apoptotic process	11	P
GO:0031146	1	1	0.6439069180859244	0.8826043522783333	-	SCF-dependent proteasomal ubiquitin-dependent protein catabolic process	11	P
GO:0032467	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of cytokinesis	11	P
GO:0032461	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of protein oligomerization	11	P
GO:0033993	1	1	0.6439069180859244	0.8826043522783333	-	response to lipid	11	P
GO:0031666	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of lipopolysaccharide-mediated signaling pathway	11	P
GO:0031638	1	1	0.6439069180859244	0.8826043522783333	-	zymogen activation	11	P
GO:0032060	1	1	0.6439069180859244	0.8826043522783333	-	bleb assembly	11	P
GO:0032024	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of insulin secretion	11	P
GO:0033617	1	1	0.6439069180859244	0.8826043522783333	-	mitochondrial respiratory chain complex IV assembly	11	P
GO:0033382	1	1	0.6439069180859244	0.8826043522783333	-	maintenance of granzyme B location in T cell secretory granule	11	P
GO:0048247	1	1	0.6439069180859244	0.8826043522783333	-	lymphocyte chemotaxis	11	P
GO:0007625	1	1	0.6439069180859244	0.8826043522783333	-	grooming behavior	11	P
GO:0008210	1	1	0.6439069180859244	0.8826043522783333	-	estrogen metabolic process	11	P
GO:0007589	1	1	0.6439069180859244	0.8826043522783333	-	body fluid secretion	11	P
GO:0008045	1	1	0.6439069180859244	0.8826043522783333	-	motor neuron axon guidance	11	P
GO:0008016	1	1	0.6439069180859244	0.8826043522783333	-	regulation of heart contraction	11	P
GO:0050672	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of lymphocyte proliferation	11	P
GO:0007195	1	1	0.6439069180859244	0.8826043522783333	-	adenylate cyclase-inhibiting dopamine receptor signaling pathway	11	P
GO:0001893	1	1	0.6439069180859244	0.8826043522783333	-	maternal placenta development	11	P
GO:0001780	1	1	0.6439069180859244	0.8826043522783333	-	neutrophil homeostasis	11	P
GO:0007520	1	1	0.6439069180859244	0.8826043522783333	-	myoblast fusion	11	P
GO:0007519	1	1	0.6439069180859244	0.8826043522783333	-	skeletal muscle tissue development	11	P
GO:0008347	1	1	0.6439069180859244	0.8826043522783333	-	glial cell migration	11	P
GO:0048266	1	1	0.6439069180859244	0.8826043522783333	-	behavioral response to pain	11	P
GO:0035020	1	1	0.6439069180859244	0.8826043522783333	-	regulation of Rac protein signal transduction	11	P
GO:0048557	1	1	0.6439069180859244	0.8826043522783333	-	embryonic digestive tract morphogenesis	11	P
GO:1990441	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of transcription from RNA polymerase II promoter in response to endoplasmic reticulum stress	11	P
GO:1990266	1	1	0.6439069180859244	0.8826043522783333	-	neutrophil migration	11	P
GO:1905908	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of amyloid fibril formation	11	P
GO:1904659	1	1	0.6439069180859244	0.8826043522783333	-	glucose transmembrane transport	11	P
GO:1904385	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to angiotensin	11	P
GO:1903902	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of viral life cycle	11	P
GO:1903898	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of PERK-mediated unfolded protein response	11	P
GO:0032502	1	1	0.6439069180859244	0.8826043522783333	-	developmental process	11	P
GO:1903670	1	1	0.6439069180859244	0.8826043522783333	-	regulation of sprouting angiogenesis	11	P
GO:1903589	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of blood vessel endothelial cell proliferation involved in sprouting angiogenesis	11	P
GO:1903553	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of extracellular exosome assembly	11	P
GO:0006370	1	1	0.6439069180859244	0.8826043522783333	-	7-methylguanosine mRNA capping	11	P
GO:0006213	1	1	0.6439069180859244	0.8826043522783333	-	pyrimidine nucleoside metabolic process	11	P
GO:0071559	1	1	0.6439069180859244	0.8826043522783333	-	response to transforming growth factor beta	11	P
GO:0071539	1	1	0.6439069180859244	0.8826043522783333	-	protein localization to centrosome	11	P
GO:0042325	1	1	0.6439069180859244	0.8826043522783333	-	regulation of phosphorylation	11	P
GO:0035278	1	1	0.6439069180859244	0.8826043522783333	-	miRNA mediated inhibition of translation	11	P
GO:0035024	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of Rho protein signal transduction	11	P
GO:0042417	1	1	0.6439069180859244	0.8826043522783333	-	dopamine metabolic process	11	P
GO:0006122	1	1	0.6439069180859244	0.8826043522783333	-	mitochondrial electron transport, ubiquinol to cytochrome c	11	P
GO:0006002	1	1	0.6439069180859244	0.8826043522783333	-	fructose 6-phosphate metabolic process	11	P
GO:0080182	1	1	0.6439069180859244	0.8826043522783333	-	histone H3-K4 trimethylation	11	P
GO:0072749	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to cytochalasin B	11	P
GO:0070528	1	1	0.6439069180859244	0.8826043522783333	-	protein kinase C signaling	11	P
GO:0070447	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of oligodendrocyte progenitor proliferation	11	P
GO:0006275	1	1	0.6439069180859244	0.8826043522783333	-	regulation of DNA replication	11	P
GO:0022417	1	1	0.6439069180859244	0.8826043522783333	-	protein maturation by protein folding	11	P
GO:0021957	1	1	0.6439069180859244	0.8826043522783333	-	corticospinal tract morphogenesis	11	P
GO:0021915	1	1	0.6439069180859244	0.8826043522783333	-	neural tube development	11	P
GO:0042594	1	1	0.6439069180859244	0.8826043522783333	-	response to starvation	11	P
GO:0042771	1	1	0.6439069180859244	0.8826043522783333	-	intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator	11	P
GO:0042759	1	1	0.6439069180859244	0.8826043522783333	-	long-chain fatty acid biosynthetic process	11	P
GO:0042699	1	1	0.6439069180859244	0.8826043522783333	-	follicle-stimulating hormone signaling pathway	11	P
GO:0019915	1	1	0.6439069180859244	0.8826043522783333	-	lipid storage	11	P
GO:0021846	1	1	0.6439069180859244	0.8826043522783333	-	cell proliferation in forebrain	11	P
GO:0030011	1	1	0.6439069180859244	0.8826043522783333	-	maintenance of cell polarity	11	P
GO:0022898	1	1	0.6439069180859244	0.8826043522783333	-	regulation of transmembrane transporter activity	11	P
GO:0022615	1	1	0.6439069180859244	0.8826043522783333	-	protein to membrane docking	11	P
GO:0030043	1	1	0.6439069180859244	0.8826043522783333	-	actin filament fragmentation	11	P
GO:0072011	1	1	0.6439069180859244	0.8826043522783333	-	glomerular endothelium development	11	P
GO:0070932	1	1	0.6439069180859244	0.8826043522783333	-	histone H3 deacetylation	11	P
GO:0043249	1	1	0.6439069180859244	0.8826043522783333	-	erythrocyte maturation	11	P
GO:0006813	1	1	0.6439069180859244	0.8826043522783333	-	potassium ion transport	11	P
GO:0006811	1	1	0.6439069180859244	0.8826043522783333	-	ion transport	11	P
GO:0006749	1	1	0.6439069180859244	0.8826043522783333	-	glutathione metabolic process	11	P
GO:0006605	1	1	0.6439069180859244	0.8826043522783333	-	protein targeting	11	P
GO:0006542	1	1	0.6439069180859244	0.8826043522783333	-	glutamine biosynthetic process	11	P
GO:0006538	1	1	0.6439069180859244	0.8826043522783333	-	glutamate catabolic process	11	P
GO:0006520	1	1	0.6439069180859244	0.8826043522783333	-	cellular amino acid metabolic process	11	P
GO:0006690	1	1	0.6439069180859244	0.8826043522783333	-	icosanoid metabolic process	11	P
GO:0043484	1	1	0.6439069180859244	0.8826043522783333	-	regulation of RNA splicing	11	P
GO:0043044	1	1	0.6439069180859244	0.8826043522783333	-	ATP-dependent chromatin remodeling	11	P
GO:0072332	1	1	0.6439069180859244	0.8826043522783333	-	intrinsic apoptotic signaling pathway by p53 class mediator	11	P
GO:0043030	1	1	0.6439069180859244	0.8826043522783333	-	regulation of macrophage activation	11	P
GO:0071801	1	1	0.6439069180859244	0.8826043522783333	-	regulation of podosome assembly	11	P
GO:0071677	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of mononuclear cell migration	11	P
GO:0006672	1	1	0.6439069180859244	0.8826043522783333	-	ceramide metabolic process	11	P
GO:0006825	1	1	0.6439069180859244	0.8826043522783333	-	copper ion transport	11	P
GO:0021756	1	1	0.6439069180859244	0.8826043522783333	-	striatum development	11	P
GO:0021549	1	1	0.6439069180859244	0.8826043522783333	-	cerebellum development	11	P
GO:0021540	1	1	0.6439069180859244	0.8826043522783333	-	corpus callosum morphogenesis	11	P
GO:0019676	1	1	0.6439069180859244	0.8826043522783333	-	ammonia assimilation cycle	11	P
GO:0019563	1	1	0.6439069180859244	0.8826043522783333	-	glycerol catabolic process	11	P
GO:0019432	1	1	0.6439069180859244	0.8826043522783333	-	triglyceride biosynthetic process	11	P
GO:0019883	1	1	0.6439069180859244	0.8826043522783333	-	antigen processing and presentation of endogenous antigen	11	P
GO:0019835	1	1	0.6439069180859244	0.8826043522783333	-	cytolysis	11	P
GO:0006306	1	1	0.6439069180859244	0.8826043522783333	-	DNA methylation	11	P
GO:0097062	1	1	0.6439069180859244	0.8826043522783333	-	dendritic spine maintenance	11	P
GO:0071353	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to interleukin-4	11	P
GO:0071476	1	1	0.6439069180859244	0.8826043522783333	-	cellular hypotonic response	11	P
GO:0040029	1	1	0.6439069180859244	0.8826043522783333	-	regulation of gene expression, epigenetic	11	P
GO:0039534	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of MDA-5 signaling pathway	11	P
GO:0042178	1	1	0.6439069180859244	0.8826043522783333	-	xenobiotic catabolic process	11	P
GO:0038027	1	1	0.6439069180859244	0.8826043522783333	-	apolipoprotein A-I-mediated signaling pathway	11	P
GO:0036499	1	1	0.6439069180859244	0.8826043522783333	-	PERK-mediated unfolded protein response	11	P
GO:0036152	1	1	0.6439069180859244	0.8826043522783333	-	phosphatidylethanolamine acyl-chain remodeling	11	P
GO:0036149	1	1	0.6439069180859244	0.8826043522783333	-	phosphatidylinositol acyl-chain remodeling	11	P
GO:0038166	1	1	0.6439069180859244	0.8826043522783333	-	angiotensin-activated signaling pathway	11	P
GO:0150077	1	1	0.6439069180859244	0.8826043522783333	-	regulation of neuroinflammatory response	11	P
GO:0140194	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of adenylate cyclase-inhibiting adrenergic receptor signaling pathway involved in heart process	11	P
GO:0002526	1	1	0.6439069180859244	0.8826043522783333	-	acute inflammatory response	11	P
GO:0071359	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to dsRNA	11	P
GO:0035821	1	1	0.6439069180859244	0.8826043522783333	-	modification of morphology or physiology of other organism	11	P
GO:0035815	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of renal sodium excretion	11	P
GO:0030833	1	1	0.6439069180859244	0.8826043522783333	-	regulation of actin filament polymerization	11	P
GO:0030728	1	1	0.6439069180859244	0.8826043522783333	-	ovulation	11	P
GO:0003382	1	1	0.6439069180859244	0.8826043522783333	-	epithelial cell morphogenesis	11	P
GO:0003333	1	1	0.6439069180859244	0.8826043522783333	-	amino acid transmembrane transport	11	P
GO:0002762	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of myeloid leukocyte differentiation	11	P
GO:0002756	1	1	0.6439069180859244	0.8826043522783333	-	MyD88-independent toll-like receptor signaling pathway	11	P
GO:0002739	1	1	0.6439069180859244	0.8826043522783333	-	regulation of cytokine secretion involved in immune response	11	P
GO:0002724	1	1	0.6439069180859244	0.8826043522783333	-	regulation of T cell cytokine production	11	P
GO:0097581	1	1	0.6439069180859244	0.8826043522783333	-	lamellipodium organization	11	P
GO:0097494	1	1	0.6439069180859244	0.8826043522783333	-	regulation of vesicle size	11	P
GO:0030199	1	1	0.6439069180859244	0.8826043522783333	-	collagen fibril organization	11	P
GO:0071673	1	1	0.6439069180859244	0.8826043522783333	-	positive regulation of smooth muscle cell chemotaxis	11	P
GO:0042982	1	1	0.6439069180859244	0.8826043522783333	-	amyloid precursor protein metabolic process	11	P
GO:0022604	1	1	0.6439069180859244	0.8826043522783333	-	regulation of cell morphogenesis	11	P
GO:0072593	1	1	0.6439069180859244	0.8826043522783333	-	reactive oxygen species metabolic process	11	P
GO:0090140	1	1	0.6439069180859244	0.8826043522783333	-	regulation of mitochondrial fission	11	P
GO:0090102	1	1	0.6439069180859244	0.8826043522783333	-	cochlea development	11	P
GO:0003332	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of extracellular matrix constituent secretion	11	P
GO:0003177	1	1	0.6439069180859244	0.8826043522783333	-	pulmonary valve development	11	P
GO:0003176	1	1	0.6439069180859244	0.8826043522783333	-	aortic valve development	11	P
GO:0030111	1	1	0.6439069180859244	0.8826043522783333	-	regulation of Wnt signaling pathway	11	P
GO:0030101	1	1	0.6439069180859244	0.8826043522783333	-	natural killer cell activation	11	P
GO:0071474	1	1	0.6439069180859244	0.8826043522783333	-	cellular hyperosmotic response	11	P
GO:0071375	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to peptide hormone stimulus	11	P
GO:0071315	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to morphine	11	P
GO:0090025	1	1	0.6439069180859244	0.8826043522783333	-	regulation of monocyte chemotaxis	11	P
GO:0090166	1	1	0.6439069180859244	0.8826043522783333	-	Golgi disassembly	11	P
GO:0090673	1	1	0.6439069180859244	0.8826043522783333	-	endothelial cell-matrix adhesion	11	P
GO:0072672	1	1	0.6439069180859244	0.8826043522783333	-	neutrophil extravasation	11	P
GO:0072656	1	1	0.6439069180859244	0.8826043522783333	-	maintenance of protein location in mitochondrion	11	P
GO:0038094	1	1	0.6439069180859244	0.8826043522783333	-	Fc-gamma receptor signaling pathway	11	P
GO:0097345	1	1	0.6439069180859244	0.8826043522783333	-	mitochondrial outer membrane permeabilization	11	P
GO:0097324	1	1	0.6439069180859244	0.8826043522783333	-	melanocyte migration	11	P
GO:0097211	1	1	0.6439069180859244	0.8826043522783333	-	cellular response to gonadotropin-releasing hormone	11	P
GO:0090324	1	1	0.6439069180859244	0.8826043522783333	-	negative regulation of oxidative phosphorylation	11	P
GO:0050900	10	16	0.6498388210564444	0.8826043522783333	-	leukocyte migration	11	P
GO:0055114	15	24	0.6542008294296476	0.8826043522783333	-	oxidation-reduction process	11	P
GO:0006914	5	8	0.6565198293732681	0.8826043522783333	-	autophagy	11	P
GO:0043547	13	21	0.6621465202763027	0.8826043522783333	-	positive regulation of GTPase activity	11	P
GO:2001237	2	3	0.6688314496926784	0.8826043522783333	-	negative regulation of extrinsic apoptotic signaling pathway	11	P
GO:0030836	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of actin filament depolymerization	11	P
GO:0001837	2	3	0.6688314496926784	0.8826043522783333	-	epithelial to mesenchymal transition	11	P
GO:0007059	2	3	0.6688314496926784	0.8826043522783333	-	chromosome segregation	11	P
GO:0045732	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of protein catabolic process	11	P
GO:1903076	2	3	0.6688314496926784	0.8826043522783333	-	regulation of protein localization to plasma membrane	11	P
GO:2001046	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of integrin-mediated signaling pathway	11	P
GO:0032967	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of collagen biosynthetic process	11	P
GO:0001895	2	3	0.6688314496926784	0.8826043522783333	-	retina homeostasis	11	P
GO:0007584	2	3	0.6688314496926784	0.8826043522783333	-	response to nutrient	11	P
GO:2000249	2	3	0.6688314496926784	0.8826043522783333	-	regulation of actin cytoskeleton reorganization	11	P
GO:0061025	2	3	0.6688314496926784	0.8826043522783333	-	membrane fusion	11	P
GO:0051893	2	3	0.6688314496926784	0.8826043522783333	-	regulation of focal adhesion assembly	11	P
GO:0032802	2	3	0.6688314496926784	0.8826043522783333	-	low-density lipoprotein particle receptor catabolic process	11	P
GO:0051091	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of DNA-binding transcription factor activity	11	P
GO:0033138	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of peptidyl-serine phosphorylation	11	P
GO:1904706	2	3	0.6688314496926784	0.8826043522783333	-	negative regulation of vascular smooth muscle cell proliferation	11	P
GO:0070935	2	3	0.6688314496926784	0.8826043522783333	-	3'-UTR-mediated mRNA stabilization	11	P
GO:0071300	2	3	0.6688314496926784	0.8826043522783333	-	cellular response to retinoic acid	11	P
GO:0048488	2	3	0.6688314496926784	0.8826043522783333	-	synaptic vesicle endocytosis	11	P
GO:0030595	2	3	0.6688314496926784	0.8826043522783333	-	leukocyte chemotaxis	11	P
GO:0034613	2	3	0.6688314496926784	0.8826043522783333	-	cellular protein localization	11	P
GO:0030889	2	3	0.6688314496926784	0.8826043522783333	-	negative regulation of B cell proliferation	11	P
GO:0006644	2	3	0.6688314496926784	0.8826043522783333	-	phospholipid metabolic process	11	P
GO:0006596	2	3	0.6688314496926784	0.8826043522783333	-	polyamine biosynthetic process	11	P
GO:0006816	2	3	0.6688314496926784	0.8826043522783333	-	calcium ion transport	11	P
GO:0002479	2	3	0.6688314496926784	0.8826043522783333	-	antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent	11	P
GO:0042327	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of phosphorylation	11	P
GO:0035162	2	3	0.6688314496926784	0.8826043522783333	-	embryonic hemopoiesis	11	P
GO:1901223	2	3	0.6688314496926784	0.8826043522783333	-	negative regulation of NIK/NF-kappaB signaling	11	P
GO:0097435	2	3	0.6688314496926784	0.8826043522783333	-	supramolecular fiber organization	11	P
GO:0045648	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of erythrocyte differentiation	11	P
GO:0045727	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of translation	11	P
GO:0019882	2	3	0.6688314496926784	0.8826043522783333	-	antigen processing and presentation	11	P
GO:0006486	2	3	0.6688314496926784	0.8826043522783333	-	protein glycosylation	11	P
GO:0002248	2	3	0.6688314496926784	0.8826043522783333	-	connective tissue replacement involved in inflammatory response wound healing	11	P
GO:0090201	2	3	0.6688314496926784	0.8826043522783333	-	negative regulation of release of cytochrome c from mitochondria	11	P
GO:0090051	2	3	0.6688314496926784	0.8826043522783333	-	negative regulation of cell migration involved in sprouting angiogenesis	11	P
GO:0035023	2	3	0.6688314496926784	0.8826043522783333	-	regulation of Rho protein signal transduction	11	P
GO:0016064	2	3	0.6688314496926784	0.8826043522783333	-	immunoglobulin mediated immune response	11	P
GO:0010507	2	3	0.6688314496926784	0.8826043522783333	-	negative regulation of autophagy	11	P
GO:0007200	2	3	0.6688314496926784	0.8826043522783333	-	phospholipase C-activating G protein-coupled receptor signaling pathway	11	P
GO:0051592	2	3	0.6688314496926784	0.8826043522783333	-	response to calcium ion	11	P
GO:0070268	2	3	0.6688314496926784	0.8826043522783333	-	cornification	11	P
GO:0007249	2	3	0.6688314496926784	0.8826043522783333	-	I-kappaB kinase/NF-kappaB signaling	11	P
GO:0055010	2	3	0.6688314496926784	0.8826043522783333	-	ventricular cardiac muscle tissue morphogenesis	11	P
GO:2000178	2	3	0.6688314496926784	0.8826043522783333	-	negative regulation of neural precursor cell proliferation	11	P
GO:0032456	2	3	0.6688314496926784	0.8826043522783333	-	endocytic recycling	11	P
GO:0000398	2	3	0.6688314496926784	0.8826043522783333	-	mRNA splicing, via spliceosome	11	P
GO:0008154	2	3	0.6688314496926784	0.8826043522783333	-	actin polymerization or depolymerization	11	P
GO:2000379	2	3	0.6688314496926784	0.8826043522783333	-	positive regulation of reactive oxygen species metabolic process	11	P
GO:0001570	2	3	0.6688314496926784	0.8826043522783333	-	vasculogenesis	11	P
GO:0050918	2	3	0.6688314496926784	0.8826043522783333	-	positive chemotaxis	11	P
GO:0001843	3	5	0.6918106156380359	0.8826043522783333	-	neural tube closure	11	P
GO:0032436	3	5	0.6918106156380359	0.8826043522783333	-	positive regulation of proteasomal ubiquitin-dependent protein catabolic process	11	P
GO:0045860	3	5	0.6918106156380359	0.8826043522783333	-	positive regulation of protein kinase activity	11	P
GO:0046718	3	5	0.6918106156380359	0.8826043522783333	-	viral entry into host cell	11	P
GO:0009653	3	5	0.6918106156380359	0.8826043522783333	-	anatomical structure morphogenesis	11	P
GO:0048678	3	5	0.6918106156380359	0.8826043522783333	-	response to axon injury	11	P
GO:0043491	3	5	0.6918106156380359	0.8826043522783333	-	protein kinase B signaling	11	P
GO:0051496	3	5	0.6918106156380359	0.8826043522783333	-	positive regulation of stress fiber assembly	11	P
GO:0021762	3	5	0.6918106156380359	0.8826043522783333	-	substantia nigra development	11	P
GO:0008361	3	5	0.6918106156380359	0.8826043522783333	-	regulation of cell size	11	P
GO:0048208	3	5	0.6918106156380359	0.8826043522783333	-	COPII vesicle coating	11	P
GO:0016525	3	5	0.6918106156380359	0.8826043522783333	-	negative regulation of angiogenesis	11	P
GO:0043149	3	5	0.6918106156380359	0.8826043522783333	-	stress fiber assembly	11	P
GO:0007507	4	7	0.7120935426440396	0.8826043522783333	-	heart development	11	P
GO:0048146	4	7	0.7120935426440396	0.8826043522783333	-	positive regulation of fibroblast proliferation	11	P
GO:0007204	4	7	0.7120935426440396	0.8826043522783333	-	positive regulation of cytosolic calcium ion concentration	11	P
GO:0030041	4	7	0.7120935426440396	0.8826043522783333	-	actin filament polymerization	11	P
GO:0014068	5	9	0.7301374558891018	0.8826043522783333	-	positive regulation of phosphatidylinositol 3-kinase signaling	11	P
GO:0120162	5	9	0.7301374558891018	0.8826043522783333	-	positive regulation of cold-induced thermogenesis	11	P
GO:0007015	5	9	0.7301374558891018	0.8826043522783333	-	actin filament organization	11	P
GO:0030334	6	11	0.746366115697487	0.8826043522783333	-	regulation of cell migration	11	P
GO:0007050	6	11	0.746366115697487	0.8826043522783333	-	cell cycle arrest	11	P
GO:0042493	7	13	0.761100221037226	0.8826043522783333	-	response to drug	11	P
GO:0007275	7	13	0.761100221037226	0.8826043522783333	-	multicellular organism development	11	P
GO:0030218	2	4	0.7722156554964141	0.8826043522783333	-	erythrocyte differentiation	11	P
GO:0061077	2	4	0.7722156554964141	0.8826043522783333	-	chaperone-mediated protein folding	11	P
GO:0010811	2	4	0.7722156554964141	0.8826043522783333	-	positive regulation of cell-substrate adhesion	11	P
GO:0006906	2	4	0.7722156554964141	0.8826043522783333	-	vesicle fusion	11	P
GO:0001764	2	4	0.7722156554964141	0.8826043522783333	-	neuron migration	11	P
GO:0051924	2	4	0.7722156554964141	0.8826043522783333	-	regulation of calcium ion transport	11	P
GO:0050679	2	4	0.7722156554964141	0.8826043522783333	-	positive regulation of epithelial cell proliferation	11	P
GO:0007612	2	4	0.7722156554964141	0.8826043522783333	-	learning	11	P
GO:1903077	2	4	0.7722156554964141	0.8826043522783333	-	negative regulation of protein localization to plasma membrane	11	P
GO:0034504	2	4	0.7722156554964141	0.8826043522783333	-	protein localization to nucleus	11	P
GO:0031647	2	4	0.7722156554964141	0.8826043522783333	-	regulation of protein stability	11	P
GO:2000145	2	4	0.7722156554964141	0.8826043522783333	-	regulation of cell motility	11	P
GO:0051216	2	4	0.7722156554964141	0.8826043522783333	-	cartilage development	11	P
GO:0030324	2	4	0.7722156554964141	0.8826043522783333	-	lung development	11	P
GO:0090307	2	4	0.7722156554964141	0.8826043522783333	-	mitotic spindle assembly	11	P
GO:0006687	2	4	0.7722156554964141	0.8826043522783333	-	glycosphingolipid metabolic process	11	P
GO:1900740	2	4	0.7722156554964141	0.8826043522783333	-	positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway	11	P
GO:0009887	3	6	0.7750390068324627	0.8826043522783333	-	animal organ morphogenesis	11	P
GO:0007159	3	6	0.7750390068324627	0.8826043522783333	-	leukocyte cell-cell adhesion	11	P
GO:0048015	3	6	0.7750390068324627	0.8826043522783333	-	phosphatidylinositol-mediated signaling	11	P
GO:0051384	3	6	0.7750390068324627	0.8826043522783333	-	response to glucocorticoid	11	P
GO:0030220	3	6	0.7750390068324627	0.8826043522783333	-	platelet formation	11	P
GO:0043406	3	6	0.7750390068324627	0.8826043522783333	-	positive regulation of MAP kinase activity	11	P
GO:0009967	3	6	0.7750390068324627	0.8826043522783333	-	positive regulation of signal transduction	11	P
GO:0001701	4	8	0.7825872922362739	0.8826043522783333	-	in utero embryonic development	11	P
GO:0031334	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of protein complex assembly	11	P
GO:0030517	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of axon extension	11	P
GO:0034765	1	2	0.7877137396281472	0.8826043522783333	-	regulation of ion transmembrane transport	11	P
GO:0002052	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of neuroblast proliferation	11	P
GO:1902230	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of intrinsic apoptotic signaling pathway in response to DNA damage	11	P
GO:0002513	1	2	0.7877137396281472	0.8826043522783333	-	tolerance induction to self antigen	11	P
GO:0018107	1	2	0.7877137396281472	0.8826043522783333	-	peptidyl-threonine phosphorylation	11	P
GO:0043473	1	2	0.7877137396281472	0.8826043522783333	-	pigmentation	11	P
GO:0006521	1	2	0.7877137396281472	0.8826043522783333	-	regulation of cellular amino acid metabolic process	11	P
GO:0048278	1	2	0.7877137396281472	0.8826043522783333	-	vesicle docking	11	P
GO:0031032	1	2	0.7877137396281472	0.8826043522783333	-	actomyosin structure organization	11	P
GO:0030949	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of vascular endothelial growth factor receptor signaling pathway	11	P
GO:0070848	1	2	0.7877137396281472	0.8826043522783333	-	response to growth factor	11	P
GO:0071243	1	2	0.7877137396281472	0.8826043522783333	-	cellular response to arsenic-containing substance	11	P
GO:0043032	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of macrophage activation	11	P
GO:0001503	1	2	0.7877137396281472	0.8826043522783333	-	ossification	11	P
GO:1905602	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of receptor-mediated endocytosis involved in cholesterol transport	11	P
GO:1902476	1	2	0.7877137396281472	0.8826043522783333	-	chloride transmembrane transport	11	P
GO:2000010	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of protein localization to cell surface	11	P
GO:0006413	1	2	0.7877137396281472	0.8826043522783333	-	translational initiation	11	P
GO:0032435	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of proteasomal ubiquitin-dependent protein catabolic process	11	P
GO:1900745	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of p38MAPK cascade	11	P
GO:0033628	1	2	0.7877137396281472	0.8826043522783333	-	regulation of cell adhesion mediated by integrin	11	P
GO:0033674	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of kinase activity	11	P
GO:0048813	1	2	0.7877137396281472	0.8826043522783333	-	dendrite morphogenesis	11	P
GO:0006364	1	2	0.7877137396281472	0.8826043522783333	-	rRNA processing	11	P
GO:0006368	1	2	0.7877137396281472	0.8826043522783333	-	transcription elongation from RNA polymerase II promoter	11	P
GO:0070560	1	2	0.7877137396281472	0.8826043522783333	-	protein secretion by platelet	11	P
GO:0006412	1	2	0.7877137396281472	0.8826043522783333	-	translation	11	P
GO:2000060	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of ubiquitin-dependent protein catabolic process	11	P
GO:0006417	1	2	0.7877137396281472	0.8826043522783333	-	regulation of translation	11	P
GO:0042246	1	2	0.7877137396281472	0.8826043522783333	-	tissue regeneration	11	P
GO:0019934	1	2	0.7877137396281472	0.8826043522783333	-	cGMP-mediated signaling	11	P
GO:0043388	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of DNA binding	11	P
GO:0043393	1	2	0.7877137396281472	0.8826043522783333	-	regulation of protein binding	11	P
GO:0019852	1	2	0.7877137396281472	0.8826043522783333	-	L-ascorbic acid metabolic process	11	P
GO:0019371	1	2	0.7877137396281472	0.8826043522783333	-	cyclooxygenase pathway	11	P
GO:0016358	1	2	0.7877137396281472	0.8826043522783333	-	dendrite development	11	P
GO:0001812	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of type I hypersensitivity	11	P
GO:0033591	1	2	0.7877137396281472	0.8826043522783333	-	response to L-ascorbic acid	11	P
GO:0031293	1	2	0.7877137396281472	0.8826043522783333	-	membrane protein intracellular domain proteolysis	11	P
GO:0033137	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of peptidyl-serine phosphorylation	11	P
GO:0033211	1	2	0.7877137396281472	0.8826043522783333	-	adiponectin-activated signaling pathway	11	P
GO:1900025	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of substrate adhesion-dependent cell spreading	11	P
GO:0032463	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of protein homooligomerization	11	P
GO:0007190	1	2	0.7877137396281472	0.8826043522783333	-	activation of adenylate cyclase activity	11	P
GO:0050901	1	2	0.7877137396281472	0.8826043522783333	-	leukocyte tethering or rolling	11	P
GO:0051209	1	2	0.7877137396281472	0.8826043522783333	-	release of sequestered calcium ion into cytosol	11	P
GO:0031629	1	2	0.7877137396281472	0.8826043522783333	-	synaptic vesicle fusion to presynaptic active zone membrane	11	P
GO:0051085	1	2	0.7877137396281472	0.8826043522783333	-	chaperone cofactor-dependent protein refolding	11	P
GO:0032464	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of protein homooligomerization	11	P
GO:0032465	1	2	0.7877137396281472	0.8826043522783333	-	regulation of cytokinesis	11	P
GO:1900024	1	2	0.7877137396281472	0.8826043522783333	-	regulation of substrate adhesion-dependent cell spreading	11	P
GO:0032804	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of low-density lipoprotein particle receptor catabolic process	11	P
GO:0001523	1	2	0.7877137396281472	0.8826043522783333	-	retinoid metabolic process	11	P
GO:0001805	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of type III hypersensitivity	11	P
GO:0051873	1	2	0.7877137396281472	0.8826043522783333	-	killing by host of symbiont cells	11	P
GO:0009629	1	2	0.7877137396281472	0.8826043522783333	-	response to gravity	11	P
GO:1903955	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of protein targeting to mitochondrion	11	P
GO:1990314	1	2	0.7877137396281472	0.8826043522783333	-	cellular response to insulin-like growth factor stimulus	11	P
GO:1905581	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of low-density lipoprotein particle clearance	11	P
GO:1903829	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of cellular protein localization	11	P
GO:0008356	1	2	0.7877137396281472	0.8826043522783333	-	asymmetric cell division	11	P
GO:0007528	1	2	0.7877137396281472	0.8826043522783333	-	neuromuscular junction development	11	P
GO:0007368	1	2	0.7877137396281472	0.8826043522783333	-	determination of left/right symmetry	11	P
GO:0048642	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of skeletal muscle tissue development	11	P
GO:0050919	1	2	0.7877137396281472	0.8826043522783333	-	negative chemotaxis	11	P
GO:0006458	1	2	0.7877137396281472	0.8826043522783333	-	'de novo' protein folding	11	P
GO:0008652	1	2	0.7877137396281472	0.8826043522783333	-	cellular amino acid biosynthetic process	11	P
GO:0050849	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of calcium-mediated signaling	11	P
GO:0009060	1	2	0.7877137396281472	0.8826043522783333	-	aerobic respiration	11	P
GO:0009306	1	2	0.7877137396281472	0.8826043522783333	-	protein secretion	11	P
GO:0008156	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of DNA replication	11	P
GO:0050732	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of peptidyl-tyrosine phosphorylation	11	P
GO:0032026	1	2	0.7877137396281472	0.8826043522783333	-	response to magnesium ion	11	P
GO:0032959	1	2	0.7877137396281472	0.8826043522783333	-	inositol trisphosphate biosynthetic process	11	P
GO:0060586	1	2	0.7877137396281472	0.8826043522783333	-	multicellular organismal iron ion homeostasis	11	P
GO:0010594	1	2	0.7877137396281472	0.8826043522783333	-	regulation of endothelial cell migration	11	P
GO:0060050	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of protein glycosylation	11	P
GO:0007183	1	2	0.7877137396281472	0.8826043522783333	-	SMAD protein complex assembly	11	P
GO:0055119	1	2	0.7877137396281472	0.8826043522783333	-	relaxation of cardiac muscle	11	P
GO:0007032	1	2	0.7877137396281472	0.8826043522783333	-	endosome organization	11	P
GO:0007034	1	2	0.7877137396281472	0.8826043522783333	-	vacuolar transport	11	P
GO:0045667	1	2	0.7877137396281472	0.8826043522783333	-	regulation of osteoblast differentiation	11	P
GO:0060996	1	2	0.7877137396281472	0.8826043522783333	-	dendritic spine development	11	P
GO:0061028	1	2	0.7877137396281472	0.8826043522783333	-	establishment of endothelial barrier	11	P
GO:0010591	1	2	0.7877137396281472	0.8826043522783333	-	regulation of lamellipodium assembly	11	P
GO:0060347	1	2	0.7877137396281472	0.8826043522783333	-	heart trabecula formation	11	P
GO:0060369	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of Fc receptor mediated stimulatory signaling pathway	11	P
GO:0010632	1	2	0.7877137396281472	0.8826043522783333	-	regulation of epithelial cell migration	11	P
GO:0010633	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of epithelial cell migration	11	P
GO:0010716	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of extracellular matrix disassembly	11	P
GO:0010572	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of platelet activation	11	P
GO:0043518	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of DNA damage response, signal transduction by p53 class mediator	11	P
GO:0002902	1	2	0.7877137396281472	0.8826043522783333	-	regulation of B cell apoptotic process	11	P
GO:0060100	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of phagocytosis, engulfment	11	P
GO:2000811	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of anoikis	11	P
GO:2000394	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of lamellipodium morphogenesis	11	P
GO:0001889	1	2	0.7877137396281472	0.8826043522783333	-	liver development	11	P
GO:0001961	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of cytokine-mediated signaling pathway	11	P
GO:0000060	1	2	0.7877137396281472	0.8826043522783333	-	obsolete protein import into nucleus, translocation	11	P
GO:2001240	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of extrinsic apoptotic signaling pathway in absence of ligand	11	P
GO:0001516	1	2	0.7877137396281472	0.8826043522783333	-	prostaglandin biosynthetic process	11	P
GO:0050892	1	2	0.7877137396281472	0.8826043522783333	-	intestinal absorption	11	P
GO:0051881	1	2	0.7877137396281472	0.8826043522783333	-	regulation of mitochondrial membrane potential	11	P
GO:1901215	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of neuron death	11	P
GO:0010761	1	2	0.7877137396281472	0.8826043522783333	-	fibroblast migration	11	P
GO:0046135	1	2	0.7877137396281472	0.8826043522783333	-	pyrimidine nucleoside catabolic process	11	P
GO:0010718	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of epithelial to mesenchymal transition	11	P
GO:0010737	1	2	0.7877137396281472	0.8826043522783333	-	protein kinase A signaling	11	P
GO:0010762	1	2	0.7877137396281472	0.8826043522783333	-	regulation of fibroblast migration	11	P
GO:0045921	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of exocytosis	11	P
GO:0045930	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of mitotic cell cycle	11	P
GO:0045777	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of blood pressure	11	P
GO:0006970	1	2	0.7877137396281472	0.8826043522783333	-	response to osmotic stress	11	P
GO:0030325	1	2	0.7877137396281472	0.8826043522783333	-	adrenal gland development	11	P
GO:0090150	1	2	0.7877137396281472	0.8826043522783333	-	establishment of protein localization to membrane	11	P
GO:0030206	1	2	0.7877137396281472	0.8826043522783333	-	chondroitin sulfate biosynthetic process	11	P
GO:0045198	1	2	0.7877137396281472	0.8826043522783333	-	establishment of epithelial cell apical/basal polarity	11	P
GO:0012501	1	2	0.7877137396281472	0.8826043522783333	-	programmed cell death	11	P
GO:0014065	1	2	0.7877137396281472	0.8826043522783333	-	phosphatidylinositol 3-kinase signaling	11	P
GO:0006930	1	2	0.7877137396281472	0.8826043522783333	-	substrate-dependent cell migration, cell extension	11	P
GO:0051492	1	2	0.7877137396281472	0.8826043522783333	-	regulation of stress fiber assembly	11	P
GO:0051280	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of release of sequestered calcium ion into cytosol	11	P
GO:0030511	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of transforming growth factor beta receptor signaling pathway	11	P
GO:0010923	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of phosphatase activity	11	P
GO:0010944	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of transcription by competitive promoter binding	11	P
GO:0014909	1	2	0.7877137396281472	0.8826043522783333	-	smooth muscle cell migration	11	P
GO:0010165	1	2	0.7877137396281472	0.8826043522783333	-	response to X-ray	11	P
GO:0009743	1	2	0.7877137396281472	0.8826043522783333	-	response to carbohydrate	11	P
GO:0048167	1	2	0.7877137396281472	0.8826043522783333	-	regulation of synaptic plasticity	11	P
GO:0015871	1	2	0.7877137396281472	0.8826043522783333	-	choline transport	11	P
GO:0030042	1	2	0.7877137396281472	0.8826043522783333	-	actin filament depolymerization	11	P
GO:0030217	1	2	0.7877137396281472	0.8826043522783333	-	T cell differentiation	11	P
GO:0099509	1	2	0.7877137396281472	0.8826043522783333	-	regulation of presynaptic cytosolic calcium ion concentration	11	P
GO:0005977	1	2	0.7877137396281472	0.8826043522783333	-	glycogen metabolic process	11	P
GO:0097368	1	2	0.7877137396281472	0.8826043522783333	-	establishment of Sertoli cell barrier	11	P
GO:0042391	1	2	0.7877137396281472	0.8826043522783333	-	regulation of membrane potential	11	P
GO:0042059	1	2	0.7877137396281472	0.8826043522783333	-	negative regulation of epidermal growth factor receptor signaling pathway	11	P
GO:0042119	1	2	0.7877137396281472	0.8826043522783333	-	neutrophil activation	11	P
GO:0035774	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of insulin secretion involved in cellular response to glucose stimulus	11	P
GO:0030208	1	2	0.7877137396281472	0.8826043522783333	-	dermatan sulfate biosynthetic process	11	P
GO:0021943	1	2	0.7877137396281472	0.8826043522783333	-	formation of radial glial scaffolds	11	P
GO:0035646	1	2	0.7877137396281472	0.8826043522783333	-	endosome to melanosome transport	11	P
GO:0035987	1	2	0.7877137396281472	0.8826043522783333	-	endodermal cell differentiation	11	P
GO:0090398	1	2	0.7877137396281472	0.8826043522783333	-	cellular senescence	11	P
GO:0097194	1	2	0.7877137396281472	0.8826043522783333	-	execution phase of apoptosis	11	P
GO:0097352	1	2	0.7877137396281472	0.8826043522783333	-	autophagosome maturation	11	P
GO:0090263	1	2	0.7877137396281472	0.8826043522783333	-	positive regulation of canonical Wnt signaling pathway	11	P
GO:0090162	1	2	0.7877137396281472	0.8826043522783333	-	establishment of epithelial cell polarity	11	P
GO:0007399	5	10	0.7915952837839347	0.8849035106090057	-	nervous system development	11	P
GO:0043410	5	10	0.7915952837839347	0.8849035106090057	-	positive regulation of MAPK cascade	11	P
GO:0030308	5	10	0.7915952837839347	0.8849035106090057	-	negative regulation of cell growth	11	P
GO:0001934	10	19	0.7984680039747563	0.89189920998258	-	positive regulation of protein phosphorylation	11	P
GO:0001666	6	12	0.8009535263136393	0.8939873589854542	-	response to hypoxia	11	P
GO:0065003	4	9	0.838617275730943	0.9231522423082265	-	protein-containing complex assembly	11	P
GO:0043154	4	9	0.838617275730943	0.9231522423082265	-	negative regulation of cysteine-type endopeptidase activity involved in apoptotic process	11	P
GO:0030032	3	7	0.8388501257346759	0.9231522423082265	-	lamellipodium assembly	11	P
GO:0016567	3	7	0.8388501257346759	0.9231522423082265	-	protein ubiquitination	11	P
GO:0007173	3	7	0.8388501257346759	0.9231522423082265	-	epidermal growth factor receptor signaling pathway	11	P
GO:0007179	5	11	0.8415362291742655	0.9231522423082265	-	transforming growth factor beta receptor signaling pathway	11	P
GO:0060271	5	11	0.8415362291742655	0.9231522423082265	-	cilium assembly	11	P
GO:0030335	11	22	0.8438860825293462	0.9231522423082265	-	positive regulation of cell migration	11	P
GO:0007613	2	5	0.8462211835119069	0.9231522423082265	-	memory	11	P
GO:0030049	2	5	0.8462211835119069	0.9231522423082265	-	muscle filament sliding	11	P
GO:0043552	2	5	0.8462211835119069	0.9231522423082265	-	positive regulation of phosphatidylinositol 3-kinase activity	11	P
GO:0017148	2	5	0.8462211835119069	0.9231522423082265	-	negative regulation of translation	11	P
GO:0045184	2	5	0.8462211835119069	0.9231522423082265	-	establishment of protein localization	11	P
GO:0007189	2	5	0.8462211835119069	0.9231522423082265	-	adenylate cyclase-activating G protein-coupled receptor signaling pathway	11	P
GO:0030100	2	5	0.8462211835119069	0.9231522423082265	-	regulation of endocytosis	11	P
GO:0042110	2	5	0.8462211835119069	0.9231522423082265	-	T cell activation	11	P
GO:0001649	2	5	0.8462211835119069	0.9231522423082265	-	osteoblast differentiation	11	P
GO:0042542	2	5	0.8462211835119069	0.9231522423082265	-	response to hydrogen peroxide	11	P
GO:0031954	2	5	0.8462211835119069	0.9231522423082265	-	positive regulation of protein autophosphorylation	11	P
GO:0043297	1	3	0.8735272423771157	0.9231522423082265	-	apical junction assembly	11	P
GO:0030509	1	3	0.8735272423771157	0.9231522423082265	-	BMP signaling pathway	11	P
GO:0051044	1	3	0.8735272423771157	0.9231522423082265	-	positive regulation of membrane protein ectodomain proteolysis	11	P
GO:0009952	1	3	0.8735272423771157	0.9231522423082265	-	anterior/posterior pattern specification	11	P
GO:0046627	1	3	0.8735272423771157	0.9231522423082265	-	negative regulation of insulin receptor signaling pathway	11	P
GO:0071850	1	3	0.8735272423771157	0.9231522423082265	-	mitotic cell cycle arrest	11	P
GO:0120163	1	3	0.8735272423771157	0.9231522423082265	-	negative regulation of cold-induced thermogenesis	11	P
GO:0002931	1	3	0.8735272423771157	0.9231522423082265	-	response to ischemia	11	P
GO:0030194	1	3	0.8735272423771157	0.9231522423082265	-	positive regulation of blood coagulation	11	P
GO:0010917	1	3	0.8735272423771157	0.9231522423082265	-	negative regulation of mitochondrial membrane potential	11	P
GO:0007052	1	3	0.8735272423771157	0.9231522423082265	-	mitotic spindle organization	11	P
GO:0060048	1	3	0.8735272423771157	0.9231522423082265	-	cardiac muscle contraction	11	P
GO:0003007	1	3	0.8735272423771157	0.9231522423082265	-	heart morphogenesis	11	P
GO:0003151	1	3	0.8735272423771157	0.9231522423082265	-	outflow tract morphogenesis	11	P
GO:0021766	1	3	0.8735272423771157	0.9231522423082265	-	hippocampus development	11	P
GO:0051968	1	3	0.8735272423771157	0.9231522423082265	-	positive regulation of synaptic transmission, glutamatergic	11	P
GO:0035264	1	3	0.8735272423771157	0.9231522423082265	-	multicellular organism growth	11	P
GO:0030033	1	3	0.8735272423771157	0.9231522423082265	-	microvillus assembly	11	P
GO:0042035	1	3	0.8735272423771157	0.9231522423082265	-	regulation of cytokine biosynthetic process	11	P
GO:0035025	1	3	0.8735272423771157	0.9231522423082265	-	positive regulation of Rho protein signal transduction	11	P
GO:0035094	1	3	0.8735272423771157	0.9231522423082265	-	response to nicotine	11	P
GO:0001836	1	3	0.8735272423771157	0.9231522423082265	-	release of cytochrome c from mitochondria	11	P
GO:0030030	1	3	0.8735272423771157	0.9231522423082265	-	cell projection organization	11	P
GO:0007599	1	3	0.8735272423771157	0.9231522423082265	-	hemostasis	11	P
GO:0006120	1	3	0.8735272423771157	0.9231522423082265	-	mitochondrial electron transport, NADH to ubiquinone	11	P
GO:0008542	1	3	0.8735272423771157	0.9231522423082265	-	visual learning	11	P
GO:0051016	1	3	0.8735272423771157	0.9231522423082265	-	barbed-end actin filament capping	11	P
GO:0032332	1	3	0.8735272423771157	0.9231522423082265	-	positive regulation of chondrocyte differentiation	11	P
GO:2000134	1	3	0.8735272423771157	0.9231522423082265	-	negative regulation of G1/S transition of mitotic cell cycle	11	P
GO:0060964	1	3	0.8735272423771157	0.9231522423082265	-	regulation of gene silencing by miRNA	11	P
GO:0061512	1	3	0.8735272423771157	0.9231522423082265	-	protein localization to cilium	11	P
GO:0001973	1	3	0.8735272423771157	0.9231522423082265	-	adenosine receptor signaling pathway	11	P
GO:0030512	1	3	0.8735272423771157	0.9231522423082265	-	negative regulation of transforming growth factor beta receptor signaling pathway	11	P
GO:0030521	1	3	0.8735272423771157	0.9231522423082265	-	androgen receptor signaling pathway	11	P
GO:0035584	1	3	0.8735272423771157	0.9231522423082265	-	calcium-mediated signaling using intracellular calcium source	11	P
GO:0006351	1	3	0.8735272423771157	0.9231522423082265	-	transcription, DNA-templated	11	P
GO:0071560	1	3	0.8735272423771157	0.9231522423082265	-	cellular response to transforming growth factor beta stimulus	11	P
GO:0000413	1	3	0.8735272423771157	0.9231522423082265	-	protein peptidyl-prolyl isomerization	11	P
GO:0060391	1	3	0.8735272423771157	0.9231522423082265	-	positive regulation of SMAD protein signal transduction	11	P
GO:0060315	1	3	0.8735272423771157	0.9231522423082265	-	negative regulation of ryanodine-sensitive calcium-release channel activity	11	P
GO:0051895	1	3	0.8735272423771157	0.9231522423082265	-	negative regulation of focal adhesion assembly	11	P
GO:0060325	1	3	0.8735272423771157	0.9231522423082265	-	face morphogenesis	11	P
GO:0010759	1	3	0.8735272423771157	0.9231522423082265	-	positive regulation of macrophage chemotaxis	11	P
GO:0006283	1	3	0.8735272423771157	0.9231522423082265	-	transcription-coupled nucleotide-excision repair	11	P
GO:0030010	1	3	0.8735272423771157	0.9231522423082265	-	establishment of cell polarity	11	P
GO:0048821	1	3	0.8735272423771157	0.9231522423082265	-	erythrocyte development	11	P
GO:0010862	1	3	0.8735272423771157	0.9231522423082265	-	positive regulation of pathway-restricted SMAD protein phosphorylation	11	P
GO:0007029	1	3	0.8735272423771157	0.9231522423082265	-	endoplasmic reticulum organization	11	P
GO:0002553	1	3	0.8735272423771157	0.9231522423082265	-	histamine secretion by mast cell	11	P
GO:0031589	1	3	0.8735272423771157	0.9231522423082265	-	cell-substrate adhesion	11	P
GO:1902004	1	3	0.8735272423771157	0.9231522423082265	-	positive regulation of amyloid-beta formation	11	P
GO:0048873	1	3	0.8735272423771157	0.9231522423082265	-	homeostasis of number of cells within a tissue	11	P
GO:0042552	1	3	0.8735272423771157	0.9231522423082265	-	myelination	11	P
GO:0042472	1	3	0.8735272423771157	0.9231522423082265	-	inner ear morphogenesis	11	P
GO:0007160	5	12	0.8811904431229233	0.9300838332280545	-	cell-matrix adhesion	11	P
GO:0016055	4	10	0.8820092036676795	0.9300838332280545	-	Wnt signaling pathway	11	P
GO:0007411	4	10	0.8820092036676795	0.9300838332280545	-	axon guidance	11	P
GO:0042060	3	8	0.8863838443936645	0.93401812506551	-	wound healing	11	P
GO:0098869	2	6	0.8976913728031972	0.9404694454421944	-	cellular oxidant detoxification	11	P
GO:0043536	2	6	0.8976913728031972	0.9404694454421944	-	positive regulation of blood vessel endothelial cell migration	11	P
GO:0045669	2	6	0.8976913728031972	0.9404694454421944	-	positive regulation of osteoblast differentiation	11	P
GO:0007409	2	6	0.8976913728031972	0.9404694454421944	-	axonogenesis	11	P
GO:0000082	2	6	0.8976913728031972	0.9404694454421944	-	G1/S transition of mitotic cell cycle	11	P
GO:0051493	2	6	0.8976913728031972	0.9404694454421944	-	regulation of cytoskeleton organization	11	P
GO:0001975	2	6	0.8976913728031972	0.9404694454421944	-	response to amphetamine	11	P
GO:0042113	2	6	0.8976913728031972	0.9404694454421944	-	B cell activation	11	P
GO:0016192	4	11	0.9148869927163139	0.9542968901034669	-	vesicle-mediated transport	11	P
GO:0032981	1	4	0.9247011905482249	0.9542968901034669	-	mitochondrial respiratory chain complex I assembly	11	P
GO:0045665	1	4	0.9247011905482249	0.9542968901034669	-	negative regulation of neuron differentiation	11	P
GO:0045599	1	4	0.9247011905482249	0.9542968901034669	-	negative regulation of fat cell differentiation	11	P
GO:0034446	1	4	0.9247011905482249	0.9542968901034669	-	substrate adhesion-dependent cell spreading	11	P
GO:0050890	1	4	0.9247011905482249	0.9542968901034669	-	cognition	11	P
GO:0008584	1	4	0.9247011905482249	0.9542968901034669	-	male gonad development	11	P
GO:0007595	1	4	0.9247011905482249	0.9542968901034669	-	lactation	11	P
GO:0033627	1	4	0.9247011905482249	0.9542968901034669	-	cell adhesion mediated by integrin	11	P
GO:0042307	1	4	0.9247011905482249	0.9542968901034669	-	positive regulation of protein import into nucleus	11	P
GO:0070493	1	4	0.9247011905482249	0.9542968901034669	-	thrombin-activated receptor signaling pathway	11	P
GO:0042593	1	4	0.9247011905482249	0.9542968901034669	-	glucose homeostasis	11	P
GO:0007005	1	4	0.9247011905482249	0.9542968901034669	-	mitochondrion organization	11	P
GO:0060021	1	4	0.9247011905482249	0.9542968901034669	-	roof of mouth development	11	P
GO:0045785	1	4	0.9247011905482249	0.9542968901034669	-	positive regulation of cell adhesion	11	P
GO:0007205	1	4	0.9247011905482249	0.9542968901034669	-	protein kinase C-activating G protein-coupled receptor signaling pathway	11	P
GO:0014823	1	4	0.9247011905482249	0.9542968901034669	-	response to activity	11	P
GO:0007626	1	4	0.9247011905482249	0.9542968901034669	-	locomotory behavior	11	P
GO:0030155	1	4	0.9247011905482249	0.9542968901034669	-	regulation of cell adhesion	11	P
GO:0016311	1	4	0.9247011905482249	0.9542968901034669	-	dephosphorylation	11	P
GO:0090557	1	4	0.9247011905482249	0.9542968901034669	-	establishment of endothelial intestinal barrier	11	P
GO:0051291	2	7	0.9327349059376928	0.961220417979824	-	protein heterooligomerization	11	P
GO:0007265	2	7	0.9327349059376928	0.961220417979824	-	Ras protein signal transduction	11	P
GO:0007596	9	22	0.9338007638071943	0.961635846901518	-	blood coagulation	11	P
GO:0030168	8	21	0.9491103887139183	0.9746784316156889	-	platelet activation	11	P
GO:0001558	1	5	0.955198297558587	0.9746784316156889	-	regulation of cell growth	11	P
GO:0006334	1	5	0.955198297558587	0.9746784316156889	-	nucleosome assembly	11	P
GO:0036498	1	5	0.955198297558587	0.9746784316156889	-	IRE1-mediated unfolded protein response	11	P
GO:2000300	1	5	0.955198297558587	0.9746784316156889	-	regulation of synaptic vesicle exocytosis	11	P
GO:0035307	1	5	0.955198297558587	0.9746784316156889	-	positive regulation of protein dephosphorylation	11	P
GO:0042632	1	5	0.955198297558587	0.9746784316156889	-	cholesterol homeostasis	11	P
GO:0042026	1	5	0.955198297558587	0.9746784316156889	-	protein refolding	11	P
GO:0006904	1	5	0.955198297558587	0.9746784316156889	-	vesicle docking involved in exocytosis	11	P
GO:0031175	1	5	0.955198297558587	0.9746784316156889	-	neuron projection development	11	P
GO:0045736	1	5	0.955198297558587	0.9746784316156889	-	negative regulation of cyclin-dependent protein serine/threonine kinase activity	11	P
GO:0035329	1	5	0.955198297558587	0.9746784316156889	-	hippo signaling	11	P
GO:0006281	1	5	0.955198297558587	0.9746784316156889	-	DNA repair	11	P
GO:0000086	2	8	0.9562061009884469	0.9750211191386061	-	G2/M transition of mitotic cell cycle	11	P
GO:0006888	3	11	0.9630604438163515	0.9813207190853414	-	endoplasmic reticulum to Golgi vesicle-mediated transport	11	P
GO:0008150	8	23	0.9713032691539665	0.9842139571459688	-	biological_process	11	P
GO:0055085	2	9	0.9717221580095097	0.9842139571459688	-	transmembrane transport	11	P
GO:0006338	1	6	0.9733611498997004	0.9842139571459688	-	chromatin remodeling	11	P
GO:0010389	1	6	0.9733611498997004	0.9842139571459688	-	regulation of G2/M transition of mitotic cell cycle	11	P
GO:0016197	1	6	0.9733611498997004	0.9842139571459688	-	endosomal transport	11	P
GO:0007017	1	6	0.9733611498997004	0.9842139571459688	-	microtubule-based process	11	P
GO:0016236	1	6	0.9733611498997004	0.9842139571459688	-	macroautophagy	11	P
GO:0007156	1	6	0.9733611498997004	0.9842139571459688	-	homophilic cell adhesion via plasma membrane adhesion molecules	11	P
GO:0007605	1	6	0.9733611498997004	0.9842139571459688	-	sensory perception of sound	11	P
GO:0008277	1	6	0.9733611498997004	0.9842139571459688	-	regulation of G protein-coupled receptor signaling pathway	11	P
GO:0010595	1	6	0.9733611498997004	0.9842139571459688	-	positive regulation of endothelial cell migration	11	P
GO:0030198	5	17	0.9762710913414018	0.9864689091478928	-	extracellular matrix organization	11	P
GO:0072659	4	15	0.9793334118874967	0.9888745863944035	-	protein localization to plasma membrane	11	P
GO:0009617	2	10	0.9818700505888527	0.9890945679732314	-	response to bacterium	11	P
GO:0006325	2	10	0.9818700505888527	0.9890945679732314	-	chromatin organization	11	P
GO:0097711	1	7	0.9841711180563437	0.9890945679732314	-	ciliary basal body-plasma membrane docking	11	P
GO:0000079	1	7	0.9841711180563437	0.9890945679732314	-	regulation of cyclin-dependent protein serine/threonine kinase activity	11	P
GO:0000209	1	7	0.9841711180563437	0.9890945679732314	-	protein polyubiquitination	11	P
GO:0006260	1	7	0.9841711180563437	0.9890945679732314	-	DNA replication	11	P
GO:0006886	8	25	0.9843229194716375	0.9890945679732314	-	intracellular protein transport	11	P
GO:0002576	14	38	0.9864519869007946	0.9905479813100714	-	platelet degranulation	11	P
GO:0032781	1	8	0.9906006445711751	0.9940259580033023	-	positive regulation of ATPase activity	11	P
GO:0070527	5	20	0.9918863800812614	0.9946282912908848	-	platelet aggregation	11	P
GO:0000278	2	12	0.9926790489185277	0.9947357044066185	-	mitotic cell cycle	11	P
GO:0000226	3	16	0.9952308786207369	0.9966045582323598	-	microtubule cytoskeleton organization	11	P
GO:0006511	1	11	0.9980397453388408	0.9987280486114882	-	ubiquitin-dependent protein catabolic process	11	P
GO:0006936	1	15	0.9997598206111092	0.9997598206111092	-	muscle contraction	11	P
GO:0042605	3	6	0.000324492175993473	0.010383749631791136	*	peptide antigen binding	12	F
GO:0023026	3	6	0.000324492175993473	0.010383749631791136	*	MHC class II protein complex binding	12	F
GO:0050786	2	5	0.005458501714150248	0.11644803656853861	-	RAGE receptor binding	12	F
GO:0005509	4	53	0.013033453740281494	0.12378147888534954	-	calcium ion binding	12	F
GO:0042802	5	92	0.01879246169829785	0.12378147888534954	-	identical protein binding	12	F
GO:0048306	2	11	0.019088022180734597	0.12378147888534954	-	calcium-dependent protein binding	12	F
GO:0019767	1	1	0.033794771086807514	0.12378147888534954	-	IgE receptor activity	12	F
GO:0003796	1	1	0.033794771086807514	0.12378147888534954	-	lysozyme activity	12	F
GO:0035718	1	1	0.033794771086807514	0.12378147888534954	-	macrophage migration inhibitory factor binding	12	F
GO:0042289	1	1	0.033794771086807514	0.12378147888534954	-	MHC class II protein binding	12	F
GO:0042658	1	1	0.033794771086807514	0.12378147888534954	-	MHC class II protein binding, via antigen binding groove	12	F
GO:0016209	1	1	0.033794771086807514	0.12378147888534954	-	antioxidant activity	12	F
GO:0008553	1	1	0.033794771086807514	0.12378147888534954	-	proton-exporting ATPase activity, phosphorylative mechanism	12	F
GO:0043015	1	1	0.033794771086807514	0.12378147888534954	-	gamma-tubulin binding	12	F
GO:0044183	1	1	0.033794771086807514	0.12378147888534954	-	protein folding chaperone	12	F
GO:0001540	2	16	0.035612653359119846	0.12378147888534954	-	amyloid-beta binding	12	F
GO:0035662	1	2	0.05028622579717325	0.12378147888534954	-	Toll-like receptor 4 binding	12	F
GO:0140031	1	2	0.05028622579717325	0.12378147888534954	-	phosphorylation-dependent protein binding	12	F
GO:0043208	1	2	0.05028622579717325	0.12378147888534954	-	glycosphingolipid binding	12	F
GO:0046875	1	2	0.05028622579717325	0.12378147888534954	-	ephrin receptor binding	12	F
GO:0051219	1	2	0.05028622579717325	0.12378147888534954	-	phosphoprotein binding	12	F
GO:0019863	1	2	0.05028622579717325	0.12378147888534954	-	IgE binding	12	F
GO:0004322	1	2	0.05028622579717325	0.12378147888534954	-	ferroxidase activity	12	F
GO:0008199	1	2	0.05028622579717325	0.12378147888534954	-	ferric iron binding	12	F
GO:0046914	1	2	0.05028622579717325	0.12378147888534954	-	transition metal ion binding	12	F
GO:0050544	1	2	0.05028622579717325	0.12378147888534954	-	arachidonic acid binding	12	F
GO:0042609	1	3	0.06651249128606576	0.14662679004960655	-	CD4 receptor binding	12	F
GO:0048487	1	3	0.06651249128606576	0.14662679004960655	-	beta-tubulin binding	12	F
GO:0032395	1	3	0.06651249128606576	0.14662679004960655	-	MHC class II receptor activity	12	F
GO:0004866	1	4	0.08247756940290368	0.14662679004960655	-	endopeptidase inhibitor activity	12	F
GO:0016301	1	4	0.08247756940290368	0.14662679004960655	-	kinase activity	12	F
GO:0008198	1	4	0.08247756940290368	0.14662679004960655	-	ferrous iron binding	12	F
GO:0004869	1	4	0.08247756940290368	0.14662679004960655	-	cysteine-type endopeptidase inhibitor activity	12	F
GO:0050998	1	4	0.08247756940290368	0.14662679004960655	-	nitric-oxide synthase binding	12	F
GO:0019955	1	4	0.08247756940290368	0.14662679004960655	-	cytokine binding	12	F
GO:0019864	1	4	0.08247756940290368	0.14662679004960655	-	IgG binding	12	F
GO:0004896	1	5	0.09818540557626787	0.16536489360213535	-	cytokine receptor activity	12	F
GO:0001965	1	5	0.09818540557626787	0.16536489360213535	-	G-protein alpha-subunit binding	12	F
GO:0046961	1	6	0.11363988955328745	0.17738909588805846	-	proton-transporting ATPase activity, rotational mechanism	12	F
GO:0005161	1	6	0.11363988955328745	0.17738909588805846	-	platelet-derived growth factor receptor binding	12	F
GO:0044548	1	6	0.11363988955328745	0.17738909588805846	-	S100 protein binding	12	F
GO:0051015	2	34	0.12260849046357994	0.18683198546831228	-	actin filament binding	12	F
GO:0004715	1	7	0.12884485613007265	0.18741069982556022	-	non-membrane spanning protein tyrosine kinase activity	12	F
GO:0098641	1	7	0.12884485613007265	0.18741069982556022	-	cadherin binding involved in cell-cell adhesion	12	F
GO:0005506	1	8	0.14380408587329357	0.20452136657535086	-	iron ion binding	12	F
GO:0004713	1	11	0.1872443612354658	0.260513893892822	-	protein tyrosine kinase activity	12	F
GO:0008017	1	16	0.2550677267285799	0.3473262661835982	-	microtubule binding	12	F
GO:0005200	1	17	0.2679768486914895	0.3522198707193651	-	structural constituent of cytoskeleton	12	F
GO:0002020	1	18	0.2806752094794941	0.3522198707193651	-	protease binding	12	F
GO:0005516	1	18	0.2806752094794941	0.3522198707193651	-	calmodulin binding	12	F
GO:0017124	1	18	0.2806752094794941	0.3522198707193651	-	SH3 domain binding	12	F
GO:0005096	1	20	0.3054525182305111	0.37594156089909064	-	GTPase activator activity	12	F
GO:0005102	1	21	0.3175377865929279	0.37634108040643305	-	signaling receptor binding	12	F
GO:0005178	1	21	0.3175377865929279	0.37634108040643305	-	integrin binding	12	F
GO:0044325	1	24	0.3526170248962908	0.4103179926065929	-	ion channel binding	12	F
GO:0044877	1	25	0.36392792990838035	0.4159176341810061	-	protein-containing complex binding	12	F
GO:0031625	1	29	0.4073382278489196	0.4497884600509951	-	ubiquitin protein ligase binding	12	F
GO:0042803	2	79	0.4076207919212143	0.4497884600509951	-	protein homodimerization activity	12	F
GO:0003674	1	37	0.48592977097561607	0.5190420504891768	-	molecular_function	12	F
GO:0005515	12	656	0.48660192233360333	0.5190420504891768	-	protein binding	12	F
GO:0003779	1	39	0.5039845445953528	0.5287706697393866	-	actin binding	12	F
GO:0008270	1	44	0.5465455981880302	0.564176101355386	-	zinc ion binding	12	F
GO:0003723	1	65	0.6905347940136946	0.7014956637599437	-	RNA binding	12	F
GO:0005524	1	80	0.7657007609837639	0.7657007609837639	-	ATP binding	12	F
GO:0012507	4	7	1.775746547167099e-05	0.0005337240856456997	***	ER to Golgi transport vesicle membrane	12	C
GO:0071556	4	7	1.775746547167099e-05	0.0005337240856456997	***	integral component of lumenal side of endoplasmic reticulum membrane	12	C
GO:0030658	4	8	2.6356744970158007e-05	0.0005337240856456997	***	transport vesicle membrane	12	C
GO:0042613	4	8	2.6356744970158007e-05	0.0005337240856456997	***	MHC class II protein complex	12	C
GO:0032588	4	10	5.218604890172082e-05	0.0008454139922078772	***	trans-Golgi network membrane	12	C
GO:0030669	4	11	7.041533225185783e-05	0.0009506069854000808	***	clathrin-coated endocytic vesicle membrane	12	C
GO:0030666	4	13	0.00012020838961269656	0.0013909827940897744	**	endocytic vesicle membrane	12	C
GO:0070062	12	269	0.0003044963628899316	0.0030830256742605576	**	extracellular exosome	12	C
GO:0005765	5	38	0.0004944088289789085	0.004449679460810176	**	lysosomal membrane	12	C
GO:1904724	3	14	0.002412949149833405	0.01954488811365058	*	tertiary granule lumen	12	C
GO:0000139	5	58	0.0029326746329855977	0.021595149570166672	*	Golgi membrane	12	C
GO:0009986	5	69	0.005989539746792755	0.040429393290851094	*	cell surface	12	C
GO:0005615	5	101	0.02685055710301901	0.13035125990625757	-	extracellular space	12	C
GO:0005576	6	142	0.02909000921251331	0.13035125990625757	-	extracellular region	12	C
GO:0005913	2	15	0.03197402001827906	0.13035125990625757	-	cell-cell adherens junction	12	C
GO:0032998	1	1	0.033794771086807514	0.13035125990625757	-	Fc-epsilon receptor I complex	12	C
GO:0035692	1	1	0.033794771086807514	0.13035125990625757	-	macrophage migration inhibitory factor receptor complex	12	C
GO:0036284	1	1	0.033794771086807514	0.13035125990625757	-	tubulobulbar complex	12	C
GO:0099091	1	1	0.033794771086807514	0.13035125990625757	-	postsynaptic specialization, intracellular component	12	C
GO:0035693	1	1	0.033794771086807514	0.13035125990625757	-	NOS2-CD74 complex	12	C
GO:0005773	1	1	0.033794771086807514	0.13035125990625757	-	vacuole	12	C
GO:0031902	2	16	0.035612653359119846	0.13111931464039578	-	late endosome membrane	12	C
GO:0043005	2	19	0.04743644019543578	0.1566609342142705	-	neuron projection	12	C
GO:0030061	1	2	0.05028622579717325	0.1566609342142705	-	mitochondrial crista	12	C
GO:0033181	1	2	0.05028622579717325	0.1566609342142705	-	plasma membrane proton-transporting V-type ATPase complex	12	C
GO:0008043	1	2	0.05028622579717325	0.1566609342142705	-	intracellular ferritin complex	12	C
GO:0033179	1	3	0.06651249128606576	0.17958372647237758	-	proton-transporting V-type ATPase, V0 domain	12	C
GO:0005771	1	3	0.06651249128606576	0.17958372647237758	-	multivesicular body	12	C
GO:0016471	1	3	0.06651249128606576	0.17958372647237758	-	vacuolar proton-transporting V-type ATPase complex	12	C
GO:0034666	1	3	0.06651249128606576	0.17958372647237758	-	integrin alpha2-beta1 complex	12	C
GO:0034774	2	25	0.07468528075149154	0.18539814560241366	-	secretory granule lumen	12	C
GO:0010008	2	25	0.07468528075149154	0.18539814560241366	-	endosome membrane	12	C
GO:0048471	3	58	0.08004466172368786	0.18539814560241366	-	perinuclear region of cytoplasm	12	C
GO:0005758	1	4	0.08247756940290368	0.18539814560241366	-	mitochondrial intermembrane space	12	C
GO:0043679	1	4	0.08247756940290368	0.18539814560241366	-	axon terminus	12	C
GO:0009898	1	4	0.08247756940290368	0.18539814560241366	-	cytoplasmic side of plasma membrane	12	C
GO:0009897	2	27	0.08468804181838649	0.18539814560241366	-	external side of plasma membrane	12	C
GO:0044754	1	5	0.09818540557626787	0.20928994346520258	-	autolysosome	12	C
GO:1904813	2	32	0.11136070726528031	0.224508074483324	-	ficolin-1-rich granule lumen	12	C
GO:0001891	1	6	0.11363988955328745	0.224508074483324	-	phagocytic cup	12	C
GO:0042629	1	6	0.11363988955328745	0.224508074483324	-	mast cell granule	12	C
GO:0005634	8	302	0.12192988571130524	0.23515049387180295	-	nucleus	12	C
GO:0005885	1	7	0.12884485613007265	0.24270775224502059	-	Arp2/3 protein complex	12	C
GO:0035580	1	9	0.15852130583300447	0.28533835049940803	-	specific granule lumen	12	C
GO:0031234	1	9	0.15852130583300447	0.28533835049940803	-	extrinsic component of cytoplasmic side of plasma membrane	12	C
GO:0005886	8	325	0.1706801721959933	0.2981492640423731	-	plasma membrane	12	C
GO:0043204	1	10	0.17300019024680907	0.2981492640423731	-	perikaryon	12	C
GO:0008021	1	11	0.1872443612354658	0.31597485958484856	-	synaptic vesicle	12	C
GO:0043202	1	14	0.22860404747686489	0.37789648664542974	-	lysosomal lumen	12	C
GO:0030670	1	17	0.2679768486914895	0.40942153628017947	-	phagocytic vesicle membrane	12	C
GO:0016324	1	17	0.2679768486914895	0.40942153628017947	-	apical plasma membrane	12	C
GO:0014069	1	17	0.2679768486914895	0.40942153628017947	-	postsynaptic density	12	C
GO:0005884	1	17	0.2679768486914895	0.40942153628017947	-	actin filament	12	C
GO:0001726	1	18	0.2806752094794941	0.40942153628017947	-	ruffle	12	C
GO:0101003	1	18	0.2806752094794941	0.40942153628017947	-	ficolin-1-rich granule membrane	12	C
GO:0042995	1	19	0.2931660383240791	0.40942153628017947	-	cell projection	12	C
GO:0070821	1	19	0.2931660383240791	0.40942153628017947	-	tertiary granule membrane	12	C
GO:0032587	1	19	0.2931660383240791	0.40942153628017947	-	ruffle membrane	12	C
GO:0035578	1	20	0.3054525182305111	0.41236089961119005	-	azurophil granule lumen	12	C
GO:0005788	1	20	0.3054525182305111	0.41236089961119005	-	endoplasmic reticulum lumen	12	C
GO:0005737	7	322	0.3130571747212183	0.41569887135112593	-	cytoplasm	12	C
GO:0030027	1	22	0.32942493580186283	0.4303777387088853	-	lamellipodium	12	C
GO:0005769	1	24	0.3526170248962908	0.4533647462952311	-	early endosome	12	C
GO:0005764	1	25	0.36392792990838035	0.45351018957813555	-	lysosome	12	C
GO:0005813	1	25	0.36392792990838035	0.45351018957813555	-	centrosome	12	C
GO:0016020	4	184	0.40223688926135637	0.49299058954129665	-	membrane	12	C
GO:0005887	2	80	0.41386864307170584	0.49299058954129665	-	integral component of plasma membrane	12	C
GO:0005794	2	80	0.41386864307170584	0.49299058954129665	-	Golgi apparatus	12	C
GO:0030054	1	32	0.43804920216972176	0.5142316721122822	-	cell junction	12	C
GO:0098978	1	37	0.48592977097561607	0.5622901635574986	-	glutamatergic synapse	12	C
GO:0015629	1	40	0.5127864890023244	0.5768848001276149	-	actin cytoskeleton	12	C
GO:0062023	1	40	0.5127864890023244	0.5768848001276149	-	collagen-containing extracellular matrix	12	C
GO:0005856	1	45	0.5546345014732381	0.6154163646483874	-	cytoskeleton	12	C
GO:0045121	1	48	0.5780940691605726	0.6243415946934184	-	membrane raft	12	C
GO:0005730	1	48	0.5780940691605726	0.6243415946934184	-	nucleolus	12	C
GO:0043231	1	63	0.6789478831068197	0.7236155069954264	-	intracellular membrane-bounded organelle	12	C
GO:0005739	1	66	0.6961795456250326	0.732344716826333	-	mitochondrion	12	C
GO:0016021	2	145	0.7402740733794841	0.768746153124849	-	integral component of membrane	12	C
GO:0005829	6	400	0.7565090077191587	0.7707960208827793	-	cytosol	12	C
GO:0005925	1	79	0.7612800206249672	0.7707960208827793	-	focal adhesion	12	C
GO:0005654	1	168	0.9585051388768182	0.9585051388768182	-	nucleoplasm	12	C
GO:0019886	5	23	5.890671980579067e-05	0.016317161386204017	*	antigen processing and presentation of exogenous peptide antigen via MHC class II	12	P
GO:0010976	3	10	0.0010588889212582674	0.08905158490946179	-	positive regulation of neuron projection development	12	P
GO:0060333	3	10	0.0010588889212582674	0.08905158490946179	-	interferon-gamma-mediated signaling pathway	12	P
GO:0002431	2	2	0.0016074293304957	0.08905158490946179	-	Fc receptor mediated stimulatory signaling pathway	12	P
GO:0002506	2	2	0.0016074293304957	0.08905158490946179	-	polysaccharide assembly with MHC class II protein complex	12	P
GO:0016064	2	3	0.0026521788187141315	0.11233293988562089	-	immunoglobulin mediated immune response	12	P
GO:0050852	3	15	0.0028649532915527576	0.11233293988562089	-	T cell receptor signaling pathway	12	P
GO:0042742	3	16	0.0033661935987093976	0.11233293988562089	-	defense response to bacterium	12	P
GO:0031668	2	5	0.005458501714150248	0.11233293988562089	-	cellular response to extracellular stimulus	12	P
GO:0006911	2	6	0.007205123515791211	0.11233293988562089	-	phagocytosis, engulfment	12	P
GO:0048678	2	6	0.007205123515791211	0.11233293988562089	-	response to axon injury	12	P
GO:0019730	2	6	0.007205123515791211	0.11233293988562089	-	antimicrobial humoral response	12	P
GO:0050900	3	22	0.007496651467301679	0.11233293988562089	-	leukocyte migration	12	P
GO:0002223	2	7	0.009171018184304849	0.11233293988562089	-	stimulatory C-type lectin receptor signaling pathway	12	P
GO:0001934	3	24	0.009333385126085706	0.11233293988562089	-	positive regulation of protein phosphorylation	12	P
GO:0043407	2	9	0.013732461694958661	0.11233293988562089	-	negative regulation of MAP kinase activity	12	P
GO:0043312	6	121	0.014205857833634157	0.11233293988562089	-	neutrophil degranulation	12	P
GO:0006879	2	11	0.019088022180734597	0.11233293988562089	-	cellular iron ion homeostasis	12	P
GO:0006952	2	11	0.019088022180734597	0.11233293988562089	-	defense response	12	P
GO:0007420	2	11	0.019088022180734597	0.11233293988562089	-	brain development	12	P
GO:0008285	3	35	0.023950009490500204	0.11233293988562089	-	negative regulation of cell population proliferation	12	P
GO:0038095	2	13	0.02518525958296673	0.11233293988562089	-	Fc-epsilon receptor signaling pathway	12	P
GO:0030593	2	15	0.03197402001827906	0.11233293988562089	-	neutrophil chemotaxis	12	P
GO:0042102	2	15	0.03197402001827906	0.11233293988562089	-	positive regulation of T cell proliferation	12	P
GO:0006955	3	40	0.033224464438036685	0.11233293988562089	-	immune response	12	P
GO:0032765	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of mast cell cytokine production	12	P
GO:0033026	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of mast cell apoptotic process	12	P
GO:0050672	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of lymphocyte proliferation	12	P
GO:0050707	1	1	0.033794771086807514	0.11233293988562089	-	regulation of cytokine secretion	12	P
GO:0050855	1	1	0.033794771086807514	0.11233293988562089	-	regulation of B cell receptor signaling pathway	12	P
GO:0050777	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of immune response	12	P
GO:0070667	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of mast cell proliferation	12	P
GO:0070668	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of mast cell proliferation	12	P
GO:0002292	1	1	0.033794771086807514	0.11233293988562089	-	T cell differentiation involved in immune response	12	P
GO:0071315	1	1	0.033794771086807514	0.11233293988562089	-	cellular response to morphine	12	P
GO:0071673	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of smooth muscle cell chemotaxis	12	P
GO:0045060	1	1	0.033794771086807514	0.11233293988562089	-	negative thymic T cell selection	12	P
GO:0043030	1	1	0.033794771086807514	0.11233293988562089	-	regulation of macrophage activation	12	P
GO:0060311	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of elastin catabolic process	12	P
GO:0060087	1	1	0.033794771086807514	0.11233293988562089	-	relaxation of vascular smooth muscle	12	P
GO:0043951	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of cAMP-mediated signaling	12	P
GO:0061052	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of cell growth involved in cardiac muscle cell development	12	P
GO:0070206	1	1	0.033794771086807514	0.11233293988562089	-	protein trimerization	12	P
GO:0002792	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of peptide secretion	12	P
GO:0060313	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of blood vessel remodeling	12	P
GO:0060397	1	1	0.033794771086807514	0.11233293988562089	-	growth hormone receptor signaling pathway via JAK-STAT	12	P
GO:0038094	1	1	0.033794771086807514	0.11233293988562089	-	Fc-gamma receptor signaling pathway	12	P
GO:0060452	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of cardiac muscle contraction	12	P
GO:0060547	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of necrotic cell death	12	P
GO:0010519	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of phospholipase activity	12	P
GO:0010711	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of collagen catabolic process	12	P
GO:0002504	1	1	0.033794771086807514	0.11233293988562089	-	antigen processing and presentation of peptide or polysaccharide antigen via MHC class II	12	P
GO:0002906	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of mature B cell apoptotic process	12	P
GO:0045059	1	1	0.033794771086807514	0.11233293988562089	-	positive thymic T cell selection	12	P
GO:0019883	1	1	0.033794771086807514	0.11233293988562089	-	antigen processing and presentation of endogenous antigen	12	P
GO:0019835	1	1	0.033794771086807514	0.11233293988562089	-	cytolysis	12	P
GO:0014739	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of muscle hyperplasia	12	P
GO:2000343	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of chemokine (C-X-C motif) ligand 2 production	12	P
GO:0001798	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of type IIa hypersensitivity	12	P
GO:0090025	1	1	0.033794771086807514	0.11233293988562089	-	regulation of monocyte chemotaxis	12	P
GO:0021549	1	1	0.033794771086807514	0.11233293988562089	-	cerebellum development	12	P
GO:0034103	1	1	0.033794771086807514	0.11233293988562089	-	regulation of tissue remodeling	12	P
GO:0034136	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of toll-like receptor 2 signaling pathway	12	P
GO:0035691	1	1	0.033794771086807514	0.11233293988562089	-	macrophage migration inhibitory factor signaling pathway	12	P
GO:0035821	1	1	0.033794771086807514	0.11233293988562089	-	modification of morphology or physiology of other organism	12	P
GO:0030728	1	1	0.033794771086807514	0.11233293988562089	-	ovulation	12	P
GO:0045581	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of T cell differentiation	12	P
GO:0045582	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of T cell differentiation	12	P
GO:0070304	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of stress-activated protein kinase signaling cascade	12	P
GO:0070447	1	1	0.033794771086807514	0.11233293988562089	-	positive regulation of oligodendrocyte progenitor proliferation	12	P
GO:0006991	1	1	0.033794771086807514	0.11233293988562089	-	response to sterol depletion	12	P
GO:0002762	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of myeloid leukocyte differentiation	12	P
GO:1900924	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of glycine import across plasma membrane	12	P
GO:1902532	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of intracellular signal transduction	12	P
GO:0140194	1	1	0.033794771086807514	0.11233293988562089	-	negative regulation of adenylate cyclase-inhibiting adrenergic receptor signaling pathway involved in heart process	12	P
GO:0045087	3	41	0.0352789071479376	0.11233293988562089	-	innate immune response	12	P
GO:0048013	2	18	0.04334946143238142	0.11233293988562089	-	ephrin receptor signaling pathway	12	P
GO:0043123	2	19	0.04743644019543578	0.11233293988562089	-	positive regulation of I-kappaB kinase/NF-kappaB signaling	12	P
GO:0010614	1	2	0.05028622579717325	0.11233293988562089	-	negative regulation of cardiac muscle hypertrophy	12	P
GO:0035606	1	2	0.05028622579717325	0.11233293988562089	-	peptidyl-cysteine S-trans-nitrosylation	12	P
GO:0016358	1	2	0.05028622579717325	0.11233293988562089	-	dendrite development	12	P
GO:0036295	1	2	0.05028622579717325	0.11233293988562089	-	cellular response to increased oxygen levels	12	P
GO:0042590	1	2	0.05028622579717325	0.11233293988562089	-	antigen processing and presentation of exogenous peptide antigen via MHC class I	12	P
GO:0060252	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of glial cell proliferation	12	P
GO:0060369	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of Fc receptor mediated stimulatory signaling pathway	12	P
GO:0043518	1	2	0.05028622579717325	0.11233293988562089	-	negative regulation of DNA damage response, signal transduction by p53 class mediator	12	P
GO:0051085	1	2	0.05028622579717325	0.11233293988562089	-	chaperone cofactor-dependent protein refolding	12	P
GO:0051272	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of cellular component movement	12	P
GO:0051602	1	2	0.05028622579717325	0.11233293988562089	-	response to electrical stimulus	12	P
GO:0055119	1	2	0.05028622579717325	0.11233293988562089	-	relaxation of cardiac muscle	12	P
GO:0045058	1	2	0.05028622579717325	0.11233293988562089	-	T cell selection	12	P
GO:0002513	1	2	0.05028622579717325	0.11233293988562089	-	tolerance induction to self antigen	12	P
GO:0009743	1	2	0.05028622579717325	0.11233293988562089	-	response to carbohydrate	12	P
GO:0010543	1	2	0.05028622579717325	0.11233293988562089	-	regulation of platelet activation	12	P
GO:0001812	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of type I hypersensitivity	12	P
GO:0010716	1	2	0.05028622579717325	0.11233293988562089	-	negative regulation of extracellular matrix disassembly	12	P
GO:0002606	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of dendritic cell antigen processing and presentation	12	P
GO:0002768	1	2	0.05028622579717325	0.11233293988562089	-	immune response-regulating cell surface receptor signaling pathway	12	P
GO:0002830	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of type 2 immune response	12	P
GO:0002902	1	2	0.05028622579717325	0.11233293988562089	-	regulation of B cell apoptotic process	12	P
GO:0007034	1	2	0.05028622579717325	0.11233293988562089	-	vacuolar transport	12	P
GO:0008156	1	2	0.05028622579717325	0.11233293988562089	-	negative regulation of DNA replication	12	P
GO:0006880	1	2	0.05028622579717325	0.11233293988562089	-	intracellular sequestering of iron ion	12	P
GO:0070488	1	2	0.05028622579717325	0.11233293988562089	-	neutrophil aggregation	12	P
GO:0090330	1	2	0.05028622579717325	0.11233293988562089	-	regulation of platelet aggregation	12	P
GO:0001516	1	2	0.05028622579717325	0.11233293988562089	-	prostaglandin biosynthetic process	12	P
GO:2000670	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of dendritic cell apoptotic process	12	P
GO:0001805	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of type III hypersensitivity	12	P
GO:2000010	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of protein localization to cell surface	12	P
GO:0033674	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of kinase activity	12	P
GO:0001961	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of cytokine-mediated signaling pathway	12	P
GO:0045657	1	2	0.05028622579717325	0.11233293988562089	-	positive regulation of monocyte differentiation	12	P
GO:1902166	1	2	0.05028622579717325	0.11233293988562089	-	negative regulation of intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator	12	P
GO:0071672	1	2	0.05028622579717325	0.11233293988562089	-	negative regulation of smooth muscle cell chemotaxis	12	P
GO:0048147	1	2	0.05028622579717325	0.11233293988562089	-	negative regulation of fibroblast proliferation	12	P
GO:0050663	1	2	0.05028622579717325	0.11233293988562089	-	cytokine secretion	12	P
GO:0032119	1	2	0.05028622579717325	0.11233293988562089	-	sequestering of zinc ion	12	P
GO:0032602	1	2	0.05028622579717325	0.11233293988562089	-	chemokine production	12	P
GO:0032623	1	2	0.05028622579717325	0.11233293988562089	-	interleukin-2 production	12	P
GO:0033003	1	2	0.05028622579717325	0.11233293988562089	-	regulation of mast cell activation	12	P
GO:0034144	1	2	0.05028622579717325	0.11233293988562089	-	negative regulation of toll-like receptor 4 signaling pathway	12	P
GO:0033628	1	2	0.05028622579717325	0.11233293988562089	-	regulation of cell adhesion mediated by integrin	12	P
GO:0045576	1	2	0.05028622579717325	0.11233293988562089	-	mast cell activation	12	P
GO:0045646	1	2	0.05028622579717325	0.11233293988562089	-	regulation of erythrocyte differentiation	12	P
GO:0044267	2	20	0.05166188261975636	0.11448273188538009	-	cellular protein metabolic process	12	P
GO:0038096	2	21	0.05602050399636576	0.12041803977934781	-	Fc-gamma receptor signaling pathway involved in phagocytosis	12	P
GO:0010466	1	3	0.06651249128606576	0.12041803977934781	-	negative regulation of peptidase activity	12	P
GO:0031640	1	3	0.06651249128606576	0.12041803977934781	-	killing of cells of other organism	12	P
GO:0046579	1	3	0.06651249128606576	0.12041803977934781	-	positive regulation of Ras protein signal transduction	12	P
GO:0001895	1	3	0.06651249128606576	0.12041803977934781	-	retina homeostasis	12	P
GO:0050873	1	3	0.06651249128606576	0.12041803977934781	-	brown fat cell differentiation	12	P
GO:0001782	1	3	0.06651249128606576	0.12041803977934781	-	B cell homeostasis	12	P
GO:0001817	1	3	0.06651249128606576	0.12041803977934781	-	regulation of cytokine production	12	P
GO:0002774	1	3	0.06651249128606576	0.12041803977934781	-	Fc receptor mediated inhibitory signaling pathway	12	P
GO:0050861	1	3	0.06651249128606576	0.12041803977934781	-	positive regulation of B cell receptor signaling pathway	12	P
GO:0071447	1	3	0.06651249128606576	0.12041803977934781	-	cellular response to hydroperoxide	12	P
GO:1900087	1	3	0.06651249128606576	0.12041803977934781	-	positive regulation of G1/S transition of mitotic cell cycle	12	P
GO:0097435	1	3	0.06651249128606576	0.12041803977934781	-	supramolecular fiber organization	12	P
GO:0045861	1	3	0.06651249128606576	0.12041803977934781	-	negative regulation of proteolysis	12	P
GO:0097028	1	3	0.06651249128606576	0.12041803977934781	-	dendritic cell differentiation	12	P
GO:0002554	1	3	0.06651249128606576	0.12041803977934781	-	serotonin secretion by platelet	12	P
GO:0002553	1	3	0.06651249128606576	0.12041803977934781	-	histamine secretion by mast cell	12	P
GO:0043304	1	3	0.06651249128606576	0.12041803977934781	-	regulation of mast cell degranulation	12	P
GO:1902961	1	3	0.06651249128606576	0.12041803977934781	-	positive regulation of aspartic-type endopeptidase activity involved in amyloid precursor protein catabolic process	12	P
GO:0060907	1	3	0.06651249128606576	0.12041803977934781	-	positive regulation of macrophage cytokine production	12	P
GO:0002503	1	3	0.06651249128606576	0.12041803977934781	-	peptide antigen assembly with MHC class II protein complex	12	P
GO:0002283	1	3	0.06651249128606576	0.12041803977934781	-	neutrophil activation involved in immune response	12	P
GO:0038156	1	3	0.06651249128606576	0.12041803977934781	-	interleukin-3-mediated signaling pathway	12	P
GO:0051290	1	3	0.06651249128606576	0.12041803977934781	-	protein heterotetramerization	12	P
GO:0046598	1	3	0.06651249128606576	0.12041803977934781	-	positive regulation of viral entry into host cell	12	P
GO:0001816	1	3	0.06651249128606576	0.12041803977934781	-	cytokine production	12	P
GO:0090026	1	3	0.06651249128606576	0.12041803977934781	-	positive regulation of monocyte chemotaxis	12	P
GO:0060135	1	3	0.06651249128606576	0.12041803977934781	-	maternal process involved in female pregnancy	12	P
GO:0030168	2	25	0.07468528075149154	0.1328272483988623	-	platelet activation	12	P
GO:0006954	3	57	0.07691931830381149	0.1328272483988623	-	inflammatory response	12	P
GO:0097178	1	4	0.08247756940290368	0.1328272483988623	-	ruffle assembly	12	P
GO:0071300	1	4	0.08247756940290368	0.1328272483988623	-	cellular response to retinoic acid	12	P
GO:2001244	1	4	0.08247756940290368	0.1328272483988623	-	positive regulation of intrinsic apoptotic signaling pathway	12	P
GO:0050829	1	4	0.08247756940290368	0.1328272483988623	-	defense response to Gram-negative bacterium	12	P
GO:0001837	1	4	0.08247756940290368	0.1328272483988623	-	epithelial to mesenchymal transition	12	P
GO:0070372	1	4	0.08247756940290368	0.1328272483988623	-	regulation of ERK1 and ERK2 cascade	12	P
GO:0016601	1	4	0.08247756940290368	0.1328272483988623	-	Rac protein signal transduction	12	P
GO:0007035	1	4	0.08247756940290368	0.1328272483988623	-	vacuolar acidification	12	P
GO:0090023	1	4	0.08247756940290368	0.1328272483988623	-	positive regulation of neutrophil chemotaxis	12	P
GO:0030890	1	4	0.08247756940290368	0.1328272483988623	-	positive regulation of B cell proliferation	12	P
GO:0001932	1	4	0.08247756940290368	0.1328272483988623	-	regulation of protein phosphorylation	12	P
GO:0002523	1	4	0.08247756940290368	0.1328272483988623	-	leukocyte migration involved in inflammatory response	12	P
GO:0051279	1	4	0.08247756940290368	0.1328272483988623	-	regulation of release of sequestered calcium ion into cytosol	12	P
GO:0032868	1	4	0.08247756940290368	0.1328272483988623	-	response to insulin	12	P
GO:0050890	1	4	0.08247756940290368	0.1328272483988623	-	cognition	12	P
GO:0043306	1	4	0.08247756940290368	0.1328272483988623	-	positive regulation of mast cell degranulation	12	P
GO:0030889	1	4	0.08247756940290368	0.1328272483988623	-	negative regulation of B cell proliferation	12	P
GO:0007596	2	28	0.08984084991084076	0.1438492221115774	-	blood coagulation	12	P
GO:0045429	1	5	0.09818540557626787	0.14390136161177883	-	positive regulation of nitric oxide biosynthetic process	12	P
GO:0017148	1	5	0.09818540557626787	0.14390136161177883	-	negative regulation of translation	12	P
GO:0045744	1	5	0.09818540557626787	0.14390136161177883	-	negative regulation of G protein-coupled receptor signaling pathway	12	P
GO:0030316	1	5	0.09818540557626787	0.14390136161177883	-	osteoclast differentiation	12	P
GO:0031175	1	5	0.09818540557626787	0.14390136161177883	-	neuron projection development	12	P
GO:0030218	1	5	0.09818540557626787	0.14390136161177883	-	erythrocyte differentiation	12	P
GO:0050766	1	5	0.09818540557626787	0.14390136161177883	-	positive regulation of phagocytosis	12	P
GO:0042531	1	5	0.09818540557626787	0.14390136161177883	-	positive regulation of tyrosine phosphorylation of STAT protein	12	P
GO:0036498	1	5	0.09818540557626787	0.14390136161177883	-	IRE1-mediated unfolded protein response	12	P
GO:0032733	1	5	0.09818540557626787	0.14390136161177883	-	positive regulation of interleukin-10 production	12	P
GO:0050870	1	5	0.09818540557626787	0.14390136161177883	-	positive regulation of T cell activation	12	P
GO:0043200	1	5	0.09818540557626787	0.14390136161177883	-	response to amino acid	12	P
GO:0009725	1	5	0.09818540557626787	0.14390136161177883	-	response to hormone	12	P
GO:2000406	1	5	0.09818540557626787	0.14390136161177883	-	positive regulation of T cell migration	12	P
GO:0001774	1	5	0.09818540557626787	0.14390136161177883	-	microglial cell activation	12	P
GO:0000902	1	5	0.09818540557626787	0.14390136161177883	-	cell morphogenesis	12	P
GO:0071404	1	6	0.11363988955328745	0.15660820600129666	-	cellular response to low-density lipoprotein particle stimulus	12	P
GO:0009653	1	6	0.11363988955328745	0.15660820600129666	-	anatomical structure morphogenesis	12	P
GO:0034605	1	6	0.11363988955328745	0.15660820600129666	-	cellular response to heat	12	P
GO:0050832	1	6	0.11363988955328745	0.15660820600129666	-	defense response to fungus	12	P
GO:0032870	1	6	0.11363988955328745	0.15660820600129666	-	cellular response to hormone stimulus	12	P
GO:0033572	1	6	0.11363988955328745	0.15660820600129666	-	transferrin transport	12	P
GO:0038083	1	6	0.11363988955328745	0.15660820600129666	-	peptidyl-tyrosine autophosphorylation	12	P
GO:0015991	1	6	0.11363988955328745	0.15660820600129666	-	ATP hydrolysis coupled proton transport	12	P
GO:0008277	1	6	0.11363988955328745	0.15660820600129666	-	regulation of G protein-coupled receptor signaling pathway	12	P
GO:0043552	1	6	0.11363988955328745	0.15660820600129666	-	positive regulation of phosphatidylinositol 3-kinase activity	12	P
GO:0014003	1	6	0.11363988955328745	0.15660820600129666	-	oligodendrocyte development	12	P
GO:0043627	1	6	0.11363988955328745	0.15660820600129666	-	response to estrogen	12	P
GO:0051493	1	7	0.12884485613007265	0.16995250070490536	-	regulation of cytoskeleton organization	12	P
GO:0032753	1	7	0.12884485613007265	0.16995250070490536	-	positive regulation of interleukin-4 production	12	P
GO:0050830	1	7	0.12884485613007265	0.16995250070490536	-	defense response to Gram-positive bacterium	12	P
GO:0061844	1	7	0.12884485613007265	0.16995250070490536	-	antimicrobial humoral immune response mediated by antimicrobial peptide	12	P
GO:0031663	1	7	0.12884485613007265	0.16995250070490536	-	lipopolysaccharide-mediated signaling pathway	12	P
GO:0098869	1	7	0.12884485613007265	0.16995250070490536	-	cellular oxidant detoxification	12	P
GO:0048662	1	7	0.12884485613007265	0.16995250070490536	-	negative regulation of smooth muscle cell proliferation	12	P
GO:0001975	1	7	0.12884485613007265	0.16995250070490536	-	response to amphetamine	12	P
GO:0050853	1	7	0.12884485613007265	0.16995250070490536	-	B cell receptor signaling pathway	12	P
GO:0016032	2	36	0.1341401862830822	0.17609872796404627	-	viral process	12	P
GO:1902600	1	8	0.14380408587329357	0.1827235403068914	-	proton transmembrane transport	12	P
GO:0014070	1	8	0.14380408587329357	0.1827235403068914	-	response to organic cyclic compound	12	P
GO:0034314	1	8	0.14380408587329357	0.1827235403068914	-	Arp2/3 complex-mediated actin nucleation	12	P
GO:0002224	1	8	0.14380408587329357	0.1827235403068914	-	toll-like receptor signaling pathway	12	P
GO:0032755	1	8	0.14380408587329357	0.1827235403068914	-	positive regulation of interleukin-6 production	12	P
GO:0051384	1	8	0.14380408587329357	0.1827235403068914	-	response to glucocorticoid	12	P
GO:0007283	1	8	0.14380408587329357	0.1827235403068914	-	spermatogenesis	12	P
GO:0001933	1	9	0.15852130583300447	0.1934378930208909	-	negative regulation of protein phosphorylation	12	P
GO:0032355	1	9	0.15852130583300447	0.1934378930208909	-	response to estradiol	12	P
GO:0070373	1	9	0.15852130583300447	0.1934378930208909	-	negative regulation of ERK1 and ERK2 cascade	12	P
GO:0050729	1	9	0.15852130583300447	0.1934378930208909	-	positive regulation of inflammatory response	12	P
GO:0030307	1	9	0.15852130583300447	0.1934378930208909	-	positive regulation of cell growth	12	P
GO:0030041	1	9	0.15852130583300447	0.1934378930208909	-	actin filament polymerization	12	P
GO:0045471	1	9	0.15852130583300447	0.1934378930208909	-	response to ethanol	12	P
GO:0048146	1	9	0.15852130583300447	0.1934378930208909	-	positive regulation of fibroblast proliferation	12	P
GO:0050727	1	9	0.15852130583300447	0.1934378930208909	-	regulation of inflammatory response	12	P
GO:0048661	1	10	0.17300019024680907	0.20479082349729108	-	positive regulation of smooth muscle cell proliferation	12	P
GO:0051017	1	10	0.17300019024680907	0.20479082349729108	-	actin filament bundle assembly	12	P
GO:0000187	1	10	0.17300019024680907	0.20479082349729108	-	activation of MAPK activity	12	P
GO:0007169	1	10	0.17300019024680907	0.20479082349729108	-	transmembrane receptor protein tyrosine kinase signaling pathway	12	P
GO:0031295	1	10	0.17300019024680907	0.20479082349729108	-	T cell costimulation	12	P
GO:0009636	1	10	0.17300019024680907	0.20479082349729108	-	response to toxic substance	12	P
GO:0032729	1	10	0.17300019024680907	0.20479082349729108	-	positive regulation of interferon-gamma production	12	P
GO:0032760	1	11	0.1872443612354658	0.21977410195857638	-	positive regulation of tumor necrosis factor production	12	P
GO:0016241	1	11	0.1872443612354658	0.21977410195857638	-	regulation of macroautophagy	12	P
GO:0065003	1	12	0.2012573894900267	0.23228457036973912	-	protein-containing complex assembly	12	P
GO:0008286	1	12	0.2012573894900267	0.23228457036973912	-	insulin receptor signaling pathway	12	P
GO:0050731	1	12	0.2012573894900267	0.23228457036973912	-	positive regulation of peptidyl-tyrosine phosphorylation	12	P
GO:0006914	1	12	0.2012573894900267	0.23228457036973912	-	autophagy	12	P
GO:0007165	3	91	0.2103220944827554	0.24173950278723338	-	signal transduction	12	P
GO:0034220	1	13	0.2150427949506049	0.24313001714823493	-	ion transmembrane transport	12	P
GO:0060271	1	13	0.2150427949506049	0.24313001714823493	-	cilium assembly	12	P
GO:0014068	1	13	0.2150427949506049	0.24313001714823493	-	positive regulation of phosphatidylinositol 3-kinase signaling	12	P
GO:0098609	1	13	0.2150427949506049	0.24313001714823493	-	cell-cell adhesion	12	P
GO:0043066	2	50	0.22053974056798273	0.2483313338915903	-	negative regulation of apoptotic process	12	P
GO:0071346	1	14	0.22860404747686489	0.252284148012317	-	cellular response to interferon-gamma	12	P
GO:0010951	1	14	0.22860404747686489	0.252284148012317	-	negative regulation of endopeptidase activity	12	P
GO:0018108	1	14	0.22860404747686489	0.252284148012317	-	peptidyl-tyrosine phosphorylation	12	P
GO:0043410	1	14	0.22860404747686489	0.252284148012317	-	positive regulation of MAPK cascade	12	P
GO:0010629	1	14	0.22860404747686489	0.252284148012317	-	negative regulation of gene expression	12	P
GO:0006919	1	15	0.24194456751032697	0.2659470047633356	-	activation of cysteine-type endopeptidase activity involved in apoptotic process	12	P
GO:0072659	1	16	0.2550677267285799	0.2792638747186428	-	protein localization to plasma membrane	12	P
GO:0051260	1	17	0.2679768486914895	0.28995932456071327	-	protein homooligomerization	12	P
GO:0031623	1	17	0.2679768486914895	0.28995932456071327	-	receptor internalization	12	P
GO:0007267	1	17	0.2679768486914895	0.28995932456071327	-	cell-cell signaling	12	P
GO:0002250	1	18	0.2806752094794941	0.3013450892473638	-	adaptive immune response	12	P
GO:0046777	1	18	0.2806752094794941	0.3013450892473638	-	protein autophosphorylation	12	P
GO:0042493	1	19	0.2931660383240791	0.31233458698373046	-	response to drug	12	P
GO:0051092	1	19	0.2931660383240791	0.31233458698373046	-	positive regulation of NF-kappaB transcription factor activity	12	P
GO:0006974	1	20	0.3054525182305111	0.322940257823861	-	cellular response to DNA damage stimulus	12	P
GO:0007229	1	20	0.3054525182305111	0.322940257823861	-	integrin-mediated signaling pathway	12	P
GO:0007049	1	21	0.3175377865929279	0.3344409387309545	-	cell cycle	12	P
GO:0070374	1	24	0.3526170248962908	0.368584588287821	-	positive regulation of ERK1 and ERK2 cascade	12	P
GO:0008360	1	24	0.3526170248962908	0.368584588287821	-	regulation of cell shape	12	P
GO:0043687	1	25	0.36392792990838035	0.37755818945551073	-	post-translational protein modification	12	P
GO:0030154	1	25	0.36392792990838035	0.37755818945551073	-	cell differentiation	12	P
GO:0006886	1	29	0.4073382278489196	0.42101749669459226	-	intracellular protein transport	12	P
GO:0030335	1	31	0.4279826441474722	0.44071075252360525	-	positive regulation of cell migration	12	P
GO:0043547	1	33	0.44794885353046243	0.45956234232569665	-	positive regulation of GTPase activity	12	P
GO:0055114	1	36	0.4766721503258123	0.4872257772702952	-	oxidation-reduction process	12	P
GO:0045893	1	37	0.48592977097561607	0.4912501699279039	-	positive regulation of transcription, DNA-templated	12	P
GO:0008283	1	37	0.48592977097561607	0.4912501699279039	-	cell population proliferation	12	P
GO:0006468	1	37	0.48592977097561607	0.4912501699279039	-	protein phosphorylation	12	P
GO:0008284	1	45	0.5546345014732381	0.556644046768431	-	positive regulation of cell population proliferation	12	P
GO:0002576	1	45	0.5546345014732381	0.556644046768431	-	platelet degranulation	12	P
GO:0006915	1	56	0.6350963121105413	0.6350963121105413	-	apoptotic process	12	P
GO:0004012	1	1	0.006109966922548835	0.021363055503865733	*	phospholipid-translocating ATPase activity	2	F
GO:0042626	1	2	0.009155595215942456	0.021363055503865733	*	ATPase activity, coupled to transmembrane movement of substances	2	F
GO:0043225	1	2	0.009155595215942456	0.021363055503865733	*	ATPase-coupled anion transmembrane transporter activity	2	F
GO:0016887	1	5	0.018255135923417543	0.0319464878659807	*	ATPase activity	2	F
GO:0005215	1	8	0.02729879424476447	0.03821831194267026	*	transporter activity	2	F
GO:0003723	1	64	0.1860738747804882	0.2170861872439029	-	RNA binding	2	F
GO:0005524	1	79	0.22545042981816807	0.22545042981816807	-	ATP binding	2	F
GO:0005774	1	2	0.009155595215942456	0.0824003569434821	-	vacuolar membrane	2	C
GO:0016323	1	17	0.054096196115428305	0.23295764126606314	-	basolateral plasma membrane	2	C
GO:0005654	2	166	0.07765254708868771	0.23295764126606314	-	nucleoplasm	2	C
GO:0005765	1	37	0.11187428668837063	0.2517171450488339	-	lysosomal membrane	2	C
GO:0005887	1	79	0.22545042981816807	0.4058107736727025	-	integral component of plasma membrane	2	C
GO:0016020	1	183	0.46412839765513275	0.6961925964826992	-	membrane	2	C
GO:0070062	1	268	0.6181171384316864	0.7891095661962436	-	extracellular exosome	2	C
GO:0005886	1	324	0.7014307255077721	0.7891095661962436	-	plasma membrane	2	C
GO:0005829	1	399	0.7926973208902908	0.7926973208902908	-	cytosol	2	C
GO:0034198	1	1	0.006109966922548835	0.01464895234550793	*	cellular response to amino acid starvation	2	P
GO:0042149	1	1	0.006109966922548835	0.01464895234550793	*	cellular response to glucose starvation	2	P
GO:0045332	1	1	0.006109966922548835	0.01464895234550793	*	phospholipid translocation	2	P
GO:0006691	1	2	0.009155595215942456	0.01464895234550793	*	leukotriene metabolic process	2	P
GO:0099133	1	2	0.009155595215942456	0.01464895234550793	*	ATP hydrolysis coupled anion transmembrane transport	2	P
GO:0060326	1	7	0.02429044000016065	0.03238725333354753	*	cell chemotaxis	2	P
GO:0055085	1	10	0.033296939020943904	0.03805364459536446	*	transmembrane transport	2	P
GO:0042493	1	18	0.05704293693749863	0.05704293693749863	-	response to drug	2	P
GO:0004843	2	2	0.005940956982317139	0.19902205890762414	-	thiol-dependent ubiquitin-specific protease activity	3	F
GO:0030544	2	2	0.005940956982317139	0.19902205890762414	-	Hsp70 protein binding	3	F
GO:0042393	2	3	0.009700066675363503	0.21532771037448628	-	histone binding	3	F
GO:0008092	2	6	0.02554139471728318	0.21532771037448628	-	cytoskeletal protein binding	3	F
GO:0061630	2	8	0.03940893383021306	0.21532771037448628	-	ubiquitin protein ligase activity	3	F
GO:0000981	5	58	0.04864089237606471	0.21532771037448628	-	DNA-binding transcription factor activity, RNA polymerase II-specific	3	F
GO:0003723	5	60	0.05454951003761957	0.21532771037448628	-	RNA binding	3	F
GO:0004726	1	1	0.06420770142919553	0.21532771037448628	-	non-membrane spanning protein tyrosine phosphatase activity	3	F
GO:0015171	1	1	0.06420770142919553	0.21532771037448628	-	amino acid transmembrane transporter activity	3	F
GO:0008177	1	1	0.06420770142919553	0.21532771037448628	-	succinate dehydrogenase (ubiquinone) activity	3	F
GO:0016505	1	1	0.06420770142919553	0.21532771037448628	-	peptidase activator activity involved in apoptotic process	3	F
GO:0000993	1	1	0.06420770142919553	0.21532771037448628	-	RNA polymerase II complex binding	3	F
GO:0034459	1	1	0.06420770142919553	0.21532771037448628	-	ATP-dependent 3'-5' RNA helicase activity	3	F
GO:0000175	1	1	0.06420770142919553	0.21532771037448628	-	3'-5'-exoribonuclease activity	3	F
GO:0036459	1	1	0.06420770142919553	0.21532771037448628	-	thiol-dependent ubiquitinyl hydrolase activity	3	F
GO:0000340	1	1	0.06420770142919553	0.21532771037448628	-	RNA 7-methylguanosine cap binding	3	F
GO:0017160	1	1	0.06420770142919553	0.21532771037448628	-	Ral GTPase binding	3	F
GO:0051537	1	1	0.06420770142919553	0.21532771037448628	-	2 iron, 2 sulfur cluster binding	3	F
GO:0070034	1	1	0.06420770142919553	0.21532771037448628	-	telomerase RNA binding	3	F
GO:0009055	2	11	0.0642769284699959	0.21532771037448628	-	electron transfer activity	3	F
GO:0015297	1	2	0.0947953965816631	0.2442804450373626	-	antiporter activity	3	F
GO:0015179	1	2	0.0947953965816631	0.2442804450373626	-	L-amino acid transmembrane transporter activity	3	F
GO:0001102	1	2	0.0947953965816631	0.2442804450373626	-	RNA polymerase II activating transcription factor binding	3	F
GO:1990763	1	2	0.0947953965816631	0.2442804450373626	-	arrestin family protein binding	3	F
GO:0003712	1	2	0.0947953965816631	0.2442804450373626	-	transcription coregulator activity	3	F
GO:0003743	1	2	0.0947953965816631	0.2442804450373626	-	translation initiation factor activity	3	F
GO:0001227	2	15	0.10340215224742876	0.2565905259473232	-	DNA-binding transcription repressor activity, RNA polymerase II-specific	3	F
GO:0050660	1	3	0.1244135431147784	0.2778569129563384	-	flavin adenine dinucleotide binding	3	F
GO:0008137	1	3	0.1244135431147784	0.2778569129563384	-	NADH dehydrogenase (ubiquinone) activity	3	F
GO:0003727	1	3	0.1244135431147784	0.2778569129563384	-	single-stranded RNA binding	3	F
GO:0000977	2	18	0.1360558004963256	0.2849210588625098	-	RNA polymerase II regulatory region sequence-specific DNA binding	3	F
GO:0035035	1	4	0.15309191222463212	0.2849210588625098	-	histone acetyltransferase binding	3	F
GO:0051287	1	4	0.15309191222463212	0.2849210588625098	-	NAD binding	3	F
GO:0046332	1	4	0.15309191222463212	0.2849210588625098	-	SMAD binding	3	F
GO:0003676	1	4	0.15309191222463212	0.2849210588625098	-	nucleic acid binding	3	F
GO:0001965	1	4	0.15309191222463212	0.2849210588625098	-	G-protein alpha-subunit binding	3	F
GO:0008234	1	5	0.18085939051234912	0.3107071580596767	-	cysteine-type peptidase activity	3	F
GO:0003697	1	5	0.18085939051234912	0.3107071580596767	-	single-stranded DNA binding	3	F
GO:0001671	1	5	0.18085939051234912	0.3107071580596767	-	ATPase activator activity	3	F
GO:0003690	1	6	0.20774400538784127	0.3394841063654967	-	double-stranded DNA binding	3	F
GO:0003729	1	6	0.20774400538784127	0.3394841063654967	-	mRNA binding	3	F
GO:0003725	1	7	0.23377294976934956	0.36425087522200983	-	double-stranded RNA binding	3	F
GO:0035091	1	7	0.23377294976934956	0.36425087522200983	-	phosphatidylinositol binding	3	F
GO:0003730	1	8	0.258972606097593	0.39434465019406206	-	mRNA 3'-UTR binding	3	F
GO:0003713	1	9	0.28336856968285745	0.4039509397606691	-	transcription coactivator activity	3	F
GO:0000980	1	9	0.28336856968285745	0.4039509397606691	-	RNA polymerase II distal enhancer sequence-specific DNA binding	3	F
GO:0004842	1	9	0.28336856968285745	0.4039509397606691	-	ubiquitin-protein transferase activity	3	F
GO:0000166	1	10	0.30698567140288696	0.428500832999863	-	nucleotide binding	3	F
GO:0044212	1	11	0.3298479997689771	0.45101665274533603	-	transcription regulatory region DNA binding	3	F
GO:0008022	1	12	0.35197892237722556	0.4624036823387081	-	protein C-terminus binding	3	F
GO:0047485	1	12	0.35197892237722556	0.4624036823387081	-	protein N-terminus binding	3	F
GO:0046872	4	92	0.38368030279192655	0.4943573132126746	-	metal ion binding	3	F
GO:0003682	1	14	0.3941365406638629	0.4982480797071474	-	chromatin binding	3	F
GO:0003700	2	40	0.4025090863877734	0.4994094219996447	-	DNA-binding transcription factor activity	3	F
GO:0008270	2	42	0.42594833823613837	0.5188825211240231	-	zinc ion binding	3	F
GO:0043565	1	18	0.47062782170539197	0.5630725723975225	-	sequence-specific DNA binding	3	F
GO:0003677	2	48	0.4934521972055033	0.5800227581187496	-	DNA binding	3	F
GO:0008134	1	22	0.5377263002054009	0.6211665881683079	-	transcription factor binding	3	F
GO:0001228	1	24	0.5681086974135597	0.6451403851984493	-	DNA-binding transcription activator activity, RNA polymerase II-specific	3	F
GO:0019904	1	29	0.6358443759838148	0.7100262198485933	-	protein domain specific binding	3	F
GO:0019899	1	31	0.659951321095509	0.7248645657934281	-	enzyme binding	3	F
GO:0000978	1	32	0.6714176622176837	0.7255642478804001	-	RNA polymerase II proximal promoter sequence-specific DNA binding	3	F
GO:0003674	1	36	0.7136504566050081	0.7589615967069134	-	molecular_function	3	F
GO:0019901	1	51	0.8299696943453369	0.8688745237677745	-	protein kinase binding	3	F
GO:0005524	1	79	0.9372171297263965	0.966054579871824	-	ATP binding	3	F
GO:0042802	1	91	0.9594305365443736	0.9667370582804408	-	identical protein binding	3	F
GO:0005515	17	639	0.9667370582804408	0.9667370582804408	-	protein binding	3	F
GO:0005654	14	154	0.00030496480492802903	0.01860285310060977	*	nucleoplasm	3	C
GO:0005929	2	8	0.03940893383021306	0.26139284244465	-	cilium	3	C
GO:0005662	1	1	0.06420770142919553	0.26139284244465	-	DNA replication factor A complex	3	C
GO:0000178	1	1	0.06420770142919553	0.26139284244465	-	exosome (RNase complex)	3	C
GO:0005749	1	1	0.06420770142919553	0.26139284244465	-	mitochondrial respiratory chain complex II, succinate dehydrogenase complex (ubiquinone)	3	C
GO:0005845	1	1	0.06420770142919553	0.26139284244465	-	mRNA cap binding complex	3	C
GO:0000176	1	1	0.06420770142919553	0.26139284244465	-	nuclear exosome (RNase complex)	3	C
GO:0016442	1	1	0.06420770142919553	0.26139284244465	-	RISC complex	3	C
GO:0045111	1	1	0.06420770142919553	0.26139284244465	-	intermediate filament cytoskeleton	3	C
GO:0070469	1	1	0.06420770142919553	0.26139284244465	-	respirasome	3	C
GO:0072546	1	1	0.06420770142919553	0.26139284244465	-	ER membrane protein complex	3	C
GO:0070578	1	1	0.06420770142919553	0.26139284244465	-	RISC-loading complex	3	C
GO:0031519	1	1	0.06420770142919553	0.26139284244465	-	PcG protein complex	3	C
GO:0035327	1	1	0.06420770142919553	0.26139284244465	-	transcriptionally active chromatin	3	C
GO:0045211	2	11	0.0642769284699959	0.26139284244465	-	postsynaptic membrane	3	C
GO:0030054	3	29	0.08231331687822357	0.3043431153411289	-	cell junction	3	C
GO:0005759	2	14	0.09308637453410512	0.3043431153411289	-	mitochondrial matrix	3	C
GO:0000145	1	2	0.0947953965816631	0.3043431153411289	-	exocyst	3	C
GO:0005844	1	2	0.0947953965816631	0.3043431153411289	-	polysome	3	C
GO:0009898	1	3	0.1244135431147784	0.3312973088255022	-	cytoplasmic side of plasma membrane	3	C
GO:0090543	1	3	0.1244135431147784	0.3312973088255022	-	Flemming body	3	C
GO:0005719	1	3	0.1244135431147784	0.3312973088255022	-	nuclear euchromatin	3	C
GO:0005743	2	17	0.12491537873748444	0.3312973088255022	-	mitochondrial inner membrane	3	C
GO:0031514	1	4	0.15309191222463212	0.37354426582810235	-	motile cilium	3	C
GO:0097546	1	4	0.15309191222463212	0.37354426582810235	-	ciliary base	3	C
GO:0005903	1	6	0.20774400538784127	0.45258515459493986	-	brush border	3	C
GO:0030136	1	6	0.20774400538784127	0.45258515459493986	-	clathrin-coated vesicle	3	C
GO:0030141	1	6	0.20774400538784127	0.45258515459493986	-	secretory granule	3	C
GO:0000932	1	7	0.23377294976934956	0.4917293081355284	-	P-body	3	C
GO:1990904	1	9	0.28336856968285745	0.5575962177630421	-	ribonucleoprotein complex	3	C
GO:0031966	1	9	0.28336856968285745	0.5575962177630421	-	mitochondrial membrane	3	C
GO:0016021	6	139	0.33112543438732633	0.6312078593008408	-	integral component of membrane	3	C
GO:0055037	1	12	0.35197892237722556	0.639086104971552	-	recycling endosome	3	C
GO:0005739	3	63	0.3663688338601743	0.639086104971552	-	mitochondrion	3	C
GO:0005815	1	13	0.3734011067614496	0.639086104971552	-	microtubule organizing center	3	C
GO:0031965	1	15	0.4142065517391799	0.639086104971552	-	nuclear membrane	3	C
GO:0000790	1	15	0.4142065517391799	0.639086104971552	-	nuclear chromatin	3	C
GO:0014069	1	16	0.4336318267074145	0.639086104971552	-	postsynaptic density	3	C
GO:0005667	1	16	0.4336318267074145	0.639086104971552	-	transcription factor complex	3	C
GO:0016604	1	16	0.4336318267074145	0.639086104971552	-	nuclear body	3	C
GO:0005856	2	43	0.4375044672474597	0.639086104971552	-	cytoskeleton	3	C
GO:0016323	1	17	0.45243242997023897	0.639086104971552	-	basolateral plasma membrane	3	C
GO:0005874	1	17	0.45243242997023897	0.639086104971552	-	microtubule	3	C
GO:0043209	1	18	0.47062782170539197	0.639086104971552	-	myelin sheath	3	C
GO:0005730	2	46	0.47145696268393184	0.639086104971552	-	nucleolus	3	C
GO:0030424	1	19	0.4882368754532374	0.6474445522314669	-	axon	3	C
GO:0032991	2	49	0.5042469430781595	0.6544481601652709	-	protein-containing complex	3	C
GO:0005634	10	292	0.5449975224234154	0.6886372074773608	-	nucleus	3	C
GO:0030425	1	23	0.5531675928916504	0.6886372074773608	-	dendrite	3	C
GO:0005813	1	24	0.5681086974135597	0.6930926108445429	-	centrosome	3	C
GO:0005829	12	388	0.7115077779790473	0.8510191069945467	-	cytosol	3	C
GO:0016020	5	179	0.7472897304483728	0.8766283376413604	-	membrane	3	C
GO:0045121	1	47	0.8044520838149739	0.9258788134474227	-	membrane raft	3	C
GO:0000139	1	57	0.8622994925804911	0.9635994509320159	-	Golgi membrane	3	C
GO:0005737	8	314	0.8766072856080593	0.9635994509320159	-	cytoplasm	3	C
GO:0043231	1	62	0.8846158893802114	0.9635994509320159	-	intracellular membrane-bounded organelle	3	C
GO:0005794	1	79	0.9372171297263965	0.9856938778156928	-	Golgi apparatus	3	C
GO:0005887	1	79	0.9372171297263965	0.9856938778156928	-	integral component of plasma membrane	3	C
GO:0005886	5	320	0.9932919619974432	0.9996058704214696	-	plasma membrane	3	C
GO:0005576	1	141	0.9938493032208819	0.9996058704214696	-	extracellular region	3	C
GO:0070062	2	267	0.9996058704214696	0.9996058704214696	-	extracellular exosome	3	C
GO:0016055	3	9	0.0056850180159135405	0.2921408051118588	-	Wnt signaling pathway	3	P
GO:0035094	2	2	0.005940956982317139	0.2921408051118588	-	response to nicotine	3	P
GO:0016567	2	8	0.03940893383021306	0.2921408051118588	-	protein ubiquitination	3	P
GO:0006325	2	10	0.055500059375893085	0.2921408051118588	-	chromatin organization	3	P
GO:0006099	1	1	0.06420770142919553	0.2921408051118588	-	tricarboxylic acid cycle	3	P
GO:0003333	1	1	0.06420770142919553	0.2921408051118588	-	amino acid transmembrane transport	3	P
GO:0060213	1	1	0.06420770142919553	0.2921408051118588	-	positive regulation of nuclear-transcribed mRNA poly(A) tail shortening	3	P
GO:0090503	1	1	0.06420770142919553	0.2921408051118588	-	RNA phosphodiester bond hydrolysis, exonucleolytic	3	P
GO:0035729	1	1	0.06420770142919553	0.2921408051118588	-	cellular response to hepatocyte growth factor stimulus	3	P
GO:0006405	1	1	0.06420770142919553	0.2921408051118588	-	RNA export from nucleus	3	P
GO:0046887	1	1	0.06420770142919553	0.2921408051118588	-	positive regulation of hormone secretion	3	P
GO:0006520	1	1	0.06420770142919553	0.2921408051118588	-	cellular amino acid metabolic process	3	P
GO:0045947	1	1	0.06420770142919553	0.2921408051118588	-	negative regulation of translational initiation	3	P
GO:0006105	1	1	0.06420770142919553	0.2921408051118588	-	succinate metabolic process	3	P
GO:0035278	1	1	0.06420770142919553	0.2921408051118588	-	miRNA mediated inhibition of translation	3	P
GO:0032502	1	1	0.06420770142919553	0.2921408051118588	-	developmental process	3	P
GO:0030433	1	1	0.06420770142919553	0.2921408051118588	-	ubiquitin-dependent ERAD pathway	3	P
GO:0000184	1	1	0.06420770142919553	0.2921408051118588	-	nuclear-transcribed mRNA catabolic process, nonsense-mediated decay	3	P
GO:2000279	1	1	0.06420770142919553	0.2921408051118588	-	negative regulation of DNA biosynthetic process	3	P
GO:1902065	1	1	0.06420770142919553	0.2921408051118588	-	response to L-glutamate	3	P
GO:0010944	1	1	0.06420770142919553	0.2921408051118588	-	negative regulation of transcription by competitive promoter binding	3	P
GO:0007223	1	1	0.06420770142919553	0.2921408051118588	-	Wnt signaling pathway, calcium modulating pathway	3	P
GO:0022904	1	1	0.06420770142919553	0.2921408051118588	-	respiratory electron transport chain	3	P
GO:0021983	1	1	0.06420770142919553	0.2921408051118588	-	pituitary gland development	3	P
GO:2000535	1	1	0.06420770142919553	0.2921408051118588	-	regulation of entry of bacterium into host cell	3	P
GO:0045930	1	2	0.0947953965816631	0.2921408051118588	-	negative regulation of mitotic cell cycle	3	P
GO:0001947	1	2	0.0947953965816631	0.2921408051118588	-	heart looping	3	P
GO:0006364	1	2	0.0947953965816631	0.2921408051118588	-	rRNA processing	3	P
GO:0071294	1	2	0.0947953965816631	0.2921408051118588	-	cellular response to zinc ion	3	P
GO:0002040	1	2	0.0947953965816631	0.2921408051118588	-	sprouting angiogenesis	3	P
GO:0090305	1	2	0.0947953965816631	0.2921408051118588	-	nucleic acid phosphodiester bond hydrolysis	3	P
GO:1900153	1	2	0.0947953965816631	0.2921408051118588	-	positive regulation of nuclear-transcribed mRNA catabolic process, deadenylation-dependent decay	3	P
GO:0009968	1	2	0.0947953965816631	0.2921408051118588	-	negative regulation of signal transduction	3	P
GO:0006413	1	2	0.0947953965816631	0.2921408051118588	-	translational initiation	3	P
GO:0036120	1	2	0.0947953965816631	0.2921408051118588	-	cellular response to platelet-derived growth factor stimulus	3	P
GO:0043473	1	2	0.0947953965816631	0.2921408051118588	-	pigmentation	3	P
GO:0042391	1	2	0.0947953965816631	0.2921408051118588	-	regulation of membrane potential	3	P
GO:0051865	1	2	0.0947953965816631	0.2921408051118588	-	protein autoubiquitination	3	P
GO:0002052	1	2	0.0947953965816631	0.2921408051118588	-	positive regulation of neuroblast proliferation	3	P
GO:0006412	1	2	0.0947953965816631	0.2921408051118588	-	translation	3	P
GO:1901215	1	2	0.0947953965816631	0.2921408051118588	-	negative regulation of neuron death	3	P
GO:0009791	1	2	0.0947953965816631	0.2921408051118588	-	post-embryonic development	3	P
GO:0009060	1	2	0.0947953965816631	0.2921408051118588	-	aerobic respiration	3	P
GO:0007368	1	2	0.0947953965816631	0.2921408051118588	-	determination of left/right symmetry	3	P
GO:1902475	1	2	0.0947953965816631	0.2921408051118588	-	L-alpha-amino acid transmembrane transport	3	P
GO:1900745	1	2	0.0947953965816631	0.2921408051118588	-	positive regulation of p38MAPK cascade	3	P
GO:1990314	1	2	0.0947953965816631	0.2921408051118588	-	cellular response to insulin-like growth factor stimulus	3	P
GO:0045766	2	15	0.10340215224742876	0.2921408051118588	-	positive regulation of angiogenesis	3	P
GO:0060828	1	3	0.1244135431147784	0.2921408051118588	-	regulation of canonical Wnt signaling pathway	3	P
GO:0060964	1	3	0.1244135431147784	0.2921408051118588	-	regulation of gene silencing by miRNA	3	P
GO:0061512	1	3	0.1244135431147784	0.2921408051118588	-	protein localization to cilium	3	P
GO:0070935	1	3	0.1244135431147784	0.2921408051118588	-	3'-UTR-mediated mRNA stabilization	3	P
GO:0071300	1	3	0.1244135431147784	0.2921408051118588	-	cellular response to retinoic acid	3	P
GO:0050890	1	3	0.1244135431147784	0.2921408051118588	-	cognition	3	P
GO:0006120	1	3	0.1244135431147784	0.2921408051118588	-	mitochondrial electron transport, NADH to ubiquinone	3	P
GO:0045732	1	3	0.1244135431147784	0.2921408051118588	-	positive regulation of protein catabolic process	3	P
GO:0010803	1	3	0.1244135431147784	0.2921408051118588	-	regulation of tumor necrosis factor-mediated signaling pathway	3	P
GO:0045637	1	3	0.1244135431147784	0.2921408051118588	-	regulation of myeloid cell differentiation	3	P
GO:0040018	1	3	0.1244135431147784	0.2921408051118588	-	positive regulation of multicellular organism growth	3	P
GO:0032802	1	3	0.1244135431147784	0.2921408051118588	-	low-density lipoprotein particle receptor catabolic process	3	P
GO:0008542	1	3	0.1244135431147784	0.2921408051118588	-	visual learning	3	P
GO:0007568	2	17	0.12491537873748444	0.2921408051118588	-	aging	3	P
GO:0007595	1	4	0.15309191222463212	0.2959776969676221	-	lactation	3	P
GO:0032981	1	4	0.15309191222463212	0.2959776969676221	-	mitochondrial respiratory chain complex I assembly	3	P
GO:0014823	1	4	0.15309191222463212	0.2959776969676221	-	response to activity	3	P
GO:0042220	1	4	0.15309191222463212	0.2959776969676221	-	response to cocaine	3	P
GO:0030177	1	4	0.15309191222463212	0.2959776969676221	-	positive regulation of Wnt signaling pathway	3	P
GO:0031648	1	4	0.15309191222463212	0.2959776969676221	-	protein destabilization	3	P
GO:0006893	1	4	0.15309191222463212	0.2959776969676221	-	Golgi to plasma membrane transport	3	P
GO:0000398	1	4	0.15309191222463212	0.2959776969676221	-	mRNA splicing, via spliceosome	3	P
GO:0001764	1	4	0.15309191222463212	0.2959776969676221	-	neuron migration	3	P
GO:0001822	1	4	0.15309191222463212	0.2959776969676221	-	kidney development	3	P
GO:1990090	1	4	0.15309191222463212	0.2959776969676221	-	cellular response to nerve growth factor stimulus	3	P
GO:1900034	1	4	0.15309191222463212	0.2959776969676221	-	regulation of cellular response to heat	3	P
GO:1900273	1	4	0.15309191222463212	0.2959776969676221	-	positive regulation of long-term synaptic potentiation	3	P
GO:0030324	1	5	0.18085939051234912	0.31145414613106476	-	lung development	3	P
GO:0045672	1	5	0.18085939051234912	0.31145414613106476	-	positive regulation of osteoclast differentiation	3	P
GO:0042632	1	5	0.18085939051234912	0.31145414613106476	-	cholesterol homeostasis	3	P
GO:0042752	1	5	0.18085939051234912	0.31145414613106476	-	regulation of circadian rhythm	3	P
GO:0006406	1	5	0.18085939051234912	0.31145414613106476	-	mRNA export from nucleus	3	P
GO:0001558	1	5	0.18085939051234912	0.31145414613106476	-	regulation of cell growth	3	P
GO:0010033	1	5	0.18085939051234912	0.31145414613106476	-	response to organic substance	3	P
GO:0010977	1	5	0.18085939051234912	0.31145414613106476	-	negative regulation of neuron projection development	3	P
GO:0045944	4	65	0.1822498262949578	0.31145414613106476	-	positive regulation of transcription by RNA polymerase II	3	P
GO:0015031	2	22	0.18257656842165865	0.31145414613106476	-	protein transport	3	P
GO:0008361	1	6	0.20774400538784127	0.32390194388426863	-	regulation of cell size	3	P
GO:0010595	1	6	0.20774400538784127	0.32390194388426863	-	positive regulation of endothelial cell migration	3	P
GO:0050728	1	6	0.20774400538784127	0.32390194388426863	-	negative regulation of inflammatory response	3	P
GO:0007264	1	6	0.20774400538784127	0.32390194388426863	-	small GTPase mediated signal transduction	3	P
GO:0042542	1	6	0.20774400538784127	0.32390194388426863	-	response to hydrogen peroxide	3	P
GO:0071345	1	6	0.20774400538784127	0.32390194388426863	-	cellular response to cytokine stimulus	3	P
GO:0006338	1	6	0.20774400538784127	0.32390194388426863	-	chromatin remodeling	3	P
GO:0007613	1	6	0.20774400538784127	0.32390194388426863	-	memory	3	P
GO:0007623	1	7	0.23377294976934956	0.33897077716555685	-	circadian rhythm	3	P
GO:0045454	1	7	0.23377294976934956	0.33897077716555685	-	cell redox homeostasis	3	P
GO:0006260	1	7	0.23377294976934956	0.33897077716555685	-	DNA replication	3	P
GO:0007409	1	7	0.23377294976934956	0.33897077716555685	-	axonogenesis	3	P
GO:0000209	1	7	0.23377294976934956	0.33897077716555685	-	protein polyubiquitination	3	P
GO:0090090	1	7	0.23377294976934956	0.33897077716555685	-	negative regulation of canonical Wnt signaling pathway	3	P
GO:1990830	1	7	0.23377294976934956	0.33897077716555685	-	cellular response to leukemia inhibitory factor	3	P
GO:0006887	1	8	0.258972606097593	0.3576288369919141	-	exocytosis	3	P
GO:0032781	1	8	0.258972606097593	0.3576288369919141	-	positive regulation of ATPase activity	3	P
GO:0051291	1	8	0.258972606097593	0.3576288369919141	-	protein heterooligomerization	3	P
GO:0045471	1	8	0.258972606097593	0.3576288369919141	-	response to ethanol	3	P
GO:0045600	1	8	0.258972606097593	0.3576288369919141	-	positive regulation of fat cell differentiation	3	P
GO:0000122	3	52	0.26487425477982135	0.3623279900290009	-	negative regulation of transcription by RNA polymerase II	3	P
GO:0006470	1	9	0.28336856968285745	0.3701661495857147	-	protein dephosphorylation	3	P
GO:0043161	1	9	0.28336856968285745	0.3701661495857147	-	proteasome-mediated ubiquitin-dependent protein catabolic process	3	P
GO:0035335	1	9	0.28336856968285745	0.3701661495857147	-	peptidyl-tyrosine dephosphorylation	3	P
GO:0002244	1	9	0.28336856968285745	0.3701661495857147	-	hematopoietic progenitor cell differentiation	3	P
GO:0001938	1	9	0.28336856968285745	0.3701661495857147	-	positive regulation of endothelial cell proliferation	3	P
GO:0055085	1	10	0.30698567140288696	0.39743680672695186	-	transmembrane transport	3	P
GO:0032088	1	11	0.3298479997689771	0.4123099997112214	-	negative regulation of NF-kappaB transcription factor activity	3	P
GO:0006914	1	11	0.3298479997689771	0.4123099997112214	-	autophagy	3	P
GO:0006511	1	11	0.3298479997689771	0.4123099997112214	-	ubiquitin-dependent protein catabolic process	3	P
GO:0065003	1	11	0.3298479997689771	0.4123099997112214	-	protein-containing complex assembly	3	P
GO:0045893	2	35	0.34230559419573026	0.42422488169556316	-	positive regulation of transcription, DNA-templated	3	P
GO:0032091	1	12	0.35197892237722556	0.4288818802075437	-	negative regulation of protein binding	3	P
GO:0060271	1	12	0.35197892237722556	0.4288818802075437	-	cilium assembly	3	P
GO:0010629	1	13	0.3734011067614496	0.4401882965887008	-	negative regulation of gene expression	3	P
GO:0007399	1	13	0.3734011067614496	0.4401882965887008	-	nervous system development	3	P
GO:0016192	1	13	0.3734011067614496	0.4401882965887008	-	vesicle-mediated transport	3	P
GO:0030334	1	13	0.3734011067614496	0.4401882965887008	-	regulation of cell migration	3	P
GO:0050821	1	14	0.3941365406638629	0.45719838717008093	-	protein stabilization	3	P
GO:0007179	1	14	0.3941365406638629	0.45719838717008093	-	transforming growth factor beta receptor signaling pathway	3	P
GO:0000226	1	17	0.45243242997023897	0.5165567113833437	-	microtubule cytoskeleton organization	3	P
GO:0001666	1	17	0.45243242997023897	0.5165567113833437	-	response to hypoxia	3	P
GO:0050900	1	21	0.5217686320355764	0.5910660284778014	-	leukocyte migration	3	P
GO:0006508	1	22	0.5377263002054009	0.6044210351145979	-	proteolysis	3	P
GO:0006366	1	23	0.5531675928916504	0.616994622840687	-	transcription by RNA polymerase II	3	P
GO:0001525	1	24	0.5681086974135597	0.6193666249997456	-	angiogenesis	3	P
GO:0043687	1	24	0.5681086974135597	0.6193666249997456	-	post-translational protein modification	3	P
GO:0030154	1	24	0.5681086974135597	0.6193666249997456	-	cell differentiation	3	P
GO:0043065	1	26	0.596552650448585	0.6455233904107822	-	positive regulation of apoptotic process	3	P
GO:0010628	1	27	0.610085475590771	0.6552769923011985	-	positive regulation of gene expression	3	P
GO:0035556	1	29	0.6358443759838148	0.6779223126298025	-	intracellular signal transduction	3	P
GO:0016032	1	35	0.7036141089723248	0.73913797291233	-	viral process	3	P
GO:0055114	1	35	0.7036141089723248	0.73913797291233	-	oxidation-reduction process	3	P
GO:0006468	1	36	0.7136504566050081	0.73913797291233	-	protein phosphorylation	3	P
GO:0008283	1	36	0.7136504566050081	0.73913797291233	-	cell population proliferation	3	P
GO:0006357	1	38	0.732744886678438	0.7535319756622234	-	regulation of transcription by RNA polymerase II	3	P
GO:0045087	1	40	0.7506032598599053	0.7664610752090582	-	innate immune response	3	P
GO:0007186	1	41	0.7590933616519087	0.7697100520246626	-	G protein-coupled receptor signaling pathway	3	P
GO:0006915	1	55	0.8522481486656879	0.8581665385869773	-	apoptotic process	3	P
GO:0007165	1	90	0.9579174563222523	0.9579174563222523	-	signal transduction	3	P
GO:0000175	1	1	0.014227464686180283	0.07469418960244649	-	3'-5'-exoribonuclease activity	4	F
GO:0004629	1	1	0.014227464686180283	0.07469418960244649	-	phospholipase C activity	4	F
GO:0005168	1	1	0.014227464686180283	0.07469418960244649	-	neurotrophin TRKA receptor binding	4	F
GO:0030023	1	1	0.014227464686180283	0.07469418960244649	-	extracellular matrix constituent conferring elasticity	4	F
GO:0004864	1	2	0.021275889351345494	0.08223421353832853	-	protein phosphatase inhibitor activity	4	F
GO:0003676	1	4	0.03524323437356937	0.08223421353832853	-	nucleic acid binding	4	F
GO:0019902	1	4	0.03524323437356937	0.08223421353832853	-	phosphatase binding	4	F
GO:0004435	1	4	0.03524323437356937	0.08223421353832853	-	phosphatidylinositol phospholipase C activity	4	F
GO:0030971	1	4	0.03524323437356937	0.08223421353832853	-	receptor tyrosine kinase binding	4	F
GO:0005509	2	51	0.05051742230318292	0.10608658683668414	-	calcium ion binding	4	F
GO:0000166	1	10	0.07612505459346187	0.13412707434443627	-	nucleotide binding	4	F
GO:0000287	1	12	0.08941804956295751	0.13412707434443627	-	magnesium ion binding	4	F
GO:0008022	1	12	0.08941804956295751	0.13412707434443627	-	protein C-terminus binding	4	F
GO:0047485	1	12	0.08941804956295751	0.13412707434443627	-	protein N-terminus binding	4	F
GO:0042803	2	77	0.10318044764230114	0.14445262669922157	-	protein homodimerization activity	4	F
GO:0005178	1	20	0.14096412955994636	0.1850154200474296	-	integrin binding	4	F
GO:0019899	1	31	0.20774400538784127	0.2566249478320392	-	enzyme binding	4	F
GO:0008270	1	43	0.27547139399024756	0.32138329298862217	-	zinc ion binding	4	F
GO:0019901	1	51	0.31781182159292876	0.3512656975500792	-	protein kinase binding	4	F
GO:0042802	1	91	0.49920189020719613	0.5241619847175559	-	identical protein binding	4	F
GO:0005515	5	651	0.5753578009264282	0.5753578009264282	-	protein binding	4	F
GO:0005671	1	1	0.014227464686180283	0.16261616484754574	-	Ada2/Gcn5/Ada3 transcription activator complex	4	C
GO:0005758	1	3	0.02828107214739926	0.16261616484754574	-	mitochondrial intermembrane space	4	C
GO:0008180	1	3	0.02828107214739926	0.16261616484754574	-	COP9 signalosome	4	C
GO:0098685	1	3	0.02828107214739926	0.16261616484754574	-	Schaffer collateral - CA1 synapse	4	C
GO:0031012	1	10	0.07612505459346187	0.327924114837899	-	extracellular matrix	4	C
GO:0005759	1	15	0.10905005052696805	0.327924114837899	-	mitochondrial matrix	4	C
GO:0001726	1	17	0.12193552407222441	0.327924114837899	-	ruffle	4	C
GO:0005911	1	18	0.12831813189309094	0.327924114837899	-	cell-cell junction	4	C
GO:0042995	1	18	0.12831813189309094	0.327924114837899	-	cell projection	4	C
GO:0030027	1	21	0.14722793278190516	0.3386242453983818	-	lamellipodium	4	C
GO:0098978	1	36	0.23659522732949753	0.48581728095398186	-	glutamatergic synapse	4	C
GO:0062023	1	39	0.25346988571512097	0.48581728095398186	-	collagen-containing extracellular matrix	4	C
GO:0005730	1	47	0.2969147560501037	0.5253107222424911	-	nucleolus	4	C
GO:0005739	1	65	0.38681583440841066	0.6084833418974686	-	mitochondrion	4	C
GO:0005737	3	319	0.4175743470502103	0.6084833418974686	-	cytoplasm	4	C
GO:0005925	1	78	0.4454449144841283	0.6084833418974686	-	focal adhesion	4	C
GO:0005794	1	79	0.44974855705465067	0.6084833418974686	-	Golgi apparatus	4	C
GO:0005615	1	100	0.5337448717396375	0.6820073361117589	-	extracellular space	4	C
GO:0005829	3	397	0.5973424117142211	0.7062165380167658	-	cytosol	4	C
GO:0070062	2	267	0.6141013374058834	0.7062165380167658	-	extracellular exosome	4	C
GO:0005576	1	141	0.6665201360295973	0.7299982442228923	-	extracellular region	4	C
GO:0005634	1	301	0.9246214221223983	0.9408421498218266	-	nucleus	4	C
GO:0005886	1	324	0.9408421498218266	0.9408421498218266	-	plasma membrane	4	C
GO:0006164	1	1	0.014227464686180283	0.041101564648965264	*	purine nucleotide biosynthetic process	4	P
GO:0010863	1	1	0.014227464686180283	0.041101564648965264	*	positive regulation of phospholipase C activity	4	P
GO:0009117	1	1	0.014227464686180283	0.041101564648965264	*	nucleotide metabolic process	4	P
GO:0007625	1	1	0.014227464686180283	0.041101564648965264	*	grooming behavior	4	P
GO:0021756	1	1	0.014227464686180283	0.041101564648965264	*	striatum development	4	P
GO:0021895	1	1	0.014227464686180283	0.041101564648965264	*	cerebral cortex neuron differentiation	4	P
GO:0021954	1	1	0.014227464686180283	0.041101564648965264	*	central nervous system neuron development	4	P
GO:0016573	1	1	0.014227464686180283	0.041101564648965264	*	histone acetylation	4	P
GO:0032264	1	1	0.014227464686180283	0.041101564648965264	*	IMP salvage	4	P
GO:0042417	1	1	0.014227464686180283	0.041101564648965264	*	dopamine metabolic process	4	P
GO:2000353	1	1	0.014227464686180283	0.041101564648965264	*	positive regulation of endothelial cell apoptotic process	4	P
GO:0051281	1	1	0.014227464686180283	0.041101564648965264	*	positive regulation of release of sequestered calcium ion into cytosol	4	P
GO:0071169	1	1	0.014227464686180283	0.041101564648965264	*	establishment of protein localization to chromatin	4	P
GO:0046903	1	1	0.014227464686180283	0.041101564648965264	*	secretion	4	P
GO:0046651	1	1	0.014227464686180283	0.041101564648965264	*	lymphocyte proliferation	4	P
GO:0043647	1	1	0.014227464686180283	0.041101564648965264	*	inositol phosphate metabolic process	4	P
GO:0090503	1	1	0.014227464686180283	0.041101564648965264	*	RNA phosphodiester bond hydrolysis, exonucleolytic	4	P
GO:2000121	1	1	0.014227464686180283	0.041101564648965264	*	regulation of removal of superoxide radicals	4	P
GO:0032959	1	2	0.021275889351345494	0.046097760261248566	*	inositol trisphosphate biosynthetic process	4	P
GO:0010923	1	2	0.021275889351345494	0.046097760261248566	*	negative regulation of phosphatase activity	4	P
GO:0001913	1	2	0.021275889351345494	0.046097760261248566	*	T cell mediated cytotoxicity	4	P
GO:0043966	1	2	0.021275889351345494	0.046097760261248566	*	histone H3 acetylation	4	P
GO:0048813	1	2	0.021275889351345494	0.046097760261248566	*	dendrite morphogenesis	4	P
GO:0051209	1	2	0.021275889351345494	0.046097760261248566	*	release of sequestered calcium ion into cytosol	4	P
GO:0050804	1	3	0.02828107214739926	0.05252199113088434	-	modulation of chemical synaptic transmission	4	P
GO:0007202	1	3	0.02828107214739926	0.05252199113088434	-	activation of phospholipase C activity	4	P
GO:0006139	1	3	0.02828107214739926	0.05252199113088434	-	nucleobase-containing compound metabolic process	4	P
GO:0043101	1	3	0.02828107214739926	0.05252199113088434	-	purine-containing compound salvage	4	P
GO:0034394	1	4	0.03524323437356937	0.059117683465342165	-	protein localization to cell surface	4	P
GO:0032515	1	4	0.03524323437356937	0.059117683465342165	-	negative regulation of phosphoprotein phosphatase activity	4	P
GO:0007626	1	4	0.03524323437356937	0.059117683465342165	-	locomotory behavior	4	P
GO:0001558	1	5	0.0421625964221196	0.06851421918594434	-	regulation of cell growth	4	P
GO:0001975	1	6	0.04903937778114027	0.07500140131233218	-	response to amphetamine	4	P
GO:0043536	1	6	0.04903937778114027	0.07500140131233218	-	positive regulation of blood vessel endothelial cell migration	4	P
GO:0071364	1	7	0.05587379703733333	0.08301249845546665	-	cellular response to epidermal growth factor stimulus	4	P
GO:0010634	1	8	0.06266607187879136	0.08575357204466187	-	positive regulation of epithelial cell migration	4	P
GO:0051289	1	8	0.06266607187879136	0.08575357204466187	-	protein homotetramerization	4	P
GO:0048015	1	8	0.06266607187879136	0.08575357204466187	-	phosphatidylinositol-mediated signaling	4	P
GO:0007173	1	9	0.06941641909777123	0.0902413448271026	-	epidermal growth factor receptor signaling pathway	4	P
GO:0019722	1	9	0.06941641909777123	0.0902413448271026	-	calcium-mediated signaling	4	P
GO:0001701	1	10	0.07612505459346187	0.09654884972829311	-	in utero embryonic development	4	P
GO:0007411	1	12	0.08941804956295751	0.10813345528543698	-	axon guidance	4	P
GO:0038095	1	12	0.08941804956295751	0.10813345528543698	-	Fc-epsilon receptor signaling pathway	4	P
GO:0007160	1	13	0.09600283639463028	0.11345789755729034	-	cell-matrix adhesion	4	P
GO:0050852	1	14	0.1025467662242452	0.11849848541468333	-	T cell receptor signaling pathway	4	P
GO:0045766	1	16	0.11551289990138387	0.1305797998885209	-	positive regulation of angiogenesis	4	P
GO:0030198	1	20	0.14096412955994636	0.15271114035660854	-	extracellular matrix organization	4	P
GO:0038096	1	20	0.14096412955994636	0.15271114035660854	-	Fc-gamma receptor signaling pathway involved in phagocytosis	4	P
GO:0050900	1	21	0.14722793278190516	0.15624188785018508	-	leukocyte migration	4	P
GO:0016477	1	23	0.1596381749865175	0.1660237019859782	-	cell migration	4	P
GO:0016032	1	35	0.23089817678718036	0.23542559201830152	-	viral process	4	P
GO:0007165	1	90	0.49523180326659644	0.49523180326659644	-	signal transduction	4	P
GO:0005524	3	77	0.004638312555135663	0.0285528250152663	*	ATP binding	5	F
GO:0043548	1	1	0.010172876490045557	0.0285528250152663	*	phosphatidylinositol 3-kinase binding	5	F
GO:0034459	1	1	0.010172876490045557	0.0285528250152663	*	ATP-dependent 3'-5' RNA helicase activity	5	F
GO:0031994	1	1	0.010172876490045557	0.0285528250152663	*	insulin-like growth factor I binding	5	F
GO:0043559	1	1	0.010172876490045557	0.0285528250152663	*	insulin binding	5	F
GO:0004386	1	2	0.01522817334147536	0.0285528250152663	*	helicase activity	5	F
GO:0004714	1	2	0.01522817334147536	0.0285528250152663	*	transmembrane receptor protein tyrosine kinase activity	5	F
GO:0043560	1	2	0.01522817334147536	0.0285528250152663	*	insulin receptor substrate binding	5	F
GO:1990841	1	3	0.02026279413216311	0.03377132355360519	*	promoter-specific chromatin binding	5	F
GO:0003723	2	63	0.037962397696573946	0.05694359654486092	-	RNA binding	5	F
GO:0004713	1	10	0.054931526617438484	0.07490662720559793	-	protein tyrosine kinase activity	5	F
GO:0003682	1	14	0.07429706765846311	0.09287133457307889	-	chromatin binding	5	F
GO:0000978	1	32	0.15755402016865414	0.1817931001946009	-	RNA polymerase II proximal promoter sequence-specific DNA binding	5	F
GO:0042802	1	91	0.3894740251010385	0.41729359832254126	-	identical protein binding	5	F
GO:0005515	3	653	0.7936427688225867	0.7936427688225867	-	protein binding	5	F
GO:0035867	1	1	0.010172876490045557	0.08534498920810235	-	alphav-beta3 integrin-IGF-1-IGF1R complex	5	C
GO:0005681	1	3	0.02026279413216311	0.08534498920810235	-	spliceosomal complex	5	C
GO:0005730	2	46	0.021336247302025588	0.08534498920810235	-	nucleolus	5	C
GO:1990904	1	9	0.05003986785251839	0.15011960355755516	-	ribonucleoprotein complex	5	C
GO:0043235	1	12	0.06465439123247298	0.15517053895793514	-	receptor complex	5	C
GO:0030424	1	19	0.09805619210921426	0.19611238421842853	-	axon	5	C
GO:0005634	3	299	0.17308295508024504	0.29671363728042005	-	nucleus	5	C
GO:0043231	1	62	0.28292664225390923	0.4243899633808639	-	intracellular membrane-bounded organelle	5	C
GO:0005887	1	79	0.3470430054035383	0.4627240072047177	-	integral component of plasma membrane	5	C
GO:0005654	1	167	0.6098875808980888	0.7056884273098591	-	nucleoplasm	5	C
GO:0016020	1	183	0.646881058367371	0.7056884273098591	-	membrane	5	C
GO:0005886	1	324	0.8669629913043844	0.8669629913043844	-	plasma membrane	5	C
GO:0001893	1	1	0.010172876490045557	0.03988331113243546	*	maternal placenta development	5	P
GO:0007628	1	1	0.010172876490045557	0.03988331113243546	*	adult walking behavior	5	P
GO:0035909	1	1	0.010172876490045557	0.03988331113243546	*	aorta morphogenesis	5	P
GO:0060041	1	1	0.010172876490045557	0.03988331113243546	*	retina development in camera-type eye	5	P
GO:0060411	1	1	0.010172876490045557	0.03988331113243546	*	cardiac septum morphogenesis	5	P
GO:0051262	1	1	0.010172876490045557	0.03988331113243546	*	protein tetramerization	5	P
GO:0046328	1	1	0.010172876490045557	0.03988331113243546	*	regulation of JNK cascade	5	P
GO:0097062	1	1	0.010172876490045557	0.03988331113243546	*	dendritic spine maintenance	5	P
GO:0071225	1	1	0.010172876490045557	0.03988331113243546	*	cellular response to muramyl dipeptide	5	P
GO:0048856	1	1	0.010172876490045557	0.03988331113243546	*	anatomical structure development	5	P
GO:0060324	1	1	0.010172876490045557	0.03988331113243546	*	face development	5	P
GO:0048026	1	1	0.010172876490045557	0.03988331113243546	*	positive regulation of mRNA splicing, via spliceosome	5	P
GO:0010880	1	2	0.01522817334147536	0.03988331113243546	*	regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum	5	P
GO:0030217	1	2	0.01522817334147536	0.03988331113243546	*	T cell differentiation	5	P
GO:0014065	1	2	0.01522817334147536	0.03988331113243546	*	phosphatidylinositol 3-kinase signaling	5	P
GO:0006364	1	2	0.01522817334147536	0.03988331113243546	*	rRNA processing	5	P
GO:0035116	1	2	0.01522817334147536	0.03988331113243546	*	embryonic hindlimb morphogenesis	5	P
GO:0097242	1	2	0.01522817334147536	0.03988331113243546	*	amyloid-beta clearance	5	P
GO:0071333	1	2	0.01522817334147536	0.03988331113243546	*	cellular response to glucose stimulus	5	P
GO:0003007	1	2	0.01522817334147536	0.03988331113243546	*	heart morphogenesis	5	P
GO:0060137	1	2	0.01522817334147536	0.03988331113243546	*	maternal process involved in parturition	5	P
GO:0042472	1	3	0.02026279413216311	0.04457814709075885	*	inner ear morphogenesis	5	P
GO:0021772	1	3	0.02026279413216311	0.04457814709075885	*	olfactory bulb development	5	P
GO:0050890	1	3	0.02026279413216311	0.04457814709075885	*	cognition	5	P
GO:0040018	1	3	0.02026279413216311	0.04457814709075885	*	positive regulation of multicellular organism growth	5	P
GO:0008015	1	4	0.02527680235052461	0.04793876307858116	*	blood circulation	5	P
GO:0042593	1	4	0.02527680235052461	0.04793876307858116	*	glucose homeostasis	5	P
GO:0000398	1	4	0.02527680235052461	0.04793876307858116	*	mRNA splicing, via spliceosome	5	P
GO:0008380	1	4	0.02527680235052461	0.04793876307858116	*	RNA splicing	5	P
GO:1904646	1	5	0.030270261354876433	0.05202701170369387	-	cellular response to amyloid-beta	5	P
GO:0038083	1	5	0.030270261354876433	0.05202701170369387	-	peptidyl-tyrosine autophosphorylation	5	P
GO:0006397	1	5	0.030270261354876433	0.05202701170369387	-	mRNA processing	5	P
GO:0007417	1	6	0.03524323437356937	0.0538438302929532	-	central nervous system development	5	P
GO:0007605	1	6	0.03524323437356937	0.0538438302929532	-	sensory perception of sound	5	P
GO:0006338	1	6	0.03524323437356937	0.0538438302929532	-	chromatin remodeling	5	P
GO:0001501	1	6	0.03524323437356937	0.0538438302929532	-	skeletal system development	5	P
GO:0048015	1	8	0.045127974718353545	0.06708212458133635	-	phosphatidylinositol-mediated signaling	5	P
GO:0007169	1	9	0.05003986785251839	0.07242612452338189	-	transmembrane receptor protein tyrosine kinase signaling pathway	5	P
GO:0001701	1	10	0.054931526617438484	0.07553084909897792	-	in utero embryonic development	5	P
GO:0009617	1	10	0.054931526617438484	0.07553084909897792	-	response to bacterium	5	P
GO:0008286	1	11	0.05980301359363726	0.08022355482073291	-	insulin receptor signaling pathway	5	P
GO:0014068	1	12	0.06465439123247298	0.0846664647091908	-	positive regulation of phosphatidylinositol 3-kinase signaling	5	P
GO:0043410	1	13	0.06948572185627218	0.08685715232034022	-	positive regulation of MAPK cascade	5	P
GO:0120162	1	13	0.06948572185627218	0.08685715232034022	-	positive regulation of cold-induced thermogenesis	5	P
GO:0046777	1	17	0.08861181613899642	0.10830333083655119	-	protein autophosphorylation	5	P
GO:0051897	1	18	0.09334384201574099	0.1116067676275164	-	positive regulation of protein kinase B signaling	5	P
GO:0006355	1	19	0.09805619210921426	0.1147466077873784	-	regulation of transcription, DNA-templated	5	P
GO:0071222	1	23	0.11671006118854332	0.13373027844520588	-	cellular response to lipopolysaccharide	5	P
GO:0030335	1	30	0.1486109170512175	0.16680817220034616	-	positive regulation of cell migration	5	P
GO:0008285	1	34	0.16642181332114345	0.18306399465325782	-	negative regulation of cell population proliferation	5	P
GO:0006955	1	39	0.188264938121611	0.20303081562134517	-	immune response	5	P
GO:0008284	1	44	0.20964774169123004	0.22174280371187793	-	positive regulation of cell population proliferation	5	P
GO:0043066	1	49	0.2305775382577759	0.23927857743731462	-	negative regulation of apoptotic process	5	P
GO:0045944	1	68	0.3061059780181605	0.31177460724071904	-	positive regulation of transcription by RNA polymerase II	5	P
GO:0007165	1	90	0.38602472580782415	0.38602472580782415	-	signal transduction	5	P
GO:0034237	2	1	0.0015345711326849273	0.11969654834942432	-	protein kinase A regulatory subunit binding	7	F
GO:0044548	2	5	0.010146317642944769	0.19425015236677864	-	S100 protein binding	7	F
GO:0042605	2	6	0.01333735971272478	0.19425015236677864	-	peptide antigen binding	7	F
GO:0003779	4	38	0.013878765301606954	0.19425015236677864	-	actin binding	7	F
GO:0051117	2	7	0.016906116966274667	0.19425015236677864	-	ATPase binding	7	F
GO:0050839	2	9	0.02510637315336162	0.19425015236677864	-	cell adhesion molecule binding	7	F
GO:0048306	2	11	0.03461178959761681	0.19425015236677864	-	calcium-dependent protein binding	7	F
GO:0044730	1	1	0.04603981311708075	0.19425015236677864	-	bone sialoprotein binding	7	F
GO:0008330	1	1	0.04603981311708075	0.19425015236677864	-	protein tyrosine/threonine phosphatase activity	7	F
GO:0046790	1	1	0.04603981311708075	0.19425015236677864	-	virion binding	7	F
GO:0031624	1	1	0.04603981311708075	0.19425015236677864	-	ubiquitin conjugating enzyme binding	7	F
GO:0005031	1	1	0.04603981311708075	0.19425015236677864	-	tumor necrosis factor-activated receptor activity	7	F
GO:0043120	1	1	0.04603981311708075	0.19425015236677864	-	tumor necrosis factor binding	7	F
GO:0099511	1	1	0.04603981311708075	0.19425015236677864	-	voltage-gated calcium channel activity involved in regulation of cytosolic calcium levels	7	F
GO:0045296	3	34	0.05109183460235402	0.19425015236677864	-	cadherin binding	7	F
GO:0042802	5	89	0.059217119662937676	0.19425015236677864	-	identical protein binding	7	F
GO:0003723	4	63	0.0642885387564624	0.19425015236677864	-	RNA binding	7	F
GO:0051018	1	2	0.06829243410927867	0.19425015236677864	-	protein kinase A binding	7	F
GO:0019960	1	2	0.06829243410927867	0.19425015236677864	-	C-X3-C chemokine binding	7	F
GO:0016791	1	2	0.06829243410927867	0.19425015236677864	-	phosphatase activity	7	F
GO:0017017	1	2	0.06829243410927867	0.19425015236677864	-	MAP kinase tyrosine/serine/threonine phosphatase activity	7	F
GO:0098772	1	2	0.06829243410927867	0.19425015236677864	-	molecular function regulator	7	F
GO:0097718	1	2	0.06829243410927867	0.19425015236677864	-	disordered domain specific binding	7	F
GO:0098639	1	2	0.06829243410927867	0.19425015236677864	-	collagen binding involved in cell-matrix adhesion	7	F
GO:0046914	1	2	0.06829243410927867	0.19425015236677864	-	transition metal ion binding	7	F
GO:0034236	1	2	0.06829243410927867	0.19425015236677864	-	protein kinase A catalytic subunit binding	7	F
GO:0005262	1	2	0.06829243410927867	0.19425015236677864	-	calcium channel activity	7	F
GO:0005516	2	17	0.06973082392653593	0.19425015236677864	-	calmodulin binding	7	F
GO:0002020	2	18	0.07639970360313647	0.2054888579670567	-	protease binding	7	F
GO:0008138	1	3	0.09004804224682848	0.22657249339524585	-	protein tyrosine/serine/threonine phosphatase activity	7	F
GO:0019834	1	3	0.09004804224682848	0.22657249339524585	-	phospholipase A2 inhibitor activity	7	F
GO:0015026	1	4	0.11131724451138433	0.2480784306253708	-	coreceptor activity	7	F
GO:0004601	1	4	0.11131724451138433	0.2480784306253708	-	peroxidase activity	7	F
GO:0001965	1	4	0.11131724451138433	0.2480784306253708	-	G-protein alpha-subunit binding	7	F
GO:0019902	1	4	0.11131724451138433	0.2480784306253708	-	phosphatase binding	7	F
GO:0001786	1	5	0.13211043207622478	0.2576153425486383	-	phosphatidylserine binding	7	F
GO:0050786	1	5	0.13211043207622478	0.2576153425486383	-	RAGE receptor binding	7	F
GO:0001671	1	5	0.13211043207622478	0.2576153425486383	-	ATPase activator activity	7	F
GO:0001968	1	5	0.13211043207622478	0.2576153425486383	-	fibronectin binding	7	F
GO:0003774	1	5	0.13211043207622478	0.2576153425486383	-	motor activity	7	F
GO:0023026	1	6	0.15243778448177148	0.2702306179449585	-	MHC class II protein complex binding	7	F
GO:0005544	1	6	0.15243778448177148	0.2702306179449585	-	calcium-dependent phospholipid binding	7	F
GO:0043236	1	6	0.15243778448177148	0.2702306179449585	-	laminin binding	7	F
GO:0001618	1	6	0.15243778448177148	0.2702306179449585	-	virus receptor activity	7	F
GO:0004725	1	7	0.17230927373449237	0.29217659459326967	-	protein tyrosine phosphatase activity	7	F
GO:0098641	1	7	0.17230927373449237	0.29217659459326967	-	cadherin binding involved in cell-cell adhesion	7	F
GO:0000981	3	60	0.17647547376171113	0.2928741904981589	-	DNA-binding transcription factor activity, RNA polymerase II-specific	7	F
GO:0005125	1	8	0.1917346683305196	0.293241257446677	-	cytokine activity	7	F
GO:0008092	1	8	0.1917346683305196	0.293241257446677	-	cytoskeletal protein binding	7	F
GO:0046983	1	8	0.1917346683305196	0.293241257446677	-	protein dimerization activity	7	F
GO:0003730	1	8	0.1917346683305196	0.293241257446677	-	mRNA 3'-UTR binding	7	F
GO:0061630	1	9	0.21072353720528791	0.3056317061361507	-	ubiquitin protein ligase activity	7	F
GO:0004842	1	9	0.21072353720528791	0.3056317061361507	-	ubiquitin-protein transferase activity	7	F
GO:0003674	2	35	0.21159118117118125	0.3056317061361507	-	molecular_function	7	F
GO:0005546	1	10	0.22928525361048036	0.32516817784759033	-	phosphatidylinositol-4,5-bisphosphate binding	7	F
GO:0044212	1	11	0.24742899891954795	0.34463324849508464	-	transcription regulatory region DNA binding	7	F
GO:0017137	1	12	0.26516376636304206	0.3505554877341912	-	Rab GTPase binding	7	F
GO:0003714	1	12	0.26516376636304206	0.3505554877341912	-	transcription corepressor activity	7	F
GO:0008022	1	12	0.26516376636304206	0.3505554877341912	-	protein C-terminus binding	7	F
GO:0008017	1	15	0.31600138818287776	0.41080180463774113	-	microtubule binding	7	F
GO:0003677	2	48	0.3262973491341961	0.41723267594208685	-	DNA binding	7	F
GO:0005509	2	53	0.3699885474474683	0.4654694629177827	-	calcium ion binding	7	F
GO:0005096	1	19	0.3785733425380989	0.4687098526662177	-	GTPase activator activity	7	F
GO:0046872	3	94	0.3967772573660839	0.48357228241491473	-	metal ion binding	7	F
GO:0005178	1	21	0.40776802607043516	0.48932163128452216	-	integrin binding	7	F
GO:0008134	1	22	0.42186878735447253	0.49857220323710394	-	transcription factor binding	7	F
GO:0044325	1	23	0.4356480805056093	0.5071723922304109	-	ion channel binding	7	F
GO:0044877	1	25	0.462270128715192	0.5302510299968378	-	protein-containing complex binding	7	F
GO:0005515	16	644	0.48558572941420186	0.5489229984682281	-	protein binding	7	F
GO:0031625	1	29	0.5119551516270642	0.5624296031959297	-	ubiquitin protein ligase binding	7	F
GO:0019904	1	29	0.5119551516270642	0.5624296031959297	-	protein domain specific binding	7	F
GO:0051015	1	33	0.5572311439981327	0.6001185934118796	-	actin filament binding	7	F
GO:0042803	2	77	0.5616494528085539	0.6001185934118796	-	protein homodimerization activity	7	F
GO:0005525	1	40	0.6269655483772069	0.660855578019218	-	GTP binding	7	F
GO:0003700	1	41	0.6360255192191546	0.6614665399879208	-	DNA-binding transcription factor activity	7	F
GO:0003924	1	42	0.6448747930647776	0.661845182355956	-	GTPase activity	7	F
GO:0046982	1	53	0.729581481303279	0.7390565654760488	-	protein heterodimerization activity	7	F
GO:0005524	1	80	0.8633952671137507	0.8633952671137507	-	ATP binding	7	F
GO:0048471	6	56	0.0020843552690810105	0.16188192354070327	-	perinuclear region of cytoplasm	7	C
GO:0016020	10	179	0.005877019709808843	0.16188192354070327	-	membrane	7	C
GO:0042470	3	15	0.0071666791156103265	0.16188192354070327	-	melanosome	7	C
GO:0001726	3	17	0.009710906837131028	0.16188192354070327	-	ruffle	7	C
GO:0030175	2	6	0.01333735971272478	0.16188192354070327	-	filopodium	7	C
GO:0012507	2	7	0.016906116966274667	0.16188192354070327	-	ER to Golgi transport vesicle membrane	7	C
GO:0071556	2	7	0.016906116966274667	0.16188192354070327	-	integral component of lumenal side of endoplasmic reticulum membrane	7	C
GO:0042383	2	8	0.020834848457606005	0.16188192354070327	-	sarcolemma	7	C
GO:0042613	2	8	0.020834848457606005	0.16188192354070327	-	MHC class II protein complex	7	C
GO:0030658	2	8	0.020834848457606005	0.16188192354070327	-	transport vesicle membrane	7	C
GO:0032588	2	10	0.029704055210737903	0.16188192354070327	-	trans-Golgi network membrane	7	C
GO:0030669	2	11	0.03461178959761681	0.16188192354070327	-	clathrin-coated endocytic vesicle membrane	7	C
GO:0030666	2	13	0.04529556534364298	0.16188192354070327	-	endocytic vesicle membrane	7	C
GO:0000139	4	56	0.0456298028791751	0.16188192354070327	-	Golgi membrane	7	C
GO:0071437	1	1	0.04603981311708075	0.16188192354070327	-	invadopodium	7	C
GO:0044354	1	1	0.04603981311708075	0.16188192354070327	-	macropinosome	7	C
GO:0043196	1	1	0.04603981311708075	0.16188192354070327	-	varicosity	7	C
GO:0034665	1	1	0.04603981311708075	0.16188192354070327	-	integrin alpha1-beta1 complex	7	C
GO:0035749	1	1	0.04603981311708075	0.16188192354070327	-	myelin sheath adaxonal region	7	C
GO:0035748	1	1	0.04603981311708075	0.16188192354070327	-	myelin sheath abaxonal region	7	C
GO:0031528	1	1	0.04603981311708075	0.16188192354070327	-	microvillus membrane	7	C
GO:0034680	1	1	0.04603981311708075	0.16188192354070327	-	integrin alpha10-beta1 complex	7	C
GO:0034677	1	1	0.04603981311708075	0.16188192354070327	-	integrin alpha7-beta1 complex	7	C
GO:0034678	1	1	0.04603981311708075	0.16188192354070327	-	integrin alpha8-beta1 complex	7	C
GO:0034667	1	1	0.04603981311708075	0.16188192354070327	-	integrin alpha3-beta1 complex	7	C
GO:0034681	1	1	0.04603981311708075	0.16188192354070327	-	integrin alpha11-beta1 complex	7	C
GO:0034704	1	1	0.04603981311708075	0.16188192354070327	-	calcium channel complex	7	C
GO:0002947	1	1	0.04603981311708075	0.16188192354070327	-	tumor necrosis factor receptor superfamily complex	7	C
GO:0098686	1	1	0.04603981311708075	0.16188192354070327	-	hippocampal mossy fiber to CA3 synapse	7	C
GO:1990667	1	1	0.04603981311708075	0.16188192354070327	-	PCSK9-AnxA2 complex	7	C
GO:0099699	1	1	0.04603981311708075	0.16188192354070327	-	integral component of synaptic membrane	7	C
GO:0031902	2	16	0.06327298474603034	0.1767595304184283	-	late endosome membrane	7	C
GO:0005765	3	38	0.06597103265193542	0.1767595304184283	-	lysosomal membrane	7	C
GO:0043220	1	2	0.06829243410927867	0.1767595304184283	-	Schmidt-Lanterman incisure	7	C
GO:0000151	1	2	0.06829243410927867	0.1767595304184283	-	ubiquitin ligase complex	7	C
GO:0045177	1	2	0.06829243410927867	0.1767595304184283	-	apical part of cell	7	C
GO:0071944	1	2	0.06829243410927867	0.1767595304184283	-	cell periphery	7	C
GO:0071438	1	2	0.06829243410927867	0.1767595304184283	-	invadopodium membrane	7	C
GO:0019897	1	2	0.06829243410927867	0.1767595304184283	-	extrinsic component of plasma membrane	7	C
GO:0016461	1	2	0.06829243410927867	0.1767595304184283	-	unconventional myosin complex	7	C
GO:0001931	1	2	0.06829243410927867	0.1767595304184283	-	uropod	7	C
GO:0016323	2	17	0.06973082392653593	0.1767595304184283	-	basolateral plasma membrane	7	C
GO:0043209	2	17	0.06973082392653593	0.1767595304184283	-	myelin sheath	7	C
GO:0032587	2	18	0.07639970360313647	0.1892629021077699	-	ruffle membrane	7	C
GO:0005615	5	98	0.08190578625754387	0.19245561970400596	-	extracellular space	7	C
GO:0009986	4	69	0.08312373452089415	0.19245561970400596	-	cell surface	7	C
GO:0097386	1	3	0.09004804224682848	0.19245561970400596	-	glial cell projection	7	C
GO:0030315	1	3	0.09004804224682848	0.19245561970400596	-	T-tubule	7	C
GO:0014704	1	3	0.09004804224682848	0.19245561970400596	-	intercalated disc	7	C
GO:0034666	1	3	0.09004804224682848	0.19245561970400596	-	integrin alpha2-beta1 complex	7	C
GO:0044853	1	3	0.09004804224682848	0.19245561970400596	-	plasma membrane raft	7	C
GO:0045121	3	48	0.11081003593915896	0.2246959194766832	-	membrane raft	7	C
GO:0030863	1	4	0.11131724451138433	0.2246959194766832	-	cortical cytoskeleton	7	C
GO:0098685	1	4	0.11131724451138433	0.2246959194766832	-	Schaffer collateral - CA1 synapse	7	C
GO:0031594	1	5	0.13211043207622478	0.24406842536116102	-	neuromuscular junction	7	C
GO:0098794	1	5	0.13211043207622478	0.24406842536116102	-	postsynapse	7	C
GO:0036064	1	5	0.13211043207622478	0.24406842536116102	-	ciliary basal body	7	C
GO:0005604	1	5	0.13211043207622478	0.24406842536116102	-	basement membrane	7	C
GO:0001650	1	5	0.13211043207622478	0.24406842536116102	-	fibrillar center	7	C
GO:0005768	2	26	0.13595218908119408	0.24697981016416926	-	endosome	7	C
GO:0005903	1	6	0.15243778448177148	0.2679954598147273	-	brush border	7	C
GO:0001772	1	6	0.15243778448177148	0.2679954598147273	-	immunological synapse	7	C
GO:0031982	2	29	0.16044059486576406	0.27620162995675984	-	vesicle	7	C
GO:0044297	1	7	0.17230927373449237	0.27620162995675984	-	cell body	7	C
GO:0005811	1	7	0.17230927373449237	0.27620162995675984	-	lipid droplet	7	C
GO:0019898	1	7	0.17230927373449237	0.27620162995675984	-	extrinsic component of membrane	7	C
GO:0008305	1	7	0.17230927373449237	0.27620162995675984	-	integrin complex	7	C
GO:0000932	1	7	0.17230927373449237	0.27620162995675984	-	P-body	7	C
GO:0031234	1	8	0.1917346683305196	0.2985582692575234	-	extrinsic component of cytoplasmic side of plasma membrane	7	C
GO:0032154	1	8	0.1917346683305196	0.2985582692575234	-	cleavage furrow	7	C
GO:0098978	2	36	0.22031091433562858	0.33822379806455655	-	glutamatergic synapse	7	C
GO:0030496	1	11	0.24742899891954795	0.37458001225320453	-	midbody	7	C
GO:0005902	1	12	0.26516376636304206	0.3959294593639943	-	microvillus	7	C
GO:0005789	2	43	0.28207765873733054	0.4051621283125527	-	endoplasmic reticulum membrane	7	C
GO:0043235	1	13	0.2824983646949909	0.4051621283125527	-	receptor complex	7	C
GO:0055037	1	13	0.2824983646949909	0.4051621283125527	-	recycling endosome	7	C
GO:0043197	1	14	0.2994414217915157	0.42388461006850925	-	dendritic spine	7	C
GO:0005913	1	15	0.31600138818287776	0.43055189139917094	-	cell-cell adherens junction	7	C
GO:0031965	1	15	0.31600138818287776	0.43055189139917094	-	nuclear membrane	7	C
GO:0005741	1	15	0.31600138818287776	0.43055189139917094	-	mitochondrial outer membrane	7	C
GO:0005884	1	16	0.33218654052011415	0.44156503556942006	-	actin filament	7	C
GO:0016324	1	16	0.33218654052011415	0.44156503556942006	-	apical plasma membrane	7	C
GO:0042995	1	18	0.3634646605913507	0.47163866671972887	-	cell projection	7	C
GO:0005938	1	18	0.3634646605913507	0.47163866671972887	-	cell cortex	7	C
GO:0043005	1	19	0.3785733425380989	0.4798197015889858	-	neuron projection	7	C
GO:0031410	1	19	0.3785733425380989	0.4798197015889858	-	cytoplasmic vesicle	7	C
GO:0005783	2	55	0.3872197762215829	0.4851374207833624	-	endoplasmic reticulum	7	C
GO:0035578	1	20	0.3933386453496935	0.48720354935359755	-	azurophil granule lumen	7	C
GO:0035579	1	21	0.40776802607043516	0.49940129035592623	-	specific granule membrane	7	C
GO:0030027	1	22	0.42186878735447253	0.5109299757959722	-	lamellipodium	7	C
GO:0005576	4	141	0.4432919998340461	0.5263796453997666	-	extracellular region	7	C
GO:0005813	1	24	0.4491129084603513	0.5263796453997666	-	centrosome	7	C
GO:0005769	1	24	0.4491129084603513	0.5263796453997666	-	early endosome	7	C
GO:0010008	1	25	0.462270128715192	0.5304470500644202	-	endosome membrane	7	C
GO:0005739	2	64	0.4623162362946781	0.5304470500644202	-	mitochondrion	7	C
GO:0009897	1	27	0.4876884660923894	0.5537296125424005	-	external side of plasma membrane	7	C
GO:0005737	8	316	0.4975718482267951	0.5591271284198007	-	cytoplasm	7	C
GO:0043025	1	33	0.5572311439981327	0.6197774968958822	-	neuronal cell body	7	C
GO:0005794	2	78	0.5687784524136823	0.6199685131309137	-	Golgi apparatus	7	C
GO:0005925	2	78	0.5687784524136823	0.6199685131309137	-	focal adhesion	7	C
GO:0005634	7	298	0.5973941583602962	0.6447125075373494	-	nucleus	7	C
GO:0015629	1	39	0.6176900971476668	0.6600805940107419	-	actin cytoskeleton	7	C
GO:0062023	1	40	0.6269655483772069	0.6634878133312189	-	collagen-containing extracellular matrix	7	C
GO:0070062	6	267	0.6453834516237256	0.6764115021825585	-	extracellular exosome	7	C
GO:0016021	3	143	0.6846432905351791	0.7107249396984241	-	integral component of membrane	7	C
GO:0005886	7	323	0.6948532970427085	0.7145189563929739	-	plasma membrane	7	C
GO:0032991	1	50	0.7086254147272911	0.7218707495820068	-	protein-containing complex	7	C
GO:0005829	8	393	0.7829302028566704	0.7901795565868247	-	cytosol	7	C
GO:0005654	2	166	0.9238438723353958	0.9238438723353958	-	nucleoplasm	7	C
GO:1901222	2	1	0.0015345711326849273	0.1617452386798705	-	regulation of NIK/NF-kappaB signaling	7	P
GO:0007159	2	7	0.016906116966274667	0.1617452386798705	-	leukocyte cell-cell adhesion	7	P
GO:0070373	2	7	0.016906116966274667	0.1617452386798705	-	negative regulation of ERK1 and ERK2 cascade	7	P
GO:1903078	2	9	0.02510637315336162	0.1617452386798705	-	positive regulation of protein localization to plasma membrane	7	P
GO:0009617	2	9	0.02510637315336162	0.1617452386798705	-	response to bacterium	7	P
GO:0060333	2	10	0.029704055210737903	0.1617452386798705	-	interferon-gamma-mediated signaling pathway	7	P
GO:0006511	2	10	0.029704055210737903	0.1617452386798705	-	ubiquitin-dependent protein catabolic process	7	P
GO:1903364	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of cellular protein catabolic process	7	P
GO:0036035	1	1	0.04603981311708075	0.1617452386798705	-	osteoclast development	7	P
GO:0036499	1	1	0.04603981311708075	0.1617452386798705	-	PERK-mediated unfolded protein response	7	P
GO:0043517	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of DNA damage response, signal transduction by p53 class mediator	7	P
GO:0044090	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of vacuole organization	7	P
GO:0044147	1	1	0.04603981311708075	0.1617452386798705	-	negative regulation of development of symbiont involved in interaction with host	7	P
GO:2000643	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of early endosome to late endosome transport	7	P
GO:1902896	1	1	0.04603981311708075	0.1617452386798705	-	terminal web assembly	7	P
GO:1902966	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of protein localization to early endosome	7	P
GO:0003176	1	1	0.04603981311708075	0.1617452386798705	-	aortic valve development	7	P
GO:0002739	1	1	0.04603981311708075	0.1617452386798705	-	regulation of cytokine secretion involved in immune response	7	P
GO:0002724	1	1	0.04603981311708075	0.1617452386798705	-	regulation of T cell cytokine production	7	P
GO:0003177	1	1	0.04603981311708075	0.1617452386798705	-	pulmonary valve development	7	P
GO:0006694	1	1	0.04603981311708075	0.1617452386798705	-	steroid biosynthetic process	7	P
GO:0007016	1	1	0.04603981311708075	0.1617452386798705	-	cytoskeletal anchoring at plasma membrane	7	P
GO:0003332	1	1	0.04603981311708075	0.1617452386798705	-	negative regulation of extracellular matrix constituent secretion	7	P
GO:0003417	1	1	0.04603981311708075	0.1617452386798705	-	growth plate cartilage development	7	P
GO:1900017	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of cytokine production involved in inflammatory response	7	P
GO:0150077	1	1	0.04603981311708075	0.1617452386798705	-	regulation of neuroinflammatory response	7	P
GO:1900227	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of NLRP3 inflammasome complex assembly	7	P
GO:1902043	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of extrinsic apoptotic signaling pathway via death domain receptors	7	P
GO:0007161	1	1	0.04603981311708075	0.1617452386798705	-	calcium-independent cell-matrix adhesion	7	P
GO:1903753	1	1	0.04603981311708075	0.1617452386798705	-	negative regulation of p38MAPK cascade	7	P
GO:1903902	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of viral life cycle	7	P
GO:0042149	1	1	0.04603981311708075	0.1617452386798705	-	cellular response to glucose starvation	7	P
GO:0010710	1	1	0.04603981311708075	0.1617452386798705	-	regulation of collagen catabolic process	7	P
GO:0072593	1	1	0.04603981311708075	0.1617452386798705	-	reactive oxygen species metabolic process	7	P
GO:0072332	1	1	0.04603981311708075	0.1617452386798705	-	intrinsic apoptotic signaling pathway by p53 class mediator	7	P
GO:0050702	1	1	0.04603981311708075	0.1617452386798705	-	interleukin-1 beta secretion	7	P
GO:0034629	1	1	0.04603981311708075	0.1617452386798705	-	cellular protein-containing complex localization	7	P
GO:0046902	1	1	0.04603981311708075	0.1617452386798705	-	regulation of mitochondrial membrane permeability	7	P
GO:0034101	1	1	0.04603981311708075	0.1617452386798705	-	erythrocyte homeostasis	7	P
GO:0033631	1	1	0.04603981311708075	0.1617452386798705	-	cell-cell adhesion mediated by integrin	7	P
GO:0052362	1	1	0.04603981311708075	0.1617452386798705	-	catabolism by host of symbiont protein	7	P
GO:0007589	1	1	0.04603981311708075	0.1617452386798705	-	body fluid secretion	7	P
GO:0055007	1	1	0.04603981311708075	0.1617452386798705	-	cardiac muscle cell differentiation	7	P
GO:0052405	1	1	0.04603981311708075	0.1617452386798705	-	negative regulation by host of symbiont molecular function	7	P
GO:0060956	1	1	0.04603981311708075	0.1617452386798705	-	endocardial cell differentiation	7	P
GO:0032461	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of protein oligomerization	7	P
GO:0032532	1	1	0.04603981311708075	0.1617452386798705	-	regulation of microvillus length	7	P
GO:0032872	1	1	0.04603981311708075	0.1617452386798705	-	regulation of stress-activated MAPK cascade	7	P
GO:1902339	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of apoptotic process involved in morphogenesis	7	P
GO:0010755	1	1	0.04603981311708075	0.1617452386798705	-	regulation of plasminogen activation	7	P
GO:0030101	1	1	0.04603981311708075	0.1617452386798705	-	natural killer cell activation	7	P
GO:0030199	1	1	0.04603981311708075	0.1617452386798705	-	collagen fibril organization	7	P
GO:0050779	1	1	0.04603981311708075	0.1617452386798705	-	RNA destabilization	7	P
GO:0030953	1	1	0.04603981311708075	0.1617452386798705	-	astral microtubule organization	7	P
GO:0019430	1	1	0.04603981311708075	0.1617452386798705	-	removal of superoxide radicals	7	P
GO:0051660	1	1	0.04603981311708075	0.1617452386798705	-	establishment of centrosome localization	7	P
GO:1905599	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of low-density lipoprotein receptor activity	7	P
GO:1905597	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of low-density lipoprotein particle receptor binding	7	P
GO:0001951	1	1	0.04603981311708075	0.1617452386798705	-	intestinal D-glucose absorption	7	P
GO:1902237	1	1	0.04603981311708075	0.1617452386798705	-	positive regulation of endoplasmic reticulum stress-induced intrinsic apoptotic signaling pathway	7	P
GO:0072659	2	15	0.057038950787094755	0.1617452386798705	-	protein localization to plasma membrane	7	P
GO:0050852	2	15	0.057038950787094755	0.1617452386798705	-	T cell receptor signaling pathway	7	P
GO:0031623	2	16	0.06327298474603034	0.1617452386798705	-	receptor internalization	7	P
GO:0043029	1	2	0.06829243410927867	0.1617452386798705	-	T cell homeostasis	7	P
GO:0001708	1	2	0.06829243410927867	0.1617452386798705	-	cell fate specification	7	P
GO:0001921	1	2	0.06829243410927867	0.1617452386798705	-	positive regulation of receptor recycling	7	P
GO:0021943	1	2	0.06829243410927867	0.1617452386798705	-	formation of radial glial scaffolds	7	P
GO:0071711	1	2	0.06829243410927867	0.1617452386798705	-	basement membrane organization	7	P
GO:0030033	1	2	0.06829243410927867	0.1617452386798705	-	microvillus assembly	7	P
GO:0035024	1	2	0.06829243410927867	0.1617452386798705	-	negative regulation of Rho protein signal transduction	7	P
GO:0097066	1	2	0.06829243410927867	0.1617452386798705	-	response to thyroid hormone	7	P
GO:0010614	1	2	0.06829243410927867	0.1617452386798705	-	negative regulation of cardiac muscle hypertrophy	7	P
GO:0097193	1	2	0.06829243410927867	0.1617452386798705	-	intrinsic apoptotic signaling pathway	7	P
GO:1902236	1	2	0.06829243410927867	0.1617452386798705	-	negative regulation of endoplasmic reticulum stress-induced intrinsic apoptotic signaling pathway	7	P
GO:0031214	1	2	0.06829243410927867	0.1617452386798705	-	biomineral tissue development	7	P
GO:0010737	1	2	0.06829243410927867	0.1617452386798705	-	protein kinase A signaling	7	P
GO:0010498	1	2	0.06829243410927867	0.1617452386798705	-	proteasomal protein catabolic process	7	P
GO:0048675	1	2	0.06829243410927867	0.1617452386798705	-	axon extension	7	P
GO:0032804	1	2	0.06829243410927867	0.1617452386798705	-	negative regulation of low-density lipoprotein particle receptor catabolic process	7	P
GO:0032469	1	2	0.06829243410927867	0.1617452386798705	-	endoplasmic reticulum calcium ion homeostasis	7	P
GO:1905581	1	2	0.06829243410927867	0.1617452386798705	-	positive regulation of low-density lipoprotein particle clearance	7	P
GO:0061028	1	2	0.06829243410927867	0.1617452386798705	-	establishment of endothelial barrier	7	P
GO:0008354	1	2	0.06829243410927867	0.1617452386798705	-	germ cell migration	7	P
GO:0009968	1	2	0.06829243410927867	0.1617452386798705	-	negative regulation of signal transduction	7	P
GO:0031340	1	2	0.06829243410927867	0.1617452386798705	-	positive regulation of vesicle fusion	7	P
GO:0031396	1	2	0.06829243410927867	0.1617452386798705	-	regulation of protein ubiquitination	7	P
GO:0031641	1	2	0.06829243410927867	0.1617452386798705	-	regulation of myelination	7	P
GO:0001765	1	2	0.06829243410927867	0.1617452386798705	-	membrane raft assembly	7	P
GO:0001895	1	2	0.06829243410927867	0.1617452386798705	-	retina homeostasis	7	P
GO:1905602	1	2	0.06829243410927867	0.1617452386798705	-	positive regulation of receptor-mediated endocytosis involved in cholesterol transport	7	P
GO:2000273	1	2	0.06829243410927867	0.1617452386798705	-	positive regulation of signaling receptor activity	7	P
GO:0045198	1	2	0.06829243410927867	0.1617452386798705	-	establishment of epithelial cell apical/basal polarity	7	P
GO:0042129	1	2	0.06829243410927867	0.1617452386798705	-	regulation of T cell proliferation	7	P
GO:0002506	1	2	0.06829243410927867	0.1617452386798705	-	polysaccharide assembly with MHC class II protein complex	7	P
GO:0042744	1	2	0.06829243410927867	0.1617452386798705	-	hydrogen peroxide catabolic process	7	P
GO:0050776	2	17	0.06973082392653593	0.16232040496489694	-	regulation of immune response	7	P
GO:0019221	3	39	0.06997813014042223	0.16232040496489694	-	cytokine-mediated signaling pathway	7	P
GO:0040018	1	3	0.09004804224682848	0.17025890340786898	-	positive regulation of multicellular organism growth	7	P
GO:0042267	1	3	0.09004804224682848	0.17025890340786898	-	natural killer cell mediated cytotoxicity	7	P
GO:0042730	1	3	0.09004804224682848	0.17025890340786898	-	fibrinolysis	7	P
GO:0002042	1	3	0.09004804224682848	0.17025890340786898	-	cell migration involved in sprouting angiogenesis	7	P
GO:2000811	1	3	0.09004804224682848	0.17025890340786898	-	negative regulation of anoikis	7	P
GO:0001836	1	3	0.09004804224682848	0.17025890340786898	-	release of cytochrome c from mitochondria	7	P
GO:0032332	1	3	0.09004804224682848	0.17025890340786898	-	positive regulation of chondrocyte differentiation	7	P
GO:0050901	1	3	0.09004804224682848	0.17025890340786898	-	leukocyte tethering or rolling	7	P
GO:0051044	1	3	0.09004804224682848	0.17025890340786898	-	positive regulation of membrane protein ectodomain proteolysis	7	P
GO:0048813	1	3	0.09004804224682848	0.17025890340786898	-	dendrite morphogenesis	7	P
GO:0048333	1	3	0.09004804224682848	0.17025890340786898	-	mesodermal cell differentiation	7	P
GO:0010917	1	3	0.09004804224682848	0.17025890340786898	-	negative regulation of mitochondrial membrane potential	7	P
GO:0045596	1	3	0.09004804224682848	0.17025890340786898	-	negative regulation of cell differentiation	7	P
GO:0097191	1	3	0.09004804224682848	0.17025890340786898	-	extrinsic apoptotic signaling pathway	7	P
GO:0051480	1	3	0.09004804224682848	0.17025890340786898	-	regulation of cytosolic calcium ion concentration	7	P
GO:0051290	1	3	0.09004804224682848	0.17025890340786898	-	protein heterotetramerization	7	P
GO:0030030	1	3	0.09004804224682848	0.17025890340786898	-	cell projection organization	7	P
GO:0044794	1	3	0.09004804224682848	0.17025890340786898	-	positive regulation by host of viral process	7	P
GO:0035970	1	3	0.09004804224682848	0.17025890340786898	-	peptidyl-threonine dephosphorylation	7	P
GO:0007029	1	3	0.09004804224682848	0.17025890340786898	-	endoplasmic reticulum organization	7	P
GO:2001244	1	3	0.09004804224682848	0.17025890340786898	-	positive regulation of intrinsic apoptotic signaling pathway	7	P
GO:0000188	1	3	0.09004804224682848	0.17025890340786898	-	inactivation of MAPK activity	7	P
GO:0001837	1	4	0.11131724451138433	0.19266446165431905	-	epithelial to mesenchymal transition	7	P
GO:0008542	1	4	0.11131724451138433	0.19266446165431905	-	visual learning	7	P
GO:0050804	1	4	0.11131724451138433	0.19266446165431905	-	modulation of chemical synaptic transmission	7	P
GO:0006900	1	4	0.11131724451138433	0.19266446165431905	-	vesicle budding from membrane	7	P
GO:0045214	1	4	0.11131724451138433	0.19266446165431905	-	sarcomere organization	7	P
GO:0090200	1	4	0.11131724451138433	0.19266446165431905	-	positive regulation of release of cytochrome c from mitochondria	7	P
GO:0006986	1	4	0.11131724451138433	0.19266446165431905	-	response to unfolded protein	7	P
GO:0045089	1	4	0.11131724451138433	0.19266446165431905	-	positive regulation of innate immune response	7	P
GO:0031589	1	4	0.11131724451138433	0.19266446165431905	-	cell-substrate adhesion	7	P
GO:0050860	1	4	0.11131724451138433	0.19266446165431905	-	negative regulation of T cell receptor signaling pathway	7	P
GO:0016311	1	4	0.11131724451138433	0.19266446165431905	-	dephosphorylation	7	P
GO:0019886	2	23	0.11249220959180894	0.1932118103676108	-	antigen processing and presentation of exogenous peptide antigen via MHC class II	7	P
GO:2001243	1	5	0.13211043207622478	0.20932990997993364	-	negative regulation of intrinsic apoptotic signaling pathway	7	P
GO:0034976	1	5	0.13211043207622478	0.20932990997993364	-	response to endoplasmic reticulum stress	7	P
GO:0007193	1	5	0.13211043207622478	0.20932990997993364	-	adenylate cyclase-inhibiting G protein-coupled receptor signaling pathway	7	P
GO:1900740	1	5	0.13211043207622478	0.20932990997993364	-	positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway	7	P
GO:0008277	1	5	0.13211043207622478	0.20932990997993364	-	regulation of G protein-coupled receptor signaling pathway	7	P
GO:0033627	1	5	0.13211043207622478	0.20932990997993364	-	cell adhesion mediated by integrin	7	P
GO:0071320	1	5	0.13211043207622478	0.20932990997993364	-	cellular response to cAMP	7	P
GO:0071363	1	5	0.13211043207622478	0.20932990997993364	-	cellular response to growth factor stimulus	7	P
GO:0050870	1	5	0.13211043207622478	0.20932990997993364	-	positive regulation of T cell activation	7	P
GO:0008630	1	5	0.13211043207622478	0.20932990997993364	-	intrinsic apoptotic signaling pathway in response to DNA damage	7	P
GO:0031532	1	5	0.13211043207622478	0.20932990997993364	-	actin cytoskeleton reorganization	7	P
GO:0043065	2	26	0.13595218908119408	0.2139107870158648	-	positive regulation of apoptotic process	7	P
GO:0071404	1	6	0.15243778448177148	0.22865667672265721	-	cellular response to low-density lipoprotein particle stimulus	7	P
GO:0007156	1	6	0.15243778448177148	0.22865667672265721	-	homophilic cell adhesion via plasma membrane adhesion molecules	7	P
GO:0034113	1	6	0.15243778448177148	0.22865667672265721	-	heterotypic cell-cell adhesion	7	P
GO:0050714	1	6	0.15243778448177148	0.22865667672265721	-	positive regulation of protein secretion	7	P
GO:0001501	1	6	0.15243778448177148	0.22865667672265721	-	skeletal system development	7	P
GO:0032092	1	6	0.15243778448177148	0.22865667672265721	-	positive regulation of protein binding	7	P
GO:0008361	1	6	0.15243778448177148	0.22865667672265721	-	regulation of cell size	7	P
GO:0008150	2	29	0.16044059486576406	0.23906711155494645	-	biological_process	7	P
GO:0050728	1	7	0.17230927373449237	0.24383387792616842	-	negative regulation of inflammatory response	7	P
GO:0043280	1	7	0.17230927373449237	0.24383387792616842	-	positive regulation of cysteine-type endopeptidase activity involved in apoptotic process	7	P
GO:0045454	1	7	0.17230927373449237	0.24383387792616842	-	cell redox homeostasis	7	P
GO:0043149	1	7	0.17230927373449237	0.24383387792616842	-	stress fiber assembly	7	P
GO:0051726	1	7	0.17230927373449237	0.24383387792616842	-	regulation of cell cycle	7	P
GO:0032436	1	7	0.17230927373449237	0.24383387792616842	-	positive regulation of proteasomal ubiquitin-dependent protein catabolic process	7	P
GO:0071456	1	7	0.17230927373449237	0.24383387792616842	-	cellular response to hypoxia	7	P
GO:0000209	1	7	0.17230927373449237	0.24383387792616842	-	protein polyubiquitination	7	P
GO:0043547	2	32	0.18572336997081487	0.25678750222837443	-	positive regulation of GTPase activity	7	P
GO:0000082	1	8	0.1917346683305196	0.25678750222837443	-	G1/S transition of mitotic cell cycle	7	P
GO:0070588	1	8	0.1917346683305196	0.25678750222837443	-	calcium ion transmembrane transport	7	P
GO:0032781	1	8	0.1917346683305196	0.25678750222837443	-	positive regulation of ATPase activity	7	P
GO:0046718	1	8	0.1917346683305196	0.25678750222837443	-	viral entry into host cell	7	P
GO:0043086	1	8	0.1917346683305196	0.25678750222837443	-	negative regulation of catalytic activity	7	P
GO:0051289	1	8	0.1917346683305196	0.25678750222837443	-	protein homotetramerization	7	P
GO:0034599	1	8	0.1917346683305196	0.25678750222837443	-	cellular response to oxidative stress	7	P
GO:0048015	1	8	0.1917346683305196	0.25678750222837443	-	phosphatidylinositol-mediated signaling	7	P
GO:0035335	1	9	0.21072353720528791	0.2724873325930447	-	peptidyl-tyrosine dephosphorylation	7	P
GO:0033209	1	9	0.21072353720528791	0.2724873325930447	-	tumor necrosis factor-mediated signaling pathway	7	P
GO:0048146	1	9	0.21072353720528791	0.2724873325930447	-	positive regulation of fibroblast proliferation	7	P
GO:0006979	1	9	0.21072353720528791	0.2724873325930447	-	response to oxidative stress	7	P
GO:0030032	1	9	0.21072353720528791	0.2724873325930447	-	lamellipodium assembly	7	P
GO:0051017	1	9	0.21072353720528791	0.2724873325930447	-	actin filament bundle assembly	7	P
GO:0043488	1	10	0.22928525361048036	0.28820772101876024	-	regulation of mRNA stability	7	P
GO:0032729	1	10	0.22928525361048036	0.28820772101876024	-	positive regulation of interferon-gamma production	7	P
GO:0035722	1	10	0.22928525361048036	0.28820772101876024	-	interleukin-12-mediated signaling pathway	7	P
GO:0030183	1	10	0.22928525361048036	0.28820772101876024	-	B cell differentiation	7	P
GO:0006952	1	10	0.22928525361048036	0.28820772101876024	-	defense response	7	P
GO:0006955	2	38	0.23785736429221785	0.2973217053652723	-	immune response	7	P
GO:0051928	1	11	0.24742899891954795	0.30421598227813273	-	positive regulation of calcium ion transport	7	P
GO:0001701	1	11	0.24742899891954795	0.30421598227813273	-	in utero embryonic development	7	P
GO:0006968	1	11	0.24742899891954795	0.30421598227813273	-	cellular defense response	7	P
GO:0007411	1	12	0.26516376636304206	0.3224964726036998	-	axon guidance	7	P
GO:0043154	1	12	0.26516376636304206	0.3224964726036998	-	negative regulation of cysteine-type endopeptidase activity involved in apoptotic process	7	P
GO:0071346	1	13	0.2824983646949909	0.3363075770178463	-	cellular response to interferon-gamma	7	P
GO:0006909	1	13	0.2824983646949909	0.3363075770178463	-	phagocytosis	7	P
GO:0098609	1	13	0.2824983646949909	0.3363075770178463	-	cell-cell adhesion	7	P
GO:0051607	1	13	0.2824983646949909	0.3363075770178463	-	defense response to virus	7	P
GO:0006919	1	14	0.2994414217915157	0.3509079161619325	-	activation of cysteine-type endopeptidase activity involved in apoptotic process	7	P
GO:0007160	1	14	0.2994414217915157	0.3509079161619325	-	cell-matrix adhesion	7	P
GO:0042981	1	14	0.2994414217915157	0.3509079161619325	-	regulation of apoptotic process	7	P
GO:0042102	1	15	0.31600138818287776	0.3646169863648589	-	positive regulation of T cell proliferation	7	P
GO:0032956	1	15	0.31600138818287776	0.3646169863648589	-	regulation of actin cytoskeleton organization	7	P
GO:0007179	1	15	0.31600138818287776	0.3646169863648589	-	transforming growth factor beta receptor signaling pathway	7	P
GO:0045766	1	17	0.3480049849774132	0.3974676224361319	-	positive regulation of angiogenesis	7	P
GO:0000226	1	17	0.3480049849774132	0.3974676224361319	-	microtubule cytoskeleton organization	7	P
GO:0000122	2	53	0.3699885474474683	0.4204415311903049	-	negative regulation of transcription by RNA polymerase II	7	P
GO:0007568	1	19	0.3785733425380989	0.42167822807461514	-	aging	7	P
GO:0006974	1	19	0.3785733425380989	0.42167822807461514	-	cellular response to DNA damage stimulus	7	P
GO:0043123	1	19	0.3785733425380989	0.42167822807461514	-	positive regulation of I-kappaB kinase/NF-kappaB signaling	7	P
GO:0051897	1	19	0.3785733425380989	0.42167822807461514	-	positive regulation of protein kinase B signaling	7	P
GO:0007229	1	20	0.3933386453496935	0.4359664788358672	-	integrin-mediated signaling pathway	7	P
GO:0006954	2	56	0.39577098397756383	0.4365121146811366	-	inflammatory response	7	P
GO:0030198	1	21	0.40776802607043516	0.44755027251633134	-	extracellular matrix organization	7	P
GO:0050900	1	22	0.42186878735447253	0.46077901531435106	-	leukocyte migration	7	P
GO:0008360	1	23	0.4356480805056093	0.4735305222887058	-	regulation of cell shape	7	P
GO:0001934	1	24	0.4491129084603513	0.4789118692112751	-	positive regulation of protein phosphorylation	7	P
GO:0071222	1	24	0.4491129084603513	0.4789118692112751	-	cellular response to lipopolysaccharide	7	P
GO:0010469	1	24	0.4491129084603513	0.4789118692112751	-	regulation of signaling receptor activity	7	P
GO:0016477	1	24	0.4491129084603513	0.4789118692112751	-	cell migration	7	P
GO:0001525	1	25	0.462270128715192	0.4906168818911236	-	angiogenesis	7	P
GO:0010628	1	27	0.4876884660923894	0.5151638726328057	-	positive regulation of gene expression	7	P
GO:0035556	1	29	0.5119551516270642	0.5382706033462124	-	intracellular signal transduction	7	P
GO:0008285	1	34	0.5679058648451377	0.5943200911170047	-	negative regulation of cell population proliferation	7	P
GO:0055114	1	35	0.578334159817396	0.6024314164764543	-	oxidation-reduction process	7	P
GO:0008283	1	36	0.5885214647797907	0.6102181086426401	-	cell population proliferation	7	P
GO:0007155	1	37	0.5984731012782812	0.617690127466116	-	cell adhesion	7	P
GO:0006357	1	38	0.6081942788262812	0.6248571357804259	-	regulation of transcription by RNA polymerase II	7	P
GO:0007186	1	41	0.6360255192191546	0.6504806446559536	-	G protein-coupled receptor signaling pathway	7	P
GO:0008284	1	45	0.6702047601860848	0.6823351630853803	-	positive regulation of cell population proliferation	7	P
GO:0006915	1	55	0.742742716864196	0.7527797806056041	-	apoptotic process	7	P
GO:0043312	2	121	0.8027758654489562	0.8099756489955836	-	neutrophil degranulation	7	P
GO:0045944	1	68	0.8144917501464298	0.8181278740310122	-	positive regulation of transcription by RNA polymerase II	7	P
GO:0007165	1	90	0.894476080691686	0.894476080691686	-	signal transduction	7	P
GO:0042609	1	2	0.0030581039755351682	0.0061162079510703364	**	CD4 receptor binding	8	F
GO:0005515	1	655	0.6687054026503567	0.6687054026503567	-	protein binding	8	F
GO:0030140	1	3	0.004077471967380225	0.020387359836901122	*	trans-Golgi network transport vesicle	8	C
GO:0010008	1	24	0.025484199796126403	0.06371049949031601	-	endosome membrane	8	C
GO:0043231	1	62	0.06422018348623854	0.10193679918450561	-	intracellular membrane-bounded organelle	8	C
GO:0005794	1	79	0.08154943934760449	0.10193679918450561	-	Golgi apparatus	8	C
GO:0005829	1	399	0.40774719673802245	0.40774719673802245	-	cytosol	8	C
GO:0050851	1	1	0.0020387359836901123	0.0020387359836901123	**	antigen receptor-mediated signaling pathway	8	P
GO:0001055	1	1	0.008142461877717446	0.06732148862128219	-	RNA polymerase II activity	9	F
GO:0051082	1	6	0.02828107214739926	0.06732148862128219	-	unfolded protein binding	9	F
GO:0035091	1	7	0.03227170429463786	0.06732148862128219	-	phosphatidylinositol binding	9	F
GO:0008092	1	7	0.03227170429463786	0.06732148862128219	-	cytoskeletal protein binding	9	F
GO:0051087	1	7	0.03227170429463786	0.06732148862128219	-	chaperone binding	9	F
GO:0005524	2	78	0.035376915663117264	0.06732148862128219	-	ATP binding	9	F
GO:0046983	1	8	0.03625003233453657	0.06732148862128219	-	protein dimerization activity	9	F
GO:0004713	1	10	0.04416987733461426	0.07177605066874818	-	protein tyrosine kinase activity	9	F
GO:0004674	1	24	0.09825050133877405	0.14191739082267363	-	protein serine/threonine kinase activity	9	F
GO:0003677	1	49	0.18907686610339494	0.23559719010638802	-	DNA binding	9	F
GO:0005515	4	652	0.19935146855155908	0.23559719010638802	-	protein binding	9	F
GO:0003723	1	64	0.24016916940141692	0.260183266851535	-	RNA binding	9	F
GO:0046872	1	95	0.3380758982477356	0.3380758982477356	-	metal ion binding	9	F
GO:0005665	1	1	0.008142461877717446	0.05699723314402212	-	RNA polymerase II, core complex	9	C
GO:0120103	1	1	0.008142461877717446	0.05699723314402212	-	centriolar subdistal appendage	9	C
GO:0090543	1	3	0.016235097499612907	0.0757637883315269	-	Flemming body	9	C
GO:0015630	1	11	0.04811144485076021	0.14876385290415084	-	microtubule cytoskeleton	9	C
GO:0005815	1	13	0.05595799610963438	0.14876385290415084	-	microtubule organizing center	9	C
GO:0005759	1	15	0.06375593695892179	0.14876385290415084	-	mitochondrial matrix	9	C
GO:0005813	1	24	0.09825050133877405	0.1965010026775481	-	centrosome	9	C
GO:0016020	2	182	0.1616732601353304	0.2829282052368282	-	membrane	9	C
GO:0005829	3	397	0.18777071984539692	0.292087786426173	-	cytosol	9	C
GO:0005739	1	65	0.24348720796298276	0.34088209114817586	-	mitochondrion	9	C
GO:0005634	2	300	0.3621002886582363	0.46085491283775526	-	nucleus	9	C
GO:0005654	1	167	0.5288753603676548	0.617021253762264	-	nucleoplasm	9	C
GO:0070062	1	268	0.7231544490266519	0.7787817143363943	-	extracellular exosome	9	C
GO:0005737	1	321	0.7969682829539793	0.7969682829539793	-	cytoplasm	9	C
GO:0006370	1	1	0.008142461877717446	0.04052558826432622	*	7-methylguanosine mRNA capping	9	P
GO:0051085	1	1	0.008142461877717446	0.04052558826432622	*	chaperone cofactor-dependent protein refolding	9	P
GO:0016070	1	2	0.01219499523061748	0.04052558826432622	*	RNA metabolic process	9	P
GO:0006368	1	2	0.01219499523061748	0.04052558826432622	*	transcription elongation from RNA polymerase II promoter	9	P
GO:0042795	1	2	0.01219499523061748	0.04052558826432622	*	snRNA transcription by RNA polymerase II	9	P
GO:0060964	1	3	0.016235097499612907	0.04052558826432622	*	regulation of gene silencing by miRNA	9	P
GO:0008543	1	3	0.016235097499612907	0.04052558826432622	*	fibroblast growth factor receptor signaling pathway	9	P
GO:0006283	1	3	0.016235097499612907	0.04052558826432622	*	transcription-coupled nucleotide-excision repair	9	P
GO:0097009	1	3	0.016235097499612907	0.04052558826432622	*	energy homeostasis	9	P
GO:0006986	1	4	0.02026279413216311	0.04052558826432622	*	response to unfolded protein	9	P
GO:0006367	1	4	0.02026279413216311	0.04052558826432622	*	transcription initiation from RNA polymerase II promoter	9	P
GO:0045727	1	4	0.02026279413216311	0.04052558826432622	*	positive regulation of translation	9	P
GO:0000398	1	4	0.02026279413216311	0.04052558826432622	*	mRNA splicing, via spliceosome	9	P
GO:0001649	1	6	0.02828107214739926	0.0459567422395238	*	osteoblast differentiation	9	P
GO:0051493	1	6	0.02828107214739926	0.0459567422395238	*	regulation of cytoskeleton organization	9	P
GO:0010389	1	6	0.02828107214739926	0.0459567422395238	*	regulation of G2/M transition of mitotic cell cycle	9	P
GO:0097711	1	7	0.03227170429463786	0.04935672421532849	*	ciliary basal body-plasma membrane docking	9	P
GO:0006457	1	8	0.03625003233453657	0.05236115781655282	-	protein folding	9	P
GO:0000086	1	9	0.0402160815841887	0.05503253269415296	-	G2/M transition of mitotic cell cycle	9	P
GO:0018108	1	13	0.05595799610963438	0.0727453949425247	-	peptidyl-tyrosine phosphorylation	9	P
GO:0006919	1	14	0.05986303025189029	0.07411613269281656	-	activation of cysteine-type endopeptidase activity involved in apoptotic process	9	P
GO:0051301	1	16	0.06763674136530969	0.07993433070445691	-	cell division	9	P
GO:0046777	1	17	0.07150546857956151	0.08083226882906953	-	protein autophosphorylation	9	P
GO:0006366	1	23	0.09446561362141322	0.10233774808986432	-	transcription by RNA polymerase II	9	P
GO:0035556	1	29	0.1169972780707816	0.12167716919361286	-	intracellular signal transduction	9	P
GO:0006468	1	36	0.14274982273396922	0.14274982273396922	-	protein phosphorylation	9	P
